{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "23660d9a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import cobra\n",
    "import memote\n",
    "import os "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "5afdca38",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/Mitocore/Human1_coverage/Statistics\n"
     ]
    }
   ],
   "source": [
    "#os.chdir('/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/neutrophil-modeling/curation')\n",
    "wd = os.getcwd()\n",
    "print(wd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3b1b7d0b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "e537b970",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Model does not contain SBML fbc package information.\n",
      "SBML package 'layout' not supported by cobrapy, information is not parsed\n",
      "SBML package 'render' not supported by cobrapy, information is not parsed\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_10fthf_c \"10-Formyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_10fthf_c \"10-Formyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_10fthf_m \"10-Formyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_10fthf_m \"10-Formyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_13dpg_c \"3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_13dpg_c \"3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_1pipdn2c_c \"delta1-Piperideine-2-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1pipdn2c_c \"delta1-Piperideine-2-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_1pyr5c_m \"(S)-1-Pyrroline-5-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1pyr5c_m \"(S)-1-Pyrroline-5-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2aobut_m \"L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2aobut_m \"L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2hb_c \"2-Hydroxybutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2hb_c \"2-Hydroxybutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2hb_e \"2-Hydroxybutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2hb_e \"2-Hydroxybutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2maacoa_m \"2-Methylacetoacetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2maacoa_m \"2-Methylacetoacetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mb2coa_m \"2-Methylbut-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mb2coa_m \"2-Methylbut-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mbcoa_m \"2-Methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mbcoa_m \"2-Methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mop_m \"2-Methyl-3-oxopropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mop_m \"2-Methyl-3-oxopropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mp2coa_m \"2-Methylprop-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mp2coa_m \"2-Methylprop-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2obut_c \"2-Oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2obut_c \"2-Oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2obut_m \"2-Oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2obut_m \"2-Oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2oxoadp_c \"2-Oxoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2oxoadp_c \"2-Oxoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2oxoadp_m \"2-Oxoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2oxoadp_m \"2-Oxoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2pg_c \"2-Phospho-D-glycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2pg_c \"2-Phospho-D-glycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_34hpp_c \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_34hpp_c \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hanthrn_c \"3-Hydroxyanthranilate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hanthrn_c \"3-Hydroxyanthranilate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hbcoa_m \"(S)-3-Hydroxybutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hbcoa_m \"(S)-3-Hydroxybutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hexdcoa_m \"(S)-3-Hydroxyhexadecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hexdcoa_m \"(S)-3-Hydroxyhexadecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hibutcoa_m \"(S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hibutcoa_m \"(S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hmbcoa_m \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hmbcoa_m \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hmp_m \"3-Hydroxy-2-methylpropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hmp_m \"3-Hydroxy-2-methylpropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mb2coa_m \"3-Methylcrotonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mb2coa_m \"3-Methylcrotonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mgcoa_m \"3-Methylglutaconyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mgcoa_m \"3-Methylglutaconyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mob_m \"3-Methyl-2-oxobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mob_m \"3-Methyl-2-oxobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mop_m \"(S)-3-Methyl-2-oxopentanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mop_m \"(S)-3-Methyl-2-oxopentanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3odcoa_m \"3-Oxodecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3odcoa_m \"3-Oxodecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3oddcoa_m \"3-Oxododecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3oddcoa_m \"3-Oxododecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3pg_c \"3-Phospho-D-glycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3pg_c \"3-Phospho-D-glycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3php_c \"3-Phosphonooxypyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3php_c \"3-Phosphonooxypyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3sala_c \"3-Sulfino-L-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3sala_c \"3-Sulfino-L-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3snpyr_c \"3-Sulfinylpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3snpyr_c \"3-Sulfinylpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3tetd7ecoa_m \"trans-Tetradec-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3tetd7ecoa_m \"trans-Tetradec-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4aabutn_m \"4-Acetamidobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4aabutn_m \"4-Acetamidobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abut_c \"4-Aminobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abut_c \"4-Aminobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abut_m \"4-Aminobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abut_m \"4-Aminobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abutn_c \"4-Aminobutyraldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abutn_c \"4-Aminobutyraldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4fumacac_c \"4-Fumarylacetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4fumacac_c \"4-Fumarylacetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4izp_c \"4-Imidazolone-5-propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4izp_c \"4-Imidazolone-5-propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4mlacac_c \"4-Maleylacetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4mlacac_c \"4-Maleylacetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4mop_m \"4-Methyl-2-oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4mop_m \"4-Methyl-2-oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5aop_c \"5-Aminolevulinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5aop_c \"5-Aminolevulinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5aop_m \"5-Aminolevulinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5aop_m \"5-Aminolevulinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5forthf_c \"5-Formiminotetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5forthf_c \"5-Formiminotetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6a2ohxnt_c \"6-Amino-2-oxohexanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6a2ohxnt_c \"6-Amino-2-oxohexanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6pgc_c \"6-Phospho-D-gluconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6pgc_c \"6-Phospho-D-gluconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6pgl_c \"D-Glucono-1,5-lactone 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6pgl_c \"D-Glucono-1,5-lactone 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aacoa_m \"Acetoacetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aacoa_m \"Acetoacetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aact_m \"Aminoacetone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aact_m \"Aminoacetone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_c \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_c \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_e \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_e \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_m \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_m \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_c \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_c \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_e \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_e \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_m \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_m \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acald_c \"Acetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acald_c \"Acetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acald_m \"Acetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acald_m \"Acetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_accoa_c \"Acetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_accoa_c \"Acetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_accoa_m \"Acetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_accoa_m \"Acetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adn_c \"Adenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adn_c \"Adenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adp_c \"ADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adp_c \"ADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adp_m \"ADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adp_m \"ADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_agm_m \"Agmatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_agm_m \"Agmatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ahcys_c \"S-Adenosyl-L-homocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ahcys_c \"S-Adenosyl-L-homocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_c \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_c \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_e \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_e \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_m \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_m \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_c \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_c \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_e \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_e \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_m \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_m \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_c \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_c \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_e \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_e \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_m \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_m \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_alpa_hs_m \"1-Acyl-sn-glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_alpa_hs_m \"1-Acyl-sn-glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_alpro_m \"S-Aminomethyldihydrolipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_alpro_m \"S-Aminomethyldihydrolipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_am6sa_c \"2-Aminomuconate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_am6sa_c \"2-Aminomuconate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amet_c \"S-Adenosyl-L-methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amet_c \"S-Adenosyl-L-methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amp_c \"AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amp_c \"AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amp_m \"AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amp_m \"AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amuco_c \"2-Aminomuconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amuco_c \"2-Aminomuconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aprut_m \"N-Acetylputrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aprut_m \"N-Acetylputrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_c \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_c \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_e \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_e \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_m \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_m \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_argsuc_c \"N-(L-Arginino)succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_argsuc_c \"N-(L-Arginino)succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_argsuc_e \"N-(L-Arginino)succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_argsuc_e \"N-(L-Arginino)succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_c \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_c \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_e \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_e \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_m \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_m \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_c \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_c \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_e \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_e \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_m \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_m \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_atp_c \"ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_atp_c \"ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_atp_m \"ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_atp_m \"ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_b2coa_m \"Crotonoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_b2coa_m \"Crotonoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_c \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_c \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_e \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_e \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_m \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_m \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bilirub_c \"Bilirubin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bilirub_c \"Bilirubin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bilirub_e \"Bilirubin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bilirub_e \"Bilirubin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biliverd_c \"Biliverdin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biliverd_c \"Biliverdin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_c \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_c \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_e \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_e \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_m \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_m \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_btcoa_m \"Butanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_btcoa_m \"Butanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_c \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_c \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_e \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_e \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_m \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_m \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cbp_m \"Carbamoyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cbp_m \"Carbamoyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cdp_m \"CDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cdp_m \"CDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cdpdag_hs_m \"CDP-diacylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cdpdag_hs_m \"CDP-diacylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cgly_c \"Cys-Gly\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cgly_c \"Cys-Gly\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_c \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_c \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_e \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_e \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_m \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_m \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_c \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_c \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_e \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_e \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_m \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_m \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_c \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_c \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_e \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_e \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_m \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_m \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_clpn_hs_m \"Cardiolipin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_clpn_hs_m \"Cardiolipin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cmp_m \"CMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cmp_m \"CMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cmusa_c \"2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cmusa_c \"2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co_c \"CO\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co_c \"CO\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co_e \"CO\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co_e \"CO\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_c \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_c \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_e \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_e \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_m \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_m \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_coa_c \"CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_coa_c \"CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_coa_m \"CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_coa_m \"CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cpppg3_c \"Coproporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cpppg3_c \"Coproporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_c \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_c \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_e \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_e \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_m \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_m \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_crn_c \"L-Carnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_crn_c \"L-Carnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_crn_m \"L-Carnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_crn_m \"L-Carnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ctp_m \"CTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ctp_m \"CTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_c \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_c \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_e \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_e \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_m \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_m \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_c \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_c \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_e \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_e \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_m \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_m \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyst_L_c \"L-Cystathionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyst_L_c \"L-Cystathionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dc2coa_m \"trans-Dec-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dc2coa_m \"trans-Dec-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dcacoa_m \"Decanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dcacoa_m \"Decanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dcamp_c \"N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dcamp_c \"N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dd2coa_m \"2-trans-Dodecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dd2coa_m \"2-trans-Dodecenoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ddcacoa_m \"Lauroyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ddcacoa_m \"Lauroyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhap_c \"Glycerone phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhap_c \"Glycerone phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhbpt_c \"Dihydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhbpt_c \"Dihydrobiopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhlpro_m \"Dihydrolipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhlpro_m \"Dihydrolipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_e4p_c \"D-Erythrose 4-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_e4p_c \"D-Erythrose 4-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_etoh_c \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_etoh_c \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_etoh_e \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_etoh_e \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_f6p_c \"D-Fructose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_f6p_c \"D-Fructose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fdp_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fdp_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_c \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_c \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_e \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_e \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_m \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_m \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ficytC_m \"Ferricytochrome c\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ficytC_m \"Ferricytochrome c\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_focytC_m \"Ferrocytochrome c\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_focytC_m \"Ferrocytochrome c\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_c \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_c \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_e \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_e \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_m \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_m \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_forglu_c \"N-Formimino-L-glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_forglu_c \"N-Formimino-L-glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_c \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_c \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_e \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_e \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_m \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_m \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_g3p_c \"D-Glyceraldehyde 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_g3p_c \"D-Glyceraldehyde 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_g6p_c \"D-Glucose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_g6p_c \"D-Glucose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gdp_c \"GDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gdp_c \"GDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gdp_m \"GDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gdp_m \"GDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glc_D_c \"D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glc_D_c \"D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glc_D_e \"D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glc_D_e \"D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_c \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_c \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_e \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_e \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_m \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_m \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_c \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_c \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_e \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_e \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_m \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_m \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu5p_m \"L-Glutamyl 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu5p_m \"L-Glutamyl 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu5sa_m \"L-Glutamate 5-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu5sa_m \"L-Glutamate 5-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glucys_c \"gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glucys_c \"gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glutcoa_m \"Glutaryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glutcoa_m \"Glutaryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_c \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_c \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_e \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_e \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_m \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_m \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_c \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_c \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_e \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_e \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_m \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_m \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc3p_c \"sn-Glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc3p_c \"sn-Glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc3p_m \"sn-Glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc3p_m \"sn-Glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthox_c \"Glutathione disulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthox_c \"Glutathione disulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthox_m \"Glutathione disulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthox_m \"Glutathione disulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthrd_c \"Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthrd_c \"Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthrd_m \"Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthrd_m \"Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gtp_c \"GTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gtp_c \"GTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gtp_m \"GTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gtp_m \"GTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_c \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_c \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_e \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_e \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_m \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_m \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_PMF_c \"Proton Motive Force\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_PMF_c \"Proton Motive Force\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_PMF_m \"Proton Motive Force\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_PMF_m \"Proton Motive Force\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_c \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_c \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_e \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_e \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_m \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_m \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o2_c \"Hydrogen peroxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o2_c \"Hydrogen peroxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o2_m \"Hydrogen peroxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o2_m \"Hydrogen peroxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC00250_c \"Hydrogen sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC00250_c \"Hydrogen sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC00250_e \"Hydrogen sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC00250_e \"Hydrogen sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01401_m \"(S)-3-Hydroxydodecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01401_m \"(S)-3-Hydroxydodecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01403_m \"(S)-Hydroxydecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01403_m \"(S)-Hydroxydecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01405_m \"(S)-3-Hydroxyoctanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01405_m \"(S)-3-Hydroxyoctanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01406_m \"3-Oxooctanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01406_m \"3-Oxooctanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01407_m \"(S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01407_m \"(S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01408_m \"3-Oxohexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01408_m \"3-Oxohexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01409_m \"Hexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01409_m \"Hexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01410_m \"trans-Hex-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01410_m \"trans-Hex-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01415_m \"trans-Oct-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01415_m \"trans-Oct-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_c \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_c \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_e \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_e \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_m \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_m \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hcys_L_c \"Homocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hcys_L_c \"Homocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_c \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_c \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_e \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_e \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_m \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_m \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hgentis_c \"Homogentisate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hgentis_c \"Homogentisate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_c \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_c \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_e \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_e \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_m \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_m \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hLkynr_c \"3-Hydroxy-L-kynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hLkynr_c \"3-Hydroxy-L-kynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hmbil_c \"Hydroxymethylbilane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hmbil_c \"Hydroxymethylbilane\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hmgcoa_m \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hmgcoa_m \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ibcoa_m \"2-Methylpropanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ibcoa_m \"2-Methylpropanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_c \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_c \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_e \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_e \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_m \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_m \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_c \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_c \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_e \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_e \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_m \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_m \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_imp_c \"IMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_imp_c \"IMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ivcoa_m \"3-Methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ivcoa_m \"3-Methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp_c \"L-2-Aminoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp_c \"L-2-Aminoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp_m \"L-2-Aminoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp_m \"L-2-Aminoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp6sa_c \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp6sa_c \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp6sa_m \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp6sa_m \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_D_m \"(R)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_D_m \"(R)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_L_c \"(S)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_L_c \"(S)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_L_e \"(S)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_L_e \"(S)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_c \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_c \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_e \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_e \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_m \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_m \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lfmkynr_c \"L-Formylkynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lfmkynr_c \"L-Formylkynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lgt_S_m \"(R)-S-Lactoylglutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lgt_S_m \"(R)-S-Lactoylglutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lkynr_c \"L-Kynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lkynr_c \"L-Kynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lpipecol_c \"L-Pipecolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lpipecol_c \"L-Pipecolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lpro_m \"Lipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lpro_m \"Lipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_c \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_c \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_e \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_e \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_m \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_m \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_c \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_c \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_e \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_e \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_m \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_m \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercplac_c \"3-Mercaptolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercplac_c \"3-Mercaptolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercplac_e \"3-Mercaptolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercplac_e \"3-Mercaptolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercppyr_c \"Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercppyr_c \"Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercppyr_m \"Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercppyr_m \"Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_c \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_c \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_e \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_e \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_m \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_m \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_methf_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_methf_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_methf_m \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_methf_m \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mlthf_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mlthf_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mlthf_m \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mlthf_m \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mmcoa_R_m \"(R)-Methylmalonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mmcoa_R_m \"(R)-Methylmalonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mmcoa_S_m \"(S)-Methylmalonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mmcoa_S_m \"(S)-Methylmalonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_msa_m \"3-Oxopropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_msa_m \"3-Oxopropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mthgxl_m \"Methylglyoxal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mthgxl_m \"Methylglyoxal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_n4abutn_m \"N4-Acetylaminobutanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_n4abutn_m \"N4-Acetylaminobutanal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_c \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_c \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_e \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_e \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_m \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_m \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_c \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_c \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_e \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_e \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_m \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_m \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadp_c \"NADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadp_c \"NADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadp_m \"NADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadp_m \"NADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadph_c \"NADPH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadph_c \"NADPH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadph_m \"NADPH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadph_m \"NADPH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_c \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_c \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_e \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_e \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_m \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_m \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_no_c \"Nitric oxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_no_c \"Nitric oxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_no_e \"Nitric oxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_no_e \"Nitric oxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nwharg_c \"N(omega)-Hydroxyarginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nwharg_c \"N(omega)-Hydroxyarginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_c \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_c \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_e \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_e \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_m \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_m \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2s_m \"O2.-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2s_m \"O2.-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_c \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_c \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_e \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_e \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_m \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_m \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_occoa_m \"Octanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_occoa_m \"Octanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_orn_c \"L-Ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_orn_c \"L-Ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_orn_m \"L-Ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_orn_m \"L-Ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pa_hs_m \"Phosphatidate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pa_hs_m \"Phosphatidate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_c \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_c \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_e \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_e \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_m \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_m \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_c \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_c \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_e \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_e \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_m \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_m \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_c \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_c \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_e \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_e \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_m \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_m \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pep_c \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pep_c \"Phosphoenolpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pep_m \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pep_m \"Phosphoenolpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pglyc_hs_m \"Phosphatidylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pglyc_hs_m \"Phosphatidylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pgp_hs_m \"Phosphatidylglycerophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pgp_hs_m \"Phosphatidylglycerophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_c \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_c \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_e \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_e \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_m \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_m \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pheme_c \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pheme_c \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pheme_m \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pheme_m \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_c \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_c \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_e \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_e \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_m \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_m \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcoa_c \"Palmitoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcoa_c \"Palmitoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcoa_m \"Palmitoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcoa_m \"Palmitoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcrn_c \"L-Palmitoylcarnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcrn_c \"L-Palmitoylcarnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcrn_m \"L-Palmitoylcarnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcrn_m \"L-Palmitoylcarnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_c \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_c \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_e \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_e \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_m \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_m \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppbng_c \"Porphobilinogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppbng_c \"Porphobilinogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppcoa_m \"Propanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppcoa_m \"Propanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppi_c \"Diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppi_c \"Diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppi_m \"Diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppi_m \"Diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppp9_m \"Protoporphyrin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppp9_m \"Protoporphyrin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pppg9_c \"Protoporphyrinogen IX\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pppg9_c \"Protoporphyrinogen IX\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_c \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_c \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_e \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_e \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_m \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_m \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_c \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_c \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_e \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_e \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_m \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_m \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pser_L_c \"O-Phospho-L-serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pser_L_c \"O-Phospho-L-serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ptrc_c \"Putrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ptrc_c \"Putrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ptrc_m \"Putrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ptrc_m \"Putrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pyr_c \"Pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pyr_c \"Pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pyr_m \"Pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pyr_m \"Pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_q10_m \"Ubiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_q10_m \"Ubiquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_q10h2_m \"Ubiquinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_q10h2_m \"Ubiquinol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_r5p_c \"D-Ribose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_r5p_c \"D-Ribose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ru5p_D_c \"D-Ribulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ru5p_D_c \"D-Ribulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_s7p_c \"Sedoheptulose 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_s7p_c \"Sedoheptulose 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_saccrp_L_m \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_saccrp_L_m \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_c \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_c \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_e \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_e \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_m \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_m \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_c \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_c \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_e \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_e \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_m \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_m \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so4_c \"Sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so4_c \"Sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so4_m \"Sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so4_m \"Sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_c \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_c \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_e \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_e \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_m \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_m \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succoa_m \"Succinyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succoa_m \"Succinyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_sucsal_m \"Succinate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_sucsal_m \"Succinate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_c \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_c \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_e \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_e \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_m \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_m \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tetd7ecoa_m \"Tetradecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tetd7ecoa_m \"Tetradecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thbpt_c \"Tetrahydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thbpt_c \"Tetrahydrobiopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thf_c \"Tetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thf_c \"Tetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thf_m \"Tetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thf_m \"Tetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thp2c_c \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thp2c_c \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_c \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_c \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_e \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_e \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_m \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_m \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_c \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_c \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_e \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_e \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_m \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_m \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_c \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_c \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_e \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_e \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_m \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_m \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_c \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_c \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_e \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_e \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_m \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_m \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_uppg3_c \"Uroporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_uppg3_c \"Uroporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urcan_c \"Urocanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urcan_c \"Urocanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_c \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_c \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_e \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_e \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_m \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_m \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_c \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_c \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_e \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_e \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_m \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_m \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_xu5p_D_c \"D-Xylulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_xu5p_D_c \"D-Xylulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_c \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_c \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_e \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_e \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_m \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_m \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhf_c \"Dihydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhf_c \"Dihydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhf_m \"Dihydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhf_m \"Dihydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_malcoa_c \"Malonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_malcoa_c \"Malonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_malACP_c \"Malonyl-[acyl-carrier protein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_malACP_c \"Malonyl-[acyl-carrier protein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ACP_c \"acyl carrier protein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ACP_c \"acyl carrier protein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acACP_c \"Acetyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acACP_c \"Acetyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01587_c \"Acetoacetyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01587_c \"Acetoacetyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01321_c \"(R)-3-Hydroxybutanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01321_c \"(R)-3-Hydroxybutanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01255_c \"But-2-enoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01255_c \"But-2-enoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01588_c \"Butyryl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01588_c \"Butyryl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01589_c \"3-Oxohexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01589_c \"3-Oxohexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01590_c \"D-3-Hydroxyhexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01590_c \"D-3-Hydroxyhexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01591_c \"(2E)-Hexenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01591_c \"(2E)-Hexenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01592_c \"Hexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01592_c \"Hexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01593_c \"3-Oxooctanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01593_c \"3-Oxooctanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01323_c \"(R)-3-Hydroxyoctanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01323_c \"(R)-3-Hydroxyoctanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01594_c \"(2E)-Octenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01594_c \"(2E)-Octenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01595_c \"Octanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01595_c \"Octanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01596_c \"3-Oxodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01596_c \"3-Oxodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01322_c \"(R)-3-Hydroxydecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01322_c \"(R)-3-Hydroxydecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01597_c \"(2E)-Decenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01597_c \"(2E)-Decenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01598_c \"Decanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01598_c \"Decanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01599_c \"3-Oxododecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01599_c \"3-Oxododecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01600_c \"D-3-Hydroxydodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01600_c \"D-3-Hydroxydodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01601_c \"(2E)-Dodecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01601_c \"(2E)-Dodecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ddcaACP_c \"Dodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ddcaACP_c \"Dodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01602_c \"3-Oxotetradecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01602_c \"3-Oxotetradecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01335_c \"(R)-3-Hydroxytetradecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01335_c \"(R)-3-Hydroxytetradecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01603_c \"(2E)-Tetradecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01603_c \"(2E)-Tetradecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_myrsACP_c \"Myristoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_myrsACP_c \"Myristoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01605_c \"3-Oxohexadecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01605_c \"3-Oxohexadecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01326_c \"(R)-3-Hydroxypalmitoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01326_c \"(R)-3-Hydroxypalmitoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01606_c \"(2E)-Hexadecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01606_c \"(2E)-Hexadecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_palmACP_c \"Palmitoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_palmACP_c \"Palmitoyl-ACP\">\n",
      "Adding exchange reaction EX_2hb_e with default bounds for boundary metabolite: 2hb_e.\n",
      "Adding exchange reaction EX_ac_e with default bounds for boundary metabolite: ac_e.\n",
      "Adding exchange reaction EX_acac_e with default bounds for boundary metabolite: acac_e.\n",
      "Adding exchange reaction EX_akg_e with default bounds for boundary metabolite: akg_e.\n",
      "Adding exchange reaction EX_ala_B_e with default bounds for boundary metabolite: ala_B_e.\n",
      "Adding exchange reaction EX_ala_L_e with default bounds for boundary metabolite: ala_L_e.\n",
      "Adding exchange reaction EX_arg_L_e with default bounds for boundary metabolite: arg_L_e.\n",
      "Adding exchange reaction EX_argsuc_e with default bounds for boundary metabolite: argsuc_e.\n",
      "Adding exchange reaction EX_asn_L_e with default bounds for boundary metabolite: asn_L_e.\n",
      "Adding exchange reaction EX_asp_L_e with default bounds for boundary metabolite: asp_L_e.\n",
      "Adding exchange reaction EX_bhb_e with default bounds for boundary metabolite: bhb_e.\n",
      "Adding exchange reaction EX_bilirub_e with default bounds for boundary metabolite: bilirub_e.\n",
      "Adding exchange reaction EX_biomass_e with default bounds for boundary metabolite: biomass_e.\n",
      "Adding exchange reaction EX_but_e with default bounds for boundary metabolite: but_e.\n",
      "Adding exchange reaction EX_chol_e with default bounds for boundary metabolite: chol_e.\n",
      "Adding exchange reaction EX_cit_e with default bounds for boundary metabolite: cit_e.\n",
      "Adding exchange reaction EX_citr_L_e with default bounds for boundary metabolite: citr_L_e.\n",
      "Adding exchange reaction EX_co_e with default bounds for boundary metabolite: co_e.\n",
      "Adding exchange reaction EX_co2_e with default bounds for boundary metabolite: co2_e.\n",
      "Adding exchange reaction EX_creat_e with default bounds for boundary metabolite: creat_e.\n",
      "Adding exchange reaction EX_cyan_e with default bounds for boundary metabolite: cyan_e.\n",
      "Adding exchange reaction EX_cys_L_e with default bounds for boundary metabolite: cys_L_e.\n",
      "Adding exchange reaction EX_etoh_e with default bounds for boundary metabolite: etoh_e.\n",
      "Adding exchange reaction EX_fe2_e with default bounds for boundary metabolite: fe2_e.\n",
      "Adding exchange reaction EX_for_e with default bounds for boundary metabolite: for_e.\n",
      "Adding exchange reaction EX_fum_e with default bounds for boundary metabolite: fum_e.\n",
      "Adding exchange reaction EX_glc_D_e with default bounds for boundary metabolite: glc_D_e.\n",
      "Adding exchange reaction EX_gln_L_e with default bounds for boundary metabolite: gln_L_e.\n",
      "Adding exchange reaction EX_glu_L_e with default bounds for boundary metabolite: glu_L_e.\n",
      "Adding exchange reaction EX_gly_e with default bounds for boundary metabolite: gly_e.\n",
      "Adding exchange reaction EX_glyc_e with default bounds for boundary metabolite: glyc_e.\n",
      "Adding exchange reaction EX_h_e with default bounds for boundary metabolite: h_e.\n",
      "Adding exchange reaction EX_h2o_e with default bounds for boundary metabolite: h2o_e.\n",
      "Adding exchange reaction EX_HC00250_e with default bounds for boundary metabolite: HC00250_e.\n",
      "Adding exchange reaction EX_hco3_e with default bounds for boundary metabolite: hco3_e.\n",
      "Adding exchange reaction EX_hdca_e with default bounds for boundary metabolite: hdca_e.\n",
      "Adding exchange reaction EX_his_L_e with default bounds for boundary metabolite: his_L_e.\n",
      "Adding exchange reaction EX_icit_e with default bounds for boundary metabolite: icit_e.\n",
      "Adding exchange reaction EX_ile_L_e with default bounds for boundary metabolite: ile_L_e.\n",
      "Adding exchange reaction EX_lac_L_e with default bounds for boundary metabolite: lac_L_e.\n",
      "Adding exchange reaction EX_leu_L_e with default bounds for boundary metabolite: leu_L_e.\n",
      "Adding exchange reaction EX_lys_L_e with default bounds for boundary metabolite: lys_L_e.\n",
      "Adding exchange reaction EX_mal_L_e with default bounds for boundary metabolite: mal_L_e.\n",
      "Adding exchange reaction EX_mercplac_e with default bounds for boundary metabolite: mercplac_e.\n",
      "Adding exchange reaction EX_met_L_e with default bounds for boundary metabolite: met_L_e.\n",
      "Adding exchange reaction EX_nad_e with default bounds for boundary metabolite: nad_e.\n",
      "Adding exchange reaction EX_nadh_e with default bounds for boundary metabolite: nadh_e.\n",
      "Adding exchange reaction EX_nh4_e with default bounds for boundary metabolite: nh4_e.\n",
      "Adding exchange reaction EX_no_e with default bounds for boundary metabolite: no_e.\n",
      "Adding exchange reaction EX_o2_e with default bounds for boundary metabolite: o2_e.\n",
      "Adding exchange reaction EX_oaa_e with default bounds for boundary metabolite: oaa_e.\n",
      "Adding exchange reaction EX_pchol_hs_e with default bounds for boundary metabolite: pchol_hs_e.\n",
      "Adding exchange reaction EX_pcreat_e with default bounds for boundary metabolite: pcreat_e.\n",
      "Adding exchange reaction EX_pe_hs_e with default bounds for boundary metabolite: pe_hs_e.\n",
      "Adding exchange reaction EX_phe_L_e with default bounds for boundary metabolite: phe_L_e.\n",
      "Adding exchange reaction EX_pi_e with default bounds for boundary metabolite: pi_e.\n",
      "Adding exchange reaction EX_ppa_e with default bounds for boundary metabolite: ppa_e.\n",
      "Adding exchange reaction EX_pro_L_e with default bounds for boundary metabolite: pro_L_e.\n",
      "Adding exchange reaction EX_ps_hs_e with default bounds for boundary metabolite: ps_hs_e.\n",
      "Adding exchange reaction EX_ser_L_e with default bounds for boundary metabolite: ser_L_e.\n",
      "Adding exchange reaction EX_so3_e with default bounds for boundary metabolite: so3_e.\n",
      "Adding exchange reaction EX_succ_e with default bounds for boundary metabolite: succ_e.\n",
      "Adding exchange reaction EX_tcynt_e with default bounds for boundary metabolite: tcynt_e.\n",
      "Adding exchange reaction EX_thr_L_e with default bounds for boundary metabolite: thr_L_e.\n",
      "Adding exchange reaction EX_trp_L_e with default bounds for boundary metabolite: trp_L_e.\n",
      "Adding exchange reaction EX_tsul_e with default bounds for boundary metabolite: tsul_e.\n",
      "Adding exchange reaction EX_tyr_L_e with default bounds for boundary metabolite: tyr_L_e.\n",
      "Adding exchange reaction EX_urea_e with default bounds for boundary metabolite: urea_e.\n",
      "Adding exchange reaction EX_val_L_e with default bounds for boundary metabolite: val_L_e.\n",
      "Adding exchange reaction EX_fol_e with default bounds for boundary metabolite: fol_e.\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_ATP_MitoCore \"R_OF_ATP_MitoCore Maximum ATP production (ATP hydrolysis)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_HEME_MitoCore \"R_OF_HEME_MitoCore Maximum Heme production\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_LIPID_MitoCore \"R_OF_LIPID_MitoCore Maximum lipid synthesis (Mito Inner Membrane)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_PROTEIN_MitoCore \"R_OF_PROTEIN_MitoCore Maxmium production of amino acids for protein synthesis\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYC3Ptm \"R_GLYC3Ptm Glycerol 3-phosphate transport - diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CO2tm \"R_CO2tm CO2 transport (diffusion), mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_H2Otm \"R_H2Otm H2O transport, mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_O2tm \"R_O2tm O2 transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYCtm \"R_GLYCtm glycerol transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYANtm \"R_CYANtm Cyanide transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TCYNTtmB_MitoCore \"R_TCYNTtmB_MitoCore Thiocyanate transport via diffusion (mitochondrial)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CREATtmdiffir \"R_CREATtmdiffir Creatine transport to/from mitochondria\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCREATtmdiffirB_MitoCore \"R_PCREATtmdiffirB_MitoCore Phosphocreatine transport to/from mitochondria \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0941 \"R_r0941 Bicarbonate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0838B_MitoCore \"R_r0838B_MitoCore Ammonia free diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Biomasst_MitoCore \"R_Biomasst_MitoCore Biomass pseudo transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Biomass_MitoCore \"R_Biomass_MitoCore Biomass Sink\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_O2t \"R_O2t o2 transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CO2t \"R_CO2t CO2 transporter via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HCO3t_MitoCore \"R_HCO3t_MitoCore Bicarbonate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLCt1r \"R_GLCt1r glucose transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HDCAtr \"R_HDCAtr fatty acid transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HDCAtm_MitoCore \"R_HDCAtm_MitoCore fatty acid accumulation - mitochondria\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_L_LACt2r \"R_L_LACt2r L-lactate reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BHBt \"R_BHBt (R)-3-Hydroxybutanoate transport via H+ symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACt2 \"R_ACACt2 acetoacetate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ETOHt \"R_ETOHt ethanol reversible transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BUTt2r \"R_BUTt2r Butyrate transport via proton symport, reversible\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYCt \"R_GLYCt glycerol transport via channel\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0942 \"R_r0942 Creatine storage\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0942b_MitoCore \"R_r0942b_MitoCore Phosphocreatine storage\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HIStiDF \"R_HIStiDF L-histidine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILEtec \"R_ILEtec L-isoleucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUtec \"R_LEUtec L-leucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYStiDF \"R_LYStiDF L-lysine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METtec \"R_METtec L-methionine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PHEtec \"R_PHEtec L-phenylalanine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2534 \"R_r2534 L-threonine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPt \"R_TRPt L-tryptophan transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALtec \"R_VALtec L-valine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGtiDF \"R_ARGtiDF L-arginine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPte \"R_ASPte L-aspartate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYStec \"R_CYStec L-cysteine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUt_MitoCore \"R_GLUt_MitoCore L-glutamate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2525 \"R_r2525 L_glutamine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYt2r \"R_GLYt2r glycine reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROt2r \"R_PROt2r L-proline reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2526 \"R_r2526 L-serine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRt \"R_TYRt L-tyrosine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2532 \"R_r2532 L-asparagine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALAt2r \"R_ALAt2r L-alanine reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMt_MitoCore \"R_FUMt_MitoCore Fumarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUMt_MitoCore \"R_SUMt_MitoCore Succinate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0817 \"R_r0817 Citrulline transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NH4t3r \"R_NH4t3r ammonia transport via proton antiport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACt2r \"R_ACt2r acetate reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAt \"R_PPAt Propanoate transport, diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2HBt2 \"R_2HBt2 2-hydroxybutyrate cotransport with proton\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CHOLtu \"R_CHOLtu Choline uniport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1088 \"R_r1088 Citrate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICITt_MitoCore \"R_ICITt_MitoCore Isocitrate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UREAt \"R_UREAt Urea transport via facilitate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1512 \"R_r1512 Bilirubin transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSUCt_MitoCore \"R_ARGSUCt_MitoCore Argininosuccinate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MAL_Lte \"R_MAL_Lte Malate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OAAt_MitoCore \"R_OAAt_MitoCore Oxaloacetate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGt_MitoCore \"R_AKGt_MitoCore 2-Oxoglutarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MERCPLACt_MitoCore \"R_MERCPLACt_MitoCore 3-Mercaptolactate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0899 \"R_r0899 Beta-alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FE2t \"R_FE2t iron (II) transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_H2Ot \"R_H2Ot H2O transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Hct_MitoCore \"R_Hct_MitoCore Proton escape - cytosolic\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Hmt_MitoCore \"R_Hmt_MitoCore Proton escape - mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SO3t_MitoCore \"R_SO3t_MitoCore Sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TSULt_MitoCore \"R_TSULt_MitoCore Thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0940 \"R_r0940 hydrogen sulfide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYANt \"R_CYANt Cyanide transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TCYNTt \"R_TCYNTt Thiocyanate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1423 \"R_r1423 Phosphate facilitated diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FORt_MitoCore \"R_FORt_MitoCore Formate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FOLt_MitoCore \"R_FOLt_MitoCore Folate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADHt_MitoCore \"R_NADHt_MitoCore NADH pool\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADt_MitoCore \"R_NADt_MitoCore NAD+ pool\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADHtm_MitoCore \"R_NADHtm_MitoCore NADH pool mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADtm_MitoCore \"R_NADtm_MitoCore NAD+ pool mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COt \"R_COt CO transporter via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOt \"R_NOt NO transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCHOLHSTDe \"R_PCHOLHSTDe Phosphatidylcholine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSt3 \"R_PSt3 Phosphatidylserine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEt \"R_PEt Phosphatidylethanolamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_ATP_MitoCore \"R_OF_ATP_MitoCore Maximum ATP production (ATP hydrolysis)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_HEME_MitoCore \"R_OF_HEME_MitoCore Maximum Heme production\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_LIPID_MitoCore \"R_OF_LIPID_MitoCore Maximum lipid synthesis (Mito Inner Membrane)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_PROTEIN_MitoCore \"R_OF_PROTEIN_MitoCore Maxmium production of amino acids for protein synthesis\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYC3Ptm \"R_GLYC3Ptm Glycerol 3-phosphate transport - diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CO2tm \"R_CO2tm CO2 transport (diffusion), mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_H2Otm \"R_H2Otm H2O transport, mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_O2tm \"R_O2tm O2 transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYCtm \"R_GLYCtm glycerol transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYANtm \"R_CYANtm Cyanide transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TCYNTtmB_MitoCore \"R_TCYNTtmB_MitoCore Thiocyanate transport via diffusion (mitochondrial)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CREATtmdiffir \"R_CREATtmdiffir Creatine transport to/from mitochondria\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCREATtmdiffirB_MitoCore \"R_PCREATtmdiffirB_MitoCore Phosphocreatine transport to/from mitochondria \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0941 \"R_r0941 Bicarbonate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0838B_MitoCore \"R_r0838B_MitoCore Ammonia free diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Biomasst_MitoCore \"R_Biomasst_MitoCore Biomass pseudo transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Biomass_MitoCore \"R_Biomass_MitoCore Biomass Sink\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_O2t \"R_O2t o2 transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CO2t \"R_CO2t CO2 transporter via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HCO3t_MitoCore \"R_HCO3t_MitoCore Bicarbonate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLCt1r \"R_GLCt1r glucose transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HDCAtr \"R_HDCAtr fatty acid transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HDCAtm_MitoCore \"R_HDCAtm_MitoCore fatty acid accumulation - mitochondria\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_L_LACt2r \"R_L_LACt2r L-lactate reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BHBt \"R_BHBt (R)-3-Hydroxybutanoate transport via H+ symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACt2 \"R_ACACt2 acetoacetate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ETOHt \"R_ETOHt ethanol reversible transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BUTt2r \"R_BUTt2r Butyrate transport via proton symport, reversible\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYCt \"R_GLYCt glycerol transport via channel\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0942 \"R_r0942 Creatine storage\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0942b_MitoCore \"R_r0942b_MitoCore Phosphocreatine storage\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HIStiDF \"R_HIStiDF L-histidine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILEtec \"R_ILEtec L-isoleucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUtec \"R_LEUtec L-leucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYStiDF \"R_LYStiDF L-lysine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METtec \"R_METtec L-methionine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PHEtec \"R_PHEtec L-phenylalanine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2534 \"R_r2534 L-threonine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPt \"R_TRPt L-tryptophan transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALtec \"R_VALtec L-valine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGtiDF \"R_ARGtiDF L-arginine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPte \"R_ASPte L-aspartate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYStec \"R_CYStec L-cysteine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUt_MitoCore \"R_GLUt_MitoCore L-glutamate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2525 \"R_r2525 L_glutamine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYt2r \"R_GLYt2r glycine reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROt2r \"R_PROt2r L-proline reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2526 \"R_r2526 L-serine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRt \"R_TYRt L-tyrosine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2532 \"R_r2532 L-asparagine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALAt2r \"R_ALAt2r L-alanine reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMt_MitoCore \"R_FUMt_MitoCore Fumarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUMt_MitoCore \"R_SUMt_MitoCore Succinate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0817 \"R_r0817 Citrulline transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NH4t3r \"R_NH4t3r ammonia transport via proton antiport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACt2r \"R_ACt2r acetate reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAt \"R_PPAt Propanoate transport, diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2HBt2 \"R_2HBt2 2-hydroxybutyrate cotransport with proton\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CHOLtu \"R_CHOLtu Choline uniport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1088 \"R_r1088 Citrate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICITt_MitoCore \"R_ICITt_MitoCore Isocitrate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UREAt \"R_UREAt Urea transport via facilitate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1512 \"R_r1512 Bilirubin transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSUCt_MitoCore \"R_ARGSUCt_MitoCore Argininosuccinate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MAL_Lte \"R_MAL_Lte Malate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OAAt_MitoCore \"R_OAAt_MitoCore Oxaloacetate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGt_MitoCore \"R_AKGt_MitoCore 2-Oxoglutarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MERCPLACt_MitoCore \"R_MERCPLACt_MitoCore 3-Mercaptolactate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0899 \"R_r0899 Beta-alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FE2t \"R_FE2t iron (II) transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_H2Ot \"R_H2Ot H2O transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Hct_MitoCore \"R_Hct_MitoCore Proton escape - cytosolic\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Hmt_MitoCore \"R_Hmt_MitoCore Proton escape - mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SO3t_MitoCore \"R_SO3t_MitoCore Sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TSULt_MitoCore \"R_TSULt_MitoCore Thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0940 \"R_r0940 hydrogen sulfide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYANt \"R_CYANt Cyanide transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TCYNTt \"R_TCYNTt Thiocyanate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1423 \"R_r1423 Phosphate facilitated diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FORt_MitoCore \"R_FORt_MitoCore Formate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FOLt_MitoCore \"R_FOLt_MitoCore Folate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADHt_MitoCore \"R_NADHt_MitoCore NADH pool\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADt_MitoCore \"R_NADt_MitoCore NAD+ pool\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADHtm_MitoCore \"R_NADHtm_MitoCore NADH pool mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADtm_MitoCore \"R_NADtm_MitoCore NAD+ pool mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COt \"R_COt CO transporter via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOt \"R_NOt NO transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCHOLHSTDe \"R_PCHOLHSTDe Phosphatidylcholine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSt3 \"R_PSt3 Phosphatidylserine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEt \"R_PEt Phosphatidylethanolamine\">\n"
     ]
    }
   ],
   "source": [
    "mitocore0= cobra.io.read_sbml_model('/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/neutrophil-modeling/curation/Mitocore_Models/Mitocore_Original.xml')\n",
    "mitocore1= cobra.io.read_sbml_model('/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/neutrophil-modeling/curation/Mitocore_Models/Mitocore_Preliminary_Curationl.xml')\n",
    "mitocore2= cobra.io.read_sbml_model('/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/Mitocore/Human1_coverage/Cleaned_Mitocore_changed_metabolites_metanetx.chemical.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "8ca81293",
   "metadata": {},
   "outputs": [],
   "source": [
    "def visualization_model_annotations(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    counter=0\n",
    "    for comp in model.__getattribute__(component):\n",
    "        annot_dict= comp.__dict__.get( '_annotation')\n",
    "        counter+=1\n",
    "        print(annot_dict)\n",
    "\n",
    "def visualization_model_general(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    for comp in model.__getattribute__(component):\n",
    "        comp_complete= comp.__dict__\n",
    "        print(comp_complete)\n",
    "\n",
    "def visualization_model_notes(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    counter=0\n",
    "    for comp in model.__getattribute__(component):\n",
    "        notes_dict= comp.__dict__.get( 'notes')\n",
    "        counter+=1\n",
    "        print(notes_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "b0750719",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{'ec-code': ['2.7.1.1', '2.7.1.2', '2.7.1.4', '2.7.1.55', '2.7.1.59', '2.7.1.60'], 'kegg.reaction': 'R01786', 'metanetx.reaction': 'MNXR191953', 'bigg.reaction': 'HEX1', 'Human1': 'MAR04394'}\n",
      "{'ec-code': ['3.1.3.1', '3.1.3.104', '3.1.3.2', '3.1.3.23', '3.1.3.25', '3.1.3.29', '3.1.3.58', '3.1.3.9'], 'kegg.reaction': 'R01788', 'metanetx.reaction': 'MNXR195425', 'bigg.reaction': 'G6PPer'}\n",
      "{'ec-code': '5.3.1.9', 'kegg.reaction': 'R02740', 'metanetx.reaction': 'MNXR191975', 'bigg.reaction': 'PGI', 'Human1': 'MAR04381'}\n",
      "{'ec-code': '2.7.1.11', 'kegg.reaction': 'R04779', 'metanetx.reaction': 'MNXR145952', 'bigg.reaction': 'PFK', 'Human1': 'MAR04379'}\n",
      "{'ec-code': '3.1.3.11', 'kegg.reaction': 'R04780', 'metanetx.reaction': 'MNXR147046', 'bigg.reaction': 'FBP', 'Human1': 'MAR04377'}\n",
      "{'ec-code': '4.1.2.13', 'kegg.reaction': 'R01070', 'metanetx.reaction': 'MNXR153517', 'bigg.reaction': 'FBA', 'Human1': 'MAR04375'}\n",
      "{'ec-code': '5.3.1.1', 'kegg.reaction': 'R01015', 'metanetx.reaction': 'MNXR146573', 'bigg.reaction': 'TPI', 'Human1': 'MAR04391'}\n",
      "{'ec-code': ['1.2.1.12', '1.2.1.59'], 'kegg.reaction': 'R01061', 'metanetx.reaction': 'MNXR144947', 'bigg.reaction': 'GAPD', 'Human1': 'MAR04373'}\n",
      "{'ec-code': ['2.7.2.10', '2.7.2.3'], 'kegg.reaction': 'R01512', 'metanetx.reaction': 'MNXR191159', 'bigg.reaction': 'PGK', 'Human1': 'MAR04368'}\n",
      "{'ec-code': ['5.4.2.1', '5.4.2.4'], 'kegg.reaction': 'R01518', 'metanetx.reaction': 'MNXR191232', 'bigg.reaction': 'DPGM', 'Human1': 'MAR04365'}\n",
      "{'ec-code': '4.2.1.11', 'kegg.reaction': 'R00658', 'metanetx.reaction': 'MNXR97932', 'bigg.reaction': 'ENO', 'Human1': 'MAR04363'}\n",
      "{'ec-code': '2.7.1.40', 'kegg.reaction': 'R00200', 'metanetx.reaction': 'MNXR103371', 'bigg.reaction': 'PYK', 'Human1': 'MAR04358'}\n",
      "{'ec-code': '2.7.1.40', 'kegg.reaction': 'R00430', 'bigg.reaction': 'r0122', 'Human1': 'MAR04171'}\n",
      "{'ec-code': '4.1.1.32', 'kegg.reaction': 'R00431', 'bigg.reaction': 'PEPCK', 'Human1': 'MAR04101'}\n",
      "{'ec-code': ['1.1.1.27', '1.1.1.28'], 'kegg.reaction': 'R00703', 'metanetx.reaction': 'MNXR101040', 'bigg.reaction': 'LDH_L', 'Human1': 'MAR04281'}\n",
      "{'ec-code': ['1.1.1.363', '1.1.1.47', '1.1.1.49'], 'kegg.reaction': 'R02736', 'metanetx.reaction': 'MNXR192434', 'bigg.reaction': 'G6PDH2r', 'Human1': 'MAR08653'}\n",
      "{'ec-code': '3.1.1.31', 'kegg.reaction': 'R02035', 'metanetx.reaction': 'MNXR188641', 'bigg.reaction': 'PGL', 'Human1': 'MAR04623'}\n",
      "{'ec-code': ['1.1.1.351', '1.1.1.44'], 'kegg.reaction': 'R01528', 'metanetx.reaction': 'MNXR100389', 'bigg.reaction': 'GND', 'Human1': 'MAR04473'}\n",
      "{'ec-code': '5.3.1.6', 'kegg.reaction': 'R01056', 'metanetx.reaction': 'MNXR189850', 'bigg.reaction': 'RPI', 'Human1': 'MAR04351'}\n",
      "{'ec-code': '5.1.3.1', 'kegg.reaction': 'R01529', 'metanetx.reaction': 'MNXR104083', 'bigg.reaction': 'RPE', 'Human1': 'MAR04477'}\n",
      "{'ec-code': '2.2.1.1', 'kegg.reaction': 'R01641', 'metanetx.reaction': 'MNXR104868', 'bigg.reaction': 'TKT1', 'Human1': 'MAR04501'}\n",
      "{'ec-code': '2.2.1.2', 'kegg.reaction': 'R01827', 'metanetx.reaction': 'MNXR146501', 'bigg.reaction': 'TALA', 'Human1': 'MAR04565'}\n",
      "{'ec-code': '2.2.1.1', 'kegg.reaction': 'R01830', 'metanetx.reaction': 'MNXR146555', 'bigg.reaction': 'TKT2', 'Human1': 'MAR04404'}\n",
      "{'ec-code': ['1.2.1.104', '1.2.1.51', '1.2.4.1', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.12', '2.3.1.168', '2.3.1.61'], 'kegg.reaction': 'R00209 (R01699 + R02569 + R07618)', 'metanetx.reaction': 'MNXR188625', 'bigg.reaction': 'PDHm'}\n",
      "{'ec-code': ['2.3.3.1', '2.3.3.14', '2.3.3.16', '2.3.3.3', '4.1.3.7'], 'kegg.reaction': 'R00351', 'metanetx.reaction': 'MNXR190844', 'bigg.reaction': 'CSm', 'Human1': 'MAR04145'}\n",
      "{'ec-code': '4.2.1.3', 'kegg.reaction': 'R01324 (R01325 + R01900)', 'metanetx.reaction': 'MNXR191571', 'bigg.reaction': 'ACONTm'}\n",
      "{'ec-code': ['1.1.1.286', '1.1.1.41', '1.1.1.42', '1.1.1.87'], 'kegg.reaction': 'R00709', 'metanetx.reaction': 'MNXR100782', 'bigg.reaction': 'ICDHxm', 'Human1': 'MAR03957'}\n",
      "{'ec-code': '1.1.1.42', 'kegg.reaction': 'R00267 (R00268 + R01899)', 'bigg.reaction': 'ICDHyrm'}\n",
      "{'ec-code': ['1.2.1.105', '1.2.1.52', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.3.1.61'], 'kegg.reaction': 'R08549 (R00621+R02570+R03316+R07618)', 'metanetx.reaction': 'MNXR190634', 'bigg.reaction': 'AKGDm'}\n",
      "{'ec-code': '6.2.1.4', 'kegg.reaction': 'R00432', 'metanetx.reaction': 'MNXR188882', 'bigg.reaction': 'SUCOAS1m', 'Human1': 'MAR04147'}\n",
      "{'ec-code': '6.2.1.5', 'kegg.reaction': 'R00405', 'metanetx.reaction': 'MNXR188878', 'bigg.reaction': 'SUCOASm', 'Human1': 'MAR04152'}\n",
      "{'ec-code': '4.2.1.2', 'kegg.reaction': 'R01082', 'metanetx.reaction': 'MNXR99705', 'bigg.reaction': 'FUMm', 'Human1': 'MAR04408'}\n",
      "{'ec-code': ['1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82'], 'kegg.reaction': 'R00342', 'metanetx.reaction': 'MNXR101439', 'bigg.reaction': 'MDHm', 'Human1': 'MAR04139'}\n",
      "{'ec-code': ['1.6.5.3', '1.6.5.8', '1.6.5.9'], 'kegg.reaction': 'R02163 (R00281 + R02166)', 'metanetx.reaction': 'MNXR101867', 'bigg.reaction': 'NADH2_u10m'}\n",
      "{'ec-code': '1.3.5.1', 'kegg.reaction': 'R02164', 'bigg.reaction': 'SUCD1m and FADH2ETC'}\n",
      "{'ec-code': '1.10.2.2', 'kegg.reaction': 'R02161', 'metanetx.reaction': 'MNXR138194', 'bigg.reaction': 'CYOR_u10m', 'Human1': 'MAR06918'}\n",
      "{'ec-code': '1.9.3.1', 'kegg.reaction': 'R00081', 'metanetx.reaction': 'MNXR151671', 'bigg.reaction': 'CYOOm2', 'Human1': 'MAR06914'}\n",
      "{'ec-code': ['3.6.3.14', '7.1.2.2'], 'kegg.reaction': 'R00086', 'metanetx.reaction': 'MNXR153059', 'bigg.reaction': 'ATPS4m', 'Human1': 'MAR06916'}\n",
      "{'ec-code': ['4.1.1.32', '4.1.1.49'], 'kegg.reaction': 'R00431', 'metanetx.reaction': 'MNXR188627', 'bigg.reaction': 'PEPCKm', 'Human1': 'MAR04101'}\n",
      "{'ec-code': '6.4.1.1', 'kegg.reaction': 'R00344', 'metanetx.reaction': 'MNXR102391', 'bigg.reaction': 'PCm', 'Human1': 'MAR04143'}\n",
      "{'ec-code': ['1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42'], 'kegg.reaction': 'R00216', 'metanetx.reaction': 'MNXR101443', 'bigg.reaction': 'ME2m', 'Human1': 'MAR04089'}\n",
      "{'ec-code': ['1.1.1.38', '1.1.1.39', '1.1.1.40'], 'kegg.reaction': 'R00214', 'metanetx.reaction': 'MNXR101446', 'bigg.reaction': 'ME1m', 'Human1': 'MAR04087'}\n",
      "{'ec-code': '2.6.1.2', 'kegg.reaction': 'R00258', 'bigg.reaction': 'r0081', 'Human1': 'MAR03899'}\n",
      "{'ec-code': '2.3.3.8', 'kegg.reaction': 'R00352', 'Human1': 'MAR04149'}\n",
      "{'ec-code': '2.7.4.6', 'kegg.reaction': 'R00330', 'metanetx.reaction': 'MNXR190718', 'bigg.reaction': 'R_NDPK1m', 'Human1': 'MAR04016'}\n",
      "{'ec-code': '1.6.1.2', 'kegg.reaction': 'R00112', 'Human1': 'MAR04271'}\n",
      "{'ec-code': ['2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8'], 'kegg.reaction': 'R00127', 'metanetx.reaction': 'MNXR95450', 'bigg.reaction': 'ADK1m', 'Human1': 'MAR04002'}\n",
      "{'ec-code': ['1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42'], 'kegg.reaction': 'R00216', 'metanetx.reaction': 'MNXR101443', 'bigg.reaction': 'ME2', 'Human1': 'MAR04089'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.2', '2.6.1.5'], 'kegg.reaction': 'R00258', 'metanetx.reaction': 'MNXR95698', 'bigg.reaction': 'ALATA_L', 'Human1': 'MAR03899'}\n",
      "{'ec-code': '2.7.4.6', 'kegg.reaction': 'R00330', 'metanetx.reaction': 'MNXR190718', 'bigg.reaction': 'NDPK1', 'Human1': 'MAR04016'}\n",
      "{'ec-code': '4.2.1.2', 'kegg.reaction': 'R01082', 'metanetx.reaction': 'MNXR99705', 'bigg.reaction': 'FUM', 'Human1': 'MAR04408'}\n",
      "{'ec-code': ['2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8'], 'kegg.reaction': 'R00127', 'metanetx.reaction': 'MNXR95450', 'bigg.reaction': 'ADK1', 'Human1': 'MAR04002'}\n",
      "{'ec-code': '1.1.1.42', 'kegg.reaction': 'R00267 (R00268 + R01899)', 'bigg.reaction': 'ICDHy'}\n",
      "{'ec-code': '4.2.1.3', 'kegg.reaction': 'R01324 (R01325 + R01900)', 'metanetx.reaction': 'MNXR191571', 'bigg.reaction': 'ACONT', 'Human1': 'MAR04454'}\n",
      "{'ec-code': ['2.3.3.8', '4.1.3.6'], 'kegg.reaction': 'R00352', 'metanetx.reaction': 'MNXR190533', 'bigg.reaction': 'ACITL', 'Human1': 'MAR04149'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57'], 'kegg.reaction': 'R00355', 'metanetx.reaction': 'MNXR192556', 'bigg.reaction': 'ASPTA', 'Human1': 'MAR03827'}\n",
      "{'ec-code': ['1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82'], 'kegg.reaction': 'R00342', 'metanetx.reaction': 'MNXR101439', 'bigg.reaction': 'MDH', 'Human1': 'MAR04139'}\n",
      "{'metanetx.reaction': 'MNXR95659', 'bigg.reaction': 'AKGMALtm', 'Human1': 'MAR04852'}\n",
      "{'metanetx.reaction': 'MNXR96084', 'bigg.reaction': 'ASPGLUm', 'Human1': 'MAR03825'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57'], 'kegg.reaction': 'R00355', 'metanetx.reaction': 'MNXR192556', 'bigg.reaction': 'ASPTAm', 'Human1': 'MAR03827'}\n",
      "{'ec-code': ['1.1.1.8', '1.1.1.94', '1.1.5.3'], 'kegg.reaction': 'R00842', 'metanetx.reaction': 'MNXR99876', 'bigg.reaction': 'G3PD1', 'Human1': 'MAR00479'}\n",
      "{'ec-code': '1.1.5.3', 'kegg.reaction': 'R08657', 'metanetx.reaction': 'MNXR163340', 'bigg.reaction': 'r0205'}\n",
      "{'ec-code': '6.2.1.3', 'kegg.reaction': 'R01280', 'bigg.reaction': 'FACOAL160i', 'Human1': 'MAR00217'}\n",
      "{'ec-code': '2.3.1.21', 'kegg.reaction': 'R01923', 'metanetx.reaction': 'MNXR190744', 'bigg.reaction': 'C160CPT1', 'Human1': 'MAR02599'}\n",
      "{'ec-code': ['3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1'], 'kegg.reaction': 'R00004', 'metanetx.reaction': 'MNXR145285', 'bigg.reaction': 'PPA', 'Human1': 'MAR03975'}\n",
      "{'bigg.reaction': 'r2435'}\n",
      "{'ec-code': '2.3.1.21', 'kegg.reaction': 'R01923', 'metanetx.reaction': 'MNXR190744', 'bigg.reaction': 'C160CPT2', 'Human1': 'MAR02599'}\n",
      "{'ec-code': ['3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1'], 'kegg.reaction': 'R00004', 'metanetx.reaction': 'MNXR145285', 'bigg.reaction': 'PPAm', 'Human1': 'MAR03975'}\n",
      "{'ec-code': '3.1.2.2', 'kegg.reaction': 'R01274', 'Human1': 'MAR00709'}\n",
      "{'ec-code': ['1.3.8.8', '1.5.5.1'], 'kegg.reaction': 'R01279', 'metanetx.reaction': 'MNXR157364', 'bigg.reaction': 'FAOXC16C16OHm', 'Human1': 'MAR03121'}\n",
      "{'ec-code': '1.3.1.38', 'kegg.reaction': 'R01278'}\n",
      "{'ec-code': ['4.2.1.74', '1.1.1.211', '2.3.1.16'], 'kegg.reaction': 'R04738, R04737, R03991'}\n",
      "{'ec-code': ['1.3.8.8', '1.5.5.1'], 'kegg.reaction': 'R03990', 'metanetx.reaction': 'MNXR157338', 'bigg.reaction': 'FAOXC141C141OHm', 'Human1': 'MAR03128'}\n",
      "{'ec-code': '1.3.1.38', 'kegg.reaction': 'R03989'}\n",
      "{'ec-code': ['4.2.1.74', '1.1.1.211', '2.3.1.16'], 'kegg.reaction': 'R04740, R04739, R03858'}\n",
      "{'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R03857', 'metanetx.reaction': 'MNXR163361', 'bigg.reaction': 'r1447', 'Human1': 'MAR03135'}\n",
      "{'ec-code': '1.3.1.38', 'kegg.reaction': 'R03856', 'bigg.reaction': 'r0638'}\n",
      "{'ec-code': '4.2.1.17', 'kegg.reaction': 'R04170', 'bigg.reaction': 'r0660', 'Human1': 'MAR03087'}\n",
      "{'ec-code': '1.1.1.35', 'kegg.reaction': 'R04741', 'bigg.reaction': 'r0722', 'Human1': 'MAR03088'}\n",
      "{'ec-code': '2.3.1.16', 'kegg.reaction': 'R04742', 'bigg.reaction': 'r0724', 'Human1': 'MAR03089'}\n",
      "{'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R04754', 'metanetx.reaction': 'MNXR163365', 'bigg.reaction': 'r1451', 'Human1': 'MAR03142'}\n",
      "{'ec-code': '1.3.1.38', 'kegg.reaction': 'R04753', 'bigg.reaction': 'r0735'}\n",
      "{'ec-code': '4.2.1.17', 'kegg.reaction': 'R04744', 'bigg.reaction': 'r0728', 'Human1': 'MAR03091'}\n",
      "{'ec-code': '1.1.1.35', 'kegg.reaction': 'R04743', 'bigg.reaction': 'r0726', 'Human1': 'MAR03092'}\n",
      "{'ec-code': '2.3.1.16', 'kegg.reaction': 'R03778', 'bigg.reaction': 'r0634', 'Human1': 'MAR03093'}\n",
      "{'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R03777', 'metanetx.reaction': 'MNXR163362', 'bigg.reaction': 'r1448', 'Human1': 'MAR03149'}\n",
      "{'ec-code': ['1.3.1.10', '1.3.1.104', '1.3.1.38', '1.3.1.8'], 'kegg.reaction': 'R03776', 'metanetx.reaction': 'MNXR192338', 'bigg.reaction': 'r0633', 'Human1': 'MAR02148'}\n",
      "{'ec-code': ['4.2.1.17', '4.2.1.74'], 'kegg.reaction': 'R04746', 'metanetx.reaction': 'MNXR153432', 'bigg.reaction': 'r0731', 'Human1': 'MAR03095'}\n",
      "{'ec-code': ['1.1.1.211', '1.1.1.35'], 'kegg.reaction': 'R04745', 'metanetx.reaction': 'MNXR145162', 'bigg.reaction': 'r0730', 'Human1': 'MAR03096'}\n",
      "{'ec-code': ['2.3.1.16', '2.3.1.9'], 'kegg.reaction': 'R04747', 'metanetx.reaction': 'MNXR190498', 'bigg.reaction': 'r0732', 'Human1': 'MAR03097'}\n",
      "{'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R04751', 'metanetx.reaction': 'MNXR163364', 'bigg.reaction': 'r1450', 'Human1': 'MAR03156'}\n",
      "{'ec-code': '1.3.1.38', 'kegg.reaction': 'R06985', 'bigg.reaction': 'r0791'}\n",
      "{'ec-code': ['4.2.1.150', '4.2.1.17', '4.2.1.74'], 'kegg.reaction': 'R04749', 'metanetx.reaction': 'MNXR153431', 'bigg.reaction': 'r0734', 'Human1': 'MAR03099'}\n",
      "{'ec-code': ['1.1.1.211', '1.1.1.35'], 'kegg.reaction': 'R04748', 'metanetx.reaction': 'MNXR188200', 'bigg.reaction': 'r0733', 'Human1': 'MAR03100'}\n",
      "{'ec-code': ['2.3.1.16', '2.3.1.9'], 'kegg.reaction': 'R01177', 'metanetx.reaction': 'MNXR152737', 'bigg.reaction': 'r0287', 'Human1': 'MAR03101'}\n",
      "{'ec-code': ['1.3.8.1', '1.5.5.1'], 'kegg.reaction': 'R01175', 'metanetx.reaction': 'MNXR163360', 'bigg.reaction': 'r1446', 'Human1': 'MAR03163'}\n",
      "{'ec-code': ['4.2.1.116', '4.2.1.150', '4.2.1.17', '4.2.1.55'], 'kegg.reaction': 'R03026', 'metanetx.reaction': 'MNXR192124', 'bigg.reaction': 'ECOAH1m', 'Human1': 'MAR03164'}\n",
      "{'ec-code': ['1.1.1.178', '1.1.1.211', '1.1.1.35'], 'kegg.reaction': 'R01975', 'metanetx.reaction': 'MNXR189650', 'bigg.reaction': 'HACD1m', 'Human1': 'MAR03166'}\n",
      "{'ec-code': '2.3.1.9', 'kegg.reaction': 'R00238', 'bigg.reaction': 'ACACT1rm', 'Human1': 'MAR01434'}\n",
      "{'ec-code': '6.4.1.2', 'kegg.reaction': 'R00742', 'metanetx.reaction': 'MNXR190515', 'bigg.reaction': 'ACCOAC', 'Human1': 'MAR04295'}\n",
      "{'ec-code': ['2.3.1.0', '2.3.1.39', '2.3.1.85', '2.3.1.86'], 'kegg.reaction': 'R01626', 'metanetx.reaction': 'MNXR145554', 'bigg.reaction': 'MCOATA', 'Human1': 'MAR02151'}\n",
      "{'ec-code': ['2.3.1.179', '2.3.1.180', '2.3.1.38', '2.3.1.41', '2.3.1.85', '2.3.1.86'], 'kegg.reaction': 'R01624', 'metanetx.reaction': 'MNXR152846', 'bigg.reaction': 'ACOATA', 'Human1': 'MAR02150'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04355', 'bigg.reaction': 'r0678', 'Human1': 'MAR02152'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04533', 'bigg.reaction': 'r0691', 'Human1': 'MAR02153'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04428', 'bigg.reaction': 'r0681', 'Human1': 'MAR02154'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04430', 'bigg.reaction': 'r0682', 'Human1': 'MAR02155'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04952', 'bigg.reaction': 'r0760', 'Human1': 'MAR02156'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04953', 'bigg.reaction': 'r0761', 'Human1': 'MAR02157'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04954', 'bigg.reaction': 'r0762', 'Human1': 'MAR02158'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04956', 'bigg.reaction': 'r0763', 'Human1': 'MAR02159'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04957', 'bigg.reaction': 'r0764', 'Human1': 'MAR02160'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04536', 'bigg.reaction': 'r0694', 'Human1': 'MAR02161'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04537', 'bigg.reaction': 'r0695', 'Human1': 'MAR02162'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04959', 'bigg.reaction': 'r0765', 'Human1': 'MAR02163'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04960', 'bigg.reaction': 'r0766', 'Human1': 'MAR02164'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04534', 'bigg.reaction': 'r0692', 'Human1': 'MAR02165'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04535', 'bigg.reaction': 'r0693', 'Human1': 'MAR02166'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04962', 'bigg.reaction': 'r0767', 'Human1': 'MAR02167'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04963', 'bigg.reaction': 'r0768', 'Human1': 'MAR02168'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04964', 'bigg.reaction': 'r0769', 'Human1': 'MAR02169'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04965', 'bigg.reaction': 'r0770', 'Human1': 'MAR02170'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04725', 'bigg.reaction': 'r0712', 'Human1': 'MAR02171'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04726', 'bigg.reaction': 'r0713', 'Human1': 'MAR02173'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04566', 'bigg.reaction': 'r0701', 'Human1': 'MAR02174'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04568', 'bigg.reaction': 'r0702', 'Human1': 'MAR02175'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04967', 'bigg.reaction': 'r0771', 'Human1': 'MAR02176'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04968', 'bigg.reaction': 'r0772', 'Human1': 'MAR02178'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04543', 'bigg.reaction': 'r0696', 'Human1': 'MAR02179'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04544', 'bigg.reaction': 'r0697', 'Human1': 'MAR02180'}\n",
      "{'ec-code': '2.3.1.85', 'kegg.reaction': 'R04970', 'bigg.reaction': 'r0773', 'Human1': 'MAR02181'}\n",
      "{'ec-code': '3.1.2.14', 'kegg.reaction': 'R01706', 'bigg.reaction': 'FA160ACPH', 'Human1': 'MAR02182'}\n",
      "{'ec-code': ['6.2.1.1', '6.2.1.17', '6.2.1.2', '6.2.1.3'], 'kegg.reaction': 'R01176', 'metanetx.reaction': 'MNXR147173', 'bigg.reaction': 'FACOAL40im', 'Human1': 'MAR00156'}\n",
      "{'ec-code': '1.1.1.30', 'kegg.reaction': 'R01361', 'metanetx.reaction': 'MNXR96232', 'bigg.reaction': 'BDHm', 'Human1': 'MAR04461'}\n",
      "{'ec-code': ['2.8.3.18', '2.8.3.5'], 'kegg.reaction': 'R00410', 'metanetx.reaction': 'MNXR188565', 'bigg.reaction': 'OCOAT1m', 'Human1': 'MAR03787'}\n",
      "{'ec-code': '2.3.3.10', 'kegg.reaction': 'R01978', 'bigg.reaction': 'HMGCOASim', 'Human1': 'MAR04604'}\n",
      "{'ec-code': '4.1.3.4', 'kegg.reaction': 'R01360', 'metanetx.reaction': 'MNXR145225', 'bigg.reaction': 'HMGLm', 'Human1': 'MAR01577'}\n",
      "{'ec-code': ['2.6.1.42', '2.6.1.6', '2.6.1.67'], 'kegg.reaction': 'R01090', 'metanetx.reaction': 'MNXR192685', 'bigg.reaction': 'LEUTAm', 'Human1': 'MAR03777'}\n",
      "{'ec-code': ['1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168'], 'kegg.reaction': 'R07601, R7602, R04097, R07618', 'metanetx.reaction': 'MNXR188585', 'bigg.reaction': 'OIVD1m', 'Human1': 'MAR06421'}\n",
      "{'ec-code': '1.3.8.4', 'kegg.reaction': 'R04095', 'metanetx.reaction': 'MNXR163348', 'bigg.reaction': 'r0655', 'Human1': 'MAR03770'}\n",
      "{'ec-code': '6.4.1.4', 'kegg.reaction': 'R04138', 'metanetx.reaction': 'MNXR195395', 'bigg.reaction': 'MCCCrm', 'Human1': 'MAR03773'}\n",
      "{'ec-code': '4.2.1.18', 'kegg.reaction': 'R02085', 'metanetx.reaction': 'MNXR195397', 'bigg.reaction': 'MGCHrm', 'Human1': 'MAR03775'}\n",
      "{'ec-code': '2.6.1.42', 'kegg.reaction': 'R02199', 'metanetx.reaction': 'MNXR190730', 'bigg.reaction': 'ILETAm', 'Human1': 'MAR03778'}\n",
      "{'ec-code': ['1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168'], 'kegg.reaction': 'R07603, R07604, R03174, R07618', 'metanetx.reaction': 'MNXR188589', 'bigg.reaction': 'OIVD3m', 'Human1': 'MAR06419'}\n",
      "{'ec-code': '1.3.8.1', 'kegg.reaction': 'R03172', 'metanetx.reaction': 'MNXR163347', 'bigg.reaction': 'r0603', 'Human1': 'MAR03784'}\n",
      "{'ec-code': ['4.2.1.150', '4.2.1.17'], 'kegg.reaction': 'R04204', 'metanetx.reaction': 'MNXR190958', 'bigg.reaction': 'ECOAH9m', 'Human1': 'MAR03785'}\n",
      "{'ec-code': ['1.1.1.178', '1.1.1.35'], 'kegg.reaction': 'R04203', 'metanetx.reaction': 'MNXR188214', 'bigg.reaction': 'HACD9m', 'Human1': 'MAR03823'}\n",
      "{'ec-code': ['2.3.1.16', '2.3.1.9'], 'kegg.reaction': 'R00927', 'metanetx.reaction': 'MNXR190496', 'bigg.reaction': 'ACACT10m', 'Human1': 'MAR03206'}\n",
      "{'ec-code': '2.6.1.42', 'kegg.reaction': 'R01214', 'bigg.reaction': 'VALTAm', 'Human1': 'MAR03747'}\n",
      "{'ec-code': ['1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.7.11.4'], 'kegg.reaction': 'R07599, R07600, R02662, R07618', 'metanetx.reaction': 'MNXR188587', 'bigg.reaction': 'OIVD2m', 'Human1': 'MAR06417'}\n",
      "{'ec-code': '1.3.8.1', 'kegg.reaction': 'R02661', 'metanetx.reaction': 'MNXR163346', 'bigg.reaction': 'r0560'}\n",
      "{'ec-code': ['4.2.1.150', '4.2.1.17'], 'kegg.reaction': 'R04224', 'metanetx.reaction': 'MNXR189351', 'bigg.reaction': 'ECOAH12m', 'Human1': 'MAR03753'}\n",
      "{'ec-code': '3.1.2.4', 'kegg.reaction': 'R05064', 'metanetx.reaction': 'MNXR190062', 'bigg.reaction': '3HBCOAHLm'}\n",
      "{'ec-code': '1.1.1.31', 'kegg.reaction': 'R05066', 'metanetx.reaction': 'MNXR145203', 'bigg.reaction': 'HIBDm', 'Human1': 'MAR03757'}\n",
      "{'ec-code': ['6.2.1.1', '6.2.1.17'], 'kegg.reaction': 'R00925', 'metanetx.reaction': 'MNXR190519', 'bigg.reaction': 'ACCOALm', 'Human1': 'MAR00153'}\n",
      "{'ec-code': '1.2.1.27', 'kegg.reaction': 'R00935', 'metanetx.reaction': 'MNXR188437', 'bigg.reaction': 'MMSAD1m', 'Human1': 'MAR03795'}\n",
      "{'ec-code': ['6.4.1.2', '6.4.1.3'], 'kegg.reaction': 'R01859', 'metanetx.reaction': 'MNXR188686', 'bigg.reaction': 'PPCOACm', 'Human1': 'MAR03208'}\n",
      "{'ec-code': '5.1.99.1', 'kegg.reaction': 'R02765', 'metanetx.reaction': 'MNXR101657', 'bigg.reaction': 'MMEm', 'Human1': 'MAR03213'}\n",
      "{'ec-code': '5.4.99.2', 'kegg.reaction': 'R00833', 'metanetx.reaction': 'MNXR188435', 'bigg.reaction': 'MMMm', 'Human1': 'MAR03215'}\n",
      "{'ec-code': ['4.1.1.41', '4.1.1.9', '4.1.1.94', '7.2.4.3'], 'kegg.reaction': 'R00923', 'metanetx.reaction': 'MNXR191992', 'bigg.reaction': 'MMCDm', 'Human1': 'MAR08787'}\n",
      "{'ec-code': ['2.3.1.226', '2.3.1.30', '2.3.3.13', '3.1.2.17', '3.1.2.18', '3.1.2.2', '3.1.2.20', '3.1.2.23'], 'metanetx.reaction': 'MNXR188790', 'bigg.reaction': 'RE2649M'}\n",
      "{'ec-code': ['4.3.1.17', '4.3.1.19'], 'kegg.reaction': 'R00996', 'metanetx.reaction': 'MNXR104714', 'bigg.reaction': 'THRD_L', 'Human1': 'MAR04348'}\n",
      "{'metanetx.reaction': 'MNXR94813', 'bigg.reaction': 'r1155', 'Human1': 'MAR05348'}\n",
      "{'ec-code': ['1.2.4.4', '2.3.1.168', '1.8.1.4'], 'kegg.reaction': 'R10996, R10997, R10997, R07618', 'bigg.reaction': 'r1154'}\n",
      "{'ec-code': ['1.1.1.27', '1.1.1.272', '1.1.1.28'], 'kegg.reaction': 'R01000', 'metanetx.reaction': 'MNXR189543', 'bigg.reaction': '2HBO', 'Human1': 'MAR05351'}\n",
      "{'ec-code': '2.5.1.6', 'kegg.reaction': 'R00177', 'metanetx.reaction': 'MNXR188373', 'bigg.reaction': 'METAT', 'Human1': 'MAR03875'}\n",
      "{'ec-code': '2.1.1.-'}\n",
      "{'ec-code': '3.3.1.1', 'kegg.reaction': 'R00192', 'bigg.reaction': 'AHC', 'Human1': 'MAR03877'}\n",
      "{'ec-code': ['2.7.1.11', '2.7.1.20', '2.7.1.74'], 'kegg.reaction': 'R00185', 'metanetx.reaction': 'MNXR190585', 'bigg.reaction': 'ADNK1', 'Human1': 'MAR04012'}\n",
      "{'ec-code': '4.2.1.22', 'kegg.reaction': 'R01290', 'metanetx.reaction': 'MNXR97019', 'bigg.reaction': 'CYSTS', 'Human1': 'MAR03879'}\n",
      "{'ec-code': '4.4.1.1', 'kegg.reaction': 'R01001', 'metanetx.reaction': 'MNXR97012', 'bigg.reaction': 'CYSTGL', 'Human1': 'MAR03881'}\n",
      "{'ec-code': '1.13.11.20', 'kegg.reaction': 'R00893', 'metanetx.reaction': 'MNXR96993', 'bigg.reaction': 'CYSO', 'Human1': 'MAR08683'}\n",
      "{'ec-code': '2.6.1.1', 'kegg.reaction': 'R02619', 'metanetx.reaction': 'MNXR147724', 'bigg.reaction': '3SALATAi', 'Human1': 'MAR03865'}\n",
      "{'kegg.reaction': 'R04861', 'metanetx.reaction': 'MNXR94981', 'bigg.reaction': '3SPYRSP', 'Human1': 'MAR07261'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.3'], 'kegg.reaction': 'R00895', 'metanetx.reaction': 'MNXR153293', 'bigg.reaction': 'CYSTA', 'Human1': 'MAR03912'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.3'], 'kegg.reaction': 'R00895', 'metanetx.reaction': 'MNXR153293', 'bigg.reaction': 'CYSTAm', 'Human1': 'MAR03912'}\n",
      "{'ec-code': '2.8.1.2', 'kegg.reaction': 'R03106', 'metanetx.reaction': 'MNXR151714', 'bigg.reaction': 'MCPST', 'Human1': 'MAR06518'}\n",
      "{'ec-code': '2.8.1.2', 'kegg.reaction': 'R03106', 'Human1': 'MAR06518'}\n",
      "{'ec-code': '2.8.1.2', 'kegg.reaction': 'R03105', 'Human1': 'MAR03928'}\n",
      "{'ec-code': '2.8.1.2', 'kegg.reaction': 'R03105', 'metanetx.reaction': 'MNXR146789', 'bigg.reaction': 'r0595', 'Human1': 'MAR03928'}\n",
      "{'ec-code': '1.1.1.27', 'kegg.reaction': 'R03104', 'bigg.reaction': 'MCLOR', 'Human1': 'MAR06519'}\n",
      "{'ec-code': ['4.1.99.1', '4.4.1.1', '4.4.1.11', '4.4.1.13', '4.4.1.28', '4.4.1.3', '4.4.1.8'], 'kegg.reaction': 'R00782', 'metanetx.reaction': 'MNXR96994', 'bigg.reaction': 'r0193', 'Human1': 'MAR04302'}\n",
      "{'ec-code': ['1.13.11.11', '1.13.11.52'], 'kegg.reaction': 'R00678', 'metanetx.reaction': 'MNXR146592', 'bigg.reaction': 'TRPO2', 'Human1': 'MAR04214'}\n",
      "{'ec-code': '3.5.1.9', 'kegg.reaction': 'R01959', 'metanetx.reaction': 'MNXR153574', 'bigg.reaction': 'FKYNH', 'Human1': 'MAR04216'}\n",
      "{'ec-code': '1.14.13.9', 'kegg.reaction': 'R01960', 'metanetx.reaction': 'MNXR100947', 'bigg.reaction': 'KYN3OX', 'Human1': 'MAR04220'}\n",
      "{'ec-code': '3.7.1.3', 'kegg.reaction': 'R02668', 'metanetx.reaction': 'MNXR100656', 'bigg.reaction': 'HKYNH', 'Human1': 'MAR04225'}\n",
      "{'ec-code': '1.13.11.6', 'kegg.reaction': 'R02665', 'metanetx.reaction': 'MNXR151079', 'bigg.reaction': '3HAO', 'Human1': 'MAR04228'}\n",
      "{'ec-code': '4.1.1.45', 'kegg.reaction': 'R04323', 'metanetx.reaction': 'MNXR189778', 'bigg.reaction': 'PCLAD', 'Human1': 'MAR04231'}\n",
      "{'ec-code': '1.2.1.32', 'kegg.reaction': 'R03889', 'bigg.reaction': 'r0645', 'Human1': 'MAR04233'}\n",
      "{'kegg.reaction': 'R01938', 'metanetx.reaction': 'MNXR152977', 'bigg.reaction': 'AMCOXO', 'Human1': 'MAR04235'}\n",
      "{'ec-code': '1.5.1.-', 'kegg.reaction': 'R01937', 'bigg.reaction': 'n/a'}\n",
      "{'metanetx.reaction': 'MNXR94819', 'bigg.reaction': '2OXOADPTm', 'kegg.reaction': 'n/a', 'Human1': 'MAR04237'}\n",
      "{'metanetx.reaction': 'MNXR94819', 'bigg.reaction': '2OXOADPTm', 'kegg.reaction': 'n/a', 'Human1': 'MAR04237'}\n",
      "{'ec-code': ['1.2.1.105', '1.2.1.52', '1.2.4.2', '1.8.1.4', '2.3.1.61'], 'kegg.reaction': 'R01933', 'metanetx.reaction': 'MNXR190426', 'bigg.reaction': '2OXOADOXm', 'Human1': 'MAR04599'}\n",
      "{'ec-code': '1.3.8.6', 'kegg.reaction': 'R02487', 'metanetx.reaction': 'MNXR163345', 'bigg.reaction': 'r0541', 'Human1': 'MAR04243'}\n",
      "{'ec-code': '1.5.1.8', 'kegg.reaction': 'R00716', 'metanetx.reaction': 'MNXR188818', 'bigg.reaction': 'SACCD3m', 'Human1': 'MAR04288'}\n",
      "{'ec-code': '1.5.1.9', 'kegg.reaction': 'R02313', 'metanetx.reaction': 'MNXR146786', 'bigg.reaction': 'r0525', 'Human1': 'MAR04667'}\n",
      "{'ec-code': '1.2.1.31', 'kegg.reaction': 'R03102', 'metanetx.reaction': 'MNXR151528', 'bigg.reaction': 'AASAD3m', 'Human1': 'MAR04737'}\n",
      "{'ec-code': '1.2.1.31', 'kegg.reaction': 'R03103', 'bigg.reaction': 'r0594'}\n",
      "{'ec-code': '2.6.1.39', 'kegg.reaction': 'R01939', 'metanetx.reaction': 'MNXR95160', 'bigg.reaction': 'r0450', 'Human1': 'MAR04596'}\n",
      "{'ec-code': ['1.4.3.14', '1.4.3.2'], 'kegg.reaction': 'R00447', 'metanetx.reaction': 'MNXR188329', 'bigg.reaction': 'LYSOXp', 'Human1': 'MAR08017'}\n",
      "{'kegg.reaction': 'R04175', 'metanetx.reaction': 'MNXR95853', 'bigg.reaction': 'PPD2CSPp', 'Human1': 'MAR08018'}\n",
      "{'ec-code': ['1.5.1.1', '1.5.1.21', '1.5.1.49'], 'kegg.reaction': 'R02201', 'metanetx.reaction': 'MNXR94712', 'bigg.reaction': '1PPDCRp', 'Human1': 'MAR08019'}\n",
      "{'ec-code': ['1.5.1.1', '1.5.1.21'], 'kegg.reaction': 'R02203'}\n",
      "{'ec-code': '1.5.3.7', 'kegg.reaction': 'R02204', 'metanetx.reaction': 'MNXR107370', 'bigg.reaction': 'LPCOXp', 'Human1': 'MAR08021'}\n",
      "{'ec-code': '1.4.1.18', 'kegg.reaction': 'R02317', 'bigg.reaction': 'RE1254C', 'Human1': 'MAR04740'}\n",
      "{'ec-code': '1.2.1.31', 'kegg.reaction': 'R03102', 'bigg.reaction': 'r0594', 'Human1': 'MAR04737'}\n",
      "{'metanetx.reaction': 'MNXR94780', 'bigg.reaction': '2AMADPTm', 'kegg.reaction': 'n/a', 'Human1': 'MAR08022'}\n",
      "{'metanetx.reaction': 'MNXR94780', 'bigg.reaction': '2AMADPTm', 'kegg.reaction': 'n/a', 'Human1': 'MAR08022'}\n",
      "{'ec-code': '1.5.99.8', 'kegg.reaction': 'R01253', 'metanetx.reaction': 'MNXR192607', 'bigg.reaction': 'PROD2m', 'Human1': 'MAR08611'}\n",
      "{'kegg.reaction': 'R03314', 'metanetx.reaction': 'MNXR153703', 'bigg.reaction': 'G5SADrm', 'Human1': 'MAR03819'}\n",
      "{'ec-code': ['1.2.1.24', '1.2.1.5', '1.2.1.88', '1.5.1.12'], 'kegg.reaction': 'R00245', 'metanetx.reaction': 'MNXR103352', 'bigg.reaction': 'r0074', 'Human1': 'MAR03806'}\n",
      "{'ec-code': '2.7.2.11', 'kegg.reaction': 'R00239', 'metanetx.reaction': 'MNXR100277', 'bigg.reaction': 'GLU5Km', 'Human1': 'MAR03895'}\n",
      "{'ec-code': '1.2.1.41', 'kegg.reaction': 'R03313', 'metanetx.reaction': 'MNXR99896', 'bigg.reaction': 'G5SDym', 'Human1': 'MAR03897'}\n",
      "{'ec-code': '1.5.1.2', 'kegg.reaction': 'R01251', 'metanetx.reaction': 'MNXR102302', 'bigg.reaction': 'P5CRm', 'Human1': 'MAR03833'}\n",
      "{'ec-code': ['1.5.1.2', '1.5.99.8'], 'kegg.reaction': 'R01248', 'metanetx.reaction': 'MNXR102303', 'bigg.reaction': 'P5CRxm', 'Human1': 'MAR03835'}\n",
      "{'ec-code': ['2.6.1.11', '2.6.1.13'], 'kegg.reaction': 'R00667', 'metanetx.reaction': 'MNXR102220', 'bigg.reaction': 'ORNTArm', 'Human1': 'MAR03807'}\n",
      "{'ec-code': ['4.1.1.17', '4.1.1.19', '4.1.1.86'], 'kegg.reaction': 'R00670', 'metanetx.reaction': 'MNXR102209', 'bigg.reaction': 'ORNDC', 'Human1': 'MAR04212'}\n",
      "{'ec-code': ['1.4.3.10', '1.4.3.21', '1.4.3.22', '1.4.3.6'], 'kegg.reaction': 'R01151', 'metanetx.reaction': 'MNXR103334', 'bigg.reaction': 'PTRCOX1', 'Human1': 'MAR04423'}\n",
      "{'ec-code': '1.2.1.3', 'kegg.reaction': 'R01986', 'bigg.reaction': 'r0464', 'Human1': 'MAR04605'}\n",
      "{'ec-code': ['1.2.1.19', '1.2.1.3'], 'kegg.reaction': 'R02549', 'metanetx.reaction': 'MNXR95191', 'bigg.reaction': 'ABUTD', 'Human1': 'MAR04688'}\n",
      "{'ec-code': '4.1.1.19', 'kegg.reaction': 'R00566', 'metanetx.reaction': 'MNXR95940', 'bigg.reaction': 'ARGDCm', 'Human1': 'MAR04191'}\n",
      "{'ec-code': ['3.5.3.1', '3.5.3.11'], 'kegg.reaction': 'R01157', 'metanetx.reaction': 'MNXR95554', 'bigg.reaction': 'AGMTm', 'Human1': 'MAR04424'}\n",
      "{'ec-code': '2.3.1.57', 'kegg.reaction': 'R01154', 'metanetx.reaction': 'MNXR188722', 'bigg.reaction': 'PTRCAT1', 'Human1': 'MAR08603'}\n",
      "{'ec-code': '1.4.3.4', 'kegg.reaction': 'R04025', 'metanetx.reaction': 'MNXR95900', 'bigg.reaction': 'APRTO2', 'Human1': 'MAR08604'}\n",
      "{'ec-code': '1.2.1.3', 'kegg.reaction': 'R05050', 'metanetx.reaction': 'MNXR101805', 'bigg.reaction': 'NABTNOm', 'Human1': 'MAR08605'}\n",
      "{'ec-code': '3.5.1.63', 'kegg.reaction': 'R01987'}\n",
      "{'ec-code': '4.1.1.15', 'kegg.reaction': 'R00261', 'metanetx.reaction': 'MNXR95942', 'bigg.reaction': 'GLUDC', 'Human1': 'MAR04690'}\n",
      "{'metanetx.reaction': 'MNXR94993', 'bigg.reaction': '4ABUTtm', 'Human1': 'MAR04692'}\n",
      "{'ec-code': '2.6.1.19', 'kegg.reaction': 'R01648', 'metanetx.reaction': 'MNXR95186', 'bigg.reaction': 'ABTArm', 'Human1': 'MAR04693'}\n",
      "{'ec-code': '1.2.1.24', 'kegg.reaction': 'R00713', 'bigg.reaction': 'r0178', 'Human1': 'MAR04285'}\n",
      "{'ec-code': '1.4.1.3', 'kegg.reaction': 'R00243', 'bigg.reaction': 'GLUDxm', 'Human1': 'MAR03802'}\n",
      "{'ec-code': ['1.4.1.13', '1.4.1.3', '1.4.1.4'], 'kegg.reaction': 'R00248', 'metanetx.reaction': 'MNXR100086', 'bigg.reaction': 'GLUDym', 'Human1': 'MAR03804'}\n",
      "{'ec-code': ['1.4.1.14', '1.4.1.2', '1.4.1.3', '1.4.1.4'], 'kegg.reaction': 'R00243', 'metanetx.reaction': 'MNXR144972', 'bigg.reaction': 'GLUDxi', 'Human1': 'MAR03802'}\n",
      "{'ec-code': ['1.4.1.13', '1.4.1.3', '1.4.1.4'], 'kegg.reaction': 'R00248', 'metanetx.reaction': 'MNXR100086', 'bigg.reaction': 'GLUDy', 'Human1': 'MAR03804'}\n",
      "{'ec-code': '6.3.1.2', 'kegg.reaction': 'R00253', 'metanetx.reaction': 'MNXR100024', 'bigg.reaction': 'GLNS', 'Human1': 'MAR03890'}\n",
      "{'ec-code': ['1.4.1.13', '1.4.1.14', '1.4.7.1', '2.4.2.14', '3.5.1.2', '3.5.1.38', '4.3.3.6', '6.3.4.2', '6.3.5.2', '6.3.5.4', '6.3.5.5', '6.3.5.7'], 'kegg.reaction': 'R00256', 'metanetx.reaction': 'MNXR100030', 'bigg.reaction': 'GLUNm', 'Human1': 'MAR03892'}\n",
      "{'ec-code': '3.5.1.2', 'kegg.reaction': 'R00256', 'Human1': 'MAR03892'}\n",
      "{'ec-code': '1.1.1.95', 'kegg.reaction': 'R01513', 'metanetx.reaction': 'MNXR102527', 'bigg.reaction': 'PGCD', 'Human1': 'MAR03839'}\n",
      "{'ec-code': '2.6.1.52', 'kegg.reaction': 'R04173', 'metanetx.reaction': 'MNXR162393', 'bigg.reaction': 'PSERT', 'Human1': 'MAR03841'}\n",
      "{'ec-code': ['3.1.3.2', '3.1.3.29', '3.1.3.3'], 'kegg.reaction': 'R00582', 'metanetx.reaction': 'MNXR162386', 'bigg.reaction': 'PSP_L', 'Human1': 'MAR03843'}\n",
      "{'ec-code': '2.1.2.1', 'kegg.reaction': 'R00945', 'metanetx.reaction': 'MNXR188117', 'bigg.reaction': 'GHMT2r', 'Human1': 'MAR03845'}\n",
      "{'ec-code': ['1.5.1.3', '1.5.1.33'], 'kegg.reaction': 'R02236', 'metanetx.reaction': 'MNXR97400', 'bigg.reaction': 'FOLR2', 'Human1': 'MAR04655'}\n",
      "{'ec-code': ['1.5.1.3', '1.5.1.5'], 'kegg.reaction': 'R00939', 'metanetx.reaction': 'MNXR190914', 'bigg.reaction': 'DHFR', 'Human1': 'MAR04333'}\n",
      "{'ec-code': ['1.5.1.15', '1.5.1.5'], 'kegg.reaction': 'R01220', 'metanetx.reaction': 'MNXR188456', 'bigg.reaction': 'MTHFD', 'Human1': 'MAR04442'}\n",
      "{'ec-code': ['1.5.1.15', '3.5.4.9'], 'kegg.reaction': 'R01655', 'metanetx.reaction': 'MNXR188454', 'bigg.reaction': 'MTHFC', 'Human1': 'MAR04503'}\n",
      "{'ec-code': '4.3.1.4', 'kegg.reaction': 'R02302', 'metanetx.reaction': 'MNXR192430', 'bigg.reaction': 'FTCD', 'Human1': 'MAR03929'}\n",
      "{'ec-code': '6.3.4.3', 'kegg.reaction': 'R00943', 'bigg.reaction': 'FTHFL', 'Human1': 'MAR04338'}\n",
      "{'ec-code': '1.5.1.6', 'kegg.reaction': 'R00941', 'metanetx.reaction': 'MNXR191097', 'bigg.reaction': 'FTHFDH', 'Human1': 'MAR04336'}\n",
      "{'ec-code': ['4.3.1.17', '4.3.1.19'], 'kegg.reaction': 'R00220', 'bigg.reaction': 'r0060', 'Human1': 'MAR03883'}\n",
      "{'ec-code': '2.1.2.1', 'kegg.reaction': 'R00945', 'metanetx.reaction': 'MNXR188117', 'bigg.reaction': 'GHMT2rm', 'Human1': 'MAR03845'}\n",
      "{'ec-code': '1.4.4.2', 'kegg.reaction': 'R03425', 'metanetx.reaction': 'MNXR144960', 'bigg.reaction': 'GCCam', 'Human1': 'MAR08434'}\n",
      "{'ec-code': '2.1.2.10', 'kegg.reaction': 'R04125', 'metanetx.reaction': 'MNXR144964', 'bigg.reaction': 'GCCbim'}\n",
      "{'ec-code': '1.8.1.4', 'kegg.reaction': 'R03815', 'metanetx.reaction': 'MNXR144966', 'bigg.reaction': 'GCCcm'}\n",
      "{'ec-code': ['1.5.1.3', '1.5.1.33'], 'kegg.reaction': 'R02236', 'metanetx.reaction': 'MNXR97400', 'bigg.reaction': 'r0514', 'Human1': 'MAR04655'}\n",
      "{'ec-code': '1.5.1.3', 'kegg.reaction': 'R00939', 'bigg.reaction': 'r0226', 'Human1': 'MAR04333'}\n",
      "{'ec-code': ['1.5.1.15', '1.5.1.5'], 'kegg.reaction': 'R01220', 'metanetx.reaction': 'MNXR188456', 'bigg.reaction': 'MTHFDm', 'Human1': 'MAR04442'}\n",
      "{'ec-code': ['1.5.1.15', '1.5.1.5'], 'kegg.reaction': 'R01218', 'metanetx.reaction': 'MNXR188458', 'bigg.reaction': 'MTHFD2m', 'Human1': 'MAR04440'}\n",
      "{'ec-code': ['1.5.1.15', '3.5.4.9'], 'kegg.reaction': 'R01655', 'metanetx.reaction': 'MNXR188454', 'bigg.reaction': 'MTHFCm', 'Human1': 'MAR04503'}\n",
      "{'ec-code': '6.3.4.3', 'kegg.reaction': 'R00943', 'bigg.reaction': 'FTHFLm', 'Human1': 'MAR04338'}\n",
      "{'ec-code': '1.5.1.6', 'kegg.reaction': 'R00941', 'Human1': 'MAR04336'}\n",
      "{'ec-code': ['2.3.1.29', '2.3.1.37'], 'kegg.reaction': 'R00371', 'metanetx.reaction': 'MNXR188139', 'bigg.reaction': 'GLYATm', 'Human1': 'MAR03847'}\n",
      "{'kegg.reaction': 'R03758', 'metanetx.reaction': 'MNXR95851', 'bigg.reaction': 'AOBUTDsm', 'Human1': 'MAR03860', 'ec-code': '1.1.1.381'}\n",
      "{'ec-code': ['1.4.3.21', '1.4.3.4', '1.4.3.6'], 'kegg.reaction': 'R02529', 'metanetx.reaction': 'MNXR95149', 'bigg.reaction': 'AACTOOR', 'Human1': 'MAR03852'}\n",
      "{'ec-code': '4.4.1.5', 'kegg.reaction': 'R02530', 'metanetx.reaction': 'MNXR100355', 'bigg.reaction': 'LGTHL', 'Human1': 'MAR03853'}\n",
      "{'ec-code': ['3.1.2.12', '3.1.2.6'], 'kegg.reaction': 'R01736', 'metanetx.reaction': 'MNXR100353', 'bigg.reaction': 'GLYOXm', 'Human1': 'MAR03857'}\n",
      "{'ec-code': ['1.1.1.27', '1.1.1.28'], 'kegg.reaction': 'R00704', 'metanetx.reaction': 'MNXR101037', 'bigg.reaction': 'LDH_D', 'Human1': 'MAR08512'}\n",
      "{'ec-code': ['6.3.4.16', '6.3.5.5'], 'kegg.reaction': 'R00149', 'metanetx.reaction': 'MNXR96486', 'bigg.reaction': 'CBPSam', 'Human1': 'MAR03873'}\n",
      "{'ec-code': ['2.1.3.3', '2.1.3.6'], 'kegg.reaction': 'R01398', 'metanetx.reaction': 'MNXR102137', 'bigg.reaction': 'OCBTm', 'Human1': 'MAR03809'}\n",
      "{'ec-code': '1.14.13.39', 'kegg.reaction': 'R00558', 'metanetx.reaction': 'MNXR102006', 'bigg.reaction': 'NOS1', 'Human1': 'MAR04190'}\n",
      "{'ec-code': '1.14.13.39', 'kegg.reaction': 'R00111', 'metanetx.reaction': 'MNXR102007', 'bigg.reaction': 'NOS2', 'Human1': 'MAR03993'}\n",
      "{'ec-code': '3.4.19.13', 'kegg.reaction': 'R00494', 'bigg.reaction': 'r0129', 'Human1': 'MAR04175'}\n",
      "{'ec-code': ['3.4.11.1', '3.4.11.2', '3.4.11.23', '3.4.13.18'], 'kegg.reaction': 'R00899', 'metanetx.reaction': 'MNXR95828', 'bigg.reaction': 'AMPTASECG', 'Human1': 'MAR04328'}\n",
      "{'ec-code': '6.3.2.2', 'kegg.reaction': 'R00894', 'metanetx.reaction': 'MNXR100074', 'bigg.reaction': 'GLUCYS', 'Human1': 'MAR04324'}\n",
      "{'ec-code': '6.3.2.3', 'kegg.reaction': 'R00497', 'metanetx.reaction': 'MNXR100450', 'bigg.reaction': 'GTHS', 'Human1': 'MAR04326'}\n",
      "{'ec-code': '1.14.16.1', 'kegg.reaction': 'R01795', 'bigg.reaction': 'r0399'}\n",
      "{'ec-code': '1.5.1.34', 'kegg.reaction': 'R01793', 'metanetx.reaction': 'MNXR189529', 'bigg.reaction': 'DHPR', 'Human1': 'MAR03925'}\n",
      "{'ec-code': ['2.6.1.1', '2.6.1.27', '2.6.1.42', '2.6.1.5', '2.6.1.57', '2.6.1.9'], 'kegg.reaction': 'R00734', 'metanetx.reaction': 'MNXR105000', 'bigg.reaction': 'TYRTA', 'Human1': 'MAR06768'}\n",
      "{'ec-code': '1.4.3.2', 'kegg.reaction': 'R00729', 'Human1': 'MAR06770'}\n",
      "{'ec-code': '1.13.11.27', 'kegg.reaction': 'R02521', 'metanetx.reaction': 'MNXR94843', 'bigg.reaction': '34HPPOR', 'Human1': 'MAR06772'}\n",
      "{'ec-code': '1.13.11.5', 'kegg.reaction': 'R02519', 'metanetx.reaction': 'MNXR100628', 'bigg.reaction': 'HGNTOR', 'Human1': 'MAR06774'}\n",
      "{'ec-code': '5.2.1.2', 'kegg.reaction': 'R03181', 'metanetx.reaction': 'MNXR101325', 'bigg.reaction': 'MACACI', 'Human1': 'MAR06776'}\n",
      "{'ec-code': '3.7.1.2', 'kegg.reaction': 'R01364', 'metanetx.reaction': 'MNXR99706', 'bigg.reaction': 'FUMAC', 'Human1': 'MAR06778'}\n",
      "{'ec-code': '6.3.5.4', 'kegg.reaction': 'R00578', 'metanetx.reaction': 'MNXR153021', 'bigg.reaction': 'ASNS1', 'Human1': 'MAR03903'}\n",
      "{'ec-code': '3.5.1.1', 'kegg.reaction': 'R00485', 'bigg.reaction': 'r0127', 'Human1': 'MAR03862'}\n",
      "{'ec-code': '4.3.1.3', 'kegg.reaction': 'R01168', 'bigg.reaction': 'HISD', 'Human1': 'MAR04437'}\n",
      "{'ec-code': '4.2.1.49', 'kegg.reaction': 'R02914', 'metanetx.reaction': 'MNXR192047', 'bigg.reaction': 'URCN', 'Human1': 'MAR04712'}\n",
      "{'ec-code': '3.5.2.7', 'kegg.reaction': 'R02288', 'metanetx.reaction': 'MNXR145318', 'bigg.reaction': 'IZPN', 'Human1': 'MAR04660'}\n",
      "{'ec-code': ['2.1.2.5', '4.3.1.4'], 'kegg.reaction': 'R02287', 'metanetx.reaction': 'MNXR192588', 'bigg.reaction': 'GluForTx', 'Human1': 'MAR04658'}\n",
      "{'ec-code': ['2.6.1.19', '2.6.1.55'], 'kegg.reaction': 'R00908', 'metanetx.reaction': 'MNXR95862', 'bigg.reaction': 'APAT2rm', 'Human1': 'MAR04330'}\n",
      "{'ec-code': ['1.2.1.18', '1.2.1.27'], 'kegg.reaction': 'R00705', 'metanetx.reaction': 'MNXR188441', 'bigg.reaction': 'MMSAD3m', 'Human1': 'MAR04282'}\n",
      "{'ec-code': '1.2.1.18', 'kegg.reaction': 'R00706'}\n",
      "{'ec-code': ['4.1.1.11', '4.1.1.15'], 'kegg.reaction': 'R00489', 'metanetx.reaction': 'MNXR96077', 'bigg.reaction': 'ASP1DC', 'Human1': 'MAR04174'}\n",
      "{'ec-code': '2.7.3.2', 'kegg.reaction': 'R01881', 'metanetx.reaction': 'MNXR96757', 'bigg.reaction': 'CKc', 'Human1': 'MAR08427'}\n",
      "{'ec-code': '2.7.3.2', 'kegg.reaction': 'R01881', 'metanetx.reaction': 'MNXR96757', 'bigg.reaction': 'CK', 'Human1': 'MAR08427'}\n",
      "{'ec-code': '3.1.2.1', 'kegg.reaction': 'R00227', 'bigg.reaction': 'ACOAHi', 'Human1': 'MAR04093'}\n",
      "{'ec-code': '1.1.1.2', 'kegg.reaction': 'R00746', 'bigg.reaction': 'ALCD2yf', 'Human1': 'MAR03907'}\n",
      "{'ec-code': '1.1.1.1', 'kegg.reaction': 'R00754', 'bigg.reaction': 'ALCD2if', 'Human1': 'MAR03905'}\n",
      "{'metanetx.reaction': 'MNXR95212', 'bigg.reaction': 'ACALDtm', 'Human1': 'MAR04948'}\n",
      "{'ec-code': ['1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88'], 'kegg.reaction': 'R00710', 'metanetx.reaction': 'MNXR95749', 'bigg.reaction': 'ALDD2xm', 'Human1': 'MAR01568'}\n",
      "{'ec-code': ['1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88'], 'kegg.reaction': 'R00710', 'metanetx.reaction': 'MNXR95749', 'bigg.reaction': 'ALDD2x', 'Human1': 'MAR01568'}\n",
      "{'ec-code': ['6.2.1.1', '6.2.1.17'], 'kegg.reaction': 'R00235 (R00316 + R00236)', 'metanetx.reaction': 'MNXR190571', 'bigg.reaction': 'ACSm', 'Human1': 'MAR04099'}\n",
      "{'ec-code': ['6.2.1.1', '6.2.1.17'], 'kegg.reaction': 'R00235 (R00316 + R00236)', 'metanetx.reaction': 'MNXR190571', 'bigg.reaction': 'ACS', 'Human1': 'MAR04097'}\n",
      "{'ec-code': '4.3.2.2', 'kegg.reaction': 'R01083', 'bigg.reaction': 'ADSL1', 'Human1': 'MAR04412'}\n",
      "{'ec-code': '6.3.4.4', 'kegg.reaction': 'R01135', 'metanetx.reaction': 'MNXR192666', 'bigg.reaction': 'ADSS', 'Human1': 'MAR04042'}\n",
      "{'ec-code': ['3.5.4.17', '3.5.4.6'], 'kegg.reaction': 'R00181', 'metanetx.reaction': 'MNXR190279', 'bigg.reaction': 'AMPDA', 'Human1': 'MAR04080'}\n",
      "{'ec-code': ['3.5.3.1', '3.5.3.7'], 'kegg.reaction': 'R00551', 'metanetx.reaction': 'MNXR95945', 'bigg.reaction': 'ARGN', 'Human1': 'MAR03816'}\n",
      "{'ec-code': '4.3.2.1', 'kegg.reaction': 'R01086', 'metanetx.reaction': 'MNXR138059', 'bigg.reaction': 'ARGSL', 'Human1': 'MAR03813'}\n",
      "{'ec-code': '6.3.4.5', 'kegg.reaction': 'R01954', 'metanetx.reaction': 'MNXR138060', 'bigg.reaction': 'ARGSS', 'Human1': 'MAR03811'}\n",
      "{'ec-code': ['3.5.3.1', '3.5.3.7'], 'kegg.reaction': 'R00551', 'metanetx.reaction': 'MNXR95945', 'bigg.reaction': 'ARGNm', 'Human1': 'MAR03816'}\n",
      "{'ec-code': '2.3.1.37', 'kegg.reaction': 'R00830', 'metanetx.reaction': 'MNXR190650', 'bigg.reaction': 'ALASm', 'Human1': 'MAR04742'}\n",
      "{'metanetx.reaction': 'MNXR95062', 'bigg.reaction': '5AOPtm', 'Human1': 'MAR04743'}\n",
      "{'ec-code': '4.2.1.24', 'kegg.reaction': 'R00036', 'metanetx.reaction': 'MNXR103095', 'bigg.reaction': 'PPBNGS', 'Human1': 'MAR04744'}\n",
      "{'ec-code': ['2.5.1.61', '4.3.1.8'], 'kegg.reaction': 'R00084', 'metanetx.reaction': 'MNXR100658', 'bigg.reaction': 'HMBS', 'Human1': 'MAR04746'}\n",
      "{'ec-code': '4.2.1.75', 'kegg.reaction': 'R03165', 'metanetx.reaction': 'MNXR105139', 'bigg.reaction': 'UPP3S', 'Human1': 'MAR04748'}\n",
      "{'ec-code': '4.1.1.37', 'kegg.reaction': 'R03197', 'metanetx.reaction': 'MNXR105140', 'bigg.reaction': 'UPPDC1', 'Human1': 'MAR04750'}\n",
      "{'ec-code': '1.3.3.3', 'kegg.reaction': 'R03220', 'metanetx.reaction': 'MNXR96880', 'bigg.reaction': 'CPPPGO', 'Human1': 'MAR04752'}\n",
      "{'ec-code': '1.3.3.4', 'kegg.reaction': 'R03222', 'metanetx.reaction': 'MNXR146109', 'bigg.reaction': 'PPPGOm', 'Human1': 'MAR04755'}\n",
      "{'ec-code': '4.99.1.1', 'kegg.reaction': 'R00310', 'metanetx.reaction': 'MNXR99471', 'bigg.reaction': 'FCLTm', 'Human1': 'MAR01044'}\n",
      "{'metanetx.reaction': 'MNXR102625', 'bigg.reaction': 'PHEMEtm', 'Human1': 'MAR07735'}\n",
      "{'ec-code': ['1.14.14.18', '1.14.99.3'], 'kegg.reaction': 'R00311', 'metanetx.reaction': 'MNXR100681', 'bigg.reaction': 'HOXG', 'Human1': 'MAR01450'}\n",
      "{'ec-code': '1.3.1.24', 'kegg.reaction': 'R02393', 'metanetx.reaction': 'MNXR153102', 'bigg.reaction': 'BILIRED', 'Human1': 'MAR08634'}\n",
      "{'ec-code': '1.3.1.24', 'kegg.reaction': 'R02391', 'Human1': 'MAR04764'}\n",
      "{'ec-code': '3.1.4.4', 'kegg.reaction': 'R01310', 'metanetx.reaction': 'MNXR192014', 'bigg.reaction': 'PCHOLPm_hs', 'Human1': 'MAR00629'}\n",
      "{'ec-code': '2.7.1.30', 'kegg.reaction': 'R00847', 'metanetx.reaction': 'MNXR100348', 'bigg.reaction': 'GLYK', 'Human1': 'MAR00448'}\n",
      "{'metanetx.reaction': 'MNXR100308', 'bigg.reaction': 'GLYC3Ptm', 'Human1': 'MAR08891'}\n",
      "{'ec-code': '2.3.1.15', 'kegg.reaction': 'R00851', 'metanetx.reaction': 'MNXR138489', 'bigg.reaction': 'GPAMm_hs', 'Human1': 'MAR00579'}\n",
      "{'ec-code': '2.3.1.51', 'kegg.reaction': 'R02241', 'Human1': 'MAR00588'}\n",
      "{'ec-code': '2.7.7.41', 'kegg.reaction': 'R01799', 'metanetx.reaction': 'MNXR189947', 'bigg.reaction': 'CDSm', 'Human1': 'MAR00121'}\n",
      "{'ec-code': '2.7.8.5', 'kegg.reaction': 'R01801'}\n",
      "{'ec-code': '3.1.3.27', 'kegg.reaction': 'R02029'}\n",
      "{'ec-code': '2.7.8.41', 'kegg.reaction': 'R02030', 'Human1': 'MAR00586'}\n",
      "{'ec-code': '2.7.8.-', 'kegg.reaction': 'R07390'}\n",
      "{'ec-code': ['2.7.4.14', '2.7.4.25'], 'kegg.reaction': 'R00512', 'metanetx.reaction': 'MNXR190870', 'bigg.reaction': 'CYTK1m', 'Human1': 'MAR04024'}\n",
      "{'ec-code': '2.7.4.6', 'kegg.reaction': 'R00570', 'metanetx.reaction': 'MNXR188503', 'bigg.reaction': 'NDPK3m', 'Human1': 'MAR04028'}\n",
      "{'ec-code': '1.15.1.1', 'kegg.reaction': 'R00275', 'metanetx.reaction': 'MNXR191196', 'bigg.reaction': 'SPODMm', 'Human1': 'MAR03960'}\n",
      "{'ec-code': '1.11.1.9', 'kegg.reaction': 'R00274', 'bigg.reaction': 'GTHP', 'Human1': 'MAR04116'}\n",
      "{'ec-code': '1.11.1.9', 'kegg.reaction': 'R00274', 'metanetx.reaction': 'MNXR100446', 'bigg.reaction': 'GTHPm', 'Human1': 'MAR04116'}\n",
      "{'ec-code': '1.8.1.7', 'kegg.reaction': 'R00115', 'bigg.reaction': 'GTHO', 'Human1': 'MAR03870'}\n",
      "{'ec-code': ['1.6.4.2', '1.8.1.7', '1.8.4.2'], 'kegg.reaction': 'R00115', 'metanetx.reaction': 'MNXR100098', 'bigg.reaction': 'GTHOm', 'Human1': 'MAR03870'}\n",
      "{'metanetx.reaction': 'MNXR96753', 'bigg.reaction': 'CITtam', 'Human1': 'MAR04964'}\n",
      "{'bigg.reaction': 'r0913'}\n",
      "{'metanetx.reaction': 'MNXR96754', 'bigg.reaction': 'CITtbm', 'Human1': 'MAR04971'}\n",
      "{'bigg.reaction': 'r0917'}\n",
      "{'bigg.reaction': 'r0917'}\n",
      "{'metanetx.reaction': 'MNXR102872', 'bigg.reaction': 'PIt2m', 'Human1': 'MAR05043'}\n",
      "{'metanetx.reaction': 'MNXR96123', 'bigg.reaction': 'ATPtm', 'Human1': 'MAR06328', 'ec-code': '2.7.4.6'}\n",
      "{'metanetx.reaction': 'MNXR03', 'bigg.reaction': 'Htm', 'Human1': 'MAR07638'}\n",
      "{'metanetx.reaction': 'MNXR101374', 'bigg.reaction': 'MALtm', 'Human1': 'MAR04865'}\n",
      "{'metanetx.reaction': 'MNXR101348', 'bigg.reaction': 'MALSO3tm', 'Human1': 'MAR04868'}\n",
      "{'metanetx.reaction': 'MNXR101357', 'bigg.reaction': 'MALTSULtm', 'Human1': 'MAR06522'}\n",
      "{'metanetx.reaction': 'MNXR101349', 'bigg.reaction': 'MALSO4tm', 'Human1': 'MAR04867'}\n",
      "{'bigg.reaction': 'SUCCt2m'}\n",
      "{'metanetx.reaction': 'MNXR105396', 'bigg.reaction': 'r0830', 'Human1': 'MAR04864'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR105395', 'bigg.reaction': 'r0829', 'Human1': 'MAR04863'}\n",
      "{}\n",
      "{'bigg.reaction': 'COAtm'}\n",
      "{'bigg.reaction': 'COAtm'}\n",
      "{'metanetx.reaction': 'MNXR100300', 'bigg.reaction': 'GLUt2m', 'Human1': 'MAR05122'}\n",
      "{'metanetx.reaction': 'MNXR102226', 'bigg.reaction': 'ORNt4m', 'Human1': 'MAR06318'}\n",
      "{'metanetx.reaction': 'MNXR106278', 'bigg.reaction': 'r2398', 'Human1': 'MAR06305'}\n",
      "{'metanetx.reaction': 'MNXR106281', 'bigg.reaction': 'r2402', 'Human1': 'MAR06311'}\n",
      "{'metanetx.reaction': 'MNXR101267', 'bigg.reaction': 'LYStm', 'Human1': 'MAR06321'}\n",
      "{'metanetx.reaction': 'MNXR102225', 'bigg.reaction': 'ORNt3m', 'Human1': 'MAR05105'}\n",
      "{'metanetx.reaction': 'MNXR95954', 'bigg.reaction': 'ARGtm', 'Human1': 'MAR06323'}\n",
      "{'bigg.reaction': 'r1427'}\n",
      "{'metanetx.reaction': 'MNXR103385', 'bigg.reaction': 'PYRt2m', 'Human1': 'MAR04928'}\n",
      "{'metanetx.reaction': 'MNXR95207', 'bigg.reaction': 'ACACt2m', 'Human1': 'MAR04976'}\n",
      "{'metanetx.reaction': 'MNXR99501', 'bigg.reaction': 'FE2tm', 'Human1': 'MAR08758'}\n",
      "{'metanetx.reaction': 'MNXR96069', 'bigg.reaction': 'ASNtm', 'Human1': 'MAR05088'}\n",
      "{'metanetx.reaction': 'MNXR97029', 'bigg.reaction': 'r1437', 'Human1': 'MAR08684'}\n",
      "{'metanetx.reaction': 'MNXR100259', 'bigg.reaction': 'GLNtm', 'Human1': 'MAR05078'}\n",
      "{'metanetx.reaction': 'MNXR146139', 'bigg.reaction': 'PROtm', 'Human1': 'MAR05125'}\n",
      "{'metanetx.reaction': 'MNXR105002', 'bigg.reaction': 'r1078', 'Human1': 'MAR05074'}\n",
      "{'bigg.reaction': 'r1436'}\n",
      "{'bigg.reaction': 'r1455'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR100371', 'bigg.reaction': 'GLYtm', 'Human1': 'MAR05124'}\n",
      "{'bigg.reaction': 'ILEt5m'}\n",
      "{'metanetx.reaction': 'MNXR191139', 'bigg.reaction': 'LEUt5m', 'Human1': 'MAR05109'}\n",
      "{'metanetx.reaction': 'MNXR105190', 'bigg.reaction': 'VALt5m', 'Human1': 'MAR05089'}\n",
      "{'bigg.reaction': 'r1434'}\n",
      "{'metanetx.reaction': 'MNXR104354', 'bigg.reaction': 'r1435', 'Human1': 'MAR05079'}\n",
      "{'bigg.reaction': 'r1440'}\n",
      "{'metanetx.reaction': 'MNXR95682', 'bigg.reaction': 'BALAtmr', 'Human1': 'MAR04952'}\n",
      "{'metanetx.reaction': 'MNXR146728', 'bigg.reaction': 'UREAtm', 'Human1': 'MAR04951'}\n",
      "{'metanetx.reaction': 'MNXR99716', 'bigg.reaction': 'FUMtm', 'Human1': 'MAR03971'}\n",
      "{'metanetx.reaction': 'MNXR96249', 'bigg.reaction': 'BHBtm', 'Human1': 'MAR05018'}\n",
      "{'metanetx.reaction': 'MNXR103094', 'bigg.reaction': 'PPAtm', 'Human1': 'MAR01640'}\n",
      "{'metanetx.reaction': 'MNXR96353', 'bigg.reaction': 'BUTt2m', 'Human1': 'MAR00097'}\n",
      "{'bigg.reaction': 'FORt2m'}\n",
      "{'metanetx.reaction': 'MNXR146809', 'bigg.reaction': 'r0962'}\n",
      "{'metanetx.reaction': 'MNXR96693', 'bigg.reaction': 'CHLtm', 'Human1': 'MAR04954'}\n",
      "{'metanetx.reaction': 'MNXR96810', 'bigg.reaction': 'CO2tm', 'Human1': 'MAR04919'}\n",
      "{'metanetx.reaction': 'MNXR98641', 'bigg.reaction': 'H2Otm', 'Human1': 'MAR04885'}\n",
      "{'metanetx.reaction': 'MNXR102090', 'bigg.reaction': 'O2tm', 'Human1': 'MAR04896'}\n",
      "{'metanetx.reaction': 'MNXR100343', 'bigg.reaction': 'GLYCtm', 'Human1': 'MAR00444'}\n",
      "{'metanetx.reaction': 'MNXR96957', 'bigg.reaction': 'CYANtm', 'Human1': 'MAR06516'}\n",
      "{'metanetx.reaction': 'MNXR104749', 'bigg.reaction': 'TCYNTtm', 'Human1': 'MAR08848'}\n",
      "{'metanetx.reaction': 'MNXR96886', 'bigg.reaction': 'CREATtmdiffir', 'Human1': 'MAR04992'}\n",
      "{'metanetx.reaction': 'MNXR102417', 'bigg.reaction': 'PCREATtmdiffir', 'Human1': 'MAR08430'}\n",
      "{'metanetx.reaction': 'MNXR100484', 'bigg.reaction': 'r0941', 'Human1': 'MAR02225'}\n",
      "{'bigg.reaction': 'r0838'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR102401', 'bigg.reaction': 'PCFLOPm', 'Human1': 'MAR01583'}\n",
      "{'metanetx.reaction': 'MNXR103249', 'bigg.reaction': 'PSFLIPm', 'Human1': 'MAR01664'}\n",
      "{'metanetx.reaction': 'MNXR102479', 'bigg.reaction': 'PEFLIPm', 'Human1': 'MAR01596'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR102090', 'bigg.reaction': 'O2t', 'Human1': 'MAR04896'}\n",
      "{'metanetx.reaction': 'MNXR96810', 'bigg.reaction': 'CO2t', 'Human1': 'MAR04919'}\n",
      "{}\n",
      "{'bigg.reaction': 'GLCt1r'}\n",
      "{'bigg.reaction': 'HDCAtr'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR101277', 'bigg.reaction': 'L_LACt2r', 'Human1': 'MAR03007'}\n",
      "{'metanetx.reaction': 'MNXR96249', 'bigg.reaction': 'BHBt', 'Human1': 'MAR05018'}\n",
      "{'metanetx.reaction': 'MNXR95207', 'bigg.reaction': 'ACACt2', 'Human1': 'MAR04976'}\n",
      "{'metanetx.reaction': 'MNXR97980', 'bigg.reaction': 'ETOHt', 'Human1': 'MAR08757'}\n",
      "{'metanetx.reaction': 'MNXR96353', 'bigg.reaction': 'BUTt2r', 'Human1': 'MAR00097'}\n",
      "{'metanetx.reaction': 'MNXR100343', 'bigg.reaction': 'GLYCt', 'Human1': 'MAR00444'}\n",
      "{'metanetx.reaction': 'MNXR96886', 'bigg.reaction': 'r0942', 'Human1': 'MAR04992'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR100649', 'bigg.reaction': 'HIStiDF', 'Human1': 'MAR05070'}\n",
      "{'metanetx.reaction': 'MNXR100824', 'bigg.reaction': 'ILEtec', 'Human1': 'MAR05092'}\n",
      "{'metanetx.reaction': 'MNXR191139', 'bigg.reaction': 'LEUtec', 'Human1': 'MAR05085'}\n",
      "{'bigg.reaction': 'LYStiDF'}\n",
      "{'metanetx.reaction': 'MNXR101493', 'bigg.reaction': 'METtec', 'Human1': 'MAR05073'}\n",
      "{'metanetx.reaction': 'MNXR102637', 'bigg.reaction': 'PHEtec', 'Human1': 'MAR05082'}\n",
      "{'metanetx.reaction': 'MNXR104852', 'bigg.reaction': 'r2534', 'Human1': 'MAR05091'}\n",
      "{'metanetx.reaction': 'MNXR146605', 'bigg.reaction': 'TRPt', 'Human1': 'MAR05080'}\n",
      "{'metanetx.reaction': 'MNXR105190', 'bigg.reaction': 'VALtec', 'Human1': 'MAR05089'}\n",
      "{'bigg.reaction': 'ARGtiDF'}\n",
      "{'metanetx.reaction': 'MNXR96107', 'bigg.reaction': 'ASPte'}\n",
      "{'metanetx.reaction': 'MNXR97029', 'bigg.reaction': 'CYStec', 'Human1': 'MAR08684'}\n",
      "{}\n",
      "{'bigg.reaction': 'r2525'}\n",
      "{'metanetx.reaction': 'MNXR100368', 'bigg.reaction': 'GLYt2r', 'Human1': 'MAR05076'}\n",
      "{'metanetx.reaction': 'MNXR146138', 'bigg.reaction': 'PROt2r', 'Human1': 'MAR05087'}\n",
      "{'metanetx.reaction': 'MNXR104354', 'bigg.reaction': 'r2526', 'Human1': 'MAR05079'}\n",
      "{'metanetx.reaction': 'MNXR105002', 'bigg.reaction': 'TYRt', 'Human1': 'MAR05074'}\n",
      "{'metanetx.reaction': 'MNXR96069', 'bigg.reaction': 'r2532', 'Human1': 'MAR05088'}\n",
      "{'metanetx.reaction': 'MNXR151244', 'bigg.reaction': 'ALAt2r', 'Human1': 'MAR05077'}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR101951', 'bigg.reaction': 'NH4t3r', 'Human1': 'MAR04873'}\n",
      "{'metanetx.reaction': 'MNXR95429', 'bigg.reaction': 'ACt2r', 'Human1': 'MAR04935'}\n",
      "{'bigg.reaction': 'PPAt'}\n",
      "{'metanetx.reaction': 'MNXR189552', 'bigg.reaction': '2HBt2', 'Human1': 'MAR08075'}\n",
      "{'metanetx.reaction': 'MNXR96693', 'bigg.reaction': 'CHOLtu', 'Human1': 'MAR04954'}\n",
      "{'bigg.reaction': 'r1088'}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR146728', 'bigg.reaction': 'UREAt', 'Human1': 'MAR04949'}\n",
      "{'metanetx.reaction': 'MNXR96262', 'bigg.reaction': 'r1512', 'Human1': 'MAR05440'}\n",
      "{}\n",
      "{'bigg.reaction': 'MAL_Lte'}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR95682', 'bigg.reaction': 'r0899', 'Human1': 'MAR04952'}\n",
      "{'metanetx.reaction': 'MNXR99505', 'bigg.reaction': 'FE2t', 'Human1': 'MAR05987'}\n",
      "{'metanetx.reaction': 'MNXR98641', 'bigg.reaction': 'H2Ot', 'Human1': 'MAR04885'}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR100494', 'bigg.reaction': 'r0940'}\n",
      "{'metanetx.reaction': 'MNXR96957', 'bigg.reaction': 'CYANt', 'Human1': 'MAR06516'}\n",
      "{'metanetx.reaction': 'MNXR104749', 'bigg.reaction': 'TCYNTt', 'Human1': 'MAR08848'}\n",
      "{'bigg.reaction': 'r1423'}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{}\n",
      "{'metanetx.reaction': 'MNXR153251', 'bigg.reaction': 'COt', 'Human1': 'MAR08261'}\n",
      "{'metanetx.reaction': 'MNXR191149', 'bigg.reaction': 'NOt', 'Human1': 'MAR05009'}\n",
      "{'metanetx.reaction': 'MNXR102406', 'bigg.reaction': 'PCHOLHSTDe', 'Human1': 'MAR07158'}\n",
      "{'metanetx.reaction': 'MNXR192024', 'bigg.reaction': 'PSt3', 'Human1': 'MAR01669'}\n",
      "{'metanetx.reaction': 'MNXR102505', 'bigg.reaction': 'PEt', 'Human1': 'MAR08528'}\n"
     ]
    }
   ],
   "source": [
    "visualization_model_annotations(mitocore2, 'reactions')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "id": "71f472a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def get_model_ids_for_databases(model, component_type, dict, databases):\n",
    "    \"\"\"\n",
    "    Extract ids from model annotations for various databases across a specified component type.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - dict: dictionary where database ids are located ('annotations', 'notes')\n",
    "    - databases: list of str, e.g., ['kegg', 'bigg', 'metanetx', 'uniprot'].\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': []}\n",
    "\n",
    "    counters= {db:0 for db in databases}\n",
    "\n",
    "    for db in databases: \n",
    "        ids_dict[db] = []\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "\n",
    "        for db in databases:\n",
    "            value = None\n",
    "            annotation = getattr(comp, dict, {})\n",
    "\n",
    "            # Use several naming conventions\n",
    "            for key in [db, \n",
    "                        f\"{db}.reaction\", \n",
    "                        f\"{db}.compound\", \n",
    "                        f\"{db}.metabolite\", \n",
    "                        f\"{db}.chemical\", \n",
    "                        f\"{db}.genes\",\n",
    "                        f\"{db}.gene\",\n",
    "                        f\"{db} id\",\n",
    "                        f\"{db.upper()} ID\",\n",
    "                        f\"{db.lower()} ID\"\n",
    "                        ]:\n",
    "                if key in annotation and annotation[key] not in [None, '', 'N/A']:\n",
    "                    counters[db] += 1\n",
    "                    value = annotation[key]\n",
    "                    break\n",
    "\n",
    "            ids_dict[db].append(value)\n",
    "            pd.DataFrame(ids_dict)\n",
    "    \n",
    "    return counters\n",
    "\n",
    "def get_model_ids_for_attr(model, component_type, attr_name):\n",
    "    \"\"\"\n",
    "    Extract attribute in the model in the target core-component.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: Core component of GEMs, str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - attr_name: name of the dictionary where target object is located ('notes', '_gpr')\n",
    "\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component id and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': [], attr_name:[]}\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "        value = comp.__dict__.get(attr_name, None)\n",
    "        ids_dict[attr_name].append(value)\n",
    "\n",
    "    return pd.DataFrame(ids_dict)\n",
    "\n",
    "def get_model_ids_for_ec_code(model, component_type, dict_name, databases):\n",
    "    \"\"\"\n",
    "    Extract ids from model annotations for various databases across a specified component type.\n",
    "        \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - dict_name: dictionary where database ids are located ('annotations', 'notes')\n",
    "    - databases: list of str, e.g., ['kegg', 'ec-code'].\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': []}\n",
    "    counters = {db: 0 for db in databases}\n",
    "\n",
    "    for db in databases:\n",
    "        ids_dict[db] = []\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "\n",
    "        annotation = getattr(comp, dict_name, {})\n",
    "\n",
    "        for db in databases:\n",
    "            value = None\n",
    "            #search for different nomenclatures\n",
    "            keys_to_try = [\n",
    "                db,\n",
    "                f\"{db}.reaction\",\n",
    "                f\"{db}.compound\",\n",
    "                f\"{db}.metabolite\",\n",
    "                f\"{db}.chemical\",\n",
    "                f\"{db}.genes\",\n",
    "                f\"{db}.gene\",\n",
    "                f\"{db} id\",\n",
    "                f\"{db.upper()} ID\",\n",
    "                f\"{db.lower()} ID\"\n",
    "            ]\n",
    "\n",
    "            for key in keys_to_try:\n",
    "                if key in annotation:\n",
    "                    if annotation[key] not in [None, \"\", \"N/A\"]:\n",
    "                        counters[db] += 1\n",
    "                        value = annotation[key]\n",
    "                    break\n",
    "\n",
    "            ids_dict[db].append(value)\n",
    "\n",
    "    # Build DataFrame **after loop**\n",
    "    df = pd.DataFrame(ids_dict)\n",
    "\n",
    "    return df\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "39b9a0e2",
   "metadata": {},
   "source": [
    "# Metabolites"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "4ae9a212",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'KEGG': 436, 'RECON2': 436, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 1st curation {'kegg': 436, 'bigg': 325, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'kegg': 436, 'bigg': 325, 'metanetx': 436, 'Human1': 421}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'metabolites', 'notes', ['KEGG', 'RECON2', 'metanetx', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'metabolites', '_annotation', ['kegg','bigg', 'metanetx','Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'metabolites', '_annotation', ['kegg','bigg', 'metanetx','Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "92809075",
   "metadata": {},
   "source": [
    "# Reactions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "154cb325",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'KEGG': 328, 'Recon2': 432, 'EC Number': 328, 'Human1': 0}\n",
      "Mitocore 1st curation {'kegg': 324, 'bigg': 0, 'ec-code': 322, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'kegg': 328, 'bigg': 432, 'ec-code': 323, 'metanetx': 335, 'Human1': 395}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'reactions', 'notes', ['KEGG', 'Recon2','EC Number', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'reactions', '_annotation', ['kegg', 'bigg', 'ec-code', 'metanetx', 'Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'reactions', '_annotation', ['kegg', 'bigg', 'ec-code', 'metanetx', 'Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "70aae5d3",
   "metadata": {},
   "source": [
    "# Genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "8dc8ff94",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'Uniprot': 0, 'kegg': 0, 'Human1': 0}\n",
      "Mitocore 1st curation {'uniprot': 387, 'kegg': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'uniprot': 595, 'kegg': 587, 'Human1': 595}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'genes', 'notes', ['Uniprot', 'kegg', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'genes', '_annotation', ['uniprot','kegg', 'Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'genes', '_annotation', ['uniprot', 'kegg', 'Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "52fd4bad",
   "metadata": {},
   "source": [
    "# --- ec-codes -------"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "43464c7e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mitocore original: 339 ['2.7.1.1 or 2.7.1.2', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '2.7.2.3', '5.4.2.1', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.49', '3.1.1.31', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.4.1, 2.3.1.12, 1.8.1.4', '2.3.3.1', '4.2.1.3', '1.1.1.41', '1.1.1.42', '1.2.4.2, 1.8.1.4, 2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.37', '1.6.5.3, 1.6.5.8, 1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14, 3.6.3.10, 3.6.3.6', '4.1.1.32', '6.4.1.1', '1.1.1.40', '1.1.1.38', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.3', '1.1.1.40', '2.6.1.2', '2.7.4.6', '4.2.1.2', '2.7.4.3', '1.1.1.42', '4.2.1.3', '2.3.3.8', '2.6.1.1', '1.1.1.37', 'N/A', 'N/A', '2.6.1.1', '1.1.1.8', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.6.1.1', 'N/A', '2.3.1.21', '3.6.1.1', '3.1.2.2', '1.3.8.8, 1.5.5.1', '1.3.1.38', '4.2.1.74, 1.1.1.211, 2.3.1.16', '1.3.8.8, 1.5.5.1', '1.3.1.38', '4.2.1.74, 1.1.1.211, 2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.1, 1.5.5.1', '4.2.1.17', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.39', '2.3.1.38', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.2', '1.1.1.30', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.1', '4.2.1.17', '1.1.1.35', '2.3.1.16', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.1', '4.2.1.17', '3.2.1.4', '1.1.1.31 or 1.1.1.35', '6.2.1.1', '1.2.1.27', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '3.1.2.2', '4.3.1.19', 'N/A', '1.2.4.4 + 2.3.1.168 + 1.8.1.4', '1.1.1.27', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.20', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', 'Non-Enzymatic', '2.6.1.1', '2.6.1.1', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.4.1.1', '1.13.11.11 or 1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.5.1.-', 'n/a', 'n/a', '1.2.4.2 and 2.3.1.61 and 1.8.1.4', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', 'Non-Enzymatic', '1.5.1.1 or 1.5.1.21', '1.5.1.1 or 1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', 'N/A', 'N/A', '1.5.5.2', 'Non-Enzymatic', '1.2.1.88', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '2.6.1.13', '4.1.1.17', '1.4.3.22', '1.2.1.3', '1.2.1.3', '4.1.1.19', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', 'N/A', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.3', '1.4.1.3', '1.4.1.3', '6.3.1.2', '3.5.1.2', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.3', '1.5.1.5', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17 or 4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.3', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', 'Non-enzymatic', '1.4.3.4', '4.4.1.5', '3.1.2.6', '1.1.1.28', '6.3.4.16', '2.1.3.3', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.2', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1 or 2.6.1.5', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '2.6.1.19', '1.2.1.18', '1.2.1.18', '4.1.1.11', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', 'N/A', '1.2.1.3', '1.2.1.3', '6.2.1.1', '6.2.1.1', '4.3.2.2', '6.3.4.4', '3.5.4.6', '3.5.3.1', '4.3.2.1', '6.3.4.5', '3.5.3.1', '2.3.1.37', 'N/A', '4.2.1.24', '2.5.1.61', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', 'N/A', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', 'N/A', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.8.1.7']\n",
      "mitocore 1 curation: 358 ['2.7.1.1', '2.7.1.2', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '2.7.2.3', '5.4.2.1', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.49', '3.1.1.31', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.4.1', '2.3.1.12', '1.8.1.4', '2.3.3.1', '4.2.1.3', '1.1.1.41', '1.1.1.42', '1.2.4.2', '1.8.1.4', '2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.37', '1.6.5.3', '1.6.5.8', '1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14', '3.6.3.10', '3.6.3.6', '4.1.1.32', '6.4.1.1', '1.1.1.40', '1.1.1.38', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.3', '1.1.1.40', '2.6.1.2', '2.7.4.6', '4.2.1.2', '2.7.4.3', '1.1.1.42', '4.2.1.3', '2.3.3.8', '2.6.1.1', '1.1.1.37', '2.6.1.1', '1.1.1.8', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.6.1.1', '2.3.1.21', '3.6.1.1', '3.1.2.2', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.1', '1.5.5.1', '4.2.1.17', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.39', '2.3.1.38', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.2', '1.1.1.30', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.1', '4.2.1.17', '1.1.1.35', '2.3.1.16', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.1', '4.2.1.17', '3.2.1.4', '1.1.1.31', '1.1.1.35', '6.2.1.1', '1.2.1.27', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '3.1.2.2', '4.3.1.19', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.1.1.27', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.20', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', '2.6.1.1', '2.6.1.1', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.4.1.1', '1.13.11.11', '1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.5.1.-', '1.2.4.2', '2.3.1.61', '1.8.1.4', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', '1.5.1.1', '1.5.1.21', '1.5.1.1', '1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', '1.5.5.2', '1.2.1.88', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '2.6.1.13', '4.1.1.17', '1.4.3.22', '1.2.1.3', '1.2.1.3', '4.1.1.19', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.3', '1.4.1.3', '1.4.1.3', '6.3.1.2', '3.5.1.2', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.3', '1.5.1.5', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17', '4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.3', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', '1.4.3.4', '4.4.1.5', '3.1.2.6', '1.1.1.28', '6.3.4.16', '2.1.3.3', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.2', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1', '2.6.1.5', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '2.6.1.19', '1.2.1.18', '1.2.1.18', '4.1.1.11', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', '1.2.1.3', '1.2.1.3', '6.2.1.1', '6.2.1.1', '4.3.2.2', '6.3.4.4', '3.5.4.6', '3.5.3.1', '4.3.2.1', '6.3.4.5', '3.5.3.1', '2.3.1.37', '4.2.1.24', '2.5.1.61', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.8.1.7']\n",
      "mitocore curation 2: 582 ['2.7.1.1', '2.7.1.2', '2.7.1.4', '2.7.1.55', '2.7.1.59', '2.7.1.60', '3.1.3.1', '3.1.3.104', '3.1.3.2', '3.1.3.23', '3.1.3.25', '3.1.3.29', '3.1.3.58', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '1.2.1.59', '2.7.2.10', '2.7.2.3', '5.4.2.1', '5.4.2.4', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.28', '1.1.1.363', '1.1.1.47', '1.1.1.49', '3.1.1.31', '1.1.1.351', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.1.104', '1.2.1.51', '1.2.4.1', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.12', '2.3.1.168', '2.3.1.61', '2.3.3.1', '2.3.3.14', '2.3.3.16', '2.3.3.3', '4.1.3.7', '4.2.1.3', '1.1.1.286', '1.1.1.41', '1.1.1.42', '1.1.1.87', '1.1.1.42', '1.2.1.105', '1.2.1.52', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82', '1.6.5.3', '1.6.5.8', '1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14', '7.1.2.2', '4.1.1.32', '4.1.1.49', '6.4.1.1', '1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42', '1.1.1.38', '1.1.1.39', '1.1.1.40', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8', '1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42', '2.6.1.1', '2.6.1.2', '2.6.1.5', '2.7.4.6', '4.2.1.2', '2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8', '1.1.1.42', '4.2.1.3', '2.3.3.8', '4.1.3.6', '2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57', '1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82', '2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57', '1.1.1.8', '1.1.1.94', '1.1.5.3', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1', '2.3.1.21', '3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1', '3.1.2.2', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.10', '1.3.1.104', '1.3.1.38', '1.3.1.8', '4.2.1.17', '4.2.1.74', '1.1.1.211', '1.1.1.35', '2.3.1.16', '2.3.1.9', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.150', '4.2.1.17', '4.2.1.74', '1.1.1.211', '1.1.1.35', '2.3.1.16', '2.3.1.9', '1.3.8.1', '1.5.5.1', '4.2.1.116', '4.2.1.150', '4.2.1.17', '4.2.1.55', '1.1.1.178', '1.1.1.211', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.0', '2.3.1.39', '2.3.1.85', '2.3.1.86', '2.3.1.179', '2.3.1.180', '2.3.1.38', '2.3.1.41', '2.3.1.85', '2.3.1.86', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.1', '6.2.1.17', '6.2.1.2', '6.2.1.3', '1.1.1.30', '2.8.3.18', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '2.6.1.6', '2.6.1.67', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '1.3.8.1', '4.2.1.150', '4.2.1.17', '1.1.1.178', '1.1.1.35', '2.3.1.16', '2.3.1.9', '2.6.1.42', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.7.11.4', '1.3.8.1', '4.2.1.150', '4.2.1.17', '3.1.2.4', '1.1.1.31', '6.2.1.1', '6.2.1.17', '1.2.1.27', '6.4.1.2', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '4.1.1.9', '4.1.1.94', '7.2.4.3', '2.3.1.226', '2.3.1.30', '2.3.3.13', '3.1.2.17', '3.1.2.18', '3.1.2.2', '3.1.2.20', '3.1.2.23', '4.3.1.17', '4.3.1.19', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.1.1.27', '1.1.1.272', '1.1.1.28', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.11', '2.7.1.20', '2.7.1.74', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', '2.6.1.1', '2.6.1.3', '2.6.1.1', '2.6.1.3', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.1.99.1', '4.4.1.1', '4.4.1.11', '4.4.1.13', '4.4.1.28', '4.4.1.3', '4.4.1.8', '1.13.11.11', '1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.2.1.105', '1.2.1.52', '1.2.4.2', '1.8.1.4', '2.3.1.61', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', '1.4.3.2', '1.5.1.1', '1.5.1.21', '1.5.1.49', '1.5.1.1', '1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', '1.5.99.8', '1.2.1.24', '1.2.1.5', '1.2.1.88', '1.5.1.12', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '1.5.99.8', '2.6.1.11', '2.6.1.13', '4.1.1.17', '4.1.1.19', '4.1.1.86', '1.4.3.10', '1.4.3.21', '1.4.3.22', '1.4.3.6', '1.2.1.3', '1.2.1.19', '1.2.1.3', '4.1.1.19', '3.5.3.1', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.13', '1.4.1.3', '1.4.1.4', '1.4.1.14', '1.4.1.2', '1.4.1.3', '1.4.1.4', '1.4.1.13', '1.4.1.3', '1.4.1.4', '6.3.1.2', '1.4.1.13', '1.4.1.14', '1.4.7.1', '2.4.2.14', '3.5.1.2', '3.5.1.38', '4.3.3.6', '6.3.4.2', '6.3.5.2', '6.3.5.4', '6.3.5.5', '6.3.5.7', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.2', '3.1.3.29', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.33', '1.5.1.3', '1.5.1.5', '1.5.1.15', '1.5.1.5', '1.5.1.15', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17', '4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.33', '1.5.1.3', '1.5.1.15', '1.5.1.5', '1.5.1.15', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', '2.3.1.37', '1.1.1.381', '1.4.3.21', '1.4.3.4', '1.4.3.6', '4.4.1.5', '3.1.2.12', '3.1.2.6', '1.1.1.27', '1.1.1.28', '6.3.4.16', '6.3.5.5', '2.1.3.3', '2.1.3.6', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.1', '3.4.11.2', '3.4.11.23', '3.4.13.18', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1', '2.6.1.27', '2.6.1.42', '2.6.1.5', '2.6.1.57', '2.6.1.9', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '4.3.1.4', '2.6.1.19', '2.6.1.55', '1.2.1.18', '1.2.1.27', '1.2.1.18', '4.1.1.11', '4.1.1.15', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', '1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88', '1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88', '6.2.1.1', '6.2.1.17', '6.2.1.1', '6.2.1.17', '4.3.2.2', '6.3.4.4', '3.5.4.17', '3.5.4.6', '3.5.3.1', '3.5.3.7', '4.3.2.1', '6.3.4.5', '3.5.3.1', '3.5.3.7', '2.3.1.37', '4.2.1.24', '2.5.1.61', '4.3.1.8', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', '1.14.14.18', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.25', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.6.4.2', '1.8.1.7', '1.8.4.2', '2.7.4.6']\n"
     ]
    }
   ],
   "source": [
    "curation_0_ec = []\n",
    "\n",
    "for reaction in mitocore0.reactions:\n",
    "    annotations = getattr(reaction, 'notes', {})\n",
    "    ec_data = annotations.get('EC Number', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "\n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_0_ec.append(ec_code)\n",
    "\n",
    "print('mitocore original:', len(curation_0_ec), curation_0_ec)\n",
    "\n",
    "curation_1_ec = []\n",
    "\n",
    "for reaction in mitocore1.reactions:\n",
    "    annotations = getattr(reaction, '_annotation', {})\n",
    "    ec_data = annotations.get('ec-code', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "\n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_1_ec.append(ec_code)\n",
    "\n",
    "print('mitocore 1 curation:', len(curation_1_ec), curation_1_ec)\n",
    "\n",
    "\n",
    "curation_2_ec=[]\n",
    "\n",
    "for reaction in mitocore2.reactions:\n",
    "    annotations = getattr(reaction, '_annotation', {})\n",
    "    ec_data = annotations.get('ec-code', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "        \n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_2_ec.append(ec_code)\n",
    "\n",
    "print('mitocore curation 2:', len(curation_2_ec), curation_2_ec)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "83b70828",
   "metadata": {},
   "source": [
    "# ---- GPR rules ----\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2e01d1af",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_model_ids_for_attr(model, component_type, attr_name):\n",
    "    \"\"\"\n",
    "    Extract attribute in the model in the target core-component.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: Core component of GEMs, str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - attr_name: name of the dictionary where target object is located ('notes', '_gpr')\n",
    "\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': [], attr_name:[]}\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "        value = comp.__dict__.get(attr_name, None)\n",
    "        ids_dict[attr_name].append(value)\n",
    "\n",
    "    return pd.DataFrame(ids_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "b440ba4d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GPR Mitocore 0:     model_id                                 _gpr\n",
      "150    r0638                      ENSG00000116353\n",
      "151    r0660                      ENSG00000127884\n",
      "152    r0722                      ENSG00000138796\n",
      "153    r0724                      ENSG00000167315\n",
      "154    r1451  ENSG00000117054 and ENSG00000171503\n",
      "GPR Mitocore 1:     model_id                                 _gpr\n",
      "150    r0638                      ENSG00000116353\n",
      "151    r0660                      ENSG00000127884\n",
      "152    r0722                      ENSG00000138796\n",
      "153    r0724                      ENSG00000167315\n",
      "154    r1451  ENSG00000117054 and ENSG00000171503\n",
      "GPR Mitocore 2:     model_id                                               _gpr\n",
      "150    r0638                                    ENSG00000116353\n",
      "151    r0660  ENSG00000127884 or (ENSG00000084754 and ENSG00...\n",
      "152    r0722  ENSG00000138796 or (ENSG00000060971 and ENSG00...\n",
      "153    r0724  ENSG00000167315 or (ENSG00000060971 and ENSG00...\n",
      "154    r1451  (ENSG00000117054 and ENSG00000171503) or ENSG0...\n"
     ]
    }
   ],
   "source": [
    "gpr_mit_0=get_model_ids_for_attr(mitocore0, 'reactions', '_gpr')\n",
    "gpr_mit_1=get_model_ids_for_attr(mitocore1, 'reactions', '_gpr')\n",
    "gpr_mit_2= get_model_ids_for_attr(mitocore2, 'reactions', '_gpr')\n",
    "\n",
    "print('GPR Mitocore 0:', gpr_mit_0[150:155])\n",
    "print('GPR Mitocore 1:', gpr_mit_1[150:155])\n",
    "print('GPR Mitocore 2:', gpr_mit_2[150:155])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4157a6c7",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Split GPR on 'or' \n",
    "def gpr_counte(df, model):\n",
    "    \"\"\" function to count the absolute number of GPR \n",
    "    splits GPR based on 'or'\n",
    "\n",
    "    -df: DataFrame with GPRs and model ids \n",
    "    -model: str SBML model\n",
    "    \n",
    "    retunr: GPR count \n",
    "    \"\"\"\n",
    "    gpr_absolute= []\n",
    "\n",
    "    for index, row in df.iterrows():\n",
    "        gpr = row['_gpr']\n",
    "        elements_in_gpr = str(gpr).split(' or ')\n",
    "\n",
    "        for element in elements_in_gpr:\n",
    "            if element not in [None, 'N/A','']:\n",
    "                gpr_absolute.append(element)\n",
    "    print(gpr_absolute)\n",
    "    return print(f\" {model}, has {len(gpr_absolute)} GPRs\" )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "8ffbded1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000102144', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000067225', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000160211', 'ENSG00000130313', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000163931', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000166411 and ENSG00000101365 and ENSG00000067829', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000163541 and ENSG00000172340', 'ENSG00000136143', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', 'ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295', 'ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581', 'ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and (ENSG00000159199', 'ENSG00000135390', 'ENSG00000154518) and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253', 'ENSG00000100889', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000082212', 'ENSG00000166123', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000065833', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000014641', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000205560', 'ENSG00000180817', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000138777', 'ENSG00000119673', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000122971 and ENSG00000171503', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000075239', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000161267', 'ENSG00000083720', 'ENSG00000134240', 'ENSG00000117305', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000128928', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000111058', 'ENSG00000119711', 'ENSG00000175198 and ENSG00000114054', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'Unknown', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000168906', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000120053', 'Non-Enzymatic', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000111716', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'Unknown', 'Unknown', 'Unknown', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'Unknown', 'Non-Enzymatic', 'Unknown', 'Unknown', 'ENSG00000179761', 'Unknown', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'Non-Enzymatic', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000183010', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000069535', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'Unknown', 'ENSG00000136750', 'Unknown', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000148672', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000228716', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000178445', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000178700', 'ENSG00000163738', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000120254', 'ENSG00000136010', 'ENSG00000100116', 'Non-enzymatic', 'ENSG00000069535', 'ENSG00000124767', 'ENSG00000063854', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000007171', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000166825', 'ENSG00000001084', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000172497', 'ENSG00000117448', 'ENSG00000248144 and ENSG00000197894', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000154930', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000023330', 'Unknown', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'Unknown', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000105223', 'ENSG00000198814', 'ENSG00000186281', 'ENSG00000155189', 'ENSG00000144559', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000103202', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000176153', 'ENSG00000233276', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000147454', 'ENSG00000155287', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)']\n",
      " Mitocore0, has 489 GPRs\n",
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000102144', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000067225', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000160211', 'ENSG00000130313', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000163931', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000166411 and ENSG00000101365 and ENSG00000067829', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000163541 and ENSG00000172340', 'ENSG00000136143', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', 'ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295', 'ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000159199)', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000135390)', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000154518)', 'ENSG00000100889', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000082212', 'ENSG00000166123', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000065833', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000014641', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000205560', 'ENSG00000180817', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000138777', 'ENSG00000119673', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000122971 and ENSG00000171503', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000075239', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000161267', 'ENSG00000083720', 'ENSG00000134240', 'ENSG00000117305', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000128928', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000111058', 'ENSG00000119711', 'ENSG00000175198 and ENSG00000114054', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000168906', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000120053', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000111716', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'ENSG00000179761', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000183010', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000069535', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000136750', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000148672', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000228716', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000178445', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000178700', 'ENSG00000163738', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000120254', 'ENSG00000136010', 'ENSG00000100116', 'ENSG00000069535', 'ENSG00000124767', 'ENSG00000063854', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000007171', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000166825', 'ENSG00000001084', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000172497', 'ENSG00000117448', 'ENSG00000248144 and ENSG00000197894', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000154930', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000023330', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000105223', 'ENSG00000198814', 'ENSG00000186281', 'ENSG00000155189', 'ENSG00000144559', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000103202', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000176153', 'ENSG00000233276', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000147454', 'ENSG00000155287', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)']\n",
      " Mitocore1, has 479 GPRs\n",
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000156510', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000067057', 'ENSG00000141959', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000136872', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000105679', 'ENSG00000102144', 'ENSG00000096006', 'ENSG00000170950', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000172331', 'ENSG00000226784', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000143627', 'ENSG00000067225', 'ENSG00000143627', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000134333', 'ENSG00000160211', 'ENSG00000049239', 'ENSG00000130313', 'ENSG00000130313', 'ENSG00000049239', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000197713', 'ENSG00000163931', 'ENSG00000007350', 'ENSG00000151005', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000007350', 'ENSG00000151005', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000067829 and ENSG00000101365 and ENSG00000166411', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', '(ENSG00000163541 and ENSG00000172340)', '(ENSG00000136143 and ENSG00000163541)', 'ENSG00000136143', '(ENSG00000163541 and ENSG00000172340)', '(ENSG00000136143 and ENSG00000163541)', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000014641', 'ENSG00000138400', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', '(ENSG00000010256 and ENSG00000011295 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000169021 and ENSG00000172115 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727)', '(ENSG00000010256 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000164405 and ENSG00000169021 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154723 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000156411 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000173915 and ENSG00000180389 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', 'ENSG00000100889', 'ENSG00000124253', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000013392', 'ENSG00000065833', 'ENSG00000082212', 'ENSG00000151376', 'ENSG00000166123', 'ENSG00000167701', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000103024', 'ENSG00000143156', 'ENSG00000172113', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000085231', 'ENSG00000106992', 'ENSG00000140057', 'ENSG00000154027', 'ENSG00000155085', 'ENSG00000162433', 'ENSG00000165695', 'ENSG00000273841', 'ENSG00000065833', 'ENSG00000013392', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000103202', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000004455', 'ENSG00000085231', 'ENSG00000140057', 'ENSG00000147853', 'ENSG00000154027', 'ENSG00000155085', 'ENSG00000162433', 'ENSG00000165695', 'ENSG00000273841', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000014641', 'ENSG00000138400', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000140284', 'ENSG00000205560', 'ENSG00000110090', 'ENSG00000169169', 'ENSG00000180817', 'ENSG00000107902', 'ENSG00000138777', 'ENSG00000143363', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000110090', 'ENSG00000169169', 'ENSG00000205560', 'ENSG00000138777', 'ENSG00000107902', 'ENSG00000143363', 'ENSG00000180817', 'ENSG00000119673', 'ENSG00000097021', 'ENSG00000131238', 'ENSG00000136881', 'ENSG00000177465', 'ENSG00000184227', 'ENSG00000205669', 'ENSG00000221988', 'ENSG00000241404', '(ENSG00000115361 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', '(ENSG00000115361 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000087008', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000161533', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000196177', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000122971 and ENSG00000171503)', 'ENSG00000117054', 'ENSG00000122971', 'ENSG00000151498', 'ENSG00000177646', 'ENSG00000196177', 'ENSG00000240303', 'ENSG00000127884', 'ENSG00000138796', '(ENSG00000072506 and ENSG00000084754 and ENSG00000138796)', 'ENSG00000075239', 'ENSG00000120437', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000100294', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000115425', 'ENSG00000116353', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000152463', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000066813', 'ENSG00000166743', 'ENSG00000183549', 'ENSG00000183747', 'ENSG00000215009', 'ENSG00000161267', 'ENSG00000164039', 'ENSG00000083720', 'ENSG00000198754', 'ENSG00000134240', 'ENSG00000112972', 'ENSG00000117305', 'ENSG00000146151', 'ENSG00000105552', 'ENSG00000060982', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', '(ENSG00000083123 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000128928', 'ENSG00000196177', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', '(ENSG00000083123 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000122971', 'ENSG00000196177', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000072506', 'ENSG00000167315', '(ENSG00000084754 and ENSG00000138029 and ENSG00000167315)', 'ENSG00000105552', 'ENSG00000060982', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000137992', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000072506', 'ENSG00000111058', 'ENSG00000131069', 'ENSG00000154930', 'ENSG00000119711', 'ENSG00000114054 and ENSG00000175198', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000103150', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000166796', 'ENSG00000166800', 'ENSG00000171989', 'ENSG00000168906', 'ENSG00000038274', 'ENSG00000151224', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000156136', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000116761', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000139133', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000125166', 'ENSG00000120053', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000111716', 'ENSG00000166796', 'ENSG00000166800', 'ENSG00000171989', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000188676', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'ENSG00000183032', 'ENSG00000183032', '(ENSG00000091140 and ENSG00000105953 and ENSG00000119689)', '(ENSG00000105953 and ENSG00000119689)', '(ENSG00000119689 and ENSG00000181192)', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000143653', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'ENSG00000179761', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'ENSG00000250799', 'ENSG00000159423', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000104524', 'ENSG00000143811', 'ENSG00000183010', 'ENSG00000104524', 'ENSG00000143811', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000131471', 'ENSG00000131480', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000072210', 'ENSG00000111275', 'ENSG00000137124', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000072210', 'ENSG00000111275', 'ENSG00000137124', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000130066', 'ENSG00000069535', 'ENSG00000002726', 'ENSG00000131471', 'ENSG00000131480', 'ENSG00000189221', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000111275', 'ENSG00000136750', 'ENSG00000128683', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000134882', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000111237', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000136010', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000176974', 'ENSG00000178445', '(ENSG00000140905 and ENSG00000145020)', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000163738', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000100714', 'ENSG00000120254', 'ENSG00000100714', 'ENSG00000136010', 'ENSG00000144908', 'ENSG00000100116', 'ENSG00000069535', 'ENSG00000131471', 'ENSG00000124767', 'ENSG00000167699', 'ENSG00000063854', 'ENSG00000103253', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000089250', 'ENSG00000164867', 'ENSG00000007171', 'ENSG00000089250', 'ENSG00000164867', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000099998', 'ENSG00000100031', 'ENSG00000100121', 'ENSG00000274252', 'ENSG00000166825', 'ENSG00000002549', 'ENSG00000141279', 'ENSG00000001084', '(ENSG00000001084 and ENSG00000023909)', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000198650', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000162174', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000128683', 'ENSG00000136750', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000223572', 'ENSG00000237289', 'ENSG00000131730', 'ENSG00000104879', 'ENSG00000166165', 'ENSG00000223572', 'ENSG00000237289', 'ENSG00000172497', 'ENSG00000117448', '(ENSG00000197894 and ENSG00000248144)', 'ENSG00000147576', 'ENSG00000172955', 'ENSG00000180011', 'ENSG00000187758', 'ENSG00000196344', 'ENSG00000196616', 'ENSG00000197894', 'ENSG00000198099', 'ENSG00000248144', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000006534', 'ENSG00000108602', 'ENSG00000132746', 'ENSG00000184254', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000006534', 'ENSG00000108602', 'ENSG00000132746', 'ENSG00000184254', 'ENSG00000154930', 'ENSG00000111058', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000100354', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000116337', 'ENSG00000133805', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000118520', 'ENSG00000023330', 'ENSG00000158578', 'ENSG00000178773', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'ENSG00000076351', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000106605', 'ENSG00000090013', 'ENSG00000105223', 'ENSG00000075651', 'ENSG00000129219', 'ENSG00000179598', 'ENSG00000198814', 'ENSG00000175066', 'ENSG00000196475', 'ENSG00000229894', 'ENSG00000186281', 'ENSG00000119927', 'ENSG00000138678', 'ENSG00000158669', 'ENSG00000155189', 'ENSG00000011198', 'ENSG00000026652', 'ENSG00000143797', 'ENSG00000160216', 'ENSG00000169692', 'ENSG00000172197', 'ENSG00000172954', 'ENSG00000204310', 'ENSG00000144559', 'ENSG00000101290', 'ENSG00000163624', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000162368', 'ENSG00000103202', 'ENSG00000103024', 'ENSG00000143156', 'ENSG00000172113', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000109610', 'ENSG00000176153', 'ENSG00000116157', 'ENSG00000117450', 'ENSG00000117592', 'ENSG00000126432', 'ENSG00000164294', 'ENSG00000165672', 'ENSG00000167468', 'ENSG00000167815', 'ENSG00000198704', 'ENSG00000211445', 'ENSG00000224586', 'ENSG00000233276', 'ENSG00000233276', 'ENSG00000116157', 'ENSG00000117450', 'ENSG00000117592', 'ENSG00000126432', 'ENSG00000164294', 'ENSG00000165672', 'ENSG00000167468', 'ENSG00000167815', 'ENSG00000176153', 'ENSG00000198704', 'ENSG00000211445', 'ENSG00000224586', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000102078', 'ENSG00000109424', 'ENSG00000153291', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000177542', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', '(ENSG00000060762 and ENSG00000143158)', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', '(ENSG00000060762 and ENSG00000143158)', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000147454', 'ENSG00000155287', 'ENSG00000136010', 'ENSG00000144908', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000125166', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000144659', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000103569', 'ENSG00000183048', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000155380', 'ENSG00000070214', 'ENSG00000112499', 'ENSG00000129353', 'ENSG00000137968', 'ENSG00000143036', 'ENSG00000175003', 'ENSG00000197375', 'ENSG00000204385', 'ENSG00000086159', 'ENSG00000100170', 'ENSG00000103375', 'ENSG00000143595', 'ENSG00000161798', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000167580', 'ENSG00000171885', 'ENSG00000240583', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000130821', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000206190', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000124406', 'ENSG00000206190', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000124406', 'ENSG00000206190', 'ENSG00000155380', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000130649', 'ENSG00000155380', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000130821', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000125166', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000164363', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000003989', 'ENSG00000092068', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000112077', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000070214', 'ENSG00000112499', 'ENSG00000129353', 'ENSG00000137968', 'ENSG00000143036', 'ENSG00000175003', 'ENSG00000197375', 'ENSG00000204385', 'ENSG00000103569', 'ENSG00000132874', 'ENSG00000141469', 'ENSG00000018280', 'ENSG00000110911', 'ENSG00000138449', 'ENSG00000086159', 'ENSG00000100170', 'ENSG00000103375', 'ENSG00000143595', 'ENSG00000161798', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000167580', 'ENSG00000171885', 'ENSG00000240583']\n",
      " Mitocore2, has 1108 GPRs\n"
     ]
    }
   ],
   "source": [
    "gpr_0=gpr_counte(gpr_mit_0, 'Mitocore0')\n",
    "gpr_0=gpr_counte(gpr_mit_1, 'Mitocore1')\n",
    "gpr_0=gpr_counte(gpr_mit_2, 'Mitocore2')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "69409060",
   "metadata": {},
   "source": [
    "# GPR and ec-code comparison  Preliminary curation, Alignment to Human1\n",
    "\n",
    "1. Extract model id with gpr from both MitoCore version\n",
    "2. Extract model id with ec-code from both MitoCore version\n",
    "3. merge based on model id"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6dee1dc1",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6a7de380",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "       model_id _gpr\n",
      "0      EX_2hb_e     \n",
      "1       EX_ac_e     \n",
      "2     EX_acac_e     \n",
      "3      EX_akg_e     \n",
      "4    EX_ala_B_e     \n",
      "..          ...  ...\n",
      "550         COt     \n",
      "551         NOt     \n",
      "552  PCHOLHSTDe     \n",
      "553        PSt3     \n",
      "554         PEt     \n",
      "\n",
      "[555 rows x 2 columns]        model_id ec-code\n",
      "0      EX_2hb_e    None\n",
      "1       EX_ac_e    None\n",
      "2     EX_acac_e    None\n",
      "3      EX_akg_e    None\n",
      "4    EX_ala_B_e    None\n",
      "..          ...     ...\n",
      "550         COt    None\n",
      "551         NOt    None\n",
      "552  PCHOLHSTDe    None\n",
      "553        PSt3    None\n",
      "554         PEt    None\n",
      "\n",
      "[555 rows x 2 columns]\n",
      "       model_id _gpr_preliminary_curation _gpr_alignment_Human1_  \\\n",
      "0      EX_2hb_e                                                    \n",
      "1       EX_ac_e                                                    \n",
      "2     EX_acac_e                                                    \n",
      "3      EX_akg_e                                                    \n",
      "4    EX_ala_B_e                                                    \n",
      "..          ...                       ...                    ...   \n",
      "550         COt                                                    \n",
      "551         NOt                                                    \n",
      "552  PCHOLHSTDe                                                    \n",
      "553        PSt3                                                    \n",
      "554         PEt                                                    \n",
      "\n",
      "    ec-code_preliminary_curation ec-code_alignment_Human1_  \n",
      "0                           None                      None  \n",
      "1                           None                      None  \n",
      "2                           None                      None  \n",
      "3                           None                      None  \n",
      "4                           None                      None  \n",
      "..                           ...                       ...  \n",
      "550                         None                      None  \n",
      "551                         None                      None  \n",
      "552                         None                      None  \n",
      "553                         None                      None  \n",
      "554                         None                      None  \n",
      "\n",
      "[555 rows x 5 columns]\n"
     ]
    }
   ],
   "source": [
    "#  curation1 // final curation into a csv file \n",
    "# for visualization of the ec code and GPR to see which enzymaatic reactions cannot introduce protein contraint\n",
    "\n",
    "#  1. Extract model id with gpr from both MitoCore version\n",
    "gpr_mitocore_1= get_model_ids_for_attr(mitocore1, 'reactions', '_gpr')\n",
    "gpr_mitocore_2= get_model_ids_for_attr(mitocore2, 'reactions', '_gpr')\n",
    "\n",
    "#2. Extract model id with ec-code from both MitoCore version\n",
    "ec_mitocore_1= get_model_ids_for_ec_code(mitocore1, 'reactions','_annotation', ['ec-code'])\n",
    "ec_mitocore_2= get_model_ids_for_ec_code(mitocore2, 'reactions','_annotation', ['ec-code'])\n",
    "print(gpr_mitocore_1, ec_mitocore_1)\n",
    "\n",
    "# 3. merge based on model id\n",
    "gpr_merged = pd.merge(\n",
    "    gpr_mitocore_1, \n",
    "    gpr_mitocore_2, \n",
    "    on='model_id', \n",
    "    suffixes=('_preliminary_curation', '_alignment_Human1_')\n",
    ")\n",
    "\n",
    "ec_merged = pd.merge(\n",
    "    ec_mitocore_1, \n",
    "    ec_mitocore_2, \n",
    "    on='model_id', \n",
    "    suffixes=('_preliminary_curation', '_alignment_Human1_')\n",
    ")\n",
    "\n",
    "\n",
    "gpr_ec_merged= pd.merge(\n",
    "    gpr_merged, \n",
    "    ec_merged,\n",
    "    on='model_id', \n",
    "    suffixes=('_preliminary_curation', '_alignment_Human1_')\n",
    ")\n",
    "\n",
    "print(gpr_ec_merged)\n",
    "gpr_ec_merged.to_csv('GPR_comparison.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4f1f5cd0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "synthesis reactions: 4\n",
      "transport reactions: 94\n",
      "Exchange reactions: 70\n",
      "Other reactions: 387\n",
      "       model_id                          _gpr_preliminary_curation  \\\n",
      "0          HEX1  ENSG00000156515 or ENSG00000159399 or ENSG0000...   \n",
      "1        G6PPer                                    ENSG00000131482   \n",
      "2           PGI                                    ENSG00000105220   \n",
      "3           PFK                                    ENSG00000152556   \n",
      "4           FBP                 ENSG00000165140 or ENSG00000130957   \n",
      "..          ...                                                ...   \n",
      "382         COt                                                      \n",
      "383         NOt                                                      \n",
      "384  PCHOLHSTDe                                                      \n",
      "385        PSt3                                                      \n",
      "386         PEt                                                      \n",
      "\n",
      "    ec-code_preliminary_curation  \\\n",
      "0             [2.7.1.1, 2.7.1.2]   \n",
      "1                        3.1.3.9   \n",
      "2                        5.3.1.9   \n",
      "3                       2.7.1.11   \n",
      "4                       3.1.3.11   \n",
      "..                           ...   \n",
      "382                         None   \n",
      "383                         None   \n",
      "384                         None   \n",
      "385                         None   \n",
      "386                         None   \n",
      "\n",
      "                                _gpr_alignment_Human1_  \\\n",
      "0    ENSG00000156515 or ENSG00000159399 or ENSG0000...   \n",
      "1                                      ENSG00000131482   \n",
      "2                                      ENSG00000105220   \n",
      "3    ENSG00000152556 or ENSG00000067057 or ENSG0000...   \n",
      "4                   ENSG00000165140 or ENSG00000130957   \n",
      "..                                                 ...   \n",
      "382                                                      \n",
      "383                                                      \n",
      "384                                                      \n",
      "385                                                      \n",
      "386                                                      \n",
      "\n",
      "                             ec-code_alignment_Human1_  \n",
      "0    [2.7.1.1, 2.7.1.2, 2.7.1.4, 2.7.1.55, 2.7.1.59...  \n",
      "1    [3.1.3.1, 3.1.3.104, 3.1.3.2, 3.1.3.23, 3.1.3....  \n",
      "2                                              5.3.1.9  \n",
      "3                                             2.7.1.11  \n",
      "4                                             3.1.3.11  \n",
      "..                                                 ...  \n",
      "382                                               None  \n",
      "383                                               None  \n",
      "384                                               None  \n",
      "385                                               None  \n",
      "386                                               None  \n",
      "\n",
      "[387 rows x 5 columns]\n"
     ]
    }
   ],
   "source": [
    "\n",
    "import re\n",
    "\n",
    "# extract synthesis, transport, exchange reactions\n",
    "synthesis = r\"^OF_[A-Za-z0-9]+_MitoCore$\"\n",
    "transport = r\"^(?!OF_)[A-Za-z0-9]+_MitoCore$\" or \n",
    "exchange = r\"^EX_[A-Za-z0-9_]+$\"\n",
    "\n",
    "df = {'model_id': [], '_gpr_preliminary_curation': [], 'ec-code_preliminary_curation':[], '_gpr_alignment_Human1_': [],'ec-code_alignment_Human1_':[]}\n",
    "\n",
    "#count all reaction for total of 555 reactio0ns in MitoCore\n",
    "counter_exchange = 0\n",
    "counter_transport = 0\n",
    "counter_core = 0\n",
    "counter_other = 0\n",
    "\n",
    "# clasify reaction to return a dataframe just with enzymatic reactions and their features\n",
    "for index, reaction in gpr_ec_merged.iterrows():\n",
    "        model_id = reaction['model_id']\n",
    "        if re.search(synthesis, model_id):\n",
    "            counter_transport += 1\n",
    "        elif re.search(transport, model_id):\n",
    "            counter_core += 1\n",
    "        elif re.search(exchange, model_id):\n",
    "            counter_exchange += 1\n",
    "        else:\n",
    "            counter_other += 1\n",
    "            df['model_id'].append(model_id)\n",
    "            gpr_0 = reaction.get('_gpr_preliminary_curation', None)\n",
    "            ec_0 = reaction.get('ec-code_preliminary_curation', None)\n",
    "            ec_1 = reaction.get('ec-code_alignment_Human1_', None)\n",
    "            gpr_1 = reaction.get('_gpr_alignment_Human1_', None)\n",
    "            df['_gpr_preliminary_curation'].append(gpr_0)\n",
    "            df['ec-code_preliminary_curation'].append(ec_0)\n",
    "            df['_gpr_alignment_Human1_'].append(gpr_1)\n",
    "            df['ec-code_alignment_Human1_'].append(ec_1)\n",
    "\n",
    "print(f\"synthesis reactions: {counter_transport}\")\n",
    "print(f\"transport reactions: {counter_core}\")\n",
    "print(f\"Exchange reactions: {counter_exchange}\")\n",
    "print(f\"Other reactions: {counter_other}\")\n",
    "\n",
    "df = pd.DataFrame(df)\n",
    "df.to_csv('enzymatic_react.csv', index=False)\n",
    "print(df)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "id": "d30e35da",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Reaction identifier</strong></td><td>r0645</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x2a5fc1b50</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Stoichiometry</strong></td>\n",
       "                <td>\n",
       "                    <p style='text-align:right'>am6sa_c + h2o_c + nad_c --> amuco_c + 2.0 h_c + nadh_c</p>\n",
       "                    <p style='text-align:right'>2-Aminomuconate semialdehyde + H2O + NAD --> 2-Aminomuconate + 2.0 H + NADH</p>\n",
       "                </td>\n",
       "            </tr><tr>\n",
       "                <td><strong>GPR</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Lower bound</strong></td><td>0.0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Upper bound</strong></td><td>1000.0</td>\n",
       "            </tr>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<Reaction r0645 at 0x2a5fc1b50>"
      ]
     },
     "execution_count": 106,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mitocore1.reactions.get_by_id('r0645')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "1966d0e7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "original MitoCore\n",
      "reactions 555\n",
      "metabolites 441\n",
      "genes 391\n",
      " MitoCore prelim curation\n",
      "reactions 555\n",
      "metabolites 441\n",
      "genes 387\n",
      " MitoCore alignment Human1\n",
      "reactions 555\n",
      "metabolites 441\n",
      "genes 595\n"
     ]
    }
   ],
   "source": [
    "print('original MitoCore')\n",
    "print('reactions',len(mitocore0.reactions))\n",
    "print('metabolites',len(mitocore0.metabolites))\n",
    "print('genes', len(mitocore0.genes))\n",
    "print(' MitoCore prelim curation')\n",
    "print('reactions',len(mitocore1.reactions))\n",
    "print('metabolites',len(mitocore1.metabolites))\n",
    "print('genes', len(mitocore1.genes))\n",
    "print(' MitoCore alignment Human1')\n",
    "print('reactions',len(mitocore2.reactions))\n",
    "print('metabolites',len(mitocore2.metabolites))\n",
    "print('genes', len(mitocore2.genes))\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c8aa2c1",
   "metadata": {},
   "source": [
    "# Classify model reactions into exchange, transport and enzymatic reactions \n",
    "\n",
    "transport and ecchange reactions do not have ec-codes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "id": "4397671b",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n",
      "Could not identify an external compartment by name and choosing one with the most boundary reactions. That might be complete nonsense or change suddenly. Consider renaming your compartments using `Model.compartments` to fix this.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "exchange 70\n",
      "transport 162\n",
      "enzymatic 323\n"
     ]
    }
   ],
   "source": [
    "exchange = []\n",
    "transport=[]\n",
    "enzymatic=[]\n",
    "\n",
    "\n",
    "def is_transport_reaction(reaction):\n",
    "    \"\"\"collect reactions that are transport steps based on the number of compartments\"\"\"\n",
    "\n",
    "    compartments = {met.id.split(\"_\")[-1] for met in reaction.metabolites}\n",
    "    return len(compartments) > 1\n",
    "\n",
    "#check exchange reactions (import export, include boundary reactions)\n",
    "for reaction in mitocore1.reactions:\n",
    "    if reaction in mitocore1.exchanges:\n",
    "        exchange.append(reaction)\n",
    "\n",
    "#check transport (between compartment)\n",
    "    elif is_transport_reaction(reaction):\n",
    "        transport.append(reaction)\n",
    "        \n",
    "#check enzymatic reactions\n",
    "    else: \n",
    "        enzymatic.append(reaction)\n",
    "\n",
    "print('exchange',len(exchange))\n",
    "print('transport',len(transport))\n",
    "print('enzymatic',len(enzymatic))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "id": "1f05b202",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reactions with gpr and ec_code 7\n",
      "reactions with no ec_code 40\n",
      "reactions with no gpr 0\n",
      "reactions with none 115\n"
     ]
    }
   ],
   "source": [
    "#check reactions with ec-code in lists\n",
    "gpr=[]\n",
    "ec_code=[]\n",
    "gpr_and_ec_code=[]\n",
    "none=[]\n",
    "\n",
    "for reaction in transport:\n",
    "    gpr_val = reaction.gene_reaction_rule if hasattr(reaction, 'gene_reaction_rule') else reaction.gpr\n",
    "    id = reaction.id\n",
    "    annotation = reaction.__dict__.get('_annotation', {})\n",
    "    ec_val = annotation.get('ec-code')\n",
    "    if gpr_val and ec_val:\n",
    "        gpr_and_ec_code.append(reaction)\n",
    "    elif not gpr_val and not ec_val:\n",
    "        none.append(reaction)\n",
    "    elif gpr_val and not ec_val:\n",
    "        gpr.append(reaction)\n",
    "    elif ec_val and not gpr_val:\n",
    "        ec_code.append(reaction)\n",
    "print('reactions with gpr and ec_code', len(gpr_and_ec_code))\n",
    "print('reactions with no ec_code', len(gpr))\n",
    "print('reactions with no gpr', len(ec_code))\n",
    "print('reactions with none', len(none))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d51dbbd8",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cc0a31b9",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b35c1c5e",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dc1a6cb4",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7e8ba5b1",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "id": "39e9276d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CIV_MitoCore\n"
     ]
    }
   ],
   "source": [
    "for reaction in ec_code:\n",
    "    print(reaction.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 192,
   "id": "4beebacb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CI_MitoCore has GPR: ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264\n",
      "CIII_MitoCore has GPR: (ENSG00000010256 and ENSG00000011295 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000169021 and ENSG00000172115 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727) or (ENSG00000010256 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000164405 and ENSG00000169021 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727)\n",
      "CV_MitoCore has GPR: (ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154723 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222) or (ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222) or (ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222) or (ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000156411 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000173915 and ENSG00000180389 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)\n",
      "NNT_MitoCore has GPR: ENSG00000112992\n",
      "r0205 has GPR: ENSG00000115159\n",
      "PPPGOmB_MitoCore has GPR: ENSG00000143224\n",
      "ATPtmB_MitoCore has GPR: ENSG00000151729 or ENSG00000005022 or ENSG00000169100\n",
      "CI_MitoCore has ec-code: ['1.6.5.3', '1.6.5.8', '1.6.5.9']\n",
      "CIII_MitoCore has ec-code: 1.10.2.2\n",
      "CV_MitoCore has ec-code: ['3.6.3.14', '7.1.2.2']\n",
      "NNT_MitoCore has ec-code: 1.6.1.2\n",
      "r0205 has ec-code: 1.1.5.3\n",
      "PPPGOmB_MitoCore has ec-code: 1.3.3.4\n",
      "ATPtmB_MitoCore has ec-code: 2.7.4.6\n"
     ]
    }
   ],
   "source": [
    "for reaction in gpr_and_ec_code:\n",
    "    id = reaction.id\n",
    "    gpr_val = reaction.gene_reaction_rule if hasattr(reaction, 'gene_reaction_rule') else reaction.gpr\n",
    "    print(id, 'has GPR:', gpr_val)\n",
    "\n",
    "for reaction in gpr_and_ec_code:\n",
    "    id = reaction.id\n",
    "    annotation = reaction.__dict__.get('_annotation', {})\n",
    "    ec_val = annotation.get('ec-code')\n",
    "    print(id, 'has ec-code:', ec_val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "id": "9247dd86",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.9.3.1\n"
     ]
    }
   ],
   "source": [
    "#this reaction can be fixed\n",
    "AMCOXO = mitocore1.reactions.get_by_id('CIV_MitoCore')\n",
    "annotation = AMCOXO.__dict__.get('_annotation', {})\n",
    "ec_val = annotation.get('ec-code')\n",
    "\n",
    "print(ec_val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "id": "5e0fd176",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "        <table>\n",
       "            <tr>\n",
       "                <td><strong>Reaction identifier</strong></td><td>CIV_MitoCore</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Name</strong></td><td>R_CIV_MitoCore cytochrome c oxidase</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Memory address</strong></td>\n",
       "                <td>0x2a6a1d950</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Stoichiometry</strong></td>\n",
       "                <td>\n",
       "                    <p style='text-align:right'>8.0 PMF_m + 4.0 focytC_m + o2_m --> 4.0 PMF_c + 4.0 ficytC_m + 2.0 h2o_m</p>\n",
       "                    <p style='text-align:right'>8.0 Proton Motive Force + 4.0 Ferrocytochrome c + Oxygen --> 4.0 Proton Motive Force + 4.0 Ferricytochrome c + 2.0 H2O</p>\n",
       "                </td>\n",
       "            </tr><tr>\n",
       "                <td><strong>GPR</strong></td><td></td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Lower bound</strong></td><td>0.0</td>\n",
       "            </tr><tr>\n",
       "                <td><strong>Upper bound</strong></td><td>1000.0</td>\n",
       "            </tr>\n",
       "        </table>\n",
       "        "
      ],
      "text/plain": [
       "<Reaction CIV_MitoCore at 0x2a6a1d950>"
      ]
     },
     "execution_count": 201,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mitocore2.reactions.get_by_id('CIV_MitoCore')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "610999a4",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0ecc3b60",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "45daaa70",
   "metadata": {},
   "source": [
    "list of interesting reactions:\n",
    "AMCOXO --> ec-code: 1.5.1.-  was deleted"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
