{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "23660d9a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import cobra\n",
    "import memote\n",
    "import os "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "5afdca38",
   "metadata": {},
   "outputs": [],
   "source": [
    "os.chdir('/Users/benjaminreyes/Desktop/Masterarbeit/neutrophil_modeling/neutrophil-modeling/curation')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "e537b970",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Model does not contain SBML fbc package information.\n",
      "SBML package 'layout' not supported by cobrapy, information is not parsed\n",
      "SBML package 'render' not supported by cobrapy, information is not parsed\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_10fthf_c \"10-Formyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_10fthf_c \"10-Formyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_10fthf_m \"10-Formyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_10fthf_m \"10-Formyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_13dpg_c \"3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_13dpg_c \"3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_1pipdn2c_c \"delta1-Piperideine-2-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1pipdn2c_c \"delta1-Piperideine-2-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_1pyr5c_m \"(S)-1-Pyrroline-5-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_1pyr5c_m \"(S)-1-Pyrroline-5-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2aobut_m \"L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2aobut_m \"L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2hb_c \"2-Hydroxybutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2hb_c \"2-Hydroxybutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2hb_e \"2-Hydroxybutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2hb_e \"2-Hydroxybutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2maacoa_m \"2-Methylacetoacetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2maacoa_m \"2-Methylacetoacetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mb2coa_m \"2-Methylbut-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mb2coa_m \"2-Methylbut-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mbcoa_m \"2-Methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mbcoa_m \"2-Methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mop_m \"2-Methyl-3-oxopropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mop_m \"2-Methyl-3-oxopropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2mp2coa_m \"2-Methylprop-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2mp2coa_m \"2-Methylprop-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2obut_c \"2-Oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2obut_c \"2-Oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2obut_m \"2-Oxobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2obut_m \"2-Oxobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2oxoadp_c \"2-Oxoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2oxoadp_c \"2-Oxoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2oxoadp_m \"2-Oxoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2oxoadp_m \"2-Oxoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_2pg_c \"2-Phospho-D-glycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_2pg_c \"2-Phospho-D-glycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_34hpp_c \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_34hpp_c \"3-(4-Hydroxyphenyl)pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hanthrn_c \"3-Hydroxyanthranilate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hanthrn_c \"3-Hydroxyanthranilate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hbcoa_m \"(S)-3-Hydroxybutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hbcoa_m \"(S)-3-Hydroxybutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hexdcoa_m \"(S)-3-Hydroxyhexadecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hexdcoa_m \"(S)-3-Hydroxyhexadecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hibutcoa_m \"(S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hibutcoa_m \"(S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hmbcoa_m \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hmbcoa_m \"(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3hmp_m \"3-Hydroxy-2-methylpropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3hmp_m \"3-Hydroxy-2-methylpropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mb2coa_m \"3-Methylcrotonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mb2coa_m \"3-Methylcrotonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mgcoa_m \"3-Methylglutaconyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mgcoa_m \"3-Methylglutaconyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mob_m \"3-Methyl-2-oxobutanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mob_m \"3-Methyl-2-oxobutanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3mop_m \"(S)-3-Methyl-2-oxopentanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3mop_m \"(S)-3-Methyl-2-oxopentanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3odcoa_m \"3-Oxodecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3odcoa_m \"3-Oxodecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3oddcoa_m \"3-Oxododecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3oddcoa_m \"3-Oxododecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3pg_c \"3-Phospho-D-glycerate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3pg_c \"3-Phospho-D-glycerate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3php_c \"3-Phosphonooxypyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3php_c \"3-Phosphonooxypyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3sala_c \"3-Sulfino-L-alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3sala_c \"3-Sulfino-L-alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3snpyr_c \"3-Sulfinylpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3snpyr_c \"3-Sulfinylpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_3tetd7ecoa_m \"trans-Tetradec-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_3tetd7ecoa_m \"trans-Tetradec-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4aabutn_m \"4-Acetamidobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4aabutn_m \"4-Acetamidobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abut_c \"4-Aminobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abut_c \"4-Aminobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abut_m \"4-Aminobutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abut_m \"4-Aminobutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4abutn_c \"4-Aminobutyraldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4abutn_c \"4-Aminobutyraldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4fumacac_c \"4-Fumarylacetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4fumacac_c \"4-Fumarylacetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4izp_c \"4-Imidazolone-5-propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4izp_c \"4-Imidazolone-5-propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4mlacac_c \"4-Maleylacetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4mlacac_c \"4-Maleylacetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_4mop_m \"4-Methyl-2-oxopentanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_4mop_m \"4-Methyl-2-oxopentanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5aop_c \"5-Aminolevulinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5aop_c \"5-Aminolevulinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5aop_m \"5-Aminolevulinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5aop_m \"5-Aminolevulinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_5forthf_c \"5-Formiminotetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_5forthf_c \"5-Formiminotetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6a2ohxnt_c \"6-Amino-2-oxohexanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6a2ohxnt_c \"6-Amino-2-oxohexanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6pgc_c \"6-Phospho-D-gluconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6pgc_c \"6-Phospho-D-gluconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_6pgl_c \"D-Glucono-1,5-lactone 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_6pgl_c \"D-Glucono-1,5-lactone 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aacoa_m \"Acetoacetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aacoa_m \"Acetoacetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aact_m \"Aminoacetone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aact_m \"Aminoacetone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_c \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_c \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_e \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_e \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ac_m \"Acetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ac_m \"Acetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_c \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_c \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_e \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_e \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acac_m \"Acetoacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acac_m \"Acetoacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acald_c \"Acetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acald_c \"Acetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acald_m \"Acetaldehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acald_m \"Acetaldehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_accoa_c \"Acetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_accoa_c \"Acetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_accoa_m \"Acetyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_accoa_m \"Acetyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adn_c \"Adenosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adn_c \"Adenosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adp_c \"ADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adp_c \"ADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_adp_m \"ADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_adp_m \"ADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_agm_m \"Agmatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_agm_m \"Agmatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ahcys_c \"S-Adenosyl-L-homocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ahcys_c \"S-Adenosyl-L-homocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_c \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_c \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_e \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_e \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_akg_m \"2-Oxoglutarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_akg_m \"2-Oxoglutarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_c \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_c \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_e \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_e \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_B_m \"beta-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_B_m \"beta-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_c \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_c \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_e \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_e \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ala_L_m \"L-Alanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ala_L_m \"L-Alanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_alpa_hs_m \"1-Acyl-sn-glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_alpa_hs_m \"1-Acyl-sn-glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_alpro_m \"S-Aminomethyldihydrolipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_alpro_m \"S-Aminomethyldihydrolipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_am6sa_c \"2-Aminomuconate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_am6sa_c \"2-Aminomuconate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amet_c \"S-Adenosyl-L-methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amet_c \"S-Adenosyl-L-methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amp_c \"AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amp_c \"AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amp_m \"AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amp_m \"AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_amuco_c \"2-Aminomuconate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_amuco_c \"2-Aminomuconate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_aprut_m \"N-Acetylputrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_aprut_m \"N-Acetylputrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_c \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_c \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_e \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_e \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_arg_L_m \"L-Arginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_arg_L_m \"L-Arginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_argsuc_c \"N-(L-Arginino)succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_argsuc_c \"N-(L-Arginino)succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_argsuc_e \"N-(L-Arginino)succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_argsuc_e \"N-(L-Arginino)succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_c \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_c \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_e \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_e \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asn_L_m \"L-Asparagine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asn_L_m \"L-Asparagine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_c \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_c \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_e \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_e \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_asp_L_m \"L-Aspartate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_asp_L_m \"L-Aspartate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_atp_c \"ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_atp_c \"ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_atp_m \"ATP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_atp_m \"ATP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_b2coa_m \"Crotonoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_b2coa_m \"Crotonoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_c \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_c \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_e \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_e \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bhb_m \"(R)-3-Hydroxybutanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bhb_m \"(R)-3-Hydroxybutanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bilirub_c \"Bilirubin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bilirub_c \"Bilirubin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_bilirub_e \"Bilirubin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_bilirub_e \"Bilirubin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biliverd_c \"Biliverdin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biliverd_c \"Biliverdin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_c \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_c \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_e \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_e \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_biomass_m \"Biomass\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_biomass_m \"Biomass\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_btcoa_m \"Butanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_btcoa_m \"Butanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_c \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_c \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_e \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_e \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_but_m \"Butanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_but_m \"Butanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cbp_m \"Carbamoyl phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cbp_m \"Carbamoyl phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cdp_m \"CDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cdp_m \"CDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cdpdag_hs_m \"CDP-diacylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cdpdag_hs_m \"CDP-diacylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cgly_c \"Cys-Gly\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cgly_c \"Cys-Gly\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_c \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_c \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_e \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_e \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_chol_m \"Choline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_chol_m \"Choline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_c \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_c \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_e \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_e \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cit_m \"Citrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cit_m \"Citrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_c \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_c \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_e \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_e \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_citr_L_m \"L-Citrulline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_citr_L_m \"L-Citrulline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_clpn_hs_m \"Cardiolipin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_clpn_hs_m \"Cardiolipin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cmp_m \"CMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cmp_m \"CMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cmusa_c \"2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cmusa_c \"2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co_c \"CO\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co_c \"CO\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co_e \"CO\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co_e \"CO\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_c \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_c \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_e \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_e \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_co2_m \"CO2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_co2_m \"CO2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_coa_c \"CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_coa_c \"CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_coa_m \"CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_coa_m \"CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cpppg3_c \"Coproporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cpppg3_c \"Coproporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_c \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_c \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_e \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_e \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_creat_m \"Creatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_creat_m \"Creatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_crn_c \"L-Carnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_crn_c \"L-Carnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_crn_m \"L-Carnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_crn_m \"L-Carnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ctp_m \"CTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ctp_m \"CTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_c \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_c \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_e \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_e \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyan_m \"Cyanide ion\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyan_m \"Cyanide ion\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_c \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_c \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_e \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_e \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cys_L_m \"L-Cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cys_L_m \"L-Cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_cyst_L_c \"L-Cystathionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_cyst_L_c \"L-Cystathionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dc2coa_m \"trans-Dec-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dc2coa_m \"trans-Dec-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dcacoa_m \"Decanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dcacoa_m \"Decanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dcamp_c \"N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dcamp_c \"N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dd2coa_m \"2-trans-Dodecenoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dd2coa_m \"2-trans-Dodecenoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ddcacoa_m \"Lauroyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ddcacoa_m \"Lauroyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhap_c \"Glycerone phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhap_c \"Glycerone phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhbpt_c \"Dihydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhbpt_c \"Dihydrobiopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhlpro_m \"Dihydrolipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhlpro_m \"Dihydrolipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_e4p_c \"D-Erythrose 4-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_e4p_c \"D-Erythrose 4-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_etoh_c \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_etoh_c \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_etoh_e \"Ethanol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_etoh_e \"Ethanol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_f6p_c \"D-Fructose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_f6p_c \"D-Fructose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fdp_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fdp_c \"D-Fructose 1,6-bisphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_c \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_c \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_e \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_e \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fe2_m \"Fe2\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fe2_m \"Fe2\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ficytC_m \"Ferricytochrome c\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ficytC_m \"Ferricytochrome c\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_focytC_m \"Ferrocytochrome c\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_focytC_m \"Ferrocytochrome c\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_c \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_c \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_e \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_e \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_for_m \"Formate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_for_m \"Formate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_forglu_c \"N-Formimino-L-glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_forglu_c \"N-Formimino-L-glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_c \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_c \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_e \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_e \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fum_m \"Fumarate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fum_m \"Fumarate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_g3p_c \"D-Glyceraldehyde 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_g3p_c \"D-Glyceraldehyde 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_g6p_c \"D-Glucose 6-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_g6p_c \"D-Glucose 6-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gdp_c \"GDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gdp_c \"GDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gdp_m \"GDP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gdp_m \"GDP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glc_D_c \"D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glc_D_c \"D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glc_D_e \"D-Glucose\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glc_D_e \"D-Glucose\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_c \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_c \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_e \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_e \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gln_L_m \"L-Glutamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gln_L_m \"L-Glutamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_c \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_c \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_e \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_e \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu_L_m \"L-Glutamate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu_L_m \"L-Glutamate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu5p_m \"L-Glutamyl 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu5p_m \"L-Glutamyl 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glu5sa_m \"L-Glutamate 5-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glu5sa_m \"L-Glutamate 5-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glucys_c \"gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glucys_c \"gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glutcoa_m \"Glutaryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glutcoa_m \"Glutaryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_c \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_c \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_e \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_e \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gly_m \"Glycine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gly_m \"Glycine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_c \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_c \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_e \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_e \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc_m \"Glycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc_m \"Glycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc3p_c \"sn-Glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc3p_c \"sn-Glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_glyc3p_m \"sn-Glycerol 3-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_glyc3p_m \"sn-Glycerol 3-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthox_c \"Glutathione disulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthox_c \"Glutathione disulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthox_m \"Glutathione disulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthox_m \"Glutathione disulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthrd_c \"Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthrd_c \"Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gthrd_m \"Glutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gthrd_m \"Glutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gtp_c \"GTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gtp_c \"GTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_gtp_m \"GTP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_gtp_m \"GTP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_c \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_c \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_e \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_e \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h_m \"H\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h_m \"H\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_PMF_c \"Proton Motive Force\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_PMF_c \"Proton Motive Force\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_PMF_m \"Proton Motive Force\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_PMF_m \"Proton Motive Force\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_c \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_c \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_e \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_e \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o_m \"H2O\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o_m \"H2O\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o2_c \"Hydrogen peroxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o2_c \"Hydrogen peroxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_h2o2_m \"Hydrogen peroxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_h2o2_m \"Hydrogen peroxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC00250_c \"Hydrogen sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC00250_c \"Hydrogen sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC00250_e \"Hydrogen sulfide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC00250_e \"Hydrogen sulfide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01401_m \"(S)-3-Hydroxydodecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01401_m \"(S)-3-Hydroxydodecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01403_m \"(S)-Hydroxydecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01403_m \"(S)-Hydroxydecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01405_m \"(S)-3-Hydroxyoctanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01405_m \"(S)-3-Hydroxyoctanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01406_m \"3-Oxooctanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01406_m \"3-Oxooctanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01407_m \"(S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01407_m \"(S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01408_m \"3-Oxohexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01408_m \"3-Oxohexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01409_m \"Hexanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01409_m \"Hexanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01410_m \"trans-Hex-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01410_m \"trans-Hex-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01415_m \"trans-Oct-2-enoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01415_m \"trans-Oct-2-enoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_c \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_c \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_e \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_e \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hco3_m \"HCO3-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hco3_m \"HCO3-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hcys_L_c \"Homocysteine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hcys_L_c \"Homocysteine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_c \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_c \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_e \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_e \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hdca_m \"Hexadecanoic acid\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hdca_m \"Hexadecanoic acid\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hgentis_c \"Homogentisate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hgentis_c \"Homogentisate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_c \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_c \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_e \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_e \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_his_L_m \"L-Histidine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_his_L_m \"L-Histidine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hLkynr_c \"3-Hydroxy-L-kynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hLkynr_c \"3-Hydroxy-L-kynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hmbil_c \"Hydroxymethylbilane\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hmbil_c \"Hydroxymethylbilane\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_hmgcoa_m \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_hmgcoa_m \"(S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ibcoa_m \"2-Methylpropanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ibcoa_m \"2-Methylpropanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_c \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_c \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_e \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_e \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_icit_m \"Isocitrate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_icit_m \"Isocitrate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_c \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_c \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_e \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_e \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ile_L_m \"L-Isoleucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ile_L_m \"L-Isoleucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_imp_c \"IMP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_imp_c \"IMP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ivcoa_m \"3-Methylbutanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ivcoa_m \"3-Methylbutanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp_c \"L-2-Aminoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp_c \"L-2-Aminoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp_m \"L-2-Aminoadipate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp_m \"L-2-Aminoadipate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp6sa_c \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp6sa_c \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_L2aadp6sa_m \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_L2aadp6sa_m \"L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_D_m \"(R)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_D_m \"(R)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_L_c \"(S)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_L_c \"(S)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lac_L_e \"(S)-Lactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lac_L_e \"(S)-Lactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_c \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_c \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_e \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_e \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_leu_L_m \"L-Leucine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_leu_L_m \"L-Leucine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lfmkynr_c \"L-Formylkynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lfmkynr_c \"L-Formylkynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lgt_S_m \"(R)-S-Lactoylglutathione\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lgt_S_m \"(R)-S-Lactoylglutathione\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lkynr_c \"L-Kynurenine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lkynr_c \"L-Kynurenine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_Lpipecol_c \"L-Pipecolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_Lpipecol_c \"L-Pipecolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lpro_m \"Lipoylprotein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lpro_m \"Lipoylprotein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_c \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_c \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_e \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_e \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_lys_L_m \"L-Lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_lys_L_m \"L-Lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_c \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_c \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_e \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_e \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mal_L_m \"(S)-Malate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mal_L_m \"(S)-Malate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercplac_c \"3-Mercaptolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercplac_c \"3-Mercaptolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercplac_e \"3-Mercaptolactate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercplac_e \"3-Mercaptolactate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercppyr_c \"Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercppyr_c \"Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mercppyr_m \"Mercaptopyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mercppyr_m \"Mercaptopyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_c \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_c \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_e \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_e \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_met_L_m \"L-Methionine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_met_L_m \"L-Methionine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_methf_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_methf_c \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_methf_m \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_methf_m \"5,10-Methenyltetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mlthf_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mlthf_c \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mlthf_m \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mlthf_m \"5,10-Methylenetetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mmcoa_R_m \"(R)-Methylmalonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mmcoa_R_m \"(R)-Methylmalonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mmcoa_S_m \"(S)-Methylmalonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mmcoa_S_m \"(S)-Methylmalonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_msa_m \"3-Oxopropanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_msa_m \"3-Oxopropanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_mthgxl_m \"Methylglyoxal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_mthgxl_m \"Methylglyoxal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_n4abutn_m \"N4-Acetylaminobutanal\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_n4abutn_m \"N4-Acetylaminobutanal\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_c \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_c \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_e \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_e \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nad_m \"NAD\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nad_m \"NAD\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_c \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_c \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_e \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_e \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadh_m \"NADH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadh_m \"NADH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadp_c \"NADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadp_c \"NADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadp_m \"NADP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadp_m \"NADP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadph_c \"NADPH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadph_c \"NADPH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nadph_m \"NADPH\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nadph_m \"NADPH\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_c \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_c \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_e \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_e \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nh4_m \"Ammonia\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nh4_m \"Ammonia\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_no_c \"Nitric oxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_no_c \"Nitric oxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_no_e \"Nitric oxide\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_no_e \"Nitric oxide\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_nwharg_c \"N(omega)-Hydroxyarginine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_nwharg_c \"N(omega)-Hydroxyarginine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_c \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_c \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_e \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_e \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2_m \"Oxygen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2_m \"Oxygen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_o2s_m \"O2.-\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_o2s_m \"O2.-\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_c \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_c \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_e \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_e \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_oaa_m \"Oxaloacetate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_oaa_m \"Oxaloacetate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_occoa_m \"Octanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_occoa_m \"Octanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_orn_c \"L-Ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_orn_c \"L-Ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_orn_m \"L-Ornithine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_orn_m \"L-Ornithine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pa_hs_m \"Phosphatidate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pa_hs_m \"Phosphatidate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_c \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_c \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_e \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_e \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pchol_hs_m \"Phosphatidylcholine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pchol_hs_m \"Phosphatidylcholine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_c \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_c \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_e \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_e \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pcreat_m \"Phosphocreatine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pcreat_m \"Phosphocreatine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_c \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_c \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_e \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_e \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pe_hs_m \"Phosphatidylethanolamine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pe_hs_m \"Phosphatidylethanolamine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pep_c \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pep_c \"Phosphoenolpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pep_m \"Phosphoenolpyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pep_m \"Phosphoenolpyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pglyc_hs_m \"Phosphatidylglycerol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pglyc_hs_m \"Phosphatidylglycerol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pgp_hs_m \"Phosphatidylglycerophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pgp_hs_m \"Phosphatidylglycerophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_c \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_c \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_e \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_e \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_phe_L_m \"L-Phenylalanine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_phe_L_m \"L-Phenylalanine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pheme_c \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pheme_c \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pheme_m \"Heme\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pheme_m \"Heme\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_c \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_c \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_e \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_e \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pi_m \"Orthophosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pi_m \"Orthophosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcoa_c \"Palmitoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcoa_c \"Palmitoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcoa_m \"Palmitoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcoa_m \"Palmitoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcrn_c \"L-Palmitoylcarnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcrn_c \"L-Palmitoylcarnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pmtcrn_m \"L-Palmitoylcarnitine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pmtcrn_m \"L-Palmitoylcarnitine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_c \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_c \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_e \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_e \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppa_m \"Propanoate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppa_m \"Propanoate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppbng_c \"Porphobilinogen\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppbng_c \"Porphobilinogen\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppcoa_m \"Propanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppcoa_m \"Propanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppi_c \"Diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppi_c \"Diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppi_m \"Diphosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppi_m \"Diphosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ppp9_m \"Protoporphyrin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ppp9_m \"Protoporphyrin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pppg9_c \"Protoporphyrinogen IX\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pppg9_c \"Protoporphyrinogen IX\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_c \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_c \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_e \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_e \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pro_L_m \"L-Proline\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pro_L_m \"L-Proline\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_c \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_c \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_e \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_e \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ps_hs_m \"Phosphatidylserine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ps_hs_m \"Phosphatidylserine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pser_L_c \"O-Phospho-L-serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pser_L_c \"O-Phospho-L-serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ptrc_c \"Putrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ptrc_c \"Putrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ptrc_m \"Putrescine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ptrc_m \"Putrescine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pyr_c \"Pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pyr_c \"Pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_pyr_m \"Pyruvate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_pyr_m \"Pyruvate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_q10_m \"Ubiquinone\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_q10_m \"Ubiquinone\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_q10h2_m \"Ubiquinol\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_q10h2_m \"Ubiquinol\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_r5p_c \"D-Ribose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_r5p_c \"D-Ribose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ru5p_D_c \"D-Ribulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ru5p_D_c \"D-Ribulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_s7p_c \"Sedoheptulose 7-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_s7p_c \"Sedoheptulose 7-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_saccrp_L_m \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_saccrp_L_m \"N6-(L-1,3-Dicarboxypropyl)-L-lysine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_c \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_c \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_e \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_e \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ser_L_m \"L-Serine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ser_L_m \"L-Serine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_c \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_c \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_e \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_e \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so3_m \"Sulfite\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so3_m \"Sulfite\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so4_c \"Sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so4_c \"Sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_so4_m \"Sulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_so4_m \"Sulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_c \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_c \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_e \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_e \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succ_m \"Succinate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succ_m \"Succinate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_succoa_m \"Succinyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_succoa_m \"Succinyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_sucsal_m \"Succinate semialdehyde\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_sucsal_m \"Succinate semialdehyde\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_c \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_c \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_e \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_e \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tcynt_m \"Thiocyanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tcynt_m \"Thiocyanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tetd7ecoa_m \"Tetradecanoyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tetd7ecoa_m \"Tetradecanoyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thbpt_c \"Tetrahydrobiopterin\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thbpt_c \"Tetrahydrobiopterin\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thf_c \"Tetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thf_c \"Tetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thf_m \"Tetrahydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thf_m \"Tetrahydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thp2c_c \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thp2c_c \"(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_c \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_c \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_e \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_e \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_thr_L_m \"L-Threonine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_thr_L_m \"L-Threonine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_c \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_c \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_e \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_e \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_trp_L_m \"L-Tryptophan\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_trp_L_m \"L-Tryptophan\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_c \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_c \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_e \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_e \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tsul_m \"Thiosulfate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tsul_m \"Thiosulfate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_c \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_c \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_e \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_e \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_tyr_L_m \"L-Tyrosine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_tyr_L_m \"L-Tyrosine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_uppg3_c \"Uroporphyrinogen III\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_uppg3_c \"Uroporphyrinogen III\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urcan_c \"Urocanate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urcan_c \"Urocanate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_c \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_c \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_e \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_e \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_urea_m \"Urea\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_urea_m \"Urea\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_c \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_c \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_e \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_e \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_val_L_m \"L-Valine\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_val_L_m \"L-Valine\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_xu5p_D_c \"D-Xylulose 5-phosphate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_xu5p_D_c \"D-Xylulose 5-phosphate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_c \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_c \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_e \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_e \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_fol_m \"Folate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_fol_m \"Folate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhf_c \"Dihydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhf_c \"Dihydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_dhf_m \"Dihydrofolate\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_dhf_m \"Dihydrofolate\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_malcoa_c \"Malonyl-CoA\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_malcoa_c \"Malonyl-CoA\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_malACP_c \"Malonyl-[acyl-carrier protein]\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_malACP_c \"Malonyl-[acyl-carrier protein]\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ACP_c \"acyl carrier protein\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ACP_c \"acyl carrier protein\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_acACP_c \"Acetyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_acACP_c \"Acetyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01587_c \"Acetoacetyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01587_c \"Acetoacetyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01321_c \"(R)-3-Hydroxybutanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01321_c \"(R)-3-Hydroxybutanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01255_c \"But-2-enoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01255_c \"But-2-enoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01588_c \"Butyryl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01588_c \"Butyryl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01589_c \"3-Oxohexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01589_c \"3-Oxohexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01590_c \"D-3-Hydroxyhexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01590_c \"D-3-Hydroxyhexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01591_c \"(2E)-Hexenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01591_c \"(2E)-Hexenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01592_c \"Hexanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01592_c \"Hexanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01593_c \"3-Oxooctanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01593_c \"3-Oxooctanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01323_c \"(R)-3-Hydroxyoctanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01323_c \"(R)-3-Hydroxyoctanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01594_c \"(2E)-Octenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01594_c \"(2E)-Octenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01595_c \"Octanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01595_c \"Octanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01596_c \"3-Oxodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01596_c \"3-Oxodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01322_c \"(R)-3-Hydroxydecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01322_c \"(R)-3-Hydroxydecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01597_c \"(2E)-Decenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01597_c \"(2E)-Decenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01598_c \"Decanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01598_c \"Decanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01599_c \"3-Oxododecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01599_c \"3-Oxododecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01600_c \"D-3-Hydroxydodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01600_c \"D-3-Hydroxydodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01601_c \"(2E)-Dodecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01601_c \"(2E)-Dodecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_ddcaACP_c \"Dodecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_ddcaACP_c \"Dodecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01602_c \"3-Oxotetradecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01602_c \"3-Oxotetradecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01335_c \"(R)-3-Hydroxytetradecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01335_c \"(R)-3-Hydroxytetradecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01603_c \"(2E)-Tetradecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01603_c \"(2E)-Tetradecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_myrsACP_c \"Myristoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_myrsACP_c \"Myristoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01605_c \"3-Oxohexadecanoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01605_c \"3-Oxohexadecanoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01326_c \"(R)-3-Hydroxypalmitoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01326_c \"(R)-3-Hydroxypalmitoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_HC01606_c \"(2E)-Hexadecenoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_HC01606_c \"(2E)-Hexadecenoyl-ACP\">\n",
      "Use of CHARGE in the notes element is discouraged, use fbc:charge instead: <Species M_palmACP_c \"Palmitoyl-ACP\">\n",
      "Use of FORMULA in the notes element is discouraged, use fbc:chemicalFormula instead: <Species M_palmACP_c \"Palmitoyl-ACP\">\n",
      "Adding exchange reaction EX_2hb_e with default bounds for boundary metabolite: 2hb_e.\n",
      "Adding exchange reaction EX_ac_e with default bounds for boundary metabolite: ac_e.\n",
      "Adding exchange reaction EX_acac_e with default bounds for boundary metabolite: acac_e.\n",
      "Adding exchange reaction EX_akg_e with default bounds for boundary metabolite: akg_e.\n",
      "Adding exchange reaction EX_ala_B_e with default bounds for boundary metabolite: ala_B_e.\n",
      "Adding exchange reaction EX_ala_L_e with default bounds for boundary metabolite: ala_L_e.\n",
      "Adding exchange reaction EX_arg_L_e with default bounds for boundary metabolite: arg_L_e.\n",
      "Adding exchange reaction EX_argsuc_e with default bounds for boundary metabolite: argsuc_e.\n",
      "Adding exchange reaction EX_asn_L_e with default bounds for boundary metabolite: asn_L_e.\n",
      "Adding exchange reaction EX_asp_L_e with default bounds for boundary metabolite: asp_L_e.\n",
      "Adding exchange reaction EX_bhb_e with default bounds for boundary metabolite: bhb_e.\n",
      "Adding exchange reaction EX_bilirub_e with default bounds for boundary metabolite: bilirub_e.\n",
      "Adding exchange reaction EX_biomass_e with default bounds for boundary metabolite: biomass_e.\n",
      "Adding exchange reaction EX_but_e with default bounds for boundary metabolite: but_e.\n",
      "Adding exchange reaction EX_chol_e with default bounds for boundary metabolite: chol_e.\n",
      "Adding exchange reaction EX_cit_e with default bounds for boundary metabolite: cit_e.\n",
      "Adding exchange reaction EX_citr_L_e with default bounds for boundary metabolite: citr_L_e.\n",
      "Adding exchange reaction EX_co_e with default bounds for boundary metabolite: co_e.\n",
      "Adding exchange reaction EX_co2_e with default bounds for boundary metabolite: co2_e.\n",
      "Adding exchange reaction EX_creat_e with default bounds for boundary metabolite: creat_e.\n",
      "Adding exchange reaction EX_cyan_e with default bounds for boundary metabolite: cyan_e.\n",
      "Adding exchange reaction EX_cys_L_e with default bounds for boundary metabolite: cys_L_e.\n",
      "Adding exchange reaction EX_etoh_e with default bounds for boundary metabolite: etoh_e.\n",
      "Adding exchange reaction EX_fe2_e with default bounds for boundary metabolite: fe2_e.\n",
      "Adding exchange reaction EX_for_e with default bounds for boundary metabolite: for_e.\n",
      "Adding exchange reaction EX_fum_e with default bounds for boundary metabolite: fum_e.\n",
      "Adding exchange reaction EX_glc_D_e with default bounds for boundary metabolite: glc_D_e.\n",
      "Adding exchange reaction EX_gln_L_e with default bounds for boundary metabolite: gln_L_e.\n",
      "Adding exchange reaction EX_glu_L_e with default bounds for boundary metabolite: glu_L_e.\n",
      "Adding exchange reaction EX_gly_e with default bounds for boundary metabolite: gly_e.\n",
      "Adding exchange reaction EX_glyc_e with default bounds for boundary metabolite: glyc_e.\n",
      "Adding exchange reaction EX_h_e with default bounds for boundary metabolite: h_e.\n",
      "Adding exchange reaction EX_h2o_e with default bounds for boundary metabolite: h2o_e.\n",
      "Adding exchange reaction EX_HC00250_e with default bounds for boundary metabolite: HC00250_e.\n",
      "Adding exchange reaction EX_hco3_e with default bounds for boundary metabolite: hco3_e.\n",
      "Adding exchange reaction EX_hdca_e with default bounds for boundary metabolite: hdca_e.\n",
      "Adding exchange reaction EX_his_L_e with default bounds for boundary metabolite: his_L_e.\n",
      "Adding exchange reaction EX_icit_e with default bounds for boundary metabolite: icit_e.\n",
      "Adding exchange reaction EX_ile_L_e with default bounds for boundary metabolite: ile_L_e.\n",
      "Adding exchange reaction EX_lac_L_e with default bounds for boundary metabolite: lac_L_e.\n",
      "Adding exchange reaction EX_leu_L_e with default bounds for boundary metabolite: leu_L_e.\n",
      "Adding exchange reaction EX_lys_L_e with default bounds for boundary metabolite: lys_L_e.\n",
      "Adding exchange reaction EX_mal_L_e with default bounds for boundary metabolite: mal_L_e.\n",
      "Adding exchange reaction EX_mercplac_e with default bounds for boundary metabolite: mercplac_e.\n",
      "Adding exchange reaction EX_met_L_e with default bounds for boundary metabolite: met_L_e.\n",
      "Adding exchange reaction EX_nad_e with default bounds for boundary metabolite: nad_e.\n",
      "Adding exchange reaction EX_nadh_e with default bounds for boundary metabolite: nadh_e.\n",
      "Adding exchange reaction EX_nh4_e with default bounds for boundary metabolite: nh4_e.\n",
      "Adding exchange reaction EX_no_e with default bounds for boundary metabolite: no_e.\n",
      "Adding exchange reaction EX_o2_e with default bounds for boundary metabolite: o2_e.\n",
      "Adding exchange reaction EX_oaa_e with default bounds for boundary metabolite: oaa_e.\n",
      "Adding exchange reaction EX_pchol_hs_e with default bounds for boundary metabolite: pchol_hs_e.\n",
      "Adding exchange reaction EX_pcreat_e with default bounds for boundary metabolite: pcreat_e.\n",
      "Adding exchange reaction EX_pe_hs_e with default bounds for boundary metabolite: pe_hs_e.\n",
      "Adding exchange reaction EX_phe_L_e with default bounds for boundary metabolite: phe_L_e.\n",
      "Adding exchange reaction EX_pi_e with default bounds for boundary metabolite: pi_e.\n",
      "Adding exchange reaction EX_ppa_e with default bounds for boundary metabolite: ppa_e.\n",
      "Adding exchange reaction EX_pro_L_e with default bounds for boundary metabolite: pro_L_e.\n",
      "Adding exchange reaction EX_ps_hs_e with default bounds for boundary metabolite: ps_hs_e.\n",
      "Adding exchange reaction EX_ser_L_e with default bounds for boundary metabolite: ser_L_e.\n",
      "Adding exchange reaction EX_so3_e with default bounds for boundary metabolite: so3_e.\n",
      "Adding exchange reaction EX_succ_e with default bounds for boundary metabolite: succ_e.\n",
      "Adding exchange reaction EX_tcynt_e with default bounds for boundary metabolite: tcynt_e.\n",
      "Adding exchange reaction EX_thr_L_e with default bounds for boundary metabolite: thr_L_e.\n",
      "Adding exchange reaction EX_trp_L_e with default bounds for boundary metabolite: trp_L_e.\n",
      "Adding exchange reaction EX_tsul_e with default bounds for boundary metabolite: tsul_e.\n",
      "Adding exchange reaction EX_tyr_L_e with default bounds for boundary metabolite: tyr_L_e.\n",
      "Adding exchange reaction EX_urea_e with default bounds for boundary metabolite: urea_e.\n",
      "Adding exchange reaction EX_val_L_e with default bounds for boundary metabolite: val_L_e.\n",
      "Adding exchange reaction EX_fol_e with default bounds for boundary metabolite: fol_e.\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_ATP_MitoCore \"R_OF_ATP_MitoCore Maximum ATP production (ATP hydrolysis)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_HEME_MitoCore \"R_OF_HEME_MitoCore Maximum Heme production\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_LIPID_MitoCore \"R_OF_LIPID_MitoCore Maximum lipid synthesis (Mito Inner Membrane)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OF_PROTEIN_MitoCore \"R_OF_PROTEIN_MitoCore Maxmium production of amino acids for protein synthesis\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYC3Ptm \"R_GLYC3Ptm Glycerol 3-phosphate transport - diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CO2tm \"R_CO2tm CO2 transport (diffusion), mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_H2Otm \"R_H2Otm H2O transport, mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_O2tm \"R_O2tm O2 transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYCtm \"R_GLYCtm glycerol transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYANtm \"R_CYANtm Cyanide transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TCYNTtmB_MitoCore \"R_TCYNTtmB_MitoCore Thiocyanate transport via diffusion (mitochondrial)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CREATtmdiffir \"R_CREATtmdiffir Creatine transport to/from mitochondria\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCREATtmdiffirB_MitoCore \"R_PCREATtmdiffirB_MitoCore Phosphocreatine transport to/from mitochondria \">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0941 \"R_r0941 Bicarbonate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0838B_MitoCore \"R_r0838B_MitoCore Ammonia free diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Biomasst_MitoCore \"R_Biomasst_MitoCore Biomass pseudo transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Use of GENE ASSOCIATION or GENE_ASSOCIATION in the notes element is discouraged, use fbc:gpr instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Biomass_MitoCore \"R_Biomass_MitoCore Biomass Sink\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_O2t \"R_O2t o2 transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CO2t \"R_CO2t CO2 transporter via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HCO3t_MitoCore \"R_HCO3t_MitoCore Bicarbonate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLCt1r \"R_GLCt1r glucose transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HDCAtr \"R_HDCAtr fatty acid transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HDCAtm_MitoCore \"R_HDCAtm_MitoCore fatty acid accumulation - mitochondria\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_L_LACt2r \"R_L_LACt2r L-lactate reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BHBt \"R_BHBt (R)-3-Hydroxybutanoate transport via H+ symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACACt2 \"R_ACACt2 acetoacetate transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ETOHt \"R_ETOHt ethanol reversible transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_BUTt2r \"R_BUTt2r Butyrate transport via proton symport, reversible\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYCt \"R_GLYCt glycerol transport via channel\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0942 \"R_r0942 Creatine storage\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0942b_MitoCore \"R_r0942b_MitoCore Phosphocreatine storage\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_HIStiDF \"R_HIStiDF L-histidine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ILEtec \"R_ILEtec L-isoleucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LEUtec \"R_LEUtec L-leucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_LYStiDF \"R_LYStiDF L-lysine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_METtec \"R_METtec L-methionine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PHEtec \"R_PHEtec L-phenylalanine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2534 \"R_r2534 L-threonine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TRPt \"R_TRPt L-tryptophan transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_VALtec \"R_VALtec L-valine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGtiDF \"R_ARGtiDF L-arginine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ASPte \"R_ASPte L-aspartate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYStec \"R_CYStec L-cysteine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLUt_MitoCore \"R_GLUt_MitoCore L-glutamate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2525 \"R_r2525 L_glutamine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_GLYt2r \"R_GLYt2r glycine reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PROt2r \"R_PROt2r L-proline reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2526 \"R_r2526 L-serine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TYRt \"R_TYRt L-tyrosine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r2532 \"R_r2532 L-asparagine transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ALAt2r \"R_ALAt2r L-alanine reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FUMt_MitoCore \"R_FUMt_MitoCore Fumarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SUMt_MitoCore \"R_SUMt_MitoCore Succinate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0817 \"R_r0817 Citrulline transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NH4t3r \"R_NH4t3r ammonia transport via proton antiport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ACt2r \"R_ACt2r acetate reversible transport via proton symport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PPAt \"R_PPAt Propanoate transport, diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_2HBt2 \"R_2HBt2 2-hydroxybutyrate cotransport with proton\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CHOLtu \"R_CHOLtu Choline uniport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1088 \"R_r1088 Citrate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ICITt_MitoCore \"R_ICITt_MitoCore Isocitrate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_UREAt \"R_UREAt Urea transport via facilitate diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1512 \"R_r1512 Bilirubin transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_ARGSUCt_MitoCore \"R_ARGSUCt_MitoCore Argininosuccinate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MAL_Lte \"R_MAL_Lte Malate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_OAAt_MitoCore \"R_OAAt_MitoCore Oxaloacetate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_AKGt_MitoCore \"R_AKGt_MitoCore 2-Oxoglutarate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_MERCPLACt_MitoCore \"R_MERCPLACt_MitoCore 3-Mercaptolactate transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0899 \"R_r0899 Beta-alanine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FE2t \"R_FE2t iron (II) transport\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_H2Ot \"R_H2Ot H2O transport via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Hct_MitoCore \"R_Hct_MitoCore Proton escape - cytosolic\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_Hmt_MitoCore \"R_Hmt_MitoCore Proton escape - mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_SO3t_MitoCore \"R_SO3t_MitoCore Sulfite\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TSULt_MitoCore \"R_TSULt_MitoCore Thiosulfate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r0940 \"R_r0940 hydrogen sulfide\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_CYANt \"R_CYANt Cyanide transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_TCYNTt \"R_TCYNTt Thiocyanate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_r1423 \"R_r1423 Phosphate facilitated diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FORt_MitoCore \"R_FORt_MitoCore Formate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_FOLt_MitoCore \"R_FOLt_MitoCore Folate\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADHt_MitoCore \"R_NADHt_MitoCore NADH pool\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADt_MitoCore \"R_NADt_MitoCore NAD+ pool\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADHtm_MitoCore \"R_NADHtm_MitoCore NADH pool mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NADtm_MitoCore \"R_NADtm_MitoCore NAD+ pool mitochondrial\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_COt \"R_COt CO transporter via diffusion\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_NOt \"R_NOt NO transport (diffusion)\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PCHOLHSTDe \"R_PCHOLHSTDe Phosphatidylcholine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PSt3 \"R_PSt3 Phosphatidylserine\">\n",
      "Encoding LOWER_BOUND and UPPER_BOUND in KineticLaw is discouraged, use fbc:fluxBounds instead: <Reaction R_PEt \"R_PEt Phosphatidylethanolamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_ATP_MitoCore \"R_OF_ATP_MitoCore Maximum ATP production (ATP hydrolysis)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_HEME_MitoCore \"R_OF_HEME_MitoCore Maximum Heme production\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_LIPID_MitoCore \"R_OF_LIPID_MitoCore Maximum lipid synthesis (Mito Inner Membrane)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OF_PROTEIN_MitoCore \"R_OF_PROTEIN_MitoCore Maxmium production of amino acids for protein synthesis\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HEX1 \"R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G6PPer \"R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGI \"R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PFK \"R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FBP \"R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FBA \"R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TPI \"R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GAPD \"R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGK \"R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGM \"R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ENO \"R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PYK \"R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0122 \"R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEPCK \"R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LDH_L \"R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G6PDH2r \"R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGL \"R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GND \"R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RPI \"R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RPE \"R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TKT1 \"R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TALA \"R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TKT2 \"R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PDHm \"R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CSm \"R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACONTm \"R_ACONTm Citrate -> Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHxm \"R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHyrm \"R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGDm \"R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCOAS1m \"R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCOASm \"R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMm \"R_FUMm Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MDHm \"R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CI_MitoCore \"R_CI_MitoCore NADH dehydrogenase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CII_MitoCore \"R_CII_MitoCore succinate dehydrogenase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CIII_MitoCore \"R_CIII_MitoCore cytochrome c reductase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CIV_MitoCore \"R_CIV_MitoCore cytochrome c oxidase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CV_MitoCore \"R_CV_MitoCore ATP synthase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEPCKm \"R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCm \"R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME2m \"R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME1m \"R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0081 \"R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACITLm_MitoCore \"R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK1m \"R_NDPK1m ATP + GDP --> ADP + GTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NNT_MitoCore \"R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADK1m \"R_ADK1m ATP + AMP --> 2 ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ME2 \"R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALATA_L \"R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK1 \"R_NDPK1 ATP + GDP --> ADP + GTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUM \"R_FUM Fumarate + H2O --> (S)-Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADK1 \"R_ADK1 ATP + AMP --> 2 ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICDHy \"R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACONT \"R_ACONT Citrate -> Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACITL \"R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPTA \"R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MDH \"R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGMALtm \"R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPGLUmB_MitoCore \"R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPTAm \"R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G3PD1 \"R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0205 \"R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FACOAL160i \"R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_C160CPT1 \"R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPA \"R_PPA Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2435 \"R_r2435 carnitine carrier - Carnitine out, acyl carnitine in \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_C160CPT2 \"R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAm \"R_PPAm Pyrophosphate+H2O-->2Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOT2_MitoCore \"R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACADLC16_MitoCore \"R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MECR16C_MitoCore \"R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTPC16_MitoCore \"R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACADLC14_MitoCore \"R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MECR14C_MitoCore \"R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTPC14_MitoCore \"R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1447 \"R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0638 \"R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0660 \"R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0722 \"R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0724 \"R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1451 \"R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0735 \"R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0728 \"R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0726 \"R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0634 \"R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1448 \"R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0633 \"R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0731 \"R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0730 \"R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0732 \"R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1450 \"R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0791 \"R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0734 \"R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0733 \"R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0287 \"R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1446 \"R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH1m \"R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HACD1m \"R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACT1rm \"R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACCOAC \"R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCOATA \"R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOATA \"R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0678 \"R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0691 \"R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0681 \"R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0682 \"R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0760 \"R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0761 \"R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0762 \"R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0763 \"R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0764 \"R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0694 \"R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0695 \"R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0765 \"R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0766 \"R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0692 \"R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0693 \"R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0767 \"R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0768 \"R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0769 \"R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0770 \"R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0712 \"R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0713 \"R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0701 \"R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0702 \"R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0771 \"R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0772 \"R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0696 \"R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0697 \"R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0773 \"R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FA160ACPH \"R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FACOAL40im \"R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BDHm \"R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OCOAT1m \"R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMGCOASim \"R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMGLm \"R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUTAm \"R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD1m \"R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0655 \"R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCCCrm \"R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MGCHrm \"R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILETAm \"R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD3m \"R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0603 \"R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH9m \"R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HACD9m \"R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACT10m \"R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALTAm \"R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OIVD2m \"R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0560 \"R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ECOAH12m \"R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3HBCOAHLm \"R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HIBDm \"R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACCOALm \"R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD1m \"R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPCOACm \"R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMEm \"R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMMm \"R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMCDm \"R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RE2649M \"R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_THRD_L \"R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1155 \"R_r1155 2-oxobutanoate transport into mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1154 \"R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2HBO \"R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METAT \"R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METAT2_MitoCore \"R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AHC \"R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADNK1 \"R_ADNK1 ATP + Adenosine --> ADP + AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTS \"R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTGL \"R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSO \"R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3SALATAi \"R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3SPYRSP \"R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTA \"R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYSTAm \"R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCPST \"R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCPSTm_MitoCore \"R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0595m_MitoCore \"R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0595B_MitoCore \"R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MCLOR \"R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0193 \"R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPO2 \"R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FKYNH \"R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_KYN3OX \"R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HKYNH \"R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_3HAO \"R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCLAD \"R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0645 \"R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMCOXO \"R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMCOXO2_MitoCore \"R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADPTmB_MitoCore \"R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADPTmC_MitoCore \"R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2OXOADOXm \"R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0541 \"R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SACCD3m \"R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0525 \"R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AASAD3m \"R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_R03103_MitoCore \"R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0450 \"R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYSOXc_MitoCore \"R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPD2CSPc_MitoCore \"R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_1PPDCRc_MitoCore \"R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_1PPDCRc_NADPH_MitoCore \"R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LPCOXc_MitoCore \"R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_RE1254C \"R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0594 \"R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2AMADPTmB_MitoCore \"R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2AMADPTmC_MitoCore \"R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROD2mB_MitoCore \"R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G5SADrm \"R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0074 \"R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLU5Km \"R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_G5SDym \"R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_P5CRm \"R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_P5CRxm \"R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNTArm \"R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNDC \"R_ORNDC L-Ornithine --> Putrescine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PTRCOX1 \"R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0464c_MitoCore \"R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ABUTD \"R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGDCm \"R_ARGDCm L-Arginine --> Agmatine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AGMTm \"R_AGMTm Agmatine + H2O --> Putrescine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PTRCAT1m_MitoCore \"R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_APRTO2m_MitoCore \"R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NABTNOm \"R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_4aabutn_MitoCore \"R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDC \"R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_4ABUTtm \"R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ABTArm \"R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0178 \"R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDxm \"R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDym \"R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDxi \"R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUDy \"R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLNS \"R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUNm \"R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUN_MitoCore \"R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGCD \"R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSERT \"R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSP_L \"R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GHMT2r \"R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FOLR2 \"R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_DHFR \"R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFD \"R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFC \"R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTCD \"R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFL \"R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFDH \"R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0060 \"R_r0060 L-Serine --> Pyruvate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GHMT2rm \"R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCam \"R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCbim \"R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GCCcm \"R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0514 \"R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0226 \"R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFDm \"R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFD2m \"R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MTHFCm \"R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFLm \"R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FTHFDHm_MitoCore \"R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYATm \"R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AOBUTDsm \"R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AACTOORm_MitoCore \"R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LGTHLm_MitoCore \"R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYOXm \"R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LDH_Dm_MitoCore \"R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CBPSam \"R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OCBTm \"R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOS1 \"R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOS2 \"R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0129 \"R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMPTASECG \"R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUCYS \"R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHS \"R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0399 \"R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_DHPR \"R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRTA \"R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRTB_MitoCore \"R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_34HPPOR \"R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HGNTOR \"R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MACACI \"R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMAC \"R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASNS1 \"R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0127 \"R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HISD \"R_HISD L-Histidine --> Urocanate + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_URCN \"R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_IZPN \"R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GluForTx \"R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_APAT2rm \"R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD3m \"R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MMSAD3m2_MitoCore \"R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASP1DC \"R_ASP1DC L-Aspartate --> beta-Alanine + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CKc \"R_CKc ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CK \"R_CK ATP + Creatine --> ADP + Phosphocreatine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACOAHi \"R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALCD2yf \"R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALCD2if \"R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACALDtm \"R_ACALDtm Acetaldehyde diffusion into mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALDD2xm \"R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALDD2x \"R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACSm \"R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACS \"R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADSL1 \"R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ADSS \"R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AMPD1 \"R_AMPD1 AMP + H2O --> IMP + Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGN \"R_ARGN L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSL \"R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSS \"R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGNm \"R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALASm \"R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_5AOPtm \"R_5AOPtm 5-Aminolevulinate transport out of mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPBNGS \"R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HMBS \"R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UPP3S \"R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UPPDC1 \"R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CPPPGO \"R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPPGOmB_MitoCore \"R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FCLTm \"R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PHEMEtm \"R_PHEMEtm Heme transport out of mitochondrion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HOXG \"R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BILIRED \"R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BILIRED2_MitoCore \"R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCHOLPm_hs \"R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYK \"R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYC3Ptm \"R_GLYC3Ptm Glycerol 3-phosphate transport - diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GPAMm_hsB_MitoCore \"R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AGPAT1B_MitoCore \"R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CDSm \"R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGPPTm \"R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PGPP_hsm_MitoCore \"R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CLS_hsm_MitoCore \"R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CLPN_MitoCore \"R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYTK1m \"R_CYTK1m ATP + CMP --> ADP + CDP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NDPK3m \"R_NDPK3m ATP + CDP --> ADP + CTP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SPODMm \"R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHP \"R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHPm \"R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHO \"R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GTHOm \"R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CITtamB \"R_CITtamB Citrate Carrier (CIC) - Citrate for Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0913 \"R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CITtbm \"R_CITtbm Citrate Carrier (CIC) - Citrate for PEP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0917 \"R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0917b_MitoCore \"R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PIt2mB_MitoCore \"R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ATPtmB_MitoCore \"R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HtmB_MitoCore \"R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALtm \"R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALSO3tm \"R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALTSULtm \"R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MALSO4tm \"R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCCt2m \"R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0830 \"R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0830B_MitoCore \"R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0829 \"R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUCCt3m_MitoCore \"R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COAtmB_MitoCore \"R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COAtmC_MitoCore \"R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUt2mB_MitoCore \"R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNt4mB_MitoCore \"R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2398B_MitoCore \"R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2402B_MitoCore \"R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYStmB_MitoCore \"R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ORNt3mB_MitoCore \"R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGtmB_MitoCore \"R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1427 \"R_r1427 Basic amino acid carrier (hBAC) - L-histidine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PYRt2m \"R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACt2mB_MitoCore \"R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FE2tm \"R_FE2tm MitoFerrin carrier\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASNtm \"R_ASNtm Unknown amino acid carrier - L-Asparagine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1437 \"R_r1437 Unknown amino acid carrier - L-Cysteine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLNtm \"R_GLNtm Unknown amino acid carrier - L-Glutamine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROtm \"R_PROtm Unknown amino acid carrier - L-Proline\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1078 \"R_r1078 Unknown amino acid carrier - L-Tyrosine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1436 \"R_r1436 Unknown amino acid carrier - L-Methionine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1455 \"R_r1455 Unknown amino acid carrier - L-Phenylalanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPtm_MitoCore \"R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYtm \"R_GLYtm Unknown amino acid carrier - Glycine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILEt5m \"R_ILEt5m Unknown amino acid carrier - L-Isoleucine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUt5m \"R_LEUt5m Unknown amino acid carrier - L-Leucine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALt5m \"R_VALt5m Unknown amino acid carrier - L-Valine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1434 \"R_r1434 Unknown amino acid carrier - L-Alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1435 \"R_r1435 Unknown amino acid carrier - L-Serine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1440 \"R_r1440 Unknown amino acid carrier - L-Threonine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BALAtmr \"R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UREAtm \"R_UREAtm Urea transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMtmB_MitoCore \"R_FUMtmB_MitoCore Fumarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BHBtmB_MitoCore \"R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAtmB_MitoCore \"R_PPAtmB_MitoCore Propanoate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BUTt2mB_MitoCore \"R_BUTt2mB_MitoCore Butyrate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FORt2mB_MitoCore \"R_FORt2mB_MitoCore Formate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0962B_MitoCore \"R_r0962B_MitoCore Folate mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CHLtmB_MitoCore \"R_CHLtmB_MitoCore Choline mitochondrial transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CO2tm \"R_CO2tm CO2 transport (diffusion), mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_H2Otm \"R_H2Otm H2O transport, mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_O2tm \"R_O2tm O2 transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYCtm \"R_GLYCtm glycerol transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYANtm \"R_CYANtm Cyanide transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TCYNTtmB_MitoCore \"R_TCYNTtmB_MitoCore Thiocyanate transport via diffusion (mitochondrial)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CREATtmdiffir \"R_CREATtmdiffir Creatine transport to/from mitochondria\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCREATtmdiffirB_MitoCore \"R_PCREATtmdiffirB_MitoCore Phosphocreatine transport to/from mitochondria \">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0941 \"R_r0941 Bicarbonate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0838B_MitoCore \"R_r0838B_MitoCore Ammonia free diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Biomasst_MitoCore \"R_Biomasst_MitoCore Biomass pseudo transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCFLOPm \"R_PCFLOPm phosphatidylcholine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSFLIPm \"R_PSFLIPm phosphatidylserine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEFLIPm \"R_PEFLIPm phosphatidylethanolamine flippase\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Biomass_MitoCore \"R_Biomass_MitoCore Biomass Sink\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_O2t \"R_O2t o2 transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CO2t \"R_CO2t CO2 transporter via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HCO3t_MitoCore \"R_HCO3t_MitoCore Bicarbonate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLCt1r \"R_GLCt1r glucose transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HDCAtr \"R_HDCAtr fatty acid transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HDCAtm_MitoCore \"R_HDCAtm_MitoCore fatty acid accumulation - mitochondria\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_L_LACt2r \"R_L_LACt2r L-lactate reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BHBt \"R_BHBt (R)-3-Hydroxybutanoate transport via H+ symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACACt2 \"R_ACACt2 acetoacetate transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ETOHt \"R_ETOHt ethanol reversible transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_BUTt2r \"R_BUTt2r Butyrate transport via proton symport, reversible\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYCt \"R_GLYCt glycerol transport via channel\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0942 \"R_r0942 Creatine storage\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0942b_MitoCore \"R_r0942b_MitoCore Phosphocreatine storage\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_HIStiDF \"R_HIStiDF L-histidine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ILEtec \"R_ILEtec L-isoleucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LEUtec \"R_LEUtec L-leucine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_LYStiDF \"R_LYStiDF L-lysine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_METtec \"R_METtec L-methionine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PHEtec \"R_PHEtec L-phenylalanine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2534 \"R_r2534 L-threonine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TRPt \"R_TRPt L-tryptophan transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_VALtec \"R_VALtec L-valine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGtiDF \"R_ARGtiDF L-arginine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ASPte \"R_ASPte L-aspartate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYStec \"R_CYStec L-cysteine transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLUt_MitoCore \"R_GLUt_MitoCore L-glutamate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2525 \"R_r2525 L_glutamine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_GLYt2r \"R_GLYt2r glycine reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PROt2r \"R_PROt2r L-proline reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2526 \"R_r2526 L-serine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TYRt \"R_TYRt L-tyrosine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r2532 \"R_r2532 L-asparagine transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ALAt2r \"R_ALAt2r L-alanine reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FUMt_MitoCore \"R_FUMt_MitoCore Fumarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SUMt_MitoCore \"R_SUMt_MitoCore Succinate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0817 \"R_r0817 Citrulline transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NH4t3r \"R_NH4t3r ammonia transport via proton antiport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ACt2r \"R_ACt2r acetate reversible transport via proton symport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PPAt \"R_PPAt Propanoate transport, diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_2HBt2 \"R_2HBt2 2-hydroxybutyrate cotransport with proton\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CHOLtu \"R_CHOLtu Choline uniport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1088 \"R_r1088 Citrate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ICITt_MitoCore \"R_ICITt_MitoCore Isocitrate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_UREAt \"R_UREAt Urea transport via facilitate diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1512 \"R_r1512 Bilirubin transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_ARGSUCt_MitoCore \"R_ARGSUCt_MitoCore Argininosuccinate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MAL_Lte \"R_MAL_Lte Malate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_OAAt_MitoCore \"R_OAAt_MitoCore Oxaloacetate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_AKGt_MitoCore \"R_AKGt_MitoCore 2-Oxoglutarate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_MERCPLACt_MitoCore \"R_MERCPLACt_MitoCore 3-Mercaptolactate transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0899 \"R_r0899 Beta-alanine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FE2t \"R_FE2t iron (II) transport\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_H2Ot \"R_H2Ot H2O transport via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Hct_MitoCore \"R_Hct_MitoCore Proton escape - cytosolic\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_Hmt_MitoCore \"R_Hmt_MitoCore Proton escape - mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_SO3t_MitoCore \"R_SO3t_MitoCore Sulfite\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TSULt_MitoCore \"R_TSULt_MitoCore Thiosulfate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r0940 \"R_r0940 hydrogen sulfide\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_CYANt \"R_CYANt Cyanide transport via diffusion (extracellular to cytosol)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_TCYNTt \"R_TCYNTt Thiocyanate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_r1423 \"R_r1423 Phosphate facilitated diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FORt_MitoCore \"R_FORt_MitoCore Formate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_FOLt_MitoCore \"R_FOLt_MitoCore Folate\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADHt_MitoCore \"R_NADHt_MitoCore NADH pool\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADt_MitoCore \"R_NADt_MitoCore NAD+ pool\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADHtm_MitoCore \"R_NADHtm_MitoCore NADH pool mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NADtm_MitoCore \"R_NADtm_MitoCore NAD+ pool mitochondrial\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_COt \"R_COt CO transporter via diffusion\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_NOt \"R_NOt NO transport (diffusion)\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PCHOLHSTDe \"R_PCHOLHSTDe Phosphatidylcholine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PSt3 \"R_PSt3 Phosphatidylserine\">\n",
      "Encoding OBJECTIVE_COEFFICIENT in KineticLaw is discouraged, use fbc:fluxObjective instead: <Reaction R_PEt \"R_PEt Phosphatidylethanolamine\">\n"
     ]
    }
   ],
   "source": [
    "mitocore0= cobra.io.read_sbml_model('Mitocore_0.xml')\n",
    "mitocore1= cobra.io.read_sbml_model('Mitocore_Curated_Emanuel.xml')\n",
    "mitocore2= cobra.io.read_sbml_model('Cleaned_Mitocore_changed_metabolites_metanetx.chemical.xml')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "8ca81293",
   "metadata": {},
   "outputs": [],
   "source": [
    "def visualization_model_annotations(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    \n",
    "    mitocore= cobra.io.read_sbml_model(model)\n",
    "    counter=0\n",
    "    for comp in mitocore.__getattribute__(component):\n",
    "        comp_complete= comp.__dict__\n",
    "        annot_dict= comp.__dict__.get( '_annotation')\n",
    "        counter+=1\n",
    "        print(annot_dict)\n",
    "\n",
    "def visualization_model_general(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    \n",
    "    mitocore= cobra.io.read_sbml_model(model)\n",
    "    for comp in mitocore.__getattribute__(component):\n",
    "        comp_complete= comp.__dict__\n",
    "        print(comp_complete)\n",
    "\n",
    "def visualization_model_notes(model, component):\n",
    "    \"\"\"\"\n",
    "    model: COBRApy model object (e.g., mitocore)\n",
    "    component: str, ('reactions', 'metabolites', 'genes')\n",
    "    \"\"\"\n",
    "    \n",
    "    mitocore= cobra.io.read_sbml_model(model)\n",
    "    counter=0\n",
    "    for comp in mitocore.__getattribute__(component):\n",
    "        comp_complete= comp.__dict__\n",
    "        notes_dict= comp.__dict__.get( 'notes')\n",
    "        counter+=1\n",
    "        print(notes_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "b0750719",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'_id': 'EX_2hb_e', 'name': 'EX_2hb_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite 2hb_e at 0x164636a90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ac_e', 'name': 'EX_ac_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ac_e at 0x169dee450>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_acac_e', 'name': 'EX_acac_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite acac_e at 0x28b116f50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_akg_e', 'name': 'EX_akg_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite akg_e at 0x16af677d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ala_B_e', 'name': 'EX_ala_B_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ala_B_e at 0x2a2ef1290>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ala_L_e', 'name': 'EX_ala_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ala_L_e at 0x1795972d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_arg_L_e', 'name': 'EX_arg_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite arg_L_e at 0x17b2591d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_argsuc_e', 'name': 'EX_argsuc_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite argsuc_e at 0x2a298b390>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_asn_L_e', 'name': 'EX_asn_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite asn_L_e at 0x1789aa550>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_asp_L_e', 'name': 'EX_asp_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite asp_L_e at 0x2919310d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_bhb_e', 'name': 'EX_bhb_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite bhb_e at 0x17b519e90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_bilirub_e', 'name': 'EX_bilirub_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite bilirub_e at 0x297bc4250>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_biomass_e', 'name': 'EX_biomass_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite biomass_e at 0x1785f9f90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_but_e', 'name': 'EX_but_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite but_e at 0x2a2a64cd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_chol_e', 'name': 'EX_chol_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite chol_e at 0x2974fafd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_cit_e', 'name': 'EX_cit_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite cit_e at 0x17802eb90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_citr_L_e', 'name': 'EX_citr_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite citr_L_e at 0x17808fa10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_co_e', 'name': 'EX_co_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite co_e at 0x29f362e50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_co2_e', 'name': 'EX_co2_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite co2_e at 0x178134490>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_creat_e', 'name': 'EX_creat_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite creat_e at 0x17b50a8d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_cyan_e', 'name': 'EX_cyan_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite cyan_e at 0x17b24aa10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_cys_L_e', 'name': 'EX_cys_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite cys_L_e at 0x133bc3910>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_etoh_e', 'name': 'EX_etoh_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite etoh_e at 0x2a2b9ac10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_fe2_e', 'name': 'EX_fe2_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite fe2_e at 0x133816390>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_for_e', 'name': 'EX_for_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite for_e at 0x13352c710>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_fum_e', 'name': 'EX_fum_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite fum_e at 0x133596fd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_glc_D_e', 'name': 'EX_glc_D_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite glc_D_e at 0x13354e010>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_gln_L_e', 'name': 'EX_gln_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite gln_L_e at 0x2ae0ac890>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_glu_L_e', 'name': 'EX_glu_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite glu_L_e at 0x29c7ff110>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_gly_e', 'name': 'EX_gly_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite gly_e at 0x29c736110>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_glyc_e', 'name': 'EX_glyc_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite glyc_e at 0x29c734090>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_h_e', 'name': 'EX_h_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_h2o_e', 'name': 'EX_h2o_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite h2o_e at 0x29c7872d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_HC00250_e', 'name': 'EX_HC00250_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite HC00250_e at 0x29c72e290>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_hco3_e', 'name': 'EX_hco3_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite hco3_e at 0x16ca19bd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_hdca_e', 'name': 'EX_hdca_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite hdca_e at 0x29c7d7f10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_his_L_e', 'name': 'EX_his_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite his_L_e at 0x29c7d5610>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_icit_e', 'name': 'EX_icit_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite icit_e at 0x29c7d1450>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ile_L_e', 'name': 'EX_ile_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ile_L_e at 0x29c762e90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_lac_L_e', 'name': 'EX_lac_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite lac_L_e at 0x29d0a8410>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_leu_L_e', 'name': 'EX_leu_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite leu_L_e at 0x29d0cee10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_lys_L_e', 'name': 'EX_lys_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite lys_L_e at 0x29d021d50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_mal_L_e', 'name': 'EX_mal_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite mal_L_e at 0x29d0200d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_mercplac_e', 'name': 'EX_mercplac_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite mercplac_e at 0x29d011f10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_met_L_e', 'name': 'EX_met_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite met_L_e at 0x29d0e7a10>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_nad_e', 'name': 'EX_nad_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite nad_e at 0x29d2e94d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_nadh_e', 'name': 'EX_nadh_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite nadh_e at 0x2a0b01bd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_nh4_e', 'name': 'EX_nh4_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite nh4_e at 0x29d497f90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_no_e', 'name': 'EX_no_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite no_e at 0x2a0565550>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_o2_e', 'name': 'EX_o2_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite o2_e at 0x29d4fa190>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_oaa_e', 'name': 'EX_oaa_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite oaa_e at 0x29e285f50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_pchol_hs_e', 'name': 'EX_pchol_hs_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite pchol_hs_e at 0x17b24d790>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_pcreat_e', 'name': 'EX_pcreat_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite pcreat_e at 0x29e24fb90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_pe_hs_e', 'name': 'EX_pe_hs_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite pe_hs_e at 0x2975456d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_phe_L_e', 'name': 'EX_phe_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite phe_L_e at 0x29e616790>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_pi_e', 'name': 'EX_pi_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite pi_e at 0x29e6c4fd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ppa_e', 'name': 'EX_ppa_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ppa_e at 0x29e605590>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_pro_L_e', 'name': 'EX_pro_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite pro_L_e at 0x29d22d590>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ps_hs_e', 'name': 'EX_ps_hs_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ps_hs_e at 0x29d28ddd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_ser_L_e', 'name': 'EX_ser_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite ser_L_e at 0x29d27d250>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_so3_e', 'name': 'EX_so3_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite so3_e at 0x29d7acb50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_succ_e', 'name': 'EX_succ_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite succ_e at 0x29d7dc7d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_tcynt_e', 'name': 'EX_tcynt_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite tcynt_e at 0x29d77ef50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_thr_L_e', 'name': 'EX_thr_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite thr_L_e at 0x29d728cd0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_trp_L_e', 'name': 'EX_trp_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite trp_L_e at 0x29d7a63d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_tsul_e', 'name': 'EX_tsul_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite tsul_e at 0x29d746c50>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_tyr_L_e', 'name': 'EX_tyr_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite tyr_L_e at 0x29d771110>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_urea_e', 'name': 'EX_urea_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite urea_e at 0x29d761490>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_val_L_e', 'name': 'EX_val_L_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite val_L_e at 0x29d751a90>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'EX_fol_e', 'name': 'EX_fol_e', 'notes': {}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': '', '_genes': set(), '_metabolites': {<Metabolite fol_e at 0x29d0067d0>: -1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OF_ATP_MitoCore', 'name': 'R_OF_ATP_MitoCore Maximum ATP production (ATP hydrolysis)', 'notes': {'Model Reaction Number': '1', 'SUBSYSTEM': 'Objective Function - ATP', 'Description': 'Maximum ATP production (ATP hydrolysis)', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Objective Function - ATP', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite biomass_c at 0x1696f8b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OF_HEME_MitoCore', 'name': 'R_OF_HEME_MitoCore Maximum Heme production', 'notes': {'Model Reaction Number': '2', 'SUBSYSTEM': 'Objective Function - Heme', 'Description': 'Maximum Heme production', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Objective Function - Heme', '_genes': set(), '_metabolites': {<Metabolite pheme_m at 0x29e63ee10>: -1.0, <Metabolite biomass_m at 0x169ca8b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OF_LIPID_MitoCore', 'name': 'R_OF_LIPID_MitoCore Maximum lipid synthesis (Mito Inner Membrane)', 'notes': {'Model Reaction Number': '3', 'SUBSYSTEM': 'Objective Function - Lipid synthesis', 'Description': 'Maximum lipid synthesis (Mito Inner Membrane)', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Objective Function - Lipid synthesis', '_genes': set(), '_metabolites': {<Metabolite pchol_hs_m at 0x29e23c350>: -0.4, <Metabolite pe_hs_m at 0x29e2bf250>: -0.34, <Metabolite ps_hs_m at 0x29d28e350>: -0.3, <Metabolite clpn_hs_m at 0x16a60aed0>: -0.18, <Metabolite biomass_m at 0x169ca8b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OF_PROTEIN_MitoCore', 'name': 'R_OF_PROTEIN_MitoCore Maxmium production of amino acids for protein synthesis', 'notes': {'Model Reaction Number': '4', 'SUBSYSTEM': 'Objective Function - Protein Synthesis', 'Description': 'Maxmium production of amino acids for protein synthesis', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Objective Function - Protein Synthesis', '_genes': set(), '_metabolites': {<Metabolite his_L_m at 0x2ae3edd50>: -0.19, <Metabolite ile_L_m at 0x2ae04e010>: -0.43, <Metabolite leu_L_m at 0x29d0ce590>: -0.82, <Metabolite lys_L_m at 0x29d021590>: -0.89, <Metabolite met_L_m at 0x29d0e7bd0>: -0.23, <Metabolite phe_L_m at 0x29e60a810>: -0.39, <Metabolite thr_L_m at 0x29d72aad0>: -0.47, <Metabolite trp_L_m at 0x29d745a10>: -0.02, <Metabolite val_L_m at 0x29d753d50>: -0.53, <Metabolite arg_L_m at 0x2ae4a81d0>: -0.54, <Metabolite asp_L_m at 0x1799e7e10>: -0.53, <Metabolite cys_L_m at 0x13356df50>: -0.07, <Metabolite glu_L_m at 0x29c7fe9d0>: -0.58, <Metabolite gln_L_m at 0x133551a10>: -0.49, <Metabolite gly_m at 0x29c735650>: -0.81, <Metabolite pro_L_m at 0x1696f8b10>: -0.62, <Metabolite ser_L_m at 0x29d27c990>: -0.59, <Metabolite tyr_L_m at 0x29d7726d0>: -0.24, <Metabolite asn_L_m at 0x1789aa950>: -0.42, <Metabolite ala_L_m at 0x2a2a29310>: -0.76, <Metabolite biomass_m at 0x169ca8b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HEX1', 'name': 'R_HEX1 ATP + alpha-D-Glucose --> ADP + alpha-D-Glucose 6-phosphate', 'notes': {'Model Reaction Number': '5', 'EC Number': '2.7.1.1 or 2.7.1.2', 'SUBSYSTEM': 'Glycolysis', 'Description': 'ATP + alpha-D-Glucose --&gt; ADP + alpha-D-Glucose 6-phosphate', 'Recon2 id': 'HEX1', 'Recon2 formula': 'atp[c] + glc_D[c] -&gt; h[c] + adp[c] + g6p[c]', 'KEGG id': 'R01786', 'GENE_ASSOCIATION': 'ENSG00000156515 or ENSG00000159399 or ENSG00000160883 or ENSG00000106633', 'GENE_LIST': 'HK1 or HK2 or HK3 or GCK', 'HGNC': '4922 or HGNC:4923 or HGNC:4925 or HGNC:4195', 'Gene name': 'Hexokinases', 'MitoCarta': 'TRUE, TRUE, FALSE, FALSE', 'MitoCarta score': '6.3802, 10.7609, -1.9426, -4.3576', 'HPA mRNA level in heart tissue': 'High, Low, Low, Low', 'HPA protein level in heart tissue': 'Medium, Medium, Low, Low'}, '_annotation': {'ec-code': ['2.7.1.1', '2.7.1.2'], 'kegg.reaction': 'R01786'}, '_gpr': cobra.core.gene.GPR('ENSG00000156515 or ENSG00000159399 or ENSG00000160883 or ENSG00000106633'), 'subsystem': 'Glycolysis', '_genes': {<Gene ENSG00000160883 at 0x28abb1590>, <Gene ENSG00000106633 at 0x28abb1810>, <Gene ENSG00000159399 at 0x28abb00d0>, <Gene ENSG00000156515 at 0x28abb1310>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite glc_D_c at 0x2ae06c410>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite g6p_c at 0x13355e850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'G6PPer', 'name': 'R_G6PPer alpha-D-Glucose 6-phosphate + H2O --> alpha-D-Glucose + Orthophosphate', 'notes': {'Model Reaction Number': '6', 'EC Number': '3.1.3.9', 'SUBSYSTEM': 'Gluconeogenesis', 'Description': 'alpha-D-Glucose 6-phosphate + H2O --&gt; alpha-D-Glucose + Orthophosphate', 'Recon2 id': 'G6PPer', 'Recon2 formula': 'h2o[r] + g6p[r] -&gt; pi[r] + glc_D[r]', 'KEGG id': 'R01788', 'GENE_ASSOCIATION': 'ENSG00000131482', 'GENE_LIST': 'G6PC', 'HGNC': '4056', 'Gene name': 'Glucose 6-phosphatase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-4.8268', 'Directionality rationale': 'No gluconeogenesis in heart', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '3.1.3.9', 'kegg.reaction': 'R01788'}, '_gpr': cobra.core.gene.GPR('ENSG00000131482'), 'subsystem': 'Gluconeogenesis', '_genes': {<Gene ENSG00000131482 at 0x28abb1b10>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite g6p_c at 0x13355e850>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite glc_D_c at 0x2ae06c410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'PGI', 'name': 'R_PGI alpha-D-Glucose 6-phosphate --> beta-D-Fructose 6-phosphate', 'notes': {'Model Reaction Number': '7', 'EC Number': '5.3.1.9', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'alpha-D-Glucose 6-phosphate --&gt; beta-D-Fructose 6-phosphate', 'Recon2 id': 'PGI', 'Recon2 formula': 'g6p[c] --&gt; f6p[c]', 'KEGG id': 'R02740', 'GENE_ASSOCIATION': 'ENSG00000105220', 'GENE_LIST': 'GPI', 'HGNC': '4458', 'Gene name': 'glucose-6-phosphate isomerase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.5459', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '5.3.1.9', 'kegg.reaction': 'R02740'}, '_gpr': cobra.core.gene.GPR('ENSG00000105220'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000105220 at 0x28abb20d0>}, '_metabolites': {<Metabolite g6p_c at 0x13355e850>: -1.0, <Metabolite f6p_c at 0x1337cf8d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PFK', 'name': 'R_PFK ATP + beta-D-Fructose 6-phosphate --> ADP + beta-D-Fructose 1,6-bisphosphate', 'notes': {'Model Reaction Number': '8', 'EC Number': '2.7.1.11', 'SUBSYSTEM': 'Glycolysis', 'Description': 'ATP + beta-D-Fructose 6-phosphate --&gt; ADP + beta-D-Fructose 1,6-bisphosphate', 'Recon2 id': 'PFK', 'Recon2 formula': 'atp[c] + f6p[c] -&gt; h[c] + adp[c] + fdp[c]', 'KEGG id': 'R04779', 'GENE_ASSOCIATION': 'ENSG00000152556', 'GENE_LIST': 'PFKM', 'HGNC': '8877', 'Gene name': 'phosphofructokinase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.2497', 'Directionality rationale': 'ATP consumption, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.7.1.11', 'kegg.reaction': 'R04779'}, '_gpr': cobra.core.gene.GPR('ENSG00000152556'), 'subsystem': 'Glycolysis', '_genes': {<Gene ENSG00000152556 at 0x28abb21d0>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite f6p_c at 0x1337cf8d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite fdp_c at 0x2a2792bd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FBP', 'name': 'R_FBP beta-D-Fructose 1,6-bisphosphate + H2O --> beta-D-Fructose 6-phosphate + Orthophosphate', 'notes': {'Model Reaction Number': '9', 'EC Number': '3.1.3.11', 'SUBSYSTEM': 'Gluconeogenesis', 'Description': 'beta-D-Fructose 1,6-bisphosphate + H2O --&gt; beta-D-Fructose 6-phosphate + Orthophosphate', 'Recon2 id': 'FBP', 'Recon2 formula': 'h2o[c] + fdp[c] -&gt; pi[c] + f6p[c]', 'KEGG id': 'R04780', 'GENE_ASSOCIATION': 'ENSG00000165140 or ENSG00000130957', 'GENE_LIST': 'FBP1 or FBP2', 'HGNC': '3606 or HGNC:3607', 'Gene name': 'fructose-bisphosphatase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.5494, -8.2811', 'Directionality rationale': 'No gluconeogenesis in heart', 'HPA mRNA level in heart tissue': 'Low or Not detected', 'HPA protein level in heart tissue': 'Not detected or Not detected'}, '_annotation': {'ec-code': '3.1.3.11', 'kegg.reaction': 'R04780'}, '_gpr': cobra.core.gene.GPR('ENSG00000165140 or ENSG00000130957'), 'subsystem': 'Gluconeogenesis', '_genes': {<Gene ENSG00000130957 at 0x28abb2510>, <Gene ENSG00000165140 at 0x28abb2850>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite fdp_c at 0x2a2792bd0>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite f6p_c at 0x1337cf8d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'FBA', 'name': 'R_FBA beta-D-Fructose 1,6-bisphosphate --> Glycerone phosphate + D-Glyceraldehyde 3-phosphate', 'notes': {'Model Reaction Number': '10', 'EC Number': '4.1.2.13', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'beta-D-Fructose 1,6-bisphosphate --&gt; Glycerone phosphate + D-Glyceraldehyde 3-phosphate', 'Recon2 id': 'FBA', 'Recon2 formula': 'fdp[c] --&gt; dhap[c] + g3p[c]', 'KEGG id': 'R01070', 'GENE_ASSOCIATION': 'ENSG00000149925 or ENSG00000109107', 'GENE_LIST': 'ALDOA or ALDOC', 'HGNC': '414 or HGNC:418', 'Gene name': 'fructose-bisphosphate aldolase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-4.8741', 'HPA mRNA level in heart tissue': 'High or High', 'HPA protein level in heart tissue': 'Not detected or Not detected'}, '_annotation': {'ec-code': '4.1.2.13', 'kegg.reaction': 'R01070'}, '_gpr': cobra.core.gene.GPR('ENSG00000149925 or ENSG00000109107'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000109107 at 0x28abb2e90>, <Gene ENSG00000149925 at 0x28abb2bd0>}, '_metabolites': {<Metabolite fdp_c at 0x2a2792bd0>: -1.0, <Metabolite dhap_c at 0x29c712310>: 1.0, <Metabolite g3p_c at 0x1335406d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TPI', 'name': 'R_TPI D-Glyceraldehyde 3-phosphate --> Glycerone phosphate', 'notes': {'Model Reaction Number': '11', 'EC Number': '5.3.1.1', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'D-Glyceraldehyde 3-phosphate --&gt; Glycerone phosphate', 'Recon2 id': 'TPI', 'Recon2 formula': 'dhap[c] --&gt; g3p[c]', 'KEGG id': 'R01015', 'GENE_ASSOCIATION': 'ENSG00000111669', 'GENE_LIST': 'TPI1', 'HGNC': '12009', 'Gene name': 'triose-phosphate isomerase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.5357', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '5.3.1.1', 'kegg.reaction': 'R01015'}, '_gpr': cobra.core.gene.GPR('ENSG00000111669'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000111669 at 0x28abb31d0>}, '_metabolites': {<Metabolite dhap_c at 0x29c712310>: -1.0, <Metabolite g3p_c at 0x1335406d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GAPD', 'name': 'R_GAPD D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --> 3-Phospho-D-glyceroyl phosphate + NADH + H+', 'notes': {'Model Reaction Number': '12', 'EC Number': '1.2.1.12', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'D-Glyceraldehyde 3-phosphate + Orthophosphate + NAD+ --&gt; 3-Phospho-D-glyceroyl phosphate + NADH + H+', 'Recon2 id': 'GAPD', 'Recon2 formula': 'pi[c] + nad[c] + g3p[c] --&gt; h[c] + nadh[c] + 13dpg[c]', 'KEGG id': 'R01061', 'GENE_ASSOCIATION': 'ENSG00000111640', 'GENE_LIST': 'GAPDH', 'HGNC': '4141', 'Gene name': 'glyceraldehyde-3-phosphate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.4139', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.2.1.12', 'kegg.reaction': 'R01061'}, '_gpr': cobra.core.gene.GPR('ENSG00000111640'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000111640 at 0x28abb3510>}, '_metabolites': {<Metabolite pi_c at 0x29e6c47d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite g3p_c at 0x1335406d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite 13dpg_c at 0x2a2988310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGK', 'name': 'R_PGK ATP + 3-Phospho-D-glycerate --> ADP + 3-Phospho-D-glyceroyl phosphate', 'notes': {'Model Reaction Number': '13', 'EC Number': '2.7.2.3', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'ATP + 3-Phospho-D-glycerate --&gt; ADP + 3-Phospho-D-glyceroyl phosphate', 'Recon2 id': 'PGK', 'Recon2 formula': 'atp[c] + 3pg[c] --&gt; adp[c] + 13dpg[c]', 'KEGG id': 'R01512', 'GENE_ASSOCIATION': 'ENSG00000102144', 'GENE_LIST': 'PGK1', 'HGNC': '8896', 'Gene name': 'Phosphoglycerate kinase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-0.9741', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.7.2.3', 'kegg.reaction': 'R01512'}, '_gpr': cobra.core.gene.GPR('ENSG00000102144'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000102144 at 0x28abb3850>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite 3pg_c at 0x2ae3ee710>: -1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite 13dpg_c at 0x2a2988310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGM', 'name': 'R_PGM 2-Phospho-D-glycerate --> 3-Phospho-D-glycerate', 'notes': {'Model Reaction Number': '14', 'EC Number': '5.4.2.1', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': '2-Phospho-D-glycerate --&gt; 3-Phospho-D-glycerate', 'Recon2 id': 'DPGM', 'Recon2 formula': '2pg[c] --&gt; 3pg[c]', 'KEGG id': 'R01518', 'GENE_ASSOCIATION': 'ENSG00000171314 or ENSG00000164708', 'GENE_LIST': 'PGAM1 or PGAM2', 'HGNC': '8888 or HGNC:8889', 'Gene name': 'Phosphoglycerate mutase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-0.9066', 'HPA mRNA level in heart tissue': 'High or High', 'HPA protein level in heart tissue': 'ND or High'}, '_annotation': {'ec-code': '5.4.2.1', 'kegg.reaction': 'R01518'}, '_gpr': cobra.core.gene.GPR('ENSG00000171314 or ENSG00000164708'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000171314 at 0x28abb3b90>, <Gene ENSG00000164708 at 0x28abb3ed0>}, '_metabolites': {<Metabolite 2pg_c at 0x17b46e950>: -1.0, <Metabolite 3pg_c at 0x2ae3ee710>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ENO', 'name': 'R_ENO 2-Phospho-D-glycerate --> Phosphoenolpyruvate + H2O', 'notes': {'Model Reaction Number': '15', 'EC Number': '4.2.1.11', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': '2-Phospho-D-glycerate --&gt; Phosphoenolpyruvate + H2O', 'Recon2 id': 'ENO', 'Recon2 formula': '2pg[c] --&gt; h2o[c] + pep[c]', 'KEGG id': 'R00658', 'GENE_ASSOCIATION': 'ENSG00000074800 or ENSG00000111674 or ENSG00000108515', 'GENE_LIST': 'ENO1 or ENO2 or ENO3', 'HGNC': '3350 or HGNC:3353 or HGNC:3354', 'Gene name': 'enolase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-8.824, -5.5041, -4.8191', 'HPA mRNA level in heart tissue': 'High, Medium, High', 'HPA protein level in heart tissue': 'Low, ND, Low'}, '_annotation': {'ec-code': '4.2.1.11', 'kegg.reaction': 'R00658'}, '_gpr': cobra.core.gene.GPR('ENSG00000074800 or ENSG00000111674 or ENSG00000108515'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000074800 at 0x28abc8790>, <Gene ENSG00000111674 at 0x28abcae10>, <Gene ENSG00000108515 at 0x28abc8d90>}, '_metabolites': {<Metabolite 2pg_c at 0x17b46e950>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite pep_c at 0x29e21af10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PYK', 'name': 'R_PYK ADP + Phosphoenolpyruvate --> ATP + Pyruvate', 'notes': {'Model Reaction Number': '16', 'EC Number': '2.7.1.40', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': 'ADP + Phosphoenolpyruvate --&gt; ATP + Pyruvate', 'Recon2 id': 'PYK', 'Recon2 formula': 'h[c] + adp[c] + pep[c] -&gt; atp[c] + pyr[c]', 'KEGG id': 'R00200', 'GENE_ASSOCIATION': 'ENSG00000067225', 'GENE_LIST': 'PKM', 'HGNC': '9021', 'Gene name': 'pyruvate kinase', 'MitoCarta': 'FALSE', 'MitoCarta score': '4.2524', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.7.1.40', 'kegg.reaction': 'R00200'}, '_gpr': cobra.core.gene.GPR('ENSG00000067225'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000067225 at 0x28abcac50>}, '_metabolites': {<Metabolite pep_c at 0x29e21af10>: -1.0, <Metabolite adp_c at 0x17918bb10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite atp_c at 0x2ae181f10>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0122', 'name': 'R_r0122  GDP + Phosphoenolpyruvate --> GTP + Pyruvate', 'notes': {'Model Reaction Number': '17', 'EC Number': '2.7.1.40', 'SUBSYSTEM': 'Glycolysis', 'Description': 'GDP + Phosphoenolpyruvate --&gt; GTP + Pyruvate', 'Recon2 id': 'r0122', 'Recon2 formula': 'h[c] + pep[c] + gdp[c] -&gt; pyr[c] + gtp[c]', 'KEGG id': 'R00430', 'GENE_ASSOCIATION': 'ENSG00000067225', 'GENE_LIST': 'PKM', 'HGNC': '9021', 'Gene name': 'pyruvate kinase', 'MitoCarta': 'FALSE', 'MitoCarta score': '4.2524', 'Directionality rationale': 'Modelling to match ATP version', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.7.1.40', 'kegg.reaction': 'R00430'}, '_gpr': cobra.core.gene.GPR('ENSG00000067225'), 'subsystem': 'Glycolysis', '_genes': {<Gene ENSG00000067225 at 0x28abcac50>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite pep_c at 0x29e21af10>: -1.0, <Metabolite gdp_c at 0x1335da410>: -1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite gtp_c at 0x29c7e6250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PEPCK', 'name': 'R_PEPCK GTP + oxaloacetate --> GDP + phosphoenolpyruvate + CO2', 'notes': {'Model Reaction Number': '18', 'EC Number': '4.1.1.32', 'SUBSYSTEM': 'Gluconeogenesis', 'Description': 'GTP + oxaloacetate --&gt; GDP + phosphoenolpyruvate + CO2', 'Recon2 id': 'PEPCK', 'Recon2 formula': 'oaa[c] + gtp[c] -&gt; co2[c] + pep[c] + gdp[c]', 'KEGG id': 'R00431', 'GENE_ASSOCIATION': 'ENSG00000124253', 'GENE_LIST': 'PCK1', 'HGNC': '8724', 'Gene name': 'phosphoenolpyruvate carboxykinase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.849', 'Directionality rationale': 'GTP consumption, CO2 production, Brenda, KEGG, PMID: 19636077', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '4.1.1.32', 'kegg.reaction': 'R00431'}, '_gpr': cobra.core.gene.GPR('ENSG00000124253'), 'subsystem': 'Gluconeogenesis', '_genes': {<Gene ENSG00000124253 at 0x28abcab10>}, '_metabolites': {<Metabolite oaa_c at 0x29f50d490>: -1.0, <Metabolite gtp_c at 0x29c7e6250>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite pep_c at 0x29e21af10>: 1.0, <Metabolite gdp_c at 0x1335da410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LDH_L', 'name': 'R_LDH_L (S)-Lactate + NAD+ --> Pyruvate + NADH + H+', 'notes': {'Model Reaction Number': '19', 'EC Number': '1.1.1.27', 'SUBSYSTEM': 'Glycolysis/gluconeogenesis', 'Description': '(S)-Lactate + NAD+ --&gt; Pyruvate + NADH + H+', 'Recon2 id': 'LDH_L', 'Recon2 formula': 'nad[c] + lac_L[c] --&gt; h[c] + nadh[c] + pyr[c]', 'KEGG id': 'R00703', 'GENE_ASSOCIATION': 'ENSG00000111716', 'GENE_LIST': 'LDHB', 'HGNC': '6541', 'Gene name': 'L-lactate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.2343', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.27', 'kegg.reaction': 'R00703'}, '_gpr': cobra.core.gene.GPR('ENSG00000111716'), 'subsystem': 'Glycolysis/gluconeogenesis', '_genes': {<Gene ENSG00000111716 at 0x28abc9990>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite lac_L_c at 0x17aeddf10>: -1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'G6PDH2r', 'name': 'R_G6PDH2r beta-D-Glucose 6-phosphate + NADP+ --> D-Glucono-1,5-lactone 6-phosphate + NADPH + H+', 'notes': {'Model Reaction Number': '20', 'EC Number': '1.1.1.49', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'beta-D-Glucose 6-phosphate + NADP+ --&gt; D-Glucono-1,5-lactone 6-phosphate + NADPH + H+', 'Recon2 id': 'G6PDH2r', 'Recon2 formula': 'nadp[c] + g6p[c] --&gt; h[c] + nadph[c] + 6pgl[c]', 'KEGG id': 'R02736', 'GENE_ASSOCIATION': 'ENSG00000160211', 'GENE_LIST': 'G6PD', 'HGNC': '4057', 'Gene name': 'glucose 6-phosphate dehydrogenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.2802', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.1.1.49', 'kegg.reaction': 'R02736'}, '_gpr': cobra.core.gene.GPR('ENSG00000160211'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000160211 at 0x28abcb5d0>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite g6p_c at 0x13355e850>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite 6pgl_c at 0x28b1dd2d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGL', 'name': 'R_PGL D-Glucono-1,5-lactone 6-phosphate + H2O --> 6-Phospho-D-gluconate', 'notes': {'Model Reaction Number': '21', 'EC Number': '3.1.1.31', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'D-Glucono-1,5-lactone 6-phosphate + H2O --&gt; 6-Phospho-D-gluconate', 'Recon2 id': 'PGL', 'Recon2 formula': 'h2o[c] + 6pgl[c] -&gt; h[c] + 6pgc[c]', 'KEGG id': 'R02035', 'GENE_ASSOCIATION': 'ENSG00000130313', 'GENE_LIST': 'PGLS', 'HGNC': '8903', 'Gene name': '6-phosphogluconolactonase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.2362', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 5.48kJ/mol', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '3.1.1.31', 'kegg.reaction': 'R02035'}, '_gpr': cobra.core.gene.GPR('ENSG00000130313'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000130313 at 0x28abc8f10>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite 6pgl_c at 0x28b1dd2d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite 6pgc_c at 0x28b1df350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GND', 'name': 'R_GND 6-Phospho-D-gluconate + NADP+ --> D-Ribulose 5-phosphate + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '22', 'EC Number': '1.1.1.44', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': '6-Phospho-D-gluconate + NADP+ --&gt; D-Ribulose 5-phosphate + CO2 + NADPH + H+', 'Recon2 id': 'GND', 'Recon2 formula': 'nadp[c] + 6pgc[c] -&gt; nadph[c] + co2[c] + ru5p_D[c]', 'KEGG id': 'R01528', 'GENE_ASSOCIATION': 'ENSG00000142657', 'GENE_LIST': 'PGD', 'HGNC': '8891', 'Gene name': 'phosphogluconate dehydrogenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.4637', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.1.1.44', 'kegg.reaction': 'R01528'}, '_gpr': cobra.core.gene.GPR('ENSG00000142657'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000142657 at 0x28abca9d0>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite 6pgc_c at 0x28b1df350>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ru5p_D_c at 0x29d2b3010>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'RPI', 'name': 'R_RPI D-Ribose 5-phosphate --> D-Ribulose 5-phosphate', 'notes': {'Model Reaction Number': '23', 'EC Number': '5.3.1.6', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'D-Ribose 5-phosphate --&gt; D-Ribulose 5-phosphate', 'Recon2 id': 'RPI', 'Recon2 formula': 'r5p[c] --&gt; ru5p_D[c]', 'KEGG id': 'R01056', 'GENE_ASSOCIATION': 'ENSG00000153574', 'GENE_LIST': 'RPIA', 'HGNC': '10297', 'Gene name': 'ribose-5-phosphate isomerase', 'MitoCarta': 'TRUE', 'MitoCarta score': '-5.8466', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '5.3.1.6', 'kegg.reaction': 'R01056'}, '_gpr': cobra.core.gene.GPR('ENSG00000153574'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000153574 at 0x28abcba10>}, '_metabolites': {<Metabolite r5p_c at 0x29d2b03d0>: -1.0, <Metabolite ru5p_D_c at 0x29d2b3010>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'RPE', 'name': 'R_RPE D-Ribulose 5-phosphate --> D-Xylulose 5-phosphate', 'notes': {'Model Reaction Number': '24', 'EC Number': '5.1.3.1', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'D-Ribulose 5-phosphate --&gt; D-Xylulose 5-phosphate', 'Recon2 id': 'RPE', 'Recon2 formula': 'ru5p_D[c] --&gt; xu5p_D[c]', 'KEGG id': 'R01529', 'GENE_ASSOCIATION': 'ENSG00000235376', 'GENE_LIST': 'RPEL1', 'HGNC': '45241', 'Gene name': 'ribulose 5-phosphate 3-epimerase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-13.4554', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '5.1.3.1', 'kegg.reaction': 'R01529'}, '_gpr': cobra.core.gene.GPR('ENSG00000235376'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000235376 at 0x28abcbd50>}, '_metabolites': {<Metabolite ru5p_D_c at 0x29d2b3010>: -1.0, <Metabolite xu5p_D_c at 0x29d070f90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TKT1', 'name': 'R_TKT1 D-Ribose 5-phosphate + D-Xylulose 5-phosphate --> Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate', 'notes': {'Model Reaction Number': '25', 'EC Number': '2.2.1.1', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'D-Ribose 5-phosphate + D-Xylulose 5-phosphate --&gt; Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate', 'Recon2 id': 'TKT1', 'Recon2 formula': 'r5p[c] + xu5p_D[c] --&gt; g3p[c] + s7p[c]', 'KEGG id': 'R01641', 'GENE_ASSOCIATION': 'ENSG00000163931', 'GENE_LIST': 'TKT', 'HGNC': '11834', 'Gene name': 'transketolase', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.1077', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.2.1.1', 'kegg.reaction': 'R01641'}, '_gpr': cobra.core.gene.GPR('ENSG00000163931'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000163931 at 0x28abcaa10>}, '_metabolites': {<Metabolite r5p_c at 0x29d2b03d0>: -1.0, <Metabolite xu5p_D_c at 0x29d070f90>: -1.0, <Metabolite g3p_c at 0x1335406d0>: 1.0, <Metabolite s7p_c at 0x29d204090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TALA', 'name': 'R_TALA Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --> D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate', 'notes': {'Model Reaction Number': '26', 'EC Number': '2.2.1.2', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'Sedoheptulose 7-phosphate + D-Glyceraldehyde 3-phosphate --&gt; D-Erythrose 4-phosphate + beta-D-Fructose 6-phosphate', 'Recon2 id': 'TALA', 'Recon2 formula': 'g3p[c] + s7p[c] --&gt; f6p[c] + e4p[c]', 'KEGG id': 'R01827', 'GENE_ASSOCIATION': 'ENSG00000177156', 'GENE_LIST': 'TALDO1', 'HGNC': '11559', 'Gene name': 'transaldolase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.3955', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.2.1.2', 'kegg.reaction': 'R01827'}, '_gpr': cobra.core.gene.GPR('ENSG00000177156'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000177156 at 0x28abcbb10>}, '_metabolites': {<Metabolite g3p_c at 0x1335406d0>: -1.0, <Metabolite s7p_c at 0x29d204090>: -1.0, <Metabolite f6p_c at 0x1337cf8d0>: 1.0, <Metabolite e4p_c at 0x1337adc10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TKT2', 'name': 'R_TKT2 D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --> beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate', 'notes': {'Model Reaction Number': '27', 'EC Number': '2.2.1.1', 'SUBSYSTEM': 'Pentose phosphate pathway', 'Description': 'D-Erythrose 4-phosphate + D-Xylulose 5-phosphate --&gt; beta-D-Fructose 6-phosphate + D-Glyceraldehyde 3-phosphate', 'Recon2 id': 'TKT2', 'Recon2 formula': 'xu5p_D[c] + e4p[c] --&gt; g3p[c] + f6p[c]', 'KEGG id': 'R01830', 'GENE_ASSOCIATION': 'ENSG00000163931', 'GENE_LIST': 'TKT', 'HGNC': '11834', 'Gene name': 'transketolase', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.1077', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.2.1.1', 'kegg.reaction': 'R01830'}, '_gpr': cobra.core.gene.GPR('ENSG00000163931'), 'subsystem': 'Pentose phosphate pathway', '_genes': {<Gene ENSG00000163931 at 0x28abcaa10>}, '_metabolites': {<Metabolite xu5p_D_c at 0x29d070f90>: -1.0, <Metabolite e4p_c at 0x1337adc10>: -1.0, <Metabolite g3p_c at 0x1335406d0>: 1.0, <Metabolite f6p_c at 0x1337cf8d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PDHm', 'name': 'R_PDHm Pyruvate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+', 'notes': {'Model Reaction Number': '28', 'EC Number': '1.2.4.1, 2.3.1.12, 1.8.1.4', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Pyruvate + CoA + NAD+ --&gt; Acetyl-CoA + CO2 + NADH + H+', 'Recon2 id': 'PDHm', 'Recon2 formula': 'nad[m] + pyr[m] + coa[m] -&gt; nadh[m] + co2[m] + accoa[m]', 'KEGG id': 'R00209 (R01699 + R02569 + R07618)', 'GENE_ASSOCIATION': 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'GENE_LIST': 'PDHA1 and PDHB and DLAT and DLD', 'HGNC': '8806 and HGNC:8808 and HGNC:2896 and HGNC:2898', 'Gene name': 'pyruvate dehydrogenase', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '42.5705, 40.5677, 37.7551, 31.2307', 'Directionality rationale': 'CO2 production, Delta G &gt;5kJ/mol, literature', 'HPA mRNA level in heart tissue': 'High and High and Medium and High', 'HPA protein level in heart tissue': 'Medium and High and High and High'}, '_annotation': {'ec-code': ['1.2.4.1', '2.3.1.12', '1.8.1.4'], 'kegg.reaction': ['R00209', 'R01699', 'R02569', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000150768 at 0x28abc8990>, <Gene ENSG00000168291 at 0x28abcb450>, <Gene ENSG00000131828 at 0x28abc9650>, <Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite pyr_m at 0x29d290b10>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CSm', 'name': 'R_CSm Acetyl-CoA + H2O + Oxaloacetate --> Citrate + CoA', 'notes': {'Model Reaction Number': '29', 'EC Number': '2.3.3.1', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Acetyl-CoA + H2O + Oxaloacetate --&gt; Citrate + CoA', 'Recon2 id': 'CSm', 'Recon2 formula': 'h2o[m] + accoa[m] + oaa[m] -&gt; h[m] + coa[m] + cit[m]', 'KEGG id': 'R00351', 'GENE_ASSOCIATION': 'ENSG00000062485', 'GENE_LIST': 'CS', 'HGNC': '2422', 'Gene name': 'citrate synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '37.826', 'Directionality rationale': 'Literature, Delta G &gt;5kJ/mol, HumanCyc', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.3.3.1', 'kegg.reaction': 'R00351'}, '_gpr': cobra.core.gene.GPR('ENSG00000062485'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000062485 at 0x28abc9c10>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite oaa_m at 0x29e2c5610>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite cit_m at 0x2a2f126d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACONTm', 'name': 'R_ACONTm Citrate -> Isocitrate', 'notes': {'Model Reaction Number': '30', 'EC Number': '4.2.1.3', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Citrate -&gt; Isocitrate', 'Recon2 id': 'ACONTm', 'Recon2 formula': 'cit[m] --&gt; icit[m]', 'KEGG id': 'R01324 (R01325 + R01900)', 'GENE_ASSOCIATION': 'ENSG00000100412', 'GENE_LIST': 'ACO2', 'HGNC': '118', 'Gene name': 'Aconitate hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '30.2787', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.3', 'kegg.reaction': ['R01324', 'R01325', 'R01900']}, '_gpr': cobra.core.gene.GPR('ENSG00000100412'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000100412 at 0x28abc82d0>}, '_metabolites': {<Metabolite cit_m at 0x2a2f126d0>: -1.0, <Metabolite icit_m at 0x29c7d1a50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ICDHxm', 'name': 'R_ICDHxm Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH', 'notes': {'Model Reaction Number': '31', 'EC Number': '1.1.1.41', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Isocitrate + NAD(+) = 2-oxoglutarate + CO(2) + NADH', 'Recon2 id': 'ICDHxm', 'Recon2 formula': 'nad[m] + icit[m] -&gt; nadh[m] + akg[m] + co2[m]', 'KEGG id': 'R00709', 'GENE_ASSOCIATION': 'ENSG00000166411 and ENSG00000101365 and ENSG00000067829', 'GENE_LIST': 'IDH3A and IDH3B and IDH3G', 'HGNC': '5384 and HGNC:5385 and HGNC:5386', 'Gene name': 'Isocitrate dehydrogenase (NAD+)', 'MitoCarta': 'TRUE, TRUE, TRUE', 'MitoCarta score': '34.6301, 32.7553, 28.8516', 'Directionality rationale': 'Brenda, Literature, PMID:27049945, 22101431, 22101433', 'HPA mRNA level in heart tissue': 'Medium and High and Medium', 'HPA protein level in heart tissue': 'Low and Medium and Medium'}, '_annotation': {'ec-code': '1.1.1.41', 'kegg.reaction': 'R00709'}, '_gpr': cobra.core.gene.GPR('ENSG00000166411 and ENSG00000101365 and ENSG00000067829'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000067829 at 0x28ab18310>, <Gene ENSG00000166411 at 0x28abca910>, <Gene ENSG00000101365 at 0x28ab18650>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite icit_m at 0x29c7d1a50>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite akg_m at 0x2919313d0>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ICDHyrm', 'name': 'R_ICDHyrm Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH', 'notes': {'Model Reaction Number': '32', 'EC Number': '1.1.1.42', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Isocitrate + NADP(+) -&gt; 2-oxoglutarate + CO(2) + NADPH', 'Recon2 id': 'ICDHyrm', 'Recon2 formula': 'nadp[m] + icit[m] --&gt; nadph[m] + akg[m] + co2[m]', 'KEGG id': 'R00267 (R00268 + R01899)', 'GENE_ASSOCIATION': 'ENSG00000182054', 'GENE_LIST': 'IDH2', 'HGNC': '5383', 'Gene name': 'Isocitrate dehydrogenase (NADP+), mitochondrial', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '9.2614, 21.7892', 'Directionality rationale': 'CO2 production, literature PMID:27049945, 22101433, 22101431', 'HPA mRNA level in heart tissue': 'Medium or High', 'HPA protein level in heart tissue': 'Not detected or High'}, '_annotation': {'ec-code': '1.1.1.42', 'kegg.reaction': ['R00267', 'R00268', 'R01899']}, '_gpr': cobra.core.gene.GPR('ENSG00000182054'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000182054 at 0x28ab18990>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite icit_m at 0x29c7d1a50>: -1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite akg_m at 0x2919313d0>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AKGDm', 'name': 'R_AKGDm 2-Oxoglutarate + CoA + NAD+ --> Succinyl-CoA + CO2 + NADH + H+', 'notes': {'Model Reaction Number': '33', 'EC Number': '1.2.4.2, 1.8.1.4, 2.3.1.61', 'SUBSYSTEM': 'TCA cycle', 'Description': '2-Oxoglutarate + CoA + NAD+ --&gt; Succinyl-CoA + CO2 + NADH + H+', 'Recon2 id': 'AKGDm', 'Recon2 formula': 'nad[m] + akg[m] + coa[m] -&gt; nadh[m] + co2[m] + succoa[m]', 'KEGG id': 'R08549 (R00621+R02570+R03316+R07618)', 'GENE_ASSOCIATION': 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'GENE_LIST': 'OGDH and DLST and DLD', 'HGNC': '8124 and HGNC:2911 and HGNC:2898', 'Gene name': '2-oxoglutarate dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE', 'MitoCarta score': '40.5677, 37.6758, 31.2307', 'Directionality rationale': 'CO2 production, Delta G &gt;5kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'High and High and High', 'HPA protein level in heart tissue': 'High and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.2', '1.8.1.4', '2.3.1.61'], 'kegg.reaction': ['R08549', 'R00621', 'R02570', 'R03316', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000105953 and ENSG00000119689 and ENSG00000091140'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000105953 at 0x28ab18cd0>, <Gene ENSG00000091140 at 0x28abca110>, <Gene ENSG00000119689 at 0x28ab18fd0>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite succoa_m at 0x29d7ddd10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SUCOAS1m', 'name': 'R_SUCOAS1m GTP + Succinate + CoA --> GDP + Orthophosphate + Succinyl-CoA', 'notes': {'Model Reaction Number': '34', 'EC Number': '6.2.1.4', 'SUBSYSTEM': 'TCA cycle', 'Description': 'GTP + Succinate + CoA --&gt; GDP + Orthophosphate + Succinyl-CoA', 'Recon2 id': 'SUCOAS1m', 'Recon2 formula': 'coa[m] + gtp[m] + succ[m] --&gt; pi[m] + gdp[m] + succoa[m]', 'KEGG id': 'R00432', 'GENE_ASSOCIATION': 'ENSG00000163541 and ENSG00000172340', 'GENE_LIST': 'SUCLG1 and SUCLG2', 'HGNC': '11449 and HGNC:11450', 'Gene name': 'Succinate--CoA ligase (GDP-forming)', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '32.1445, 26.2777', 'HPA mRNA level in heart tissue': 'High and Medium', 'HPA protein level in heart tissue': 'Medium and High'}, '_annotation': {'ec-code': '6.2.1.4', 'kegg.reaction': 'R00432'}, '_gpr': cobra.core.gene.GPR('ENSG00000163541 and ENSG00000172340'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000163541 at 0x28ab19310>, <Gene ENSG00000172340 at 0x28ab19650>}, '_metabolites': {<Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite gtp_m at 0x29c7e5a90>: -1.0, <Metabolite succ_m at 0x29d7de4d0>: -1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite gdp_m at 0x1335dab10>: 1.0, <Metabolite succoa_m at 0x29d7ddd10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SUCOASm', 'name': 'R_SUCOASm ATP + Succinate + CoA --> ADP + Orthophosphate + Succinyl-CoA', 'notes': {'Model Reaction Number': '35', 'EC Number': '6.2.1.5', 'SUBSYSTEM': 'TCA cycle', 'Description': 'ATP + Succinate + CoA --&gt; ADP + Orthophosphate + Succinyl-CoA', 'Recon2 id': 'SUCOASm', 'Recon2 formula': 'atp[m] + coa[m] + succ[m] --&gt; adp[m] + pi[m] + succoa[m]', 'KEGG id': 'R00405', 'GENE_ASSOCIATION': 'ENSG00000136143', 'GENE_LIST': 'SUCLA2', 'HGNC': '11448', 'Gene name': 'Succinate--CoA ligase (ADP-forming)', 'MitoCarta': 'TRUE', 'MitoCarta score': '34.9705', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '6.2.1.5', 'kegg.reaction': 'R00405'}, '_gpr': cobra.core.gene.GPR('ENSG00000136143'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000136143 at 0x28ab19990>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite succ_m at 0x29d7de4d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite succoa_m at 0x29d7ddd10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FUMm', 'name': 'R_FUMm Fumarate + H2O --> (S)-Malate', 'notes': {'Model Reaction Number': '36', 'EC Number': '4.2.1.2', 'SUBSYSTEM': 'TCA cycle', 'Description': 'Fumarate + H2O --&gt; (S)-Malate', 'Recon2 id': 'FUMm', 'Recon2 formula': 'h2o[m] + fum[m] --&gt; mal_L[m]', 'KEGG id': 'R01082', 'GENE_ASSOCIATION': 'ENSG00000091483', 'GENE_LIST': 'FH', 'HGNC': '3700', 'Gene name': 'fumarate hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '35.7616', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.2.1.2', 'kegg.reaction': 'R01082'}, '_gpr': cobra.core.gene.GPR('ENSG00000091483'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000091483 at 0x28ab19cd0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite fum_m at 0x133525390>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MDHm', 'name': 'R_MDHm (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+', 'notes': {'Model Reaction Number': '37', 'EC Number': '1.1.1.37', 'SUBSYSTEM': 'TCA cycle', 'Description': '(S)-Malate + NAD+ --&gt; Oxaloacetate + NADH + H+', 'Recon2 id': 'MDHm', 'Recon2 formula': 'nad[m] + mal_L[m] --&gt; h[m] + nadh[m] + oaa[m]', 'KEGG id': 'R00342', 'GENE_ASSOCIATION': 'ENSG00000146701', 'GENE_LIST': 'MDH2', 'HGNC': '6971', 'Gene name': 'malate dehydrogenase 2, NAD (mitochondrial)', 'MitoCarta': 'TRUE', 'MitoCarta score': '38.3734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.37', 'kegg.reaction': 'R00342'}, '_gpr': cobra.core.gene.GPR('ENSG00000146701'), 'subsystem': 'TCA cycle', '_genes': {<Gene ENSG00000146701 at 0x28ab1a010>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite oaa_m at 0x29e2c5610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CI_MitoCore', 'name': 'R_CI_MitoCore NADH dehydrogenase', 'notes': {'Model Reaction Number': '38', 'EC Number': '1.6.5.3, 1.6.5.8, 1.6.5.9', 'SUBSYSTEM': 'Electron transport chain CI', 'Description': 'NADH dehydrogenase', 'Recon2 id': 'NADH2_u10m', 'Recon2 formula': '5.0 h[m] + nadh[m] + q10[m] -&gt; 4.0 h[c] + nad[m] + q10h2[m]', 'KEGG id': 'R02163 (R00281 + R02166)', 'GENE_ASSOCIATION': 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'GENE_LIST': 'NDUFS7 and NDUFS8 and NDUFV2 and NDUFS3 and NDUFS2 and NDUFV1 and NDUFS1 and MT-ND1 and MT-ND2 and MT-ND3 and MT-ND4 and MT-ND4L and MT-ND5and MT-ND6 and NDUFS6 and NDUFA12 and NDUFS4 and NDUFA9 and NDUFAB1 and NDUFA2 and NDUFB3 and NDUFA5 and NDUFA6 and NDUFA11 and NDUFB11 and NDUFS5 and NDUFB4 and NDUFA13 and NDUFB7 and NDUFA8 and NDUFB9 and NDUFB10 and NDUFB8 and NDUFC2 and NDUFB2 and NDUFA7 and NDUFA3 and NDUFA4 and NDUFB5 and NDUFB1 and NDUFC1 and NDUFA10 and NDUFA4L2 and NDUFV3 and NDUFB6', 'HGNC': '7714 and HGNC:7715 and HGNC:7717 and HGNC:7710 and HGNC:7708 and HGNC:7716 and HGNC:7707 and HGNC:7455 and HGNC:7456 and HGNC:7458 and HGNC:7459 and HGNC:7460 and HGNC:7461 and HGNC:7462 and HGNC:7713 and HGNC:23987 and HGNC:7711 and HGNC:7693 and HGNC:7694 and HGNC:7685 and HGNC:7698 and HGNC:7688 and HGNC:7690 and HGNC:20371 and HGNC:20372 and HGNC:7712 and HGNC:7699 and HGNC:17194 and HGNC:7702 and HGNC:7692 and HGNC:7704 and HGNC:7696 and HGNC:7703 and HGNC:7706 and HGNC:7697 and HGNC:7691 and HGNC:7686 and HGNC:7687 and HGNC:7700 and HGNC:7695 and HGNC:7705 and HGNC:7684 and HGNC:29836 and HGNC:7719 and HGNC:7701', 'Gene name': 'NADH dehydrogenase', 'MitoCarta': 'TRUE except NDUFA4L2, NDUFA7 not in MitoCarta at all', 'MitoCarta score': '33.0033, 30.6672, 32.0934, 18.2867, 30.334, 30.334, 36.8031, 3.5915, 12.0861, 8.7965, 12.0861, 1.4501, 12.0861, -6.2297, 26.466, 21.467, 27.8713, 27.4053, 18.5522, 25.0692, 16.6293, 26.2914, 28.3203, 15.7163, 20.8256, 25.0273, 10.644, 16.014, 17.9816, 21.0863, 28.397, 25.3606, 31.6796, 21.219, 22.2978, N/A, 17.0508, 16.014, 28.7877, -3.784, 17.1335, 21.261, 1.2151, 20.4205, 26.2914', 'HPA mRNA level in heart tissue': 'Medium and High and Medium and High and High and High and High and High and High and High and High and High and High and High and Medium and High and High and Medium and High and High and High and High and High and High and Medium and High and High and High and High and High and High and High and High and High and High and High and High and High and High and High and Medium and High and Medium and Medium and High', 'HPA protein level in heart tissue': 'ND and High and High and High and High and High and ND and High and ND and High and Medium and ND and ND and ND and High and High and High and High and Medium and High and High and High and Not detected and ND and High and High and High and High and High and High and High and ND and High and Medium and Medium and High and High and ND and High and High and High and ND and ND and High and High'}, '_annotation': {'ec-code': ['1.6.5.3', '1.6.5.8', '1.6.5.9'], 'kegg.reaction': ['R02163', 'R00281', 'R02166']}, '_gpr': cobra.core.gene.GPR('ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264'), 'subsystem': 'Electron transport chain CI', '_genes': {<Gene ENSG00000178127 at 0x28ab09010>, <Gene ENSG00000186010 at 0x28ab08050>, <Gene ENSG00000198886 at 0x28ab1b050>, <Gene ENSG00000151366 at 0x28ab0a050>, <Gene ENSG00000090266 at 0x28ab0b090>, <Gene ENSG00000184983 at 0x28ab290d0>, <Gene ENSG00000184752 at 0x28ab280d0>, <Gene ENSG00000170906 at 0x28ab2a110>, <Gene ENSG00000147684 at 0x28ab2b150>, <Gene ENSG00000189043 at 0x28ab089d0>, <Gene ENSG00000160194 at 0x28ab1a9d0>, <Gene ENSG00000130414 at 0x28ab099d0>, <Gene ENSG00000131495 at 0x28ab1b9d0>, <Gene ENSG00000128609 at 0x28ab0aa10>, <Gene ENSG00000198763 at 0x28ab28a50>, <Gene ENSG00000147123 at 0x28ab0ba50>, <Gene ENSG00000099795 at 0x28ab29a90>, <Gene ENSG00000119421 at 0x28ab2aad0>, <Gene ENSG00000174886 at 0x28ab1b350>, <Gene ENSG00000065518 at 0x28ab1a350>, <Gene ENSG00000166136 at 0x28ab09350>, <Gene ENSG00000267855 at 0x28ab08350>, <Gene ENSG00000198888 at 0x28ab0a390>, <Gene ENSG00000185633 at 0x28ab0b3d0>, <Gene ENSG00000139180 at 0x28ab29410>, <Gene ENSG00000140990 at 0x28ab28410>, <Gene ENSG00000167792 at 0x28ab2a450>, <Gene ENSG00000183648 at 0x28ab1bc90>, <Gene ENSG00000136521 at 0x28ab08cd0>, <Gene ENSG00000119013 at 0x28ab09d10>, <Gene ENSG00000115286 at 0x28ab1ad10>, <Gene ENSG00000158864 at 0x28ab0ad50>, <Gene ENSG00000165264 at 0x28ab0bd90>, <Gene ENSG00000213619 at 0x28ab28d90>, <Gene ENSG00000110717 at 0x28ab29dd0>, <Gene ENSG00000023228 at 0x28ab2ae10>, <Gene ENSG00000004779 at 0x28ab09690>, <Gene ENSG00000198695 at 0x28ab1a690>, <Gene ENSG00000145494 at 0x28ab1b690>, <Gene ENSG00000109390 at 0x28ab08690>, <Gene ENSG00000168653 at 0x28ab0a6d0>, <Gene ENSG00000164258 at 0x28ab28690>, <Gene ENSG00000198786 at 0x28ab0b710>, <Gene ENSG00000212907 at 0x28ab29750>, <Gene ENSG00000198840 at 0x28ab2a790>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -3.996, <Metabolite nadh_m at 0x29d2af310>: -1.0, <Metabolite q10_m at 0x29d292950>: -0.999, <Metabolite o2_m at 0x17b2554d0>: -0.002, <Metabolite PMF_c at 0x29c7879d0>: 3.996, <Metabolite nad_m at 0x29c7a06d0>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 0.999, <Metabolite o2s_m at 0x29d4ba350>: 0.002}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CII_MitoCore', 'name': 'R_CII_MitoCore succinate dehydrogenase', 'notes': {'Model Reaction Number': '39', 'EC Number': '1.3.5.1', 'SUBSYSTEM': 'Electron transport chain CII', 'Description': 'succinate dehydrogenase', 'Recon2 id': 'SUCD1m and FADH2ETC', 'Recon2 formula': 'fad[m] + succ[m] --&gt; fadh2[m] + fum[m]', 'KEGG id': 'R02164', 'GENE_ASSOCIATION': 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', 'GENE_LIST': 'SDHA and SDHB and SDHC', 'HGNC': '10680 and HGNC:10681 and HGNC:10682', 'Gene name': 'succinate dehydrogenase', 'MitoCarta': 'TRUE, TRUE, TRUE', 'MitoCarta score': '42.9701, 43.2272, 28.8474', 'HPA mRNA level in heart tissue': 'High and High and Medium', 'HPA protein level in heart tissue': 'High and High and ND'}, '_annotation': {'ec-code': '1.3.5.1', 'kegg.reaction': 'R02164'}, '_gpr': cobra.core.gene.GPR('ENSG00000073578 and ENSG00000117118 and ENSG00000143252'), 'subsystem': 'Electron transport chain CII', '_genes': {<Gene ENSG00000117118 at 0x28ab2b790>, <Gene ENSG00000073578 at 0x28ab2b450>, <Gene ENSG00000143252 at 0x28ab2bad0>}, '_metabolites': {<Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite succ_m at 0x29d7de4d0>: -1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite fum_m at 0x133525390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CIII_MitoCore', 'name': 'R_CIII_MitoCore cytochrome c reductase', 'notes': {'Model Reaction Number': '40', 'EC Number': '1.10.2.2', 'SUBSYSTEM': 'Electron transport chain CIII', 'Description': 'cytochrome c reductase', 'Recon2 id': 'CYOR_u10m', 'Recon2 formula': '2.0 h[m] + 2.0 ficytC[m] + q10h2[m] -&gt; 4.0 h[c] + q10[m] + 2.0 focytC[m]', 'KEGG id': 'R02161', 'GENE_ASSOCIATION': 'ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295', 'GENE_LIST': 'MT-CYB and CYC1 and CYCS and UQCRFS1 and UQCRB and UQCRH and UQCRC2 and UQCRC1 and UQCR11 and UQCR10 and TTC19', 'HGNC': '7427 and HGNC:2579 and HGNC:19986 and HGNC:12587 and HGNC:12582 and HGNC:12590 and HGNC:12586 and HGNC:12585 and HGNC:30862 and HGNC:30863 and HGNC:26006', 'Gene name': 'cytochrome c reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.4861, 43.6442, 24.9447, 40.7667, 19.6799, 34.0633, 40.5677, 42.9037, 10.644, 13.0076', 'HPA mRNA level in heart tissue': 'High and High and High and Medium and High and High and High and Medium and High and High and Low', 'HPA protein level in heart tissue': 'Medium and High and Medium and High and High and High and High and High and ND and ND and Medium'}, '_annotation': {'ec-code': '1.10.2.2', 'kegg.reaction': 'R02161'}, '_gpr': cobra.core.gene.GPR('ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295'), 'subsystem': 'Electron transport chain CIII', '_genes': {<Gene ENSG00000127540 at 0x28abd4810>, <Gene ENSG00000198727 at 0x28ab2be10>, <Gene ENSG00000172115 at 0x28abd5850>, <Gene ENSG00000179091 at 0x28abd4e90>, <Gene ENSG00000173660 at 0x28abd44d0>, <Gene ENSG00000184076 at 0x28abd5ed0>, <Gene ENSG00000156467 at 0x28abd5510>, <Gene ENSG00000011295 at 0x28abd4b50>, <Gene ENSG00000010256 at 0x28abd5b90>, <Gene ENSG00000140740 at 0x28abd51d0>, <Gene ENSG00000169021 at 0x28abd4190>}, '_metabolites': {<Metabolite PMF_m at 0x29c785ad0>: -2.0, <Metabolite ficytC_m at 0x1335ac1d0>: -2.0, <Metabolite q10h2_m at 0x29d2b0790>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: 4.0, <Metabolite q10_m at 0x29d292950>: 1.0, <Metabolite focytC_m at 0x1335878d0>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CIV_MitoCore', 'name': 'R_CIV_MitoCore cytochrome c oxidase', 'notes': {'Model Reaction Number': '41', 'EC Number': '1.9.3.1', 'SUBSYSTEM': 'Electron transport chain CIV', 'Description': 'cytochrome c oxidase', 'Recon2 id': 'CYOOm2', 'Recon2 formula': 'o2[m] + 4.0 focytC[m] -&gt; 2.0 h2o[m] + 4.0 h[m] + 4.0 ficytC[m]', 'KEGG id': 'R00081', 'GENE_ASSOCIATION': 'ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581', 'GENE_LIST': 'MT-CO1 and MT-CO2 and MT-CO3 and COX4I1 and COX4I2 and COX5A and COX5B and COX6A1 and COX6A2 and COX6B1 and COX6B2 and COX6C and COX7A1 and COX7A2 and COX7A2L and COX7B and COX7C and COX7A2L and COX8A and COX8C', 'HGNC': '7419 and HGNC:7421 and HGNC:7422 and HGNC:2265 and HGNC:16232 and HGNC:2267 and HGNC:2269 and HGNC:2277 and HGNC:2279 and HGNC:2280 and HGNC:24380 and HGNC:2285 and HGNC:2287 and HGNC:2288 and HGNC:2289 and HGNC:2291 and HGNC:2292 and HGNC:2289 and HGNC:2294 and HGNC:24382', 'Gene name': 'cytochrome c oxidase', 'MitoCarta': 'TRUE', 'MitoCarta score': '12.6275, 21.1221, 9.5597, 26.466, 13.7961, 41.9796, 26.3322, 33.6921, 18.761, 28.175, 14.0781, 28.397, 21.5213, 24.0965, 7.1222, 15.8766, 22.1144, 7.1222, 17.7556, 13.1755', 'Directionality rationale': 'Literature', 'HPA mRNA level in heart tissue': 'High and High and High and High and Medium and High and High and Medium and High and High and Not detected and High and High and High and High and High and High and High and High and Not detected', 'HPA protein level in heart tissue': 'High and High and Medium and High and ND and High and High and Medium and ND and High and ND and High and ND and ND and ND and High and High and ND and Medium and Not detected'}, '_annotation': {'ec-code': '1.9.3.1', 'kegg.reaction': 'R00081'}, '_gpr': cobra.core.gene.GPR('ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581'), 'subsystem': 'Electron transport chain CIV', '_genes': {<Gene ENSG00000126267 at 0x28abd6890>, <Gene ENSG00000135940 at 0x28abd78d0>, <Gene ENSG00000178741 at 0x28ab60950>, <Gene ENSG00000198712 at 0x28ab61990>, <Gene ENSG00000164919 at 0x28abd6210>, <Gene ENSG00000111775 at 0x28abd7250>, <Gene ENSG00000160471 at 0x28ab602d0>, <Gene ENSG00000161281 at 0x28ab61310>, <Gene ENSG00000127184 at 0x28abd6bd0>, <Gene ENSG00000112695 at 0x28abd7c10>, <Gene ENSG00000187581 at 0x28ab60c90>, <Gene ENSG00000176340 at 0x28ab61cd0>, <Gene ENSG00000131174 at 0x28abd6550>, <Gene ENSG00000156885 at 0x28abd7590>, <Gene ENSG00000131055 at 0x28ab60610>, <Gene ENSG00000131143 at 0x28ab61650>, <Gene ENSG00000198938 at 0x28abd6f10>, <Gene ENSG00000115944 at 0x28abd7f50>, <Gene ENSG00000198804 at 0x28ab60fd0>}, '_metabolites': {<Metabolite o2_m at 0x17b2554d0>: -1.0, <Metabolite focytC_m at 0x1335878d0>: -4.0, <Metabolite PMF_m at 0x29c785ad0>: -8.0, <Metabolite h2o_m at 0x29c7848d0>: 2.0, <Metabolite PMF_c at 0x29c7879d0>: 4.0, <Metabolite ficytC_m at 0x1335ac1d0>: 4.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CV_MitoCore', 'name': 'R_CV_MitoCore ATP synthase', 'notes': {'Model Reaction Number': '42', 'EC Number': '3.6.3.14, 3.6.3.10, 3.6.3.6', 'SUBSYSTEM': 'Electron transport chain CV', 'Description': 'ATP synthase', 'Recon2 id': 'ATPS4m', 'Recon2 formula': '4.0 h[c] + adp[m] + pi[m] -&gt; h2o[m] + 3.0 h[m] + atp[m]', 'KEGG id': 'R00086', 'GENE_ASSOCIATION': 'ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and (ENSG00000159199 or ENSG00000135390 or ENSG00000154518) and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253', 'GENE_LIST': 'ATP5A1 and ATP5B and ATP5C1 and ATP5D and ATP5E and ATP5F1 and (ATP5G1 or ATP5G2 or ATP5G3) and ATP5H and ATP5I and ATP5J and ATP5J2 and ATP5L and ATP5L2 and ATP5O and MT-ATP6 and MT-ATP8', 'HGNC': '823 and HGNC:830 and HGNC:833 and HGNC:837 and HGNC:838 and HGNC:840 and (HGNC:841 or HGNC:842 or HGNC:843) and HGNC:845 and HGNC:846 and HGNC:847 and HGNC:848 and HGNC:14247 and HGNC:13213 and HGNC:850 and HGNC:7414 and HGNC:7415', 'Gene name': 'ATP synthase', 'MitoCarta': 'TRUE, ATP5L2 - False', 'MitoCarta score': '40.7095, 35.6372, 27.2465, 42.0165, 36.4978, 22.1144, 6.1649, 3.7304, 19.5524, 19.6799, 32.2839, 31.6796, 13.2425, 19.6799, -3.784, 34.0, 21.1221, 9.7131', 'HPA mRNA level in heart tissue': 'High and High and High and High and Medium and Medium and (High or High or High) and High and High and High and High and High and Not detected and High and High and High', 'HPA protein level in heart tissue': 'High and High and Medium and High and Low and Medium and (ND or Medium or ND) and High and High and High and ND and High and ND and High and ND and Not detected'}, '_annotation': {'ec-code': ['3.6.3.14', '3.6.3.10', '3.6.3.6'], 'kegg.reaction': 'R00086'}, '_gpr': cobra.core.gene.GPR('(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000159199) or (ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000135390) or (ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000154518)'), 'subsystem': 'Electron transport chain CV', '_genes': {<Gene ENSG00000152234 at 0x28ab62010>, <Gene ENSG00000228253 at 0x28ab00410>, <Gene ENSG00000154723 at 0x28ab63a10>, <Gene ENSG00000167863 at 0x28ab63050>, <Gene ENSG00000154518 at 0x28ab01450>, <Gene ENSG00000099624 at 0x28ab62690>, <Gene ENSG00000116459 at 0x28ab00a90>, <Gene ENSG00000241837 at 0x28ab636d0>, <Gene ENSG00000167283 at 0x28ab000d0>, <Gene ENSG00000249222 at 0x28ab01110>, <Gene ENSG00000135390 at 0x28ab62d10>, <Gene ENSG00000124172 at 0x28ab00dd0>, <Gene ENSG00000241468 at 0x28ab00750>, <Gene ENSG00000198899 at 0x28ab62350>, <Gene ENSG00000159199 at 0x28ab63390>, <Gene ENSG00000169020 at 0x28ab63d50>, <Gene ENSG00000165629 at 0x28ab01790>, <Gene ENSG00000110955 at 0x28ab629d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -2.7, <Metabolite adp_m at 0x2a2a88e50>: -1.0, <Metabolite pi_m at 0x29e6c7690>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 2.7, <Metabolite atp_m at 0x28b1339d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PEPCKm', 'name': 'R_PEPCKm GTP + Oxaloacetate --> GDP + Phosphoenolpyruvate + CO2', 'notes': {'Model Reaction Number': '43', 'EC Number': '4.1.1.32', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'GTP + Oxaloacetate --&gt; GDP + Phosphoenolpyruvate + CO2', 'Recon2 id': 'PEPCKm', 'Recon2 formula': 'oaa[m] + gtp[m] -&gt; co2[m] + pep[m] + gdp[m]', 'KEGG id': 'R00431', 'GENE_ASSOCIATION': 'ENSG00000100889', 'GENE_LIST': 'PCK2', 'HGNC': '8725', 'Gene name': 'Phosphoenolpyruvate carboxykinase (GTP)', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.7634', 'Directionality rationale': 'GTP consumption, CO2 production Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.1.1.32', 'kegg.reaction': 'R00431'}, '_gpr': cobra.core.gene.GPR('ENSG00000100889'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000100889 at 0x28ab01ad0>}, '_metabolites': {<Metabolite oaa_m at 0x29e2c5610>: -1.0, <Metabolite gtp_m at 0x29c7e5a90>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite pep_m at 0x29e2bc110>: 1.0, <Metabolite gdp_m at 0x1335dab10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCm', 'name': 'R_PCm ATP + Pyruvate + HCO3- --> ADP + Orthophosphate + Oxaloacetate', 'notes': {'Model Reaction Number': '44', 'EC Number': '6.4.1.1', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'ATP + Pyruvate + HCO3- --&gt; ADP + Orthophosphate + Oxaloacetate', 'Recon2 id': 'PCm', 'Recon2 formula': 'atp[m] + pyr[m] + hco3[m] -&gt; h[m] + adp[m] + pi[m] + oaa[m]', 'KEGG id': 'R00344', 'GENE_ASSOCIATION': 'ENSG00000173599', 'GENE_LIST': 'PC', 'HGNC': '8636', 'Gene name': 'Pyruvate carboxylase, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '19.3217', 'Directionality rationale': 'ATP consumption, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '6.4.1.1', 'kegg.reaction': 'R00344'}, '_gpr': cobra.core.gene.GPR('ENSG00000173599'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000173599 at 0x28ab01e10>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite pyr_m at 0x29d290b10>: -1.0, <Metabolite hco3_m at 0x29c745e50>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite oaa_m at 0x29e2c5610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ME2m', 'name': 'R_ME2m (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '45', 'EC Number': '1.1.1.40', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': '(S)-Malate + NADP+ --&gt; Pyruvate + CO2 + NADPH + H+', 'Recon2 id': 'ME2m', 'Recon2 formula': 'nadp[m] + mal_L[m] -&gt; nadph[m] + pyr[m] + co2[m]', 'KEGG id': 'R00216', 'GENE_ASSOCIATION': 'ENSG00000151376', 'GENE_LIST': 'ME3', 'HGNC': '6985', 'Gene name': 'Malic Enzyme 3', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.8086', 'Directionality rationale': 'Modelling purposes to prevent cycling, CO2 production', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.40', 'kegg.reaction': 'R00216'}, '_gpr': cobra.core.gene.GPR('ENSG00000151376'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000151376 at 0x28ab02150>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: -1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ME1m', 'name': 'R_ME1m (S)-malate + NAD+ = pyruvate + CO(2) + NADH.', 'notes': {'Model Reaction Number': '46', 'EC Number': '1.1.1.38', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': '(S)-malate + NAD+ = pyruvate + CO(2) + NADH.', 'Recon2 id': 'ME1m', 'Recon2 formula': 'nad[m] + mal_L[m] -&gt; nadh[m] + pyr[m] + co2[m]', 'KEGG id': 'R00214', 'GENE_ASSOCIATION': 'ENSG00000082212', 'GENE_LIST': 'ME2', 'HGNC': '6984', 'Gene name': 'Malic Enzyme 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '18.7687', 'Directionality rationale': 'Modelling purposes to prevent cycling, CO2 production', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.38', 'kegg.reaction': 'R00214'}, '_gpr': cobra.core.gene.GPR('ENSG00000082212'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000082212 at 0x28ab02490>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0081', 'name': 'R_r0081 L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate', 'notes': {'Model Reaction Number': '47', 'EC Number': '2.6.1.2', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'L-Alanine + 2-Oxoglutarate --&gt; Pyruvate + L-Glutamate', 'Recon2 id': 'r0081', 'Recon2 formula': 'akg[m] + ala_L[m] --&gt; pyr[m] + glu_L[m]', 'KEGG id': 'R00258', 'GENE_ASSOCIATION': 'ENSG00000166123', 'GENE_LIST': 'GPT2', 'HGNC': '18062', 'Gene name': 'glutamic pyruvate transaminase (alanine aminotransferase) 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '12.7592', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.6.1.2', 'kegg.reaction': 'R00258'}, '_gpr': cobra.core.gene.GPR('ENSG00000166123'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000166123 at 0x28ab027d0>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite ala_L_m at 0x2a2a29310>: -1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACITLm_MitoCore', 'name': 'R_ACITLm_MitoCore ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate', 'notes': {'Model Reaction Number': '48', 'EC Number': '2.3.3.8', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'ATP + Citrate + CoA --&gt; ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00352', 'GENE_ASSOCIATION': 'ENSG00000131473', 'GENE_LIST': 'ACLY', 'HGNC': '115', 'Gene name': 'ATP citrate lyase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.8771', 'Directionality rationale': 'ATP consumption, HumanCyc, Brenda, Literature', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.3.3.8', 'kegg.reaction': 'R00352'}, '_gpr': cobra.core.gene.GPR('ENSG00000131473'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000131473 at 0x28ab02ad0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite cit_m at 0x2a2f126d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0, <Metabolite oaa_m at 0x29e2c5610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NDPK1m', 'name': 'R_NDPK1m ATP + GDP --> ADP + GTP', 'notes': {'Model Reaction Number': '49', 'EC Number': '2.7.4.6', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'ATP + GDP --&gt; ADP + GTP', 'Recon2 id': 'R_NDPK1m', 'Recon2 formula': 'atp[m] + gdp[m] --&gt; adp[m] + gtp[m]', 'KEGG id': 'R00330', 'GENE_ASSOCIATION': 'ENSG00000103202', 'GENE_LIST': 'NME4', 'HGNC': '7852', 'Gene name': 'nucleoside diphosphate kinase 4', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.3094', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.7.4.6', 'kegg.reaction': 'R00330'}, '_gpr': cobra.core.gene.GPR('ENSG00000103202'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000103202 at 0x28ab02e10>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite gdp_m at 0x1335dab10>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite gtp_m at 0x29c7e5a90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NNT_MitoCore', 'name': 'R_NNT_MitoCore NADP+ + NADH --> NADPH + NAD+', 'notes': {'Model Reaction Number': '50', 'EC Number': '1.6.1.2', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'NADP+ + NADH --&gt; NADPH + NAD+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00112', 'GENE_ASSOCIATION': 'ENSG00000112992', 'GENE_LIST': 'NNT', 'HGNC': '7863', 'Gene name': 'nicotinamide nucleotide transhydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.3432', 'Directionality rationale': 'Brenda, Literature - see notes', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.6.1.2', 'kegg.reaction': 'R00112'}, '_gpr': cobra.core.gene.GPR('ENSG00000112992'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000112992 at 0x28ab03150>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite nadh_m at 0x29d2af310>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite nad_m at 0x29c7a06d0>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ADK1m', 'name': 'R_ADK1m ATP + AMP --> 2 ADP', 'notes': {'Model Reaction Number': '51', 'EC Number': '2.7.4.3', 'SUBSYSTEM': 'TCA cycle periphery', 'Description': 'ATP + AMP --&gt; 2 ADP', 'Recon2 id': 'ADK1m', 'Recon2 formula': 'atp[m] + amp[m] --&gt; 2.0 adp[m]', 'KEGG id': 'R00127', 'GENE_ASSOCIATION': 'ENSG00000004455 or ENSG00000147853', 'GENE_LIST': 'AK2 or AK3', 'HGNC': '362 or HGNC:17376', 'Gene name': 'adenylate kinase 2 or 3', 'MitoCarta': 'TRUE', 'MitoCarta score': '17.2313, 25.0021', 'HPA mRNA level in heart tissue': 'Medium or High', 'HPA protein level in heart tissue': 'Medium or High'}, '_annotation': {'ec-code': '2.7.4.3', 'kegg.reaction': 'R00127'}, '_gpr': cobra.core.gene.GPR('ENSG00000004455 or ENSG00000147853'), 'subsystem': 'TCA cycle periphery', '_genes': {<Gene ENSG00000004455 at 0x28ab03490>, <Gene ENSG00000147853 at 0x28ab037d0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite amp_m at 0x2975598d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ME2', 'name': 'R_ME2 (S)-Malate + NADP+ --> Pyruvate + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '52', 'EC Number': '1.1.1.40', 'SUBSYSTEM': 'Cytosolic misc', 'Description': '(S)-Malate + NADP+ --&gt; Pyruvate + CO2 + NADPH + H+', 'Recon2 id': 'ME2', 'Recon2 formula': 'nadp[c] + mal_L[c] -&gt; nadph[c] + pyr[c] + co2[c]', 'KEGG id': 'R00216', 'GENE_ASSOCIATION': 'ENSG00000065833', 'GENE_LIST': 'ME1', 'HGNC': '6983', 'Gene name': 'Malic Enzyme 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.431', 'Directionality rationale': 'CO2 production, modelling purposes', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.40', 'kegg.reaction': 'R00216'}, '_gpr': cobra.core.gene.GPR('ENSG00000065833'), 'subsystem': 'Cytosolic misc', '_genes': {<Gene ENSG00000065833 at 0x28ab03b10>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite co2_c at 0x17810ba50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALATA_L', 'name': 'R_ALATA_L L-Alanine + 2-Oxoglutarate --> Pyruvate + L-Glutamate', 'notes': {'Model Reaction Number': '53', 'EC Number': '2.6.1.2', 'SUBSYSTEM': 'Cytosolic misc', 'Description': 'L-Alanine + 2-Oxoglutarate --&gt; Pyruvate + L-Glutamate', 'Recon2 id': 'ALATA_L', 'Recon2 formula': 'akg[c] + ala_L[c] --&gt; pyr[c] + glu_L[c]', 'KEGG id': 'R00258', 'GENE_ASSOCIATION': 'ENSG00000167701', 'GENE_LIST': 'GPT', 'HGNC': '4552', 'Gene name': 'glutamic-pyruvate transaminase (alanine aminotransferase)', 'MitoCarta': 'FALSE', 'MitoCarta score': '2.9978', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.6.1.2', 'kegg.reaction': 'R00258'}, '_gpr': cobra.core.gene.GPR('ENSG00000167701'), 'subsystem': 'Cytosolic misc', '_genes': {<Gene ENSG00000167701 at 0x28ab03e50>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite ala_L_c at 0x17aef1250>: -1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NDPK1', 'name': 'R_NDPK1 ATP + GDP --> ADP + GTP', 'notes': {'Model Reaction Number': '54', 'EC Number': '2.7.4.6', 'SUBSYSTEM': 'Cytosolic misc', 'Description': 'ATP + GDP --&gt; ADP + GTP', 'Recon2 id': 'NDPK1', 'Recon2 formula': 'atp[c] + gdp[c] --&gt; adp[c] + gtp[c]', 'KEGG id': 'R00330', 'GENE_ASSOCIATION': 'ENSG00000011052 or ENSG00000103024 or ENSG00000172113 or ENSG00000143156 or ENSG00000162368 or ENSG00000106992 or ENSG00000162433 or ENSG00000154027 or ENSG00000140057 or ENSG00000165695 or ENSG00000155085', 'GENE_LIST': 'NME2 or NME3 or NME6 or NME7 or CMPK1 or AK1 or AK4 or AK5 or AK7 or AK8 or AK9', 'HGNC': '7850 or HGNC:7851 or HGNC:20567 or HGNC:20461 or HGNC:18170 or HGNC:361 or HGNC:363 or HGNC:365 or HGNC:20091 or HGNC:26526 or HGNC:33814', 'Gene name': 'nucleoside diphosphate kinase (mulitple)', 'MitoCarta': 'FALSE, TRUE, TRUE, FALSE, FALSE, FALSE, TRUE, FALSE, FALSE, FALSE, FALSE', 'MitoCarta score': '2.0341, 1.4126, 3.9472, 1.0616, 1.8707, 2.062, 19.9135, -5.0662, -6.409, -3.1941, -8.5792', 'HPA mRNA level in heart tissue': 'Low or Low or Medium or Low or Low or Medium or High', 'HPA protein level in heart tissue': 'Not detected or Not detected or Medium or Low or Not detected or High or Medium'}, '_annotation': {'ec-code': '2.7.4.6', 'kegg.reaction': 'R00330'}, '_gpr': cobra.core.gene.GPR('ENSG00000011052 or ENSG00000103024 or ENSG00000172113 or ENSG00000143156 or ENSG00000162368 or ENSG00000106992 or ENSG00000162433 or ENSG00000154027 or ENSG00000140057 or ENSG00000165695 or ENSG00000155085'), 'subsystem': 'Cytosolic misc', '_genes': {<Gene ENSG00000140057 at 0x176eb5210>, <Gene ENSG00000106992 at 0x176eb4850>, <Gene ENSG00000103024 at 0x176eb6250>, <Gene ENSG00000162433 at 0x176eb5890>, <Gene ENSG00000172113 at 0x176eb4ed0>, <Gene ENSG00000011052 at 0x176eb4510>, <Gene ENSG00000162368 at 0x176eb41d0>, <Gene ENSG00000143156 at 0x176eb5f10>, <Gene ENSG00000154027 at 0x176eb5550>, <Gene ENSG00000165695 at 0x176eb4b90>, <Gene ENSG00000155085 at 0x176eb5bd0>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite gdp_c at 0x1335da410>: -1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite gtp_c at 0x29c7e6250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FUM', 'name': 'R_FUM Fumarate + H2O --> (S)-Malate', 'notes': {'Model Reaction Number': '55', 'EC Number': '4.2.1.2', 'SUBSYSTEM': 'Cytosolic misc', 'Description': 'Fumarate + H2O --&gt; (S)-Malate', 'Recon2 id': 'FUM', 'Recon2 formula': 'h2o[c] + fum[c] --&gt; mal_L[c]', 'KEGG id': 'R01082', 'GENE_ASSOCIATION': 'ENSG00000091483', 'GENE_LIST': 'FH', 'HGNC': '3700', 'Gene name': 'fumarate hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '35.7616', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.2.1.2', 'kegg.reaction': 'R01082'}, '_gpr': cobra.core.gene.GPR('ENSG00000091483'), 'subsystem': 'Cytosolic misc', '_genes': {<Gene ENSG00000091483 at 0x28ab19cd0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite fum_c at 0x133597890>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ADK1', 'name': 'R_ADK1 ATP + AMP --> 2 ADP', 'notes': {'Model Reaction Number': '56', 'EC Number': '2.7.4.3', 'SUBSYSTEM': 'Cytosolic misc', 'Description': 'ATP + AMP --&gt; 2 ADP', 'Recon2 id': 'ADK1', 'Recon2 formula': 'atp[c] + amp[c] --&gt; 2.0 adp[c]', 'KEGG id': 'R00127', 'GENE_ASSOCIATION': 'ENSG00000106992', 'GENE_LIST': 'AK1', 'HGNC': '361', 'Gene name': 'adenylate kinase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '2.062', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.7.4.3', 'kegg.reaction': 'R00127'}, '_gpr': cobra.core.gene.GPR('ENSG00000106992'), 'subsystem': 'Cytosolic misc', '_genes': {<Gene ENSG00000106992 at 0x176eb4850>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite amp_c at 0x2ae087b50>: -1.0, <Metabolite adp_c at 0x17918bb10>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ICDHy', 'name': 'R_ICDHy Isocitrate + NADP(+) -> 2-oxoglutarate + CO(2) + NADPH', 'notes': {'Model Reaction Number': '57', 'EC Number': '1.1.1.42', 'SUBSYSTEM': 'Reductive carboxylation', 'Description': 'Isocitrate + NADP(+) -&gt; 2-oxoglutarate + CO(2) + NADPH', 'Recon2 id': 'ICDHy', 'Recon2 formula': 'nadp[c] + icit[c] -&gt; nadph[c] + akg[c] + co2[c]', 'KEGG id': 'R00267 (R00268 + R01899)', 'GENE_ASSOCIATION': 'ENSG00000138413', 'GENE_LIST': 'IDH1', 'HGNC': '5382', 'Gene name': 'Isocitrate dehydrogenase (NADP+)', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.2614', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.1.1.42', 'kegg.reaction': ['R00267', 'R00268', 'R01899']}, '_gpr': cobra.core.gene.GPR('ENSG00000138413'), 'subsystem': 'Reductive carboxylation', '_genes': {<Gene ENSG00000138413 at 0x176eb6590>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite icit_c at 0x29c7d0350>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite akg_c at 0x16ac21410>: 1.0, <Metabolite co2_c at 0x17810ba50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -0.006, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACONT', 'name': 'R_ACONT Citrate -> Isocitrate', 'notes': {'Model Reaction Number': '58', 'EC Number': '4.2.1.3', 'SUBSYSTEM': 'Reductive carboxylation', 'Description': 'Citrate -&gt; Isocitrate', 'Recon2 id': 'ACONT', 'Recon2 formula': 'cit[c] --&gt; icit[c]', 'KEGG id': 'R01324 (R01325 + R01900)', 'GENE_ASSOCIATION': 'ENSG00000122729', 'GENE_LIST': 'ACO1', 'HGNC': '117', 'Gene name': 'aconitase 1, soluble', 'MitoCarta': 'TRUE', 'MitoCarta score': '30.2787', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.2.1.3', 'kegg.reaction': ['R01324', 'R01325', 'R01900']}, '_gpr': cobra.core.gene.GPR('ENSG00000122729'), 'subsystem': 'Reductive carboxylation', '_genes': {<Gene ENSG00000122729 at 0x176eb68d0>}, '_metabolites': {<Metabolite cit_c at 0x17b24a710>: -1.0, <Metabolite icit_c at 0x29c7d0350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACITL', 'name': 'R_ACITL ATP + Citrate + CoA --> ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate', 'notes': {'Model Reaction Number': '59', 'EC Number': '2.3.3.8', 'SUBSYSTEM': 'Reductive carboxylation / Acetyl-CoA production for biosynthesis', 'Description': 'ATP + Citrate + CoA --&gt; ADP + Orthophosphate + Acetyl-CoA + Oxaloacetate', 'Recon2 id': 'ACITL', 'Recon2 formula': 'atp[c] + coa[c] + cit[c] -&gt; adp[c] + pi[c] + accoa[c] + oaa[c]', 'KEGG id': 'R00352', 'GENE_ASSOCIATION': 'ENSG00000131473', 'GENE_LIST': 'ACLY', 'HGNC': '115', 'Gene name': 'ATP citrate lyase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.8771', 'Directionality rationale': 'ATP consumption, HumanCyc, Brenda, Literature', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.3.3.8', 'kegg.reaction': 'R00352'}, '_gpr': cobra.core.gene.GPR('ENSG00000131473'), 'subsystem': 'Reductive carboxylation / Acetyl-CoA production for biosynthesis', '_genes': {<Gene ENSG00000131473 at 0x28ab02ad0>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite coa_c at 0x17818b0d0>: -1.0, <Metabolite cit_c at 0x17b24a710>: -1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite accoa_c at 0x17b2485d0>: 1.0, <Metabolite oaa_c at 0x29f50d490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASPTA', 'name': 'R_ASPTA L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate', 'notes': {'Model Reaction Number': '60', 'EC Number': '2.6.1.1', 'SUBSYSTEM': 'Malate aspartate shuttle', 'Description': 'L-Aspartate + 2-Oxoglutarate --&gt; Oxaloacetate + L-Glutamate', 'Recon2 id': 'ASPTA', 'Recon2 formula': 'akg[c] + asp_L[c] --&gt; glu_L[c] + oaa[c]', 'KEGG id': 'R00355', 'GENE_ASSOCIATION': 'ENSG00000120053', 'GENE_LIST': 'GOT1', 'HGNC': '4432', 'Gene name': 'glutamic-oxaloacetic transaminase 1, soluble', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.9098', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '2.6.1.1', 'kegg.reaction': 'R00355'}, '_gpr': cobra.core.gene.GPR('ENSG00000120053'), 'subsystem': 'Malate aspartate shuttle', '_genes': {<Gene ENSG00000120053 at 0x176eb6c10>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite oaa_c at 0x29f50d490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MDH', 'name': 'R_MDH (S)-Malate + NAD+ --> Oxaloacetate + NADH + H+', 'notes': {'Model Reaction Number': '61', 'EC Number': '1.1.1.37', 'SUBSYSTEM': 'Malate aspartate shuttle', 'Description': '(S)-Malate + NAD+ --&gt; Oxaloacetate + NADH + H+', 'Recon2 id': 'MDH', 'Recon2 formula': 'nad[c] + mal_L[c] --&gt; h[c] + nadh[c] + oaa[c]', 'KEGG id': 'R00342', 'GENE_ASSOCIATION': 'ENSG00000014641', 'GENE_LIST': 'MDH1', 'HGNC': '6970', 'Gene name': 'malate dehydrogenase 1, NAD (soluble)', 'MitoCarta': 'TRUE', 'MitoCarta score': '12.0828', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.37', 'kegg.reaction': 'R00342'}, '_gpr': cobra.core.gene.GPR('ENSG00000014641'), 'subsystem': 'Malate aspartate shuttle', '_genes': {<Gene ENSG00000014641 at 0x176eb6f50>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite oaa_c at 0x29f50d490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AKGMALtm', 'name': 'R_AKGMALtm Malate - Oxoglutarate (OGC) Antiporter', 'notes': {'Model Reaction Number': '62', 'EC Number': 'N/A', 'SUBSYSTEM': 'Malate aspartate shuttle', 'Description': 'Malate - Oxoglutarate (OGC) Antiporter', 'Recon2 id': 'AKGMALtm', 'Recon2 formula': 'akg[m] + mal_L[c] --&gt; akg[c] + mal_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000108528', 'GENE_LIST': 'SLC25A11', 'HGNC': '10981', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; oxoglutarate carrier)', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.0399', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000108528'), 'subsystem': 'Malate aspartate shuttle', '_genes': {<Gene ENSG00000108528 at 0x176eb7290>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite akg_c at 0x16ac21410>: 1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASPGLUmB_MitoCore', 'name': 'R_ASPGLUmB_MitoCore Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)', 'notes': {'Model Reaction Number': '63', 'EC Number': 'N/A', 'SUBSYSTEM': 'Malate aspartate shuttle', 'Description': 'Glutamate/Aspartate Antiporter (AGC1) - glutamate (in) for aspartate (out)', 'Recon2 id': 'ASPGLUm', 'Recon2 formula': 'h[c] + glu_L[c] + asp_L[m] -&gt; h[m] + glu_L[m] + asp_L[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000115840 or ENSG00000004864', 'GENE_LIST': 'SLC25A12 or SLC25A13', 'HGNC': '10982 or HGNC:10983', 'Gene name': 'solute carrier family 25 (aspartate/glutamate carrier), member 12 or 13', 'MitoCarta': 'FALSE, TRUE', 'MitoCarta score': '-1.0247, 21.9806', 'Directionality rationale': 'Literature', 'HPA mRNA level in heart tissue': 'High, Low', 'HPA protein level in heart tissue': 'High, Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000115840 or ENSG00000004864'), 'subsystem': 'Malate aspartate shuttle', '_genes': {<Gene ENSG00000115840 at 0x176eb7910>, <Gene ENSG00000004864 at 0x176eb75d0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite asp_L_m at 0x1799e7e10>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite asp_L_c at 0x16a60b310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASPTAm', 'name': 'R_ASPTAm L-Aspartate + 2-Oxoglutarate --> Oxaloacetate + L-Glutamate', 'notes': {'Model Reaction Number': '64', 'EC Number': '2.6.1.1', 'SUBSYSTEM': 'Malate aspartate shuttle', 'Description': 'L-Aspartate + 2-Oxoglutarate --&gt; Oxaloacetate + L-Glutamate', 'Recon2 id': 'ASPTAm', 'Recon2 formula': 'akg[m] + asp_L[m] --&gt; glu_L[m] + oaa[m]', 'KEGG id': 'R00355', 'GENE_ASSOCIATION': 'ENSG00000125166', 'GENE_LIST': 'GOT2', 'HGNC': '4433', 'Gene name': 'glutamic-oxaloacetic transaminase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '8.8086', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.6.1.1', 'kegg.reaction': 'R00355'}, '_gpr': cobra.core.gene.GPR('ENSG00000125166'), 'subsystem': 'Malate aspartate shuttle', '_genes': {<Gene ENSG00000125166 at 0x176eb7c50>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite asp_L_m at 0x1799e7e10>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite oaa_m at 0x29e2c5610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'G3PD1', 'name': 'R_G3PD1 sn-Glycerol 3-phosphate + NAD+ --> Glycerone phosphate + NADH + H+', 'notes': {'Model Reaction Number': '65', 'EC Number': '1.1.1.8', 'SUBSYSTEM': 'Glycerol phosphate shuttle', 'Description': 'sn-Glycerol 3-phosphate + NAD+ --&gt; Glycerone phosphate + NADH + H+', 'Recon2 id': 'G3PD1', 'Recon2 formula': 'nad[c] + glyc3p[c] --&gt; h[c] + nadh[c] + dhap[c]', 'KEGG id': 'R00842', 'GENE_ASSOCIATION': 'ENSG00000167588 or ENSG00000152642', 'GENE_LIST': 'GPD1 or GPD1L', 'HGNC': '4455 or HGNC:28956', 'Gene name': 'glycerol-3-phosphate dehydrogenase 1', 'MitoCarta': 'FALSE, FALSE', 'MitoCarta score': '4.5427, 0.0944', 'HPA mRNA level in heart tissue': 'Low or High', 'HPA protein level in heart tissue': 'Medium or Not detected'}, '_annotation': {'ec-code': '1.1.1.8', 'kegg.reaction': 'R00842'}, '_gpr': cobra.core.gene.GPR('ENSG00000167588 or ENSG00000152642'), 'subsystem': 'Glycerol phosphate shuttle', '_genes': {<Gene ENSG00000152642 at 0x176e90310>, <Gene ENSG00000167588 at 0x176eb7f90>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite glyc3p_c at 0x29c7f5a10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite dhap_c at 0x29c712310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0205', 'name': 'R_r0205 sn-Glycerol 3-phosphate + Ubiquinone --> Glycerone phosphate + Ubiquinol', 'notes': {'Model Reaction Number': '66', 'EC Number': '1.1.5.3', 'SUBSYSTEM': 'Glycerol phosphate shuttle', 'Description': 'sn-Glycerol 3-phosphate + Ubiquinone --&gt; Glycerone phosphate + Ubiquinol', 'Recon2 id': 'r0205', 'Recon2 formula': 'q10[m] + glyc3p[c] -&gt; dhap[c] + q10h2[m]', 'KEGG id': 'R08657', 'GENE_ASSOCIATION': 'ENSG00000115159', 'GENE_LIST': 'GPD2', 'HGNC': '4456', 'Gene name': 'glycerol-3-phosphate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.2901', 'Directionality rationale': 'HumanCyc, Literature (e.g. David Metzler, Biochemistry 2nd ed. P 1050)', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.5.3', 'kegg.reaction': 'R08657'}, '_gpr': cobra.core.gene.GPR('ENSG00000115159'), 'subsystem': 'Glycerol phosphate shuttle', '_genes': {<Gene ENSG00000115159 at 0x176e90650>}, '_metabolites': {<Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite glyc3p_c at 0x29c7f5a10>: -1.0, <Metabolite dhap_c at 0x29c712310>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FACOAL160i', 'name': 'R_FACOAL160i ATP + Hexadecanoic acid + CoA --> AMP + Palmitoyl-CoA + Diphosphate', 'notes': {'Model Reaction Number': '67', 'EC Number': '6.2.1.3', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'ATP + Hexadecanoic acid + CoA --&gt; AMP + Palmitoyl-CoA + Diphosphate', 'Recon2 id': 'FACOAL160i', 'Recon2 formula': 'atp[c] + coa[c] + hdca[c] -&gt; amp[c] + ppi[c] + pmtcoa[c]', 'KEGG id': 'R01280', 'GENE_ASSOCIATION': 'ENSG00000151726 or ENSG00000123983 or ENSG00000068366 or ENSG00000103740 or ENSG00000164398 or ENSG00000197142 or ENSG00000130377', 'GENE_LIST': 'ACSL1 or ACSL3 or ACSL4 or ACSBG1 or ACSL6 or ACSL5 or ACSBG2', 'HGNC': '3569 or HGNC:3570 or HGNC:3571 or HGNC:29567 or HGNC:16496 or HGNC:16526 or HGNC:24174', 'Gene name': 'acyl-CoA synthetase long-chain family (various)', 'MitoCarta': 'TRUE, FALSE, FALSE, FALSE, N/A, FALSE, FALSE', 'MitoCarta score': '7.5981, -1.7369, -1.7369, 2.3512, N/A, 5.2209, -1.3535', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium or Medium or Medium or Not detected or Not detected or Low or Not detected', 'HPA protein level in heart tissue': 'High or Medium or Low or Medium or Not detected or Not detected or Not detected'}, '_annotation': {'ec-code': '6.2.1.3', 'kegg.reaction': 'R01280'}, '_gpr': cobra.core.gene.GPR('ENSG00000151726 or ENSG00000123983 or ENSG00000068366 or ENSG00000103740 or ENSG00000164398 or ENSG00000197142 or ENSG00000130377'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000130377 at 0x176e91010>, <Gene ENSG00000068366 at 0x176e91690>, <Gene ENSG00000164398 at 0x176e90cd0>, <Gene ENSG00000103740 at 0x176e91d10>, <Gene ENSG00000197142 at 0x176e91350>, <Gene ENSG00000151726 at 0x176e90990>, <Gene ENSG00000123983 at 0x176e919d0>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite coa_c at 0x17818b0d0>: -1.0, <Metabolite hdca_c at 0x164ccde10>: -1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite ppi_c at 0x29e6fdf10>: 1.0, <Metabolite pmtcoa_c at 0x29e6c50d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'C160CPT1', 'name': 'R_C160CPT1 Palmitoyl-CoA + L-Carnitine --> CoA + L-Palmitoylcarnitine', 'notes': {'Model Reaction Number': '68', 'EC Number': '2.3.1.21', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'Palmitoyl-CoA + L-Carnitine --&gt; CoA + L-Palmitoylcarnitine', 'Recon2 id': 'C160CPT1', 'Recon2 formula': 'pmtcoa[c] + crn[c] -&gt; coa[c] + pmtcrn[c]', 'KEGG id': 'R01923', 'GENE_ASSOCIATION': 'ENSG00000205560', 'GENE_LIST': 'CPT1B', 'HGNC': '2329', 'Gene name': 'carnitine palmitoyltransferase 1B (muscle)', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.4641', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.3.1.21', 'kegg.reaction': 'R01923'}, '_gpr': cobra.core.gene.GPR('ENSG00000205560'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000205560 at 0x176e92010>}, '_metabolites': {<Metabolite pmtcoa_c at 0x29e6c50d0>: -1.0, <Metabolite crn_c at 0x28b173310>: -1.0, <Metabolite coa_c at 0x17818b0d0>: 1.0, <Metabolite pmtcrn_c at 0x29e61b950>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPA', 'name': 'R_PPA Pyrophosphate+H2O-->2Orthophosphate', 'notes': {'Model Reaction Number': '69', 'EC Number': '3.6.1.1', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'Pyrophosphate+H2O--&gt;2Orthophosphate', 'Recon2 id': 'PPA', 'Recon2 formula': 'h2o[c] + ppi[c] -&gt; h[c] + 2.0 pi[c]', 'KEGG id': 'R00004', 'GENE_ASSOCIATION': 'ENSG00000180817', 'GENE_LIST': 'PPA1', 'HGNC': '9226', 'Gene name': 'pyrophosphatase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '2.4061', 'Directionality rationale': 'HumanCyc, Literature, Actual Delta G 19 kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.6.1.1', 'kegg.reaction': 'R00004'}, '_gpr': cobra.core.gene.GPR('ENSG00000180817'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000180817 at 0x176e922d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite ppi_c at 0x29e6fdf10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r2435', 'name': 'R_r2435 carnitine carrier - Carnitine out, acyl carnitine in', 'notes': {'Model Reaction Number': '70', 'EC Number': 'N/A', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'carnitine carrier - Carnitine out, acyl carnitine in', 'Recon2 id': 'r2435', 'Recon2 formula': 'crn[m] + pmtcrn[c] -&gt; crn[c] + pmtcrn[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000178537', 'GENE_LIST': 'SLC25A20', 'HGNC': '1421', 'Gene name': 'carnitine carrier', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.3039', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000178537'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000178537 at 0x176e92510>}, '_metabolites': {<Metabolite crn_m at 0x28b171a90>: -1.0, <Metabolite pmtcrn_c at 0x29e61b950>: -1.0, <Metabolite crn_c at 0x28b173310>: 1.0, <Metabolite pmtcrn_m at 0x29e657d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'C160CPT2', 'name': 'R_C160CPT2 CoA + L-Palmitoylcarnitine --> Palmitoyl-CoA + L-Carnitine', 'notes': {'Model Reaction Number': '71', 'EC Number': '2.3.1.21', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'CoA + L-Palmitoylcarnitine --&gt; Palmitoyl-CoA + L-Carnitine', 'Recon2 id': 'C160CPT2', 'Recon2 formula': 'coa[m] + pmtcrn[m] -&gt; pmtcoa[m] + crn[m]', 'KEGG id': 'R01923', 'GENE_ASSOCIATION': 'ENSG00000157184', 'GENE_LIST': 'CPT2', 'HGNC': '2330', 'Gene name': 'carnitine palmitoyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '32.7824', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.21', 'kegg.reaction': 'R01923'}, '_gpr': cobra.core.gene.GPR('ENSG00000157184'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000157184 at 0x176e92850>}, '_metabolites': {<Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite pmtcrn_m at 0x29e657d50>: -1.0, <Metabolite pmtcoa_m at 0x29e61b310>: 1.0, <Metabolite crn_m at 0x28b171a90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPAm', 'name': 'R_PPAm Pyrophosphate+H2O-->2Orthophosphate', 'notes': {'Model Reaction Number': '72', 'EC Number': '3.6.1.1', 'SUBSYSTEM': 'Carnitine shuttle', 'Description': 'Pyrophosphate+H2O--&gt;2Orthophosphate', 'Recon2 id': 'PPAm', 'Recon2 formula': 'h2o[m] + ppi[m] -&gt; h[m] + 2.0 pi[m]', 'KEGG id': 'R00004', 'GENE_ASSOCIATION': 'ENSG00000138777', 'GENE_LIST': 'PPA2', 'HGNC': '28883', 'Gene name': 'pyrophosphatase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.6578', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '3.6.1.1', 'kegg.reaction': 'R00004'}, '_gpr': cobra.core.gene.GPR('ENSG00000138777'), 'subsystem': 'Carnitine shuttle', '_genes': {<Gene ENSG00000138777 at 0x176e92b50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite ppi_m at 0x29e6fcb90>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACOT2_MitoCore', 'name': 'R_ACOT2_MitoCore Palmitoyl-CoA + H2O --> CoA + Hexadecanoic acid', 'notes': {'Model Reaction Number': '73', 'EC Number': '3.1.2.2', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'Palmitoyl-CoA + H2O --&gt; CoA + Hexadecanoic acid', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R01274', 'GENE_ASSOCIATION': 'ENSG00000119673', 'GENE_LIST': 'ACOT2', 'HGNC': '18431', 'Gene name': 'acyl-CoA thioesterase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.6365', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '3.1.2.2', 'kegg.reaction': 'R01274'}, '_gpr': cobra.core.gene.GPR('ENSG00000119673'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000119673 at 0x176e92e50>}, '_metabolites': {<Metabolite pmtcoa_m at 0x29e61b310>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite hdca_m at 0x16d63ddd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACADLC16_MitoCore', 'name': 'R_ACADLC16_MitoCore Palmitoyl-CoA + Q --> trans-Hexadec-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '74', 'EC Number': '1.3.8.8, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Palmitoyl-CoA + Q --&gt; trans-Hexadec-2-enoyl-CoA + QH2', 'Recon2 id': 'FAOXC16C16OHm', 'Recon2 formula': 'h2o[m] + pmtcoa[m] + fad[m] -&gt; fadh2[m] + 3hexdcoa[m]', 'KEGG id': 'R01279', 'GENE_ASSOCIATION': 'ENSG00000115361 and ENSG00000171503', 'GENE_LIST': 'ACADL and ETFDH', 'HGNC': '88 and HGNC:3483', 'Gene name': 'long-chain acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '28.0206, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'Low and Medium', 'HPA protein level in heart tissue': 'Low and High'}, '_annotation': {'ec-code': ['1.3.8.8', '1.5.5.1'], 'kegg.reaction': 'R01279'}, '_gpr': cobra.core.gene.GPR('ENSG00000115361 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000115361 at 0x17b75fc50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite pmtcoa_m at 0x29e61b310>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite 3hexdcoa_m at 0x16c52e9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MECR16C_MitoCore', 'name': 'R_MECR16C_MitoCore trans-Hexadec-2-enoyl-CoA + NADPH + H+ --> Palmitoyl-CoA + NADP+', 'notes': {'Model Reaction Number': '75', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'trans-Hexadec-2-enoyl-CoA + NADPH + H+ --&gt; Palmitoyl-CoA + NADP+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R01278', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'KEGG, Brenda, Literature, Delta G 16.6kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R01278'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite 3hexdcoa_m at 0x16c52e9d0>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite pmtcoa_m at 0x29e61b310>: 1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MTPC16_MitoCore', 'name': 'R_MTPC16_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Tetradecanoyl-CoA + NADH + Acetyl-CoA', 'notes': {'Model Reaction Number': '76', 'EC Number': '4.2.1.74, 1.1.1.211, 2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -&gt; Tetradecanoyl-CoA + NADH + Acetyl-CoA', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R04738, R04737, R03991', 'GENE_ASSOCIATION': 'ENSG00000084754 and ENSG00000138029', 'GENE_LIST': 'HADHA and HADHB', 'HGNC': '4801 and HGNC:4803', 'Gene name': 'Mitochondrial trifunctional protein', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '33.5571, 11.8128', 'HPA mRNA level in heart tissue': 'High and High', 'HPA protein level in heart tissue': 'Medium and Medium'}, '_annotation': {'ec-code': ['4.2.1.74', '1.1.1.211', '2.3.1.16'], 'kegg.reaction': ['R04738', 'R04737', 'R03991']}, '_gpr': cobra.core.gene.GPR('ENSG00000084754 and ENSG00000138029'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000084754 at 0x176e93990>, <Gene ENSG00000138029 at 0x176e93cd0>}, '_metabolites': {<Metabolite 3hexdcoa_m at 0x16c52e9d0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite tetd7ecoa_m at 0x29d7f0f10>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACADLC14_MitoCore', 'name': 'R_ACADLC14_MitoCore Tetradecanoyl-CoA + Q --> trans-Tetradec-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '77', 'EC Number': '1.3.8.8, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Tetradecanoyl-CoA + Q --&gt; trans-Tetradec-2-enoyl-CoA + QH2', 'Recon2 id': 'FAOXC141C141OHm', 'Recon2 formula': 'h2o[m] + fad[m] + tetd7ecoa[m] -&gt; fadh2[m] + 3tetd7ecoa[m]', 'KEGG id': 'R03990', 'GENE_ASSOCIATION': 'ENSG00000115361 and ENSG00000171503', 'GENE_LIST': 'ACADL and ETFDH', 'HGNC': '88 and HGNC:3483', 'Gene name': 'long-chain acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '28.0206, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'Low and Medium', 'HPA protein level in heart tissue': 'Low and High'}, '_annotation': {'ec-code': ['1.3.8.8', '1.5.5.1'], 'kegg.reaction': 'R03990'}, '_gpr': cobra.core.gene.GPR('ENSG00000115361 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000115361 at 0x17b75fc50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite tetd7ecoa_m at 0x29d7f0f10>: -1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite 3tetd7ecoa_m at 0x2ae45e690>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MECR14C_MitoCore', 'name': 'R_MECR14C_MitoCore trans-Tetradec-2-enoyl-CoA + NADPH + H+ --> Tetradecanoyl-CoA + NADP+', 'notes': {'Model Reaction Number': '78', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'trans-Tetradec-2-enoyl-CoA + NADPH + H+ --&gt; Tetradecanoyl-CoA + NADP+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R03989', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'Delta G 16.6kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R03989'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite 3tetd7ecoa_m at 0x2ae45e690>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite tetd7ecoa_m at 0x29d7f0f10>: 1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MTPC14_MitoCore', 'name': 'R_MTPC14_MitoCore trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -> Lauroyl-CoA + NADH + Acetyl-CoA', 'notes': {'Model Reaction Number': '79', 'EC Number': '4.2.1.74, 1.1.1.211, 2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'trans-Hexadec-2-enoyl-CoA + H2O + NAD+ + CoA -&gt; Lauroyl-CoA + NADH + Acetyl-CoA', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R04740, R04739, R03858', 'GENE_ASSOCIATION': 'ENSG00000084754 and ENSG00000138029', 'GENE_LIST': 'HADHA and HADHB', 'HGNC': '4801 and HGNC:4803', 'Gene name': 'Mitochondrial trifunctional protein', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '33.5571, 11.8128', 'HPA mRNA level in heart tissue': 'High and High', 'HPA protein level in heart tissue': 'Medium and Medium'}, '_annotation': {'ec-code': ['4.2.1.74', '1.1.1.211', '2.3.1.16'], 'kegg.reaction': ['R04740', 'R04739', 'R03858']}, '_gpr': cobra.core.gene.GPR('ENSG00000084754 and ENSG00000138029'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000084754 at 0x176e93990>, <Gene ENSG00000138029 at 0x176e93cd0>}, '_metabolites': {<Metabolite 3tetd7ecoa_m at 0x2ae45e690>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite ddcacoa_m at 0x29c713650>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1447', 'name': 'R_r1447 Lauroyl-CoA + Q --> 2-trans-Dodecenoyl-CoA + QH2', 'notes': {'Model Reaction Number': '80', 'EC Number': '1.3.8.7, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Lauroyl-CoA + Q --&gt; 2-trans-Dodecenoyl-CoA + QH2', 'Recon2 id': 'r1447', 'Recon2 formula': 'ddcacoa[m] + q10[m] -&gt; q10h2[m] + dd2coa[m]', 'KEGG id': 'R03857', 'GENE_ASSOCIATION': 'ENSG00000117054 and ENSG00000171503', 'GENE_LIST': 'ACADM and ETFDH', 'HGNC': '89 and HGNC:3483', 'Gene name': 'medium acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '30.6075, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'High and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R03857'}, '_gpr': cobra.core.gene.GPR('ENSG00000117054 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000117054 at 0x176ecc050>}, '_metabolites': {<Metabolite ddcacoa_m at 0x29c713650>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite dd2coa_m at 0x29c711c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0638', 'name': 'R_r0638 Lauroyl-CoA + NADP+ --> 2-trans-Dodecenoyl-CoA + NADPH + H+', 'notes': {'Model Reaction Number': '81', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'Lauroyl-CoA + NADP+ --&gt; 2-trans-Dodecenoyl-CoA + NADPH + H+', 'Recon2 id': 'r0638', 'Recon2 formula': 'nadp[m] + ddcacoa[m] --&gt; h[m] + nadph[m] + dd2coa[m]', 'KEGG id': 'R03856', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'Delta G 16.6kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R03856'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite ddcacoa_m at 0x29c713650>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite dd2coa_m at 0x29c711c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0660', 'name': 'R_r0660 (S)-3-Hydroxydodecanoyl-CoA --> 2-trans-Dodecenoyl-CoA + H2O', 'notes': {'Model Reaction Number': '82', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-3-Hydroxydodecanoyl-CoA --&gt; 2-trans-Dodecenoyl-CoA + H2O', 'Recon2 id': 'r0660', 'Recon2 formula': 'HC01401[m] --&gt; h2o[m] + dd2coa[m]', 'KEGG id': 'R04170', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04170'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite HC01401_m at 0x29c72d7d0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite dd2coa_m at 0x29c711c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0722', 'name': 'R_r0722 (S)-3-Hydroxydodecanoyl-CoA + NAD+ --> 3-Oxododecanoyl-CoA + NADH + H+', 'notes': {'Model Reaction Number': '83', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-3-Hydroxydodecanoyl-CoA + NAD+ --&gt; 3-Oxododecanoyl-CoA + NADH + H+', 'Recon2 id': 'r0722', 'Recon2 formula': 'nad[m] + HC01401[m] --&gt; h[m] + nadh[m] + 3oddcoa[m]', 'KEGG id': 'R04741', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R04741'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite HC01401_m at 0x29c72d7d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite 3oddcoa_m at 0x179570850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0724', 'name': 'R_r0724 Decanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxododecanoyl-CoA', 'notes': {'Model Reaction Number': '84', 'EC Number': '2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Decanoyl-CoA + Acetyl-CoA --&gt; CoA + 3-Oxododecanoyl-CoA', 'Recon2 id': 'r0724', 'Recon2 formula': 'accoa[m] + dcacoa[m] --&gt; coa[m] + 3oddcoa[m]', 'KEGG id': 'R04742', 'GENE_ASSOCIATION': 'ENSG00000167315', 'GENE_LIST': 'ACAA2', 'HGNC': '83', 'Gene name': 'acetyl-CoA acyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.5372', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.16', 'kegg.reaction': 'R04742'}, '_gpr': cobra.core.gene.GPR('ENSG00000167315'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000167315 at 0x28b1736d0>}, '_metabolites': {<Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite dcacoa_m at 0x133be95d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite 3oddcoa_m at 0x179570850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1451', 'name': 'R_r1451 Decanoyl-CoA + Q -->trans-Dec-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '85', 'EC Number': '1.3.8.7, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Decanoyl-CoA + Q --&gt;trans-Dec-2-enoyl-CoA + QH2', 'Recon2 id': 'r1451', 'Recon2 formula': 'dcacoa[m] + q10[m] -&gt; q10h2[m] + dc2coa[m]', 'KEGG id': 'R04754', 'GENE_ASSOCIATION': 'ENSG00000117054 and ENSG00000171503', 'GENE_LIST': 'ACADM and ETFDH', 'HGNC': '89 and HGNC:3483', 'Gene name': 'medium acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '30.6075, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'High and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R04754'}, '_gpr': cobra.core.gene.GPR('ENSG00000117054 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000117054 at 0x176ecc050>}, '_metabolites': {<Metabolite dcacoa_m at 0x133be95d0>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite dc2coa_m at 0x133be9c10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0735', 'name': 'R_r0735 Decanoyl-CoA + NADP+ --> trans-Dec-2-enoyl-CoA + NADPH + H+', 'notes': {'Model Reaction Number': '86', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'Decanoyl-CoA + NADP+ --&gt; trans-Dec-2-enoyl-CoA + NADPH + H+', 'Recon2 id': 'r0735', 'Recon2 formula': 'nadp[m] + dcacoa[m] --&gt; h[m] + nadph[m] + dc2coa[m]', 'KEGG id': 'R04753', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'Delta G 16.55kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R04753'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite dcacoa_m at 0x133be95d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite dc2coa_m at 0x133be9c10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0728', 'name': 'R_r0728 (S)-Hydroxydecanoyl-CoA --> trans-Dec-2-enoyl-CoA + H2O', 'notes': {'Model Reaction Number': '87', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-Hydroxydecanoyl-CoA --&gt; trans-Dec-2-enoyl-CoA + H2O', 'Recon2 id': 'r0728', 'Recon2 formula': 'HC01403[m] --&gt; h2o[m] + dc2coa[m]', 'KEGG id': 'R04744', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04744'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite HC01403_m at 0x179d65b90>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite dc2coa_m at 0x133be9c10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0726', 'name': 'R_r0726 (S)-Hydroxydecanoyl-CoA + NAD+ --> 3-Oxodecanoyl-CoA + NADH + H+', 'notes': {'Model Reaction Number': '88', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-Hydroxydecanoyl-CoA + NAD+ --&gt; 3-Oxodecanoyl-CoA + NADH + H+', 'Recon2 id': 'r0726', 'Recon2 formula': 'nad[m] + HC01403[m] --&gt; h[m] + nadh[m] + 3odcoa[m]', 'KEGG id': 'R04743', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R04743'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite HC01403_m at 0x179d65b90>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite 3odcoa_m at 0x29df668d0>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0634', 'name': 'R_r0634 Octanoyl-CoA + Acetyl-CoA --> CoA + 3-Oxodecanoyl-CoA', 'notes': {'Model Reaction Number': '89', 'EC Number': '2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Octanoyl-CoA + Acetyl-CoA --&gt; CoA + 3-Oxodecanoyl-CoA', 'Recon2 id': 'r0634', 'Recon2 formula': 'accoa[m] + occoa[m] --&gt; coa[m] + 3odcoa[m]', 'KEGG id': 'R03778', 'GENE_ASSOCIATION': 'ENSG00000167315', 'GENE_LIST': 'ACAA2', 'HGNC': '83', 'Gene name': 'acetyl-CoA acyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.5372', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.16', 'kegg.reaction': 'R03778'}, '_gpr': cobra.core.gene.GPR('ENSG00000167315'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000167315 at 0x28b1736d0>}, '_metabolites': {<Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite occoa_m at 0x29e2d1510>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite 3odcoa_m at 0x29df668d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1448', 'name': 'R_r1448 Octanoyl-CoA + Q --> trans-Oct-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '90', 'EC Number': '1.3.8.7, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Octanoyl-CoA + Q --&gt; trans-Oct-2-enoyl-CoA + QH2', 'Recon2 id': 'r1448', 'Recon2 formula': 'occoa[m] + q10[m] -&gt; q10h2[m] + HC01415[m]', 'KEGG id': 'R03777', 'GENE_ASSOCIATION': 'ENSG00000117054 and ENSG00000171503', 'GENE_LIST': 'ACADM and ETFDH', 'HGNC': '89 and HGNC:3483', 'Gene name': 'medium acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '30.6075, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'High and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R03777'}, '_gpr': cobra.core.gene.GPR('ENSG00000117054 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000117054 at 0x176ecc050>}, '_metabolites': {<Metabolite occoa_m at 0x29e2d1510>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite HC01415_m at 0x29c747d50>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0633', 'name': 'R_r0633 Octanoyl-CoA + NADP+ --> trans-Oct-2-enoyl-CoA + NADPH + H+', 'notes': {'Model Reaction Number': '91', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'Octanoyl-CoA + NADP+ --&gt; trans-Oct-2-enoyl-CoA + NADPH + H+', 'Recon2 id': 'r0633', 'Recon2 formula': 'nadp[m] + occoa[m] --&gt; h[m] + nadph[m] + HC01415[m]', 'KEGG id': 'R03776', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'Delta G 16.6kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R03776'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite occoa_m at 0x29e2d1510>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite HC01415_m at 0x29c747d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0731', 'name': 'R_r0731 trans-Oct-2-enoyl-CoA + H2O --> (S)-Hydroxyoctanoyl-CoA', 'notes': {'Model Reaction Number': '92', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'FA metabolism', 'Description': 'trans-Oct-2-enoyl-CoA + H2O --&gt; (S)-Hydroxyoctanoyl-CoA', 'Recon2 id': 'r0731', 'Recon2 formula': 'h2o[m] + HC01415[m] -&gt; HC01405[m]', 'KEGG id': 'R04746', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04746'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite HC01415_m at 0x29c747d50>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite HC01405_m at 0x29c72c150>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0730', 'name': 'R_r0730 (S)-Hydroxyoctanoyl-CoA + NAD+ --> 3-Oxooctanoyl-CoA + NADH + H+', 'notes': {'Model Reaction Number': '93', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-Hydroxyoctanoyl-CoA + NAD+ --&gt; 3-Oxooctanoyl-CoA + NADH + H+', 'Recon2 id': 'r0730', 'Recon2 formula': 'nad[m] + HC01405[m] -&gt; h[m] + nadh[m] + HC01406[m]', 'KEGG id': 'R04745', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R04745'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite HC01405_m at 0x29c72c150>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite HC01406_m at 0x29c7fa850>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0732', 'name': 'R_r0732 CoA + 3-Oxooctanoyl-CoA  --> Hexanoyl-CoA + Acetyl-CoA', 'notes': {'Model Reaction Number': '94', 'EC Number': '2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'CoA + 3-Oxooctanoyl-CoA  --&gt; Hexanoyl-CoA + Acetyl-CoA', 'Recon2 id': 'r0732', 'Recon2 formula': 'coa[m] + HC01406[m] -&gt; accoa[m] + HC01409[m]', 'KEGG id': 'R04747', 'GENE_ASSOCIATION': 'ENSG00000167315', 'GENE_LIST': 'ACAA2', 'HGNC': '83', 'Gene name': 'acetyl-CoA acyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.5372', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.16', 'kegg.reaction': 'R04747'}, '_gpr': cobra.core.gene.GPR('ENSG00000167315'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000167315 at 0x28b1736d0>}, '_metabolites': {<Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite HC01406_m at 0x29c7fa850>: -1.0, <Metabolite HC01409_m at 0x29c7fb210>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1450', 'name': 'R_r1450 Hexanoyl-CoA + Q --> trans-Hex-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '95', 'EC Number': '1.3.8.7, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Hexanoyl-CoA + Q --&gt; trans-Hex-2-enoyl-CoA + QH2', 'Recon2 id': 'r1450', 'Recon2 formula': 'q10[m] + HC01409[m] -&gt; q10h2[m] + HC01410[m]', 'KEGG id': 'R04751', 'GENE_ASSOCIATION': 'ENSG00000117054 and ENSG00000171503', 'GENE_LIST': 'ACADM and ETFDH', 'HGNC': '89 and HGNC:3483', 'Gene name': 'medium acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '30.6075, 34.3966', 'Directionality rationale': 'KEGG, Brenda, Literature', 'HPA mRNA level in heart tissue': 'High and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': ['1.3.8.7', '1.5.5.1'], 'kegg.reaction': 'R04751'}, '_gpr': cobra.core.gene.GPR('ENSG00000117054 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000117054 at 0x176ecc050>}, '_metabolites': {<Metabolite HC01409_m at 0x29c7fb210>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite HC01410_m at 0x29df9dad0>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0791', 'name': 'R_r0791 trans-Hex-2-enoyl-CoA + NADPH + H+ --> Hexanoyl-CoA + NADP+', 'notes': {'Model Reaction Number': '96', 'EC Number': '1.3.1.38', 'SUBSYSTEM': 'Fatty acid elongation in mitochondria', 'Description': 'trans-Hex-2-enoyl-CoA + NADPH + H+ --&gt; Hexanoyl-CoA + NADP+', 'Recon2 id': 'r0791', 'Recon2 formula': 'h[m] + nadph[m] + HC01410[m] --&gt; nadp[m] + HC01409[m]', 'KEGG id': 'R06985', 'GENE_ASSOCIATION': 'ENSG00000116353', 'GENE_LIST': 'MECR', 'HGNC': '19691', 'Gene name': 'mitochondrial trans-2-enoyl-CoA reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '27.6896', 'Directionality rationale': 'Delta G 16.6kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.1.38', 'kegg.reaction': 'R06985'}, '_gpr': cobra.core.gene.GPR('ENSG00000116353'), 'subsystem': 'Fatty acid elongation in mitochondria', '_genes': {<Gene ENSG00000116353 at 0x176e93710>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite HC01410_m at 0x29df9dad0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite HC01409_m at 0x29c7fb210>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0734', 'name': 'R_r0734 trans-Hex-2-enoyl-CoA + H2O --> (S)-Hydroxyhexanoyl-CoA', 'notes': {'Model Reaction Number': '97', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'FA metabolism', 'Description': 'trans-Hex-2-enoyl-CoA + H2O --&gt; (S)-Hydroxyhexanoyl-CoA', 'Recon2 id': 'r0734', 'Recon2 formula': 'h2o[m] + HC01410[m] -&gt; HC01407[m]', 'KEGG id': 'R04749', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04749'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite HC01410_m at 0x29df9dad0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite HC01407_m at 0x29c72e190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0733', 'name': 'R_r0733 (S)-Hydroxyhexanoyl-CoA + NAD+ --> 3-Oxohexanoyl-CoA + NADH + H+', 'notes': {'Model Reaction Number': '98', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-Hydroxyhexanoyl-CoA + NAD+ --&gt; 3-Oxohexanoyl-CoA + NADH + H+', 'Recon2 id': 'r0733', 'Recon2 formula': 'nad[m] + HC01407[m] -&gt; h[m] + nadh[m] + HC01408[m]', 'KEGG id': 'R04748', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R04748'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite HC01407_m at 0x29c72e190>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite HC01408_m at 0x2a2bf5b10>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0287', 'name': 'R_r0287 CoA + 3-Oxohexanoyl-CoA --> Acetyl-CoA + Butanoyl-CoA', 'notes': {'Model Reaction Number': '99', 'EC Number': '2.3.1.16', 'SUBSYSTEM': 'FA metabolism', 'Description': 'CoA + 3-Oxohexanoyl-CoA --&gt; Acetyl-CoA + Butanoyl-CoA', 'Recon2 id': 'r0287', 'Recon2 formula': 'coa[m] + HC01408[m] -&gt; accoa[m] + btcoa[m]', 'KEGG id': 'R01177', 'GENE_ASSOCIATION': 'ENSG00000167315', 'GENE_LIST': 'ACAA2', 'HGNC': '83', 'Gene name': 'acetyl-CoA acyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.5372', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.16', 'kegg.reaction': 'R01177'}, '_gpr': cobra.core.gene.GPR('ENSG00000167315'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000167315 at 0x28b1736d0>}, '_metabolites': {<Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite HC01408_m at 0x2a2bf5b10>: -1.0, <Metabolite accoa_m at 0x164b26550>: 1.0, <Metabolite btcoa_m at 0x17861ee50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1446', 'name': 'R_r1446 Butanoyl-CoA + Q --> QH2 + Crotonoyl-CoA', 'notes': {'Model Reaction Number': '100', 'EC Number': '1.3.8.1, 1.5.5.1', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Butanoyl-CoA + Q --&gt; QH2 + Crotonoyl-CoA', 'Recon2 id': 'r1446', 'Recon2 formula': 'btcoa[m] + q10[m] -&gt; b2coa[m] + q10h2[m]', 'KEGG id': 'R01175', 'GENE_ASSOCIATION': 'ENSG00000122971 and ENSG00000171503', 'GENE_LIST': 'ACADS and ETFDH', 'HGNC': '89 and HGNC:3483', 'Gene name': 'short acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '32.6104, 34.3966', 'Directionality rationale': 'Brenda, Literature, KEGG', 'HPA mRNA level in heart tissue': 'Medium and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': ['1.3.8.1', '1.5.5.1'], 'kegg.reaction': 'R01175'}, '_gpr': cobra.core.gene.GPR('ENSG00000122971 and ENSG00000171503'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000171503 at 0x176e93410>, <Gene ENSG00000122971 at 0x16c8b7150>}, '_metabolites': {<Metabolite btcoa_m at 0x17861ee50>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite b2coa_m at 0x2976c9ed0>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ECOAH1m', 'name': 'R_ECOAH1m (S)-3-Hydroxybutanoyl-CoA --> Crotonoyl-CoA + H2O', 'notes': {'Model Reaction Number': '101', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'FA metabolism', 'Description': '(S)-3-Hydroxybutanoyl-CoA --&gt; Crotonoyl-CoA + H2O', 'Recon2 id': 'ECOAH1m', 'Recon2 formula': '3hbcoa[m] --&gt; h2o[m] + b2coa[m]', 'KEGG id': 'R03026', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R03026'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite 3hbcoa_m at 0x2974f80d0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite b2coa_m at 0x2976c9ed0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HACD1m', 'name': 'R_HACD1m Acetoacetyl-CoA + NADH + H+ --> (S)-3-Hydroxybutanoyl-CoA + NAD+', 'notes': {'Model Reaction Number': '102', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'FA metabolism', 'Description': 'Acetoacetyl-CoA + NADH + H+ --&gt; (S)-3-Hydroxybutanoyl-CoA + NAD+', 'Recon2 id': 'HACD1m', 'Recon2 formula': 'h[m] + nadh[m] + aacoa[m] --&gt; nad[m] + 3hbcoa[m]', 'KEGG id': 'R01975', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R01975'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'FA metabolism', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadh_m at 0x29d2af310>: -1.0, <Metabolite aacoa_m at 0x16d328510>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: 1.0, <Metabolite 3hbcoa_m at 0x2974f80d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACACT1rm', 'name': 'R_ACACT1rm 2 Acetyl-CoA --> CoA + Acetoacetyl-CoA', 'notes': {'Model Reaction Number': '103', 'EC Number': '2.3.1.9', 'SUBSYSTEM': 'FA and ketone body metabolism / Ketogenesis', 'Description': '2 Acetyl-CoA --&gt; CoA + Acetoacetyl-CoA', 'Recon2 id': 'ACACT1rm', 'Recon2 formula': '2.0 accoa[m] -&gt; coa[m] + aacoa[m]', 'KEGG id': 'R00238', 'GENE_ASSOCIATION': 'ENSG00000075239', 'GENE_LIST': 'ACAT1', 'HGNC': '93', 'Gene name': 'acetyl-CoA acetyltransferase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0146', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.3.1.9', 'kegg.reaction': 'R00238'}, '_gpr': cobra.core.gene.GPR('ENSG00000075239'), 'subsystem': 'FA and ketone body metabolism / Ketogenesis', '_genes': {<Gene ENSG00000075239 at 0x176ecce50>}, '_metabolites': {<Metabolite accoa_m at 0x164b26550>: -2.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite aacoa_m at 0x16d328510>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACCOAC', 'name': 'R_ACCOAC ATP + Acetyl-CoA + HCO3- --> ADP + Orthophosphate + Malonyl-CoA', 'notes': {'Model Reaction Number': '104', 'EC Number': '6.4.1.2', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'ATP + Acetyl-CoA + HCO3- --&gt; ADP + Orthophosphate + Malonyl-CoA', 'Recon2 id': 'ACCOAC', 'Recon2 formula': 'atp[c] + hco3[c] + accoa[c] -&gt; h[c] + adp[c] + pi[c] + malcoa[c]', 'KEGG id': 'R00742', 'GENE_ASSOCIATION': 'ENSG00000278540 or ENSG00000076555', 'GENE_LIST': 'ACACA or ACACB', 'HGNC': '84 or HGNC:85', 'Gene name': 'acetyl-CoA carboxylase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '5.7466, 7.977', 'Directionality rationale': 'KEGG, HumanCyc, Brenda', 'HPA mRNA level in heart tissue': 'Low or Low', 'HPA protein level in heart tissue': 'High or Medium'}, '_annotation': {'ec-code': '6.4.1.2', 'kegg.reaction': 'R00742'}, '_gpr': cobra.core.gene.GPR('ENSG00000278540 or ENSG00000076555'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000278540 at 0x176ecd190>, <Gene ENSG00000076555 at 0x176ecd150>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite hco3_c at 0x29c747e50>: -1.0, <Metabolite accoa_c at 0x17b2485d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite malcoa_c at 0x29d07d150>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MCOATA', 'name': 'R_MCOATA Malonyl-CoA + Acyl-carrier protein --> CoA + Malonyl-[acyl-carrier protein]', 'notes': {'Model Reaction Number': '105', 'EC Number': '2.3.1.39', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Malonyl-CoA + Acyl-carrier protein --&gt; CoA + Malonyl-[acyl-carrier protein]', 'Recon2 id': 'MCOATA', 'Recon2 formula': 'malcoa[c] + ACP[c] --&gt; coa[c] + malACP[c]', 'KEGG id': 'R01626', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.39', 'kegg.reaction': 'R01626'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite malcoa_c at 0x29d07d150>: -1.0, <Metabolite ACP_c at 0x29d00fdd0>: -1.0, <Metabolite coa_c at 0x17818b0d0>: 1.0, <Metabolite malACP_c at 0x29d07c550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACOATA', 'name': 'R_ACOATA Acetyl-CoA + Acyl-carrier protein --> CoA + Acetyl-[acyl-carrier protein]', 'notes': {'Model Reaction Number': '106', 'EC Number': '2.3.1.38', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Acetyl-CoA + Acyl-carrier protein --&gt; CoA + Acetyl-[acyl-carrier protein]', 'Recon2 id': 'ACOATA', 'Recon2 formula': 'accoa[c] + ACP[c] --&gt; coa[c] + acACP[c]', 'KEGG id': 'R01624', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.38', 'kegg.reaction': 'R01624'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite accoa_c at 0x17b2485d0>: -1.0, <Metabolite ACP_c at 0x29d00fdd0>: -1.0, <Metabolite coa_c at 0x17818b0d0>: 1.0, <Metabolite acACP_c at 0x29d00dad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0678', 'name': 'R_r0678 Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> Acetoacetyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '107', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Acetyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --&gt; Acetoacetyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0678', 'Recon2 formula': 'h[c] + acACP[c] + malACP[c] -&gt; co2[c] + ACP[c] + HC01587[c]', 'KEGG id': 'R04355', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04355'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite acACP_c at 0x29d00dad0>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01587_c at 0x29d00ec90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0691', 'name': 'R_r0691 (3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --> Acetoacetyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '108', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxybutanoyl-[acyl-carrier protein] + NADP+ --&gt; Acetoacetyl-[acp] + NADPH + H+', 'Recon2 id': 'r0691', 'Recon2 formula': 'nadp[c] + HC01321[c] -&gt; h[c] + nadph[c] + HC01587[c]', 'KEGG id': 'R04533', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04533'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01321_c at 0x29d024410>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01587_c at 0x29d00ec90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0681', 'name': 'R_r0681 3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --> But-2-enoyl-[acyl-carrier protein] + H2O', 'notes': {'Model Reaction Number': '109', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '3R)-3-Hydroxybutanoyl-[acyl-carrier protein] --&gt; But-2-enoyl-[acyl-carrier protein] + H2O', 'Recon2 id': 'r0681', 'Recon2 formula': 'HC01321[c] -&gt; h2o[c] + HC01255[c]', 'KEGG id': 'R04428', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04428'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01321_c at 0x29d024410>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01255_c at 0x29d074190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0682', 'name': 'R_r0682 Butyryl-[acp] + NADP+ --> But-2-enoyl-[acyl-carrier protein] + NADPH + H+', 'notes': {'Model Reaction Number': '110', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Butyryl-[acp] + NADP+ --&gt; But-2-enoyl-[acyl-carrier protein] + NADPH + H+', 'Recon2 id': 'r0682', 'Recon2 formula': 'nadp[c] + HC01588[c] -&gt; h[c] + nadph[c] + HC01255[c]', 'KEGG id': 'R04430', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04430'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01588_c at 0x29d0278d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01255_c at 0x29d074190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0760', 'name': 'R_r0760 Butyryl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '111', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Butyryl-[acp] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxohexanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0760', 'Recon2 formula': 'h[c] + malACP[c] + HC01588[c] -&gt; co2[c] + ACP[c] + HC01589[c]', 'KEGG id': 'R04952', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04952'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite HC01588_c at 0x29d0278d0>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01589_c at 0x29d077f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0761', 'name': 'R_r0761 (R)-3-Hydroxyhexanoyl-[acp] + NADP+ --> 3-Oxohexanoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '112', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(R)-3-Hydroxyhexanoyl-[acp] + NADP+ --&gt; 3-Oxohexanoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0761', 'Recon2 formula': 'nadp[c] + HC01590[c] -&gt; h[c] + nadph[c] + HC01589[c]', 'KEGG id': 'R04953', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04953'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01590_c at 0x29d063890>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01589_c at 0x29d077f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0762', 'name': 'R_r0762 (R)-3-Hydroxyhexanoyl-[acp] --> trans-Hex-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '113', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(R)-3-Hydroxyhexanoyl-[acp] --&gt; trans-Hex-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0762', 'Recon2 formula': 'HC01590[c] -&gt; h2o[c] + HC01591[c]', 'KEGG id': 'R04954', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04954'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01590_c at 0x29d063890>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01591_c at 0x29d061cd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0763', 'name': 'R_r0763 Hexanoyl-[acp] + NADP+ --> trans-Hex-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '114', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Hexanoyl-[acp] + NADP+ --&gt; trans-Hex-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0763', 'Recon2 formula': 'nadp[c] + HC01592[c] -&gt; h[c] + nadph[c] + HC01591[c]', 'KEGG id': 'R04956', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04956'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01592_c at 0x29d061110>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01591_c at 0x29d061cd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0764', 'name': 'R_r0764 Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '115', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Hexanoyl-[acp] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxooctanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0764', 'Recon2 formula': 'h[c] + malACP[c] + HC01592[c] -&gt; co2[c] + ACP[c] + HC01593[c]', 'KEGG id': 'R04957', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04957'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite HC01592_c at 0x29d061110>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01593_c at 0x28b1d8790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0694', 'name': 'R_r0694 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxooctanoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '116', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] + NADP+ --&gt; 3-Oxooctanoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0694', 'Recon2 formula': 'nadp[c] + HC01323[c] -&gt; h[c] + nadph[c] + HC01593[c]', 'KEGG id': 'R04536', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04536'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01323_c at 0x28b1d86d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01593_c at 0x28b1d8790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0695', 'name': 'R_r0695 (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --> trans-Oct-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '117', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxyoctanoyl-[acyl-carrier protein] --&gt; trans-Oct-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0695', 'Recon2 formula': 'HC01323[c] -&gt; h2o[c] + HC01594[c]', 'KEGG id': 'R04537', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04537'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01323_c at 0x28b1d86d0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01594_c at 0x28b1d8f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0765', 'name': 'R_r0765 Octanoyl-[acp] + NADP+ --> trans-Oct-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '118', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Octanoyl-[acp] + NADP+ --&gt; trans-Oct-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0765', 'Recon2 formula': 'nadp[c] + HC01595[c] -&gt; h[c] + nadph[c] + HC01594[c]', 'KEGG id': 'R04959', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04959'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01595_c at 0x28b1d8e50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01594_c at 0x28b1d8f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0766', 'name': 'R_r0766 Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '119', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Octanoyl-[acp] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxodecanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0766', 'Recon2 formula': 'h[c] + malACP[c] + HC01595[c] -&gt; co2[c] + ACP[c] + HC01596[c]', 'KEGG id': 'R04960', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04960'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite HC01595_c at 0x28b1d8e50>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01596_c at 0x28b1d96d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0692', 'name': 'R_r0692 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxodecanoyl-[acp] + NADPH', 'notes': {'Model Reaction Number': '120', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxydecanoyl-[acyl-carrier protein] + NADP+ --&gt; 3-Oxodecanoyl-[acp] + NADPH', 'Recon2 id': 'r0692', 'Recon2 formula': 'nadp[c] + HC01322[c] -&gt; h[c] + nadph[c] + HC01596[c]', 'KEGG id': 'R04534', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04534'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01322_c at 0x28b1d9610>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01596_c at 0x28b1d96d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0693', 'name': 'R_r0693 (3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --> trans-Dec-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '121', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxydecanoyl-[acyl-carrier protein] --&gt; trans-Dec-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0693', 'Recon2 formula': 'HC01322[c] -&gt; h2o[c] + HC01597[c]', 'KEGG id': 'R04535', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04535'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01322_c at 0x28b1d9610>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01597_c at 0x28b1d9ed0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0767', 'name': 'R_r0767 Decanoyl-[acp] + NADP+ --> trans-Dec-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '122', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Decanoyl-[acp] + NADP+ --&gt; trans-Dec-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0767', 'Recon2 formula': 'nadp[c] + HC01598[c] -&gt; h[c] + nadph[c] + HC01597[c]', 'KEGG id': 'R04962', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04962'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01598_c at 0x28b1da190>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01597_c at 0x28b1d9ed0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0768', 'name': 'R_r0768 Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '123', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Decanoyl-[acp] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxododecanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0768', 'Recon2 formula': 'h[c] + malACP[c] + HC01598[c] -&gt; co2[c] + ACP[c] + HC01599[c]', 'KEGG id': 'R04963', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04963'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite HC01598_c at 0x28b1da190>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01599_c at 0x28b1da250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0769', 'name': 'R_r0769 (R)-3-Hydroxydodecanoyl-[acp] + NADP+ --> 3-Oxododecanoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '124', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(R)-3-Hydroxydodecanoyl-[acp] + NADP+ --&gt; 3-Oxododecanoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0769', 'Recon2 formula': 'nadp[c] + HC01600[c] -&gt; h[c] + nadph[c] + HC01599[c]', 'KEGG id': 'R04964', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04964'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01600_c at 0x28b1daa10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01599_c at 0x28b1da250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0770', 'name': 'R_r0770 (R)-3-Hydroxydodecanoyl-[acp] --> trans-Dodec-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '125', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(R)-3-Hydroxydodecanoyl-[acp] --&gt; trans-Dodec-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0770', 'Recon2 formula': 'HC01600[c] -&gt; h2o[c] + HC01601[c]', 'KEGG id': 'R04965', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04965'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01600_c at 0x28b1daa10>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01601_c at 0x28b1da9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0712', 'name': 'R_r0712 Dodecanoyl-[acyl-carrier protein] + NADP+ --> trans-Dodec-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '126', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Dodecanoyl-[acyl-carrier protein] + NADP+ --&gt; trans-Dodec-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0712', 'Recon2 formula': 'nadp[c] + ddcaACP[c] -&gt; h[c] + nadph[c] + HC01601[c]', 'KEGG id': 'R04725', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04725'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite ddcaACP_c at 0x28b1db250>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01601_c at 0x28b1da9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0713', 'name': 'R_r0713 Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --> 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '127', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Dodecanoyl-[acyl-carrier protein] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxotetradecanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0713', 'Recon2 formula': 'h[c] + ddcaACP[c] + malACP[c] -&gt; co2[c] + ACP[c] + HC01602[c]', 'KEGG id': 'R04726', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04726'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite ddcaACP_c at 0x28b1db250>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01602_c at 0x28b1db650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0701', 'name': 'R_r0701 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --> 3-Oxotetradecanoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '128', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] + NADP+ --&gt; 3-Oxotetradecanoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0701', 'Recon2 formula': 'nadp[c] + HC01335[c] -&gt; nadph[c] + HC01602[c]', 'KEGG id': 'R04566', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04566'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01335_c at 0x28b1dbad0>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01602_c at 0x28b1db650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0702', 'name': 'R_r0702 (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --> trans-Tetradec-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '129', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein] --&gt; trans-Tetradec-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0702', 'Recon2 formula': 'HC01335[c] -&gt; h2o[c] + HC01603[c]', 'KEGG id': 'R04568', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04568'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01335_c at 0x28b1dbad0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01603_c at 0x28b1dba10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0771', 'name': 'R_r0771 Tetradecanoyl-[acp] + NADP+ --> trans-Tetradec-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '130', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Tetradecanoyl-[acp] + NADP+ --&gt; trans-Tetradec-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0771', 'Recon2 formula': 'nadp[c] + myrsACP[c] -&gt; h[c] + nadph[c] + HC01603[c]', 'KEGG id': 'R04967', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04967'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite myrsACP_c at 0x28b1dbe10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01603_c at 0x28b1dba10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0772', 'name': 'R_r0772 Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --> 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein', 'notes': {'Model Reaction Number': '131', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Tetradecanoyl-[acp] + Malonyl-[acyl-carrier protein] --&gt; 3-Oxohexadecanoyl-[acp] + CO2 + Acyl-carrier protein', 'Recon2 id': 'r0772', 'Recon2 formula': 'h[c] + myrsACP[c] + malACP[c] -&gt; co2[c] + ACP[c] + HC01605[c]', 'KEGG id': 'R04968', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04968'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite myrsACP_c at 0x28b1dbe10>: -1.0, <Metabolite malACP_c at 0x29d07c550>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite HC01605_c at 0x28ab68710>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0696', 'name': 'R_r0696 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --> 3-Oxohexadecanoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '132', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] + NADP+ --&gt; 3-Oxohexadecanoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0696', 'Recon2 formula': 'nadp[c] + HC01326[c] -&gt; h[c] + nadph[c] + HC01605[c]', 'KEGG id': 'R04543', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04543'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite HC01326_c at 0x28ab68610>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01605_c at 0x28ab68710>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0697', 'name': 'R_r0697 (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --> trans-Hexadec-2-enoyl-[acp] + H2O', 'notes': {'Model Reaction Number': '133', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': '(3R)-3-Hydroxypalmitoyl-[acyl-carrier protein] --&gt; trans-Hexadec-2-enoyl-[acp] + H2O', 'Recon2 id': 'r0697', 'Recon2 formula': 'HC01326[c] -&gt; h2o[c] + HC01606[c]', 'KEGG id': 'R04544', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04544'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite HC01326_c at 0x28ab68610>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite HC01606_c at 0x28ab68f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0773', 'name': 'R_r0773 Hexadecanoyl-[acp] + NADP+ --> trans-Hexadec-2-enoyl-[acp] + NADPH + H+', 'notes': {'Model Reaction Number': '134', 'EC Number': '2.3.1.85', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Hexadecanoyl-[acp] + NADP+ --&gt; trans-Hexadec-2-enoyl-[acp] + NADPH + H+', 'Recon2 id': 'r0773', 'Recon2 formula': 'nadp[c] + palmACP[c] -&gt; h[c] + nadph[c] + HC01606[c]', 'KEGG id': 'R04970', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '2.3.1.85', 'kegg.reaction': 'R04970'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite palmACP_c at 0x28ab69350>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite HC01606_c at 0x28ab68f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FA160ACPH', 'name': 'R_FA160ACPH Hexadecanoyl-[acp] + H2O --> Acyl-carrier protein + Hexadecanoic acid', 'notes': {'Model Reaction Number': '135', 'EC Number': '3.1.2.14', 'SUBSYSTEM': 'FA synthesis - cytosolic', 'Description': 'Hexadecanoyl-[acp] + H2O --&gt; Acyl-carrier protein + Hexadecanoic acid', 'Recon2 id': 'FA160ACPH', 'Recon2 formula': 'h2o[c] + palmACP[c] --&gt; h[c] + ACP[c] + hdca[c]', 'KEGG id': 'R01706', 'GENE_ASSOCIATION': 'ENSG00000169710', 'GENE_LIST': 'FASN', 'HGNC': '3594', 'Gene name': 'fatty acid synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.0321', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'None'}, '_annotation': {'ec-code': '3.1.2.14', 'kegg.reaction': 'R01706'}, '_gpr': cobra.core.gene.GPR('ENSG00000169710'), 'subsystem': 'FA synthesis - cytosolic', '_genes': {<Gene ENSG00000169710 at 0x176ecd690>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite palmACP_c at 0x28ab69350>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite ACP_c at 0x29d00fdd0>: 1.0, <Metabolite hdca_c at 0x164ccde10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FACOAL40im', 'name': 'R_FACOAL40im ATP + Butanoic acid + CoA --> AMP + Diphosphate + Butanoyl-CoA', 'notes': {'Model Reaction Number': '136', 'EC Number': '6.2.1.2', 'SUBSYSTEM': 'Butanoate metabolism', 'Description': 'ATP + Butanoic acid + CoA --&gt; AMP + Diphosphate + Butanoyl-CoA', 'Recon2 id': 'FACOAL40im', 'Recon2 formula': 'atp[m] + coa[m] + but[m] -&gt; amp[m] + ppi[m] + btcoa[m]', 'KEGG id': 'R01176', 'GENE_ASSOCIATION': 'ENSG00000005187 or ENSG00000173124', 'GENE_LIST': 'ACSM3 or ACSM6', 'HGNC': '10522 or HGNC:31665', 'Gene name': 'acyl-CoA synthetase medium-chain family member 3', 'MitoCarta': 'TRUE, FALSE', 'MitoCarta score': '16.9674, -0.7431', 'Directionality rationale': 'ATP consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low or Not detected', 'HPA protein level in heart tissue': 'Medium or Low'}, '_annotation': {'ec-code': '6.2.1.2', 'kegg.reaction': 'R01176'}, '_gpr': cobra.core.gene.GPR('ENSG00000005187 or ENSG00000173124'), 'subsystem': 'Butanoate metabolism', '_genes': {<Gene ENSG00000005187 at 0x176ecdc90>, <Gene ENSG00000173124 at 0x176ecd6d0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite but_m at 0x16b692610>: -1.0, <Metabolite amp_m at 0x2975598d0>: 1.0, <Metabolite ppi_m at 0x29e6fcb90>: 1.0, <Metabolite btcoa_m at 0x17861ee50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'BDHm', 'name': 'R_BDHm (R)-3-Hydroxybutanoate + NAD+ --> Acetoacetate + NADH + H+', 'notes': {'Model Reaction Number': '137', 'EC Number': '1.1.1.30', 'SUBSYSTEM': 'Ketone bodies - degradation', 'Description': '(R)-3-Hydroxybutanoate + NAD+ --&gt; Acetoacetate + NADH + H+', 'Recon2 id': 'BDHm', 'Recon2 formula': 'nad[m] + bhb[m] --&gt; h[m] + nadh[m] + acac[m]', 'KEGG id': 'R01361', 'GENE_ASSOCIATION': 'ENSG00000161267', 'GENE_LIST': 'BDH1', 'HGNC': '1027', 'Gene name': '3-hydroxybutyrate dehydrogenase, type 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '23.9841', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.30', 'kegg.reaction': 'R01361'}, '_gpr': cobra.core.gene.GPR('ENSG00000161267'), 'subsystem': 'Ketone bodies - degradation', '_genes': {<Gene ENSG00000161267 at 0x29d0ab610>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite bhb_m at 0x178671290>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite acac_m at 0x2a02a65d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OCOAT1m', 'name': 'R_OCOAT1m Succinyl-CoA + Acetoacetate --> Succinate + Acetoacetyl-CoA', 'notes': {'Model Reaction Number': '138', 'EC Number': '2.8.3.5', 'SUBSYSTEM': 'Ketone bodies - degradation', 'Description': 'Succinyl-CoA + Acetoacetate --&gt; Succinate + Acetoacetyl-CoA', 'Recon2 id': 'OCOAT1m', 'Recon2 formula': 'acac[m] + succoa[m] -&gt; aacoa[m] + succ[m]', 'KEGG id': 'R00410', 'GENE_ASSOCIATION': 'ENSG00000083720', 'GENE_LIST': 'OXCT1', 'HGNC': '8527', 'Gene name': '3-oxoacid CoA transferase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.3333', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.8.3.5', 'kegg.reaction': 'R00410'}, '_gpr': cobra.core.gene.GPR('ENSG00000083720'), 'subsystem': 'Ketone bodies - degradation', '_genes': {<Gene ENSG00000083720 at 0x176ece210>}, '_metabolites': {<Metabolite acac_m at 0x2a02a65d0>: -1.0, <Metabolite succoa_m at 0x29d7ddd10>: -1.0, <Metabolite aacoa_m at 0x16d328510>: 1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HMGCOASim', 'name': 'R_HMGCOASim Acetyl-CoA + H2O + Acetoacetyl-CoA --> S)-3-Hydroxy-3-methylglutaryl-CoA + CoA', 'notes': {'Model Reaction Number': '139', 'EC Number': '2.3.3.10', 'SUBSYSTEM': 'Ketogenesis', 'Description': 'Acetyl-CoA + H2O + Acetoacetyl-CoA --&gt; S)-3-Hydroxy-3-methylglutaryl-CoA + CoA', 'Recon2 id': 'HMGCOASim', 'Recon2 formula': 'h2o[m] + accoa[m] + aacoa[m] -&gt; h[m] + coa[m] + hmgcoa[m]', 'KEGG id': 'R01978', 'GENE_ASSOCIATION': 'ENSG00000134240', 'GENE_LIST': 'HMGCS2', 'HGNC': '5008', 'Gene name': '3-hydroxy-3-methylglutaryl-CoA synthase 2 (mitochondrial)', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.6859', 'Directionality rationale': 'Heart doesn&apos;t synthesize ketone bodies', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.3.3.10', 'kegg.reaction': 'R01978'}, '_gpr': cobra.core.gene.GPR('ENSG00000134240'), 'subsystem': 'Ketogenesis', '_genes': {<Gene ENSG00000134240 at 0x176ece550>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite aacoa_m at 0x16d328510>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite hmgcoa_m at 0x29c7d2f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'HMGLm', 'name': 'R_HMGLm (S)-3-Hydroxy-3-methylglutaryl-CoA --> Acetyl-CoA + Acetoacetate', 'notes': {'Model Reaction Number': '140', 'EC Number': '4.1.3.4', 'SUBSYSTEM': 'Ketogenesis / Leucine degradation', 'Description': '(S)-3-Hydroxy-3-methylglutaryl-CoA --&gt; Acetyl-CoA + Acetoacetate', 'Recon2 id': 'HMGLm', 'Recon2 formula': 'hmgcoa[m] -&gt; accoa[m] + acac[m]', 'KEGG id': 'R01360', 'GENE_ASSOCIATION': 'ENSG00000117305', 'GENE_LIST': 'HMGCL', 'HGNC': '5005', 'Gene name': '3-hydroxymethyl-3-methylglutaryl-CoA lyase', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.9768', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.1.3.4', 'kegg.reaction': 'R01360'}, '_gpr': cobra.core.gene.GPR('ENSG00000117305'), 'subsystem': 'Ketogenesis / Leucine degradation', '_genes': {<Gene ENSG00000117305 at 0x176ece890>}, '_metabolites': {<Metabolite hmgcoa_m at 0x29c7d2f10>: -1.0, <Metabolite accoa_m at 0x164b26550>: 1.0, <Metabolite acac_m at 0x2a02a65d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LEUTAm', 'name': 'R_LEUTAm L-Leucine + 2-Oxoglutarate --> 4-Methyl-2-oxopentanoate + L-Glutamate', 'notes': {'Model Reaction Number': '141', 'EC Number': '2.6.1.42', 'SUBSYSTEM': 'Leucine degradation', 'Description': 'L-Leucine + 2-Oxoglutarate --&gt; 4-Methyl-2-oxopentanoate + L-Glutamate', 'Recon2 id': 'LEUTAm', 'Recon2 formula': 'akg[m] + leu_L[m] --&gt; glu_L[m] + 4mop[m]', 'KEGG id': 'R01090', 'GENE_ASSOCIATION': 'ENSG00000105552', 'GENE_LIST': '42542', 'HGNC': '977', 'Gene name': 'branched chain amino-acid transaminase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.2627', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.6.1.42', 'kegg.reaction': 'R01090'}, '_gpr': cobra.core.gene.GPR('ENSG00000105552'), 'subsystem': 'Leucine degradation', '_genes': {<Gene ENSG00000105552 at 0x176eceb90>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite leu_L_m at 0x29d0ce590>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite 4mop_m at 0x2ae45e790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OIVD1m', 'name': 'R_OIVD1m 4-Methyl-2-oxopentanoate + NAD+ + CoA -> NADH + CO2 + 3-Methylbutanoyl-CoA', 'notes': {'Model Reaction Number': '142', 'EC Number': '1.2.4.4, 2.3.1.168, 1.8.1.4', 'SUBSYSTEM': 'Leucine degradation', 'Description': '4-Methyl-2-oxopentanoate + NAD+ + CoA -&gt; NADH + CO2 + 3-Methylbutanoyl-CoA', 'Recon2 id': 'OIVD1m', 'Recon2 formula': 'nad[m] + coa[m] + 4mop[m] -&gt; nadh[m] + co2[m] + ivcoa[m]', 'KEGG id': 'R07601, R7602, R04097, R07618', 'GENE_ASSOCIATION': 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'GENE_LIST': 'BCKDHA and BCKDHB and DBT and DLD', 'HGNC': '986 and HGNC:987 and HGNC:2698 and HGNC:2898', 'Gene name': 'branched-chain -ketoacid dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '34.1724, 32.7672, 39.2448, 31.2307', 'Directionality rationale': 'CO2 production, Delta G &gt;5kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium and Medium and Low and High', 'HPA protein level in heart tissue': 'High and Low and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.4', '2.3.1.168', '1.8.1.4'], 'kegg.reaction': ['R07601', 'R7602', 'R04097', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140'), 'subsystem': 'Leucine degradation', '_genes': {<Gene ENSG00000083123 at 0x176ecee90>, <Gene ENSG00000137992 at 0x176ecf1d0>, <Gene ENSG00000248098 at 0x176ecf4d0>, <Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite 4mop_m at 0x2ae45e790>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite ivcoa_m at 0x164635fd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0655', 'name': 'R_r0655 3-Methylbutanoyl-CoA + Q --> 3-Methylcrotonyl-CoA + QH2', 'notes': {'Model Reaction Number': '143', 'EC Number': '1.3.8.4', 'SUBSYSTEM': 'Leucine degradation', 'Description': '3-Methylbutanoyl-CoA + Q --&gt; 3-Methylcrotonyl-CoA + QH2', 'Recon2 id': 'r0655', 'Recon2 formula': 'ivcoa[m] + q10[m] --&gt; 3mb2coa[m] + q10h2[m]', 'KEGG id': 'R04095', 'GENE_ASSOCIATION': 'ENSG00000128928', 'GENE_LIST': 'IVD', 'HGNC': '6186', 'Gene name': 'isovaleryl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.807', 'Directionality rationale': 'HumanCyc, KEGG, Literature', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.3.8.4', 'kegg.reaction': 'R04095'}, '_gpr': cobra.core.gene.GPR('ENSG00000128928'), 'subsystem': 'Leucine degradation', '_genes': {<Gene ENSG00000128928 at 0x2a296eb50>}, '_metabolites': {<Metabolite ivcoa_m at 0x164635fd0>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite 3mb2coa_m at 0x164cce850>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MCCCrm', 'name': 'R_MCCCrm ATP + 3-Methylcrotonyl-CoA + HCO3 ---> ADP + Orthophosphate + 3-Methylglutaconyl-CoA', 'notes': {'Model Reaction Number': '144', 'EC Number': '6.4.1.4', 'SUBSYSTEM': 'Leucine degradation', 'Description': 'ATP + 3-Methylcrotonyl-CoA + HCO3 ---&gt; ADP + Orthophosphate + 3-Methylglutaconyl-CoA', 'Recon2 id': 'MCCCrm', 'Recon2 formula': 'atp[m] + hco3[m] + 3mb2coa[m] --&gt; h[m] + adp[m] + pi[m] + 3mgcoa[m]', 'KEGG id': 'R04138', 'GENE_ASSOCIATION': 'ENSG00000078070 and ENSG00000131844', 'GENE_LIST': 'MCCC1 and MCCC2', 'HGNC': '6936 and HGNC:6937', 'Gene name': 'methylcrotonoyl-CoA carboxylase', 'MitoCarta': 'TRUE', 'MitoCarta score': '24.4144', 'Directionality rationale': 'Phosphate production, ATP consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium and Medium', 'HPA protein level in heart tissue': 'High and High'}, '_annotation': {'ec-code': '6.4.1.4', 'kegg.reaction': 'R04138'}, '_gpr': cobra.core.gene.GPR('ENSG00000078070 and ENSG00000131844'), 'subsystem': 'Leucine degradation', '_genes': {<Gene ENSG00000078070 at 0x176ecfd50>, <Gene ENSG00000131844 at 0x16cc472d0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite hco3_m at 0x29c745e50>: -1.0, <Metabolite 3mb2coa_m at 0x164cce850>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite 3mgcoa_m at 0x179766b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MGCHrm', 'name': 'R_MGCHrm 3-Methylglutaconyl-CoA + H2O --> (S)-3-Hydroxy-3-methylglutaryl-CoA', 'notes': {'Model Reaction Number': '145', 'EC Number': '4.2.1.18', 'SUBSYSTEM': 'Leucine degradation', 'Description': '3-Methylglutaconyl-CoA + H2O --&gt; (S)-3-Hydroxy-3-methylglutaryl-CoA', 'Recon2 id': 'MGCHrm', 'Recon2 formula': 'h2o[m] + 3mgcoa[m] --&gt; hmgcoa[m]', 'KEGG id': 'R02085', 'GENE_ASSOCIATION': 'ENSG00000148090', 'GENE_LIST': 'AUH', 'HGNC': '890', 'Gene name': 'Methylglutaconyl-CoA hydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '32.2168', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.2.1.18', 'kegg.reaction': 'R02085'}, '_gpr': cobra.core.gene.GPR('ENSG00000148090'), 'subsystem': 'Leucine degradation', '_genes': {<Gene ENSG00000148090 at 0x176ea00d0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite 3mgcoa_m at 0x179766b90>: -1.0, <Metabolite hmgcoa_m at 0x29c7d2f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ILETAm', 'name': 'R_ILETAm L-Isoleucine + 2-Oxoglutarate --> (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate', 'notes': {'Model Reaction Number': '146', 'EC Number': '2.6.1.42', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': 'L-Isoleucine + 2-Oxoglutarate --&gt; (S)-3-Methyl-2-oxopentanoicacid + L-Glutamate', 'Recon2 id': 'ILETAm', 'Recon2 formula': 'akg[m] + ile_L[m] --&gt; glu_L[m] + 3mop[m]', 'KEGG id': 'R02199', 'GENE_ASSOCIATION': 'ENSG00000105552', 'GENE_LIST': 'BCAT2', 'HGNC': '977', 'Gene name': 'branched chain amino-acid transaminase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.2627', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.6.1.42', 'kegg.reaction': 'R02199'}, '_gpr': cobra.core.gene.GPR('ENSG00000105552'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000105552 at 0x176eceb90>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite ile_L_m at 0x2ae04e010>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite 3mop_m at 0x17982cb10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OIVD3m', 'name': 'R_OIVD3m NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -> NADH + CO2 + (S)-2-Methylbutanoyl-CoA', 'notes': {'Model Reaction Number': '147', 'EC Number': '1.2.4.4, 2.3.1.168, 1.8.1.4', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': 'NAD+ + CoA + (S)-3-Methyl-2-oxopentanoicacid -&gt; NADH + CO2 + (S)-2-Methylbutanoyl-CoA', 'Recon2 id': 'OIVD3m', 'Recon2 formula': 'nad[m] + coa[m] + 3mop[m] -&gt; nadh[m] + co2[m] + 2mbcoa[m]', 'KEGG id': 'R07603, R07604, R03174, R07618', 'GENE_ASSOCIATION': 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'GENE_LIST': 'BCKDHA and BCKDHB and DBT and DLD', 'HGNC': '986 and HGNC:987 and HGNC:2698 and HGNC:2898', 'Gene name': 'branched-chain -ketoacid dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '34.1724, 32.7672, 39.2448, 31.2307', 'Directionality rationale': 'CO2 production, Delta G &gt;5kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium and Medium and Low and High', 'HPA protein level in heart tissue': 'High and Low and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.4', '2.3.1.168', '1.8.1.4'], 'kegg.reaction': ['R07603', 'R07604', 'R03174', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000083123 at 0x176ecee90>, <Gene ENSG00000137992 at 0x176ecf1d0>, <Gene ENSG00000248098 at 0x176ecf4d0>, <Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite 3mop_m at 0x17982cb10>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite 2mbcoa_m at 0x1796c8550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0603', 'name': 'R_r0603 (S)-2-Methylbutanoyl-CoA + Q --> 2-Methylbut-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '148', 'EC Number': '1.3.8.1', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': '(S)-2-Methylbutanoyl-CoA + Q --&gt; 2-Methylbut-2-enoyl-CoA + QH2', 'Recon2 id': 'r0603', 'Recon2 formula': '2mbcoa[m] + q10[m] --&gt; 2mb2coa[m] + q10h2[m]', 'KEGG id': 'R03172', 'GENE_ASSOCIATION': 'ENSG00000122971', 'GENE_LIST': 'ACADS', 'HGNC': '90', 'Gene name': 'acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '32.6104', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.8.1', 'kegg.reaction': 'R03172'}, '_gpr': cobra.core.gene.GPR('ENSG00000122971'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000122971 at 0x16c8b7150>}, '_metabolites': {<Metabolite 2mbcoa_m at 0x1796c8550>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite 2mb2coa_m at 0x17b23ea50>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ECOAH9m', 'name': 'R_ECOAH9m 2-Methylbut-2-enoyl-CoA + H2O --> (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA', 'notes': {'Model Reaction Number': '149', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': '2-Methylbut-2-enoyl-CoA + H2O --&gt; (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA', 'Recon2 id': 'ECOAH9m', 'Recon2 formula': 'h2o[m] + 2mb2coa[m] --&gt; 3hmbcoa[m]', 'KEGG id': 'R04204', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase, short chain', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04204'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite 2mb2coa_m at 0x17b23ea50>: -1.0, <Metabolite 3hmbcoa_m at 0x16d65e890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HACD9m', 'name': 'R_HACD9m (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --> 2-Methylacetoacetyl-CoA + NADH + H+', 'notes': {'Model Reaction Number': '150', 'EC Number': '1.1.1.35', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': '(2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA + NAD+ --&gt; 2-Methylacetoacetyl-CoA + NADH + H+', 'Recon2 id': 'HACD9m', 'Recon2 formula': 'nad[m] + 3hmbcoa[m] --&gt; h[m] + nadh[m] + 2maacoa[m]', 'KEGG id': 'R04203', 'GENE_ASSOCIATION': 'ENSG00000138796', 'GENE_LIST': 'HADH', 'HGNC': '4799', 'Gene name': '3-hydroxyacyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.0734', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.35', 'kegg.reaction': 'R04203'}, '_gpr': cobra.core.gene.GPR('ENSG00000138796'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite 3hmbcoa_m at 0x16d65e890>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite 2maacoa_m at 0x1796e90d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACACT10m', 'name': 'R_ACACT10m CoA + 2-Methylacetoacetyl-CoA --> Propanoyl-CoA + Acetyl-CoA', 'notes': {'Model Reaction Number': '151', 'EC Number': '2.3.1.16', 'SUBSYSTEM': 'Isoleucine degradation', 'Description': 'CoA + 2-Methylacetoacetyl-CoA --&gt; Propanoyl-CoA + Acetyl-CoA', 'Recon2 id': 'ACACT10m', 'Recon2 formula': 'coa[m] + 2maacoa[m] --&gt; accoa[m] + ppcoa[m]', 'KEGG id': 'R00927', 'GENE_ASSOCIATION': 'ENSG00000167315', 'GENE_LIST': 'ACAA2', 'HGNC': '83', 'Gene name': 'acetyl-CoA acyltransferase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '22.5372', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.16', 'kegg.reaction': 'R00927'}, '_gpr': cobra.core.gene.GPR('ENSG00000167315'), 'subsystem': 'Isoleucine degradation', '_genes': {<Gene ENSG00000167315 at 0x28b1736d0>}, '_metabolites': {<Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite 2maacoa_m at 0x1796e90d0>: -1.0, <Metabolite accoa_m at 0x164b26550>: 1.0, <Metabolite ppcoa_m at 0x29e61e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'VALTAm', 'name': 'R_VALTAm L-Valine + 2-Oxoglutarate --> 3-Methyl-2-oxobutanoicacid + L-Glutamate', 'notes': {'Model Reaction Number': '152', 'EC Number': '2.6.1.42', 'SUBSYSTEM': 'Valine degradation', 'Description': 'L-Valine + 2-Oxoglutarate --&gt; 3-Methyl-2-oxobutanoicacid + L-Glutamate', 'Recon2 id': 'VALTAm', 'Recon2 formula': 'akg[m] + val_L[m] --&gt; glu_L[m] + 3mob[m]', 'KEGG id': 'R01214', 'GENE_ASSOCIATION': 'ENSG00000105552', 'GENE_LIST': 'BCAT2', 'HGNC': '977', 'Gene name': 'branched chain amino-acid transaminase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.2627', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.6.1.42', 'kegg.reaction': 'R01214'}, '_gpr': cobra.core.gene.GPR('ENSG00000105552'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000105552 at 0x176eceb90>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite val_L_m at 0x29d753d50>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite 3mob_m at 0x2a2586190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OIVD2m', 'name': 'R_OIVD2m NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -> NADH + CO2 + 2-Methylpropanoyl-CoA', 'notes': {'Model Reaction Number': '153', 'EC Number': '1.2.4.4, 2.3.1.168, 1.8.1.4', 'SUBSYSTEM': 'Valine degradation', 'Description': 'NAD+ + CoA + 3-Methyl-2-oxobutanoicacid -&gt; NADH + CO2 + 2-Methylpropanoyl-CoA', 'Recon2 id': 'OIVD2m', 'Recon2 formula': 'nad[m] + coa[m] + 3mob[m] -&gt; nadh[m] + co2[m] + ibcoa[m]', 'KEGG id': 'R07599, R07600, R02662, R07618', 'GENE_ASSOCIATION': 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'GENE_LIST': 'BCKDHA and BCKDHB and DBT and DLD', 'HGNC': '986 and HGNC:987 and HGNC:2698 and HGNC:2898', 'Gene name': 'branched-chain -ketoacid dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '34.1724, 32.7672, 39.2448, 31.2307', 'Directionality rationale': 'CO2 production, Delta G &gt;5kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium and Medium and Low and High', 'HPA protein level in heart tissue': 'High and Low and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.4', '2.3.1.168', '1.8.1.4'], 'kegg.reaction': ['R07599', 'R07600', 'R02662', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000083123 at 0x176ecee90>, <Gene ENSG00000137992 at 0x176ecf1d0>, <Gene ENSG00000248098 at 0x176ecf4d0>, <Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite 3mob_m at 0x2a2586190>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite ibcoa_m at 0x29c7d3c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0560', 'name': 'R_r0560 2-Methylpropanoyl-CoA + Q --> 2-Methylprop-2-enoyl-CoA + QH2', 'notes': {'Model Reaction Number': '154', 'EC Number': '1.3.8.1', 'SUBSYSTEM': 'Valine degradation', 'Description': '2-Methylpropanoyl-CoA + Q --&gt; 2-Methylprop-2-enoyl-CoA + QH2', 'Recon2 id': 'r0560', 'Recon2 formula': 'ibcoa[m] + q10[m] -&gt; 2mp2coa[m] + q10h2[m]', 'KEGG id': 'R02661', 'GENE_ASSOCIATION': 'ENSG00000122971', 'GENE_LIST': 'ACADS', 'HGNC': '90', 'Gene name': 'acyl-CoA dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '32.6104', 'Directionality rationale': 'KEGG, Literature', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.8.1', 'kegg.reaction': 'R02661'}, '_gpr': cobra.core.gene.GPR('ENSG00000122971'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000122971 at 0x16c8b7150>}, '_metabolites': {<Metabolite ibcoa_m at 0x29c7d3c90>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite 2mp2coa_m at 0x29c7a03d0>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ECOAH12m', 'name': 'R_ECOAH12m 2-Methylprop-2-enoyl-CoA + H2O --> (S)-3-Hydroxyisobutyryl-CoA', 'notes': {'Model Reaction Number': '155', 'EC Number': '4.2.1.17', 'SUBSYSTEM': 'Valine degradation', 'Description': '2-Methylprop-2-enoyl-CoA + H2O --&gt; (S)-3-Hydroxyisobutyryl-CoA', 'Recon2 id': 'ECOAH12m', 'Recon2 formula': 'h2o[m] + 2mp2coa[m] --&gt; 3hibutcoa[m]', 'KEGG id': 'R04224', 'GENE_ASSOCIATION': 'ENSG00000127884', 'GENE_LIST': 'ECHS1', 'HGNC': '3151', 'Gene name': 'enoyl CoA hydratase, short chain', 'MitoCarta': 'TRUE', 'MitoCarta score': '33.7065', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '4.2.1.17', 'kegg.reaction': 'R04224'}, '_gpr': cobra.core.gene.GPR('ENSG00000127884'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000127884 at 0x28b16f610>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite 2mp2coa_m at 0x29c7a03d0>: -1.0, <Metabolite 3hibutcoa_m at 0x16d950090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': '3HBCOAHLm', 'name': 'R_3HBCOAHLm (S)-3-Hydroxyisobutyryl-CoA + H2O --> CoA + (S)-3-Hydroxyisobutyrate', 'notes': {'Model Reaction Number': '156', 'EC Number': '3.2.1.4', 'SUBSYSTEM': 'Valine degradation', 'Description': '(S)-3-Hydroxyisobutyryl-CoA + H2O --&gt; CoA + (S)-3-Hydroxyisobutyrate', 'Recon2 id': '3HBCOAHLm', 'Recon2 formula': 'h2o[m] + 3hibutcoa[m] -&gt; h[m] + coa[m] + 3hmp[m]', 'KEGG id': 'R05064', 'GENE_ASSOCIATION': 'ENSG00000198130', 'GENE_LIST': 'HIBCH', 'HGNC': '4908', 'Gene name': '3-hydroxyisobutyryl-CoA hydrolase', 'MitoCarta': 'TRUE', 'MitoCarta score': '29.7511', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.2.1.4', 'kegg.reaction': 'R05064'}, '_gpr': cobra.core.gene.GPR('ENSG00000198130'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000198130 at 0x176ea03d0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite 3hibutcoa_m at 0x16d950090>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite 3hmp_m at 0x179757490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HIBDm', 'name': 'R_HIBDm (S)-3-Hydroxyisobutyrate + NAD+ --> (S)-Methylmalonatesemialdehyde + NADH + H+', 'notes': {'Model Reaction Number': '157', 'EC Number': '1.1.1.31 or 1.1.1.35', 'SUBSYSTEM': 'Valine degradation', 'Description': '(S)-3-Hydroxyisobutyrate + NAD+ --&gt; (S)-Methylmalonatesemialdehyde + NADH + H+', 'Recon2 id': 'HIBDm', 'Recon2 formula': 'nad[m] + 3hmp[m] --&gt; h[m] + nadh[m] + 2mop[m]', 'KEGG id': 'R05066', 'GENE_ASSOCIATION': 'ENSG00000106049 or ENSG00000138796', 'GENE_LIST': 'HIBADH or HADH', 'HGNC': '4907 or HGNC:4799', 'Gene name': '3-hydroxyisobutyrate dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '24.0796, 28.0734', 'HPA mRNA level in heart tissue': 'Medium or High', 'HPA protein level in heart tissue': 'High or High'}, '_annotation': {'ec-code': ['1.1.1.31', '1.1.1.35'], 'kegg.reaction': 'R05066'}, '_gpr': cobra.core.gene.GPR('ENSG00000106049 or ENSG00000138796'), 'subsystem': 'Valine degradation', '_genes': {<Gene ENSG00000106049 at 0x176ea0710>, <Gene ENSG00000138796 at 0x176ecc610>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite 3hmp_m at 0x179757490>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite 2mop_m at 0x16da26b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACCOALm', 'name': 'R_ACCOALm ATP + Propanoate + CoA --> AMP + Diphosphate + Propanoyl-CoA', 'notes': {'Model Reaction Number': '158', 'EC Number': '6.2.1.1', 'SUBSYSTEM': 'Propanoate metabolism', 'Description': 'ATP + Propanoate + CoA --&gt; AMP + Diphosphate + Propanoyl-CoA', 'Recon2 id': 'ACCOALm', 'Recon2 formula': 'atp[m] + coa[m] + ppa[m] -&gt; amp[m] + ppi[m] + ppcoa[m]', 'KEGG id': 'R00925', 'GENE_ASSOCIATION': 'ENSG00000111058', 'GENE_LIST': 'ACSS3', 'HGNC': '24723', 'Gene name': 'acyl-CoA synthetase short-chain family member 3', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.6981', 'Directionality rationale': 'ATP consumption, Delta G &gt;20kJ/mol', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '6.2.1.1', 'kegg.reaction': 'R00925'}, '_gpr': cobra.core.gene.GPR('ENSG00000111058'), 'subsystem': 'Propanoate metabolism', '_genes': {<Gene ENSG00000111058 at 0x176ea0a50>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite ppa_m at 0x29e604450>: -1.0, <Metabolite amp_m at 0x2975598d0>: 1.0, <Metabolite ppi_m at 0x29e6fcb90>: 1.0, <Metabolite ppcoa_m at 0x29e61e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMSAD1m', 'name': 'R_MMSAD1m (S)-Methylmalonatesemialdehyde + CoA + NAD+ --> Propanoyl-CoA + CO2 + NADH + H+', 'notes': {'Model Reaction Number': '159', 'EC Number': '1.2.1.27', 'SUBSYSTEM': 'Propanoate metabolism', 'Description': '(S)-Methylmalonatesemialdehyde + CoA + NAD+ --&gt; Propanoyl-CoA + CO2 + NADH + H+', 'Recon2 id': 'MMSAD1m', 'Recon2 formula': 'nad[m] + coa[m] + 2mop[m] -&gt; nadh[m] + co2[m] + ppcoa[m]', 'KEGG id': 'R00935', 'GENE_ASSOCIATION': 'ENSG00000119711', 'GENE_LIST': 'ALDH6A1', 'HGNC': '7179', 'Gene name': 'aldehyde dehydrogenase 6 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '23.2735', 'Directionality rationale': 'CO2 production, Delta G 7.1kJ/mol, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.2.1.27', 'kegg.reaction': 'R00935'}, '_gpr': cobra.core.gene.GPR('ENSG00000119711'), 'subsystem': 'Propanoate metabolism', '_genes': {<Gene ENSG00000119711 at 0x176ea0d90>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite 2mop_m at 0x16da26b10>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite ppcoa_m at 0x29e61e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPCOACm', 'name': 'R_PPCOACm ATP + Propanoyl-CoA + HCO3- --> ADP + Orthophosphate + (S)-Methylmalonyl-CoA', 'notes': {'Model Reaction Number': '160', 'EC Number': '6.4.1.3', 'SUBSYSTEM': 'Propanoate metabolism', 'Description': 'ATP + Propanoyl-CoA + HCO3- --&gt; ADP + Orthophosphate + (S)-Methylmalonyl-CoA', 'Recon2 id': 'PPCOACm', 'Recon2 formula': 'atp[m] + hco3[m] + ppcoa[m] -&gt; h[m] + adp[m] + pi[m] + mmcoa_S[m]', 'KEGG id': 'R01859', 'GENE_ASSOCIATION': 'ENSG00000175198 and ENSG00000114054', 'GENE_LIST': 'PCCA and PCCB', 'HGNC': '8653 and HGNC:8654', 'Gene name': 'propionyl CoA carboxylase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '24.4144, 24.8143', 'Directionality rationale': 'Phosphate production, ATP consumption, KEGG', 'HPA mRNA level in heart tissue': 'Medium and Medium', 'HPA protein level in heart tissue': 'Medium and Medium'}, '_annotation': {'ec-code': '6.4.1.3', 'kegg.reaction': 'R01859'}, '_gpr': cobra.core.gene.GPR('ENSG00000175198 and ENSG00000114054'), 'subsystem': 'Propanoate metabolism', '_genes': {<Gene ENSG00000114054 at 0x176ea1090>, <Gene ENSG00000175198 at 0x176ea1350>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite hco3_m at 0x29c745e50>: -1.0, <Metabolite ppcoa_m at 0x29e61e590>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite mmcoa_S_m at 0x29d0ed090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMEm', 'name': 'R_MMEm (R)-Methylmalonyl-CoA --> (S)-Methylmalonyl-CoA', 'notes': {'Model Reaction Number': '161', 'EC Number': '5.1.99.1', 'SUBSYSTEM': 'Propanoate metabolism', 'Description': '(R)-Methylmalonyl-CoA --&gt; (S)-Methylmalonyl-CoA', 'Recon2 id': 'MMEm', 'Recon2 formula': 'mmcoa_R[m] --&gt; mmcoa_S[m]', 'KEGG id': 'R02765', 'GENE_ASSOCIATION': 'ENSG00000124370', 'GENE_LIST': 'MCEE', 'HGNC': '16732', 'Gene name': 'methylmalonyl CoA epimerase', 'MitoCarta': 'TRUE', 'MitoCarta score': '19.672', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '5.1.99.1', 'kegg.reaction': 'R02765'}, '_gpr': cobra.core.gene.GPR('ENSG00000124370'), 'subsystem': 'Propanoate metabolism', '_genes': {<Gene ENSG00000124370 at 0x176ea1650>}, '_metabolites': {<Metabolite mmcoa_R_m at 0x29d0e4590>: -1.0, <Metabolite mmcoa_S_m at 0x29d0ed090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMMm', 'name': 'R_MMMm (R)-Methylmalonyl-CoA --> Succinyl-CoA', 'notes': {'Model Reaction Number': '162', 'EC Number': '5.4.99.2', 'SUBSYSTEM': 'Propanoate metabolism', 'Description': '(R)-Methylmalonyl-CoA --&gt; Succinyl-CoA', 'Recon2 id': 'MMMm', 'Recon2 formula': 'mmcoa_R[m] --&gt; succoa[m]', 'KEGG id': 'R00833', 'GENE_ASSOCIATION': 'ENSG00000146085', 'GENE_LIST': 'MUT', 'HGNC': '7526', 'Gene name': 'methylmalonyl CoA mutase', 'MitoCarta': 'TRUE', 'MitoCarta score': '19.6453', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '5.4.99.2', 'kegg.reaction': 'R00833'}, '_gpr': cobra.core.gene.GPR('ENSG00000146085'), 'subsystem': 'Propanoate metabolism', '_genes': {<Gene ENSG00000146085 at 0x176ea1950>}, '_metabolites': {<Metabolite mmcoa_R_m at 0x29d0e4590>: -1.0, <Metabolite succoa_m at 0x29d7ddd10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMCDm', 'name': 'R_MMCDm (S)-Methylmalonyl-CoA --> Propanoyl-CoA + CO2', 'notes': {'Model Reaction Number': '163', 'EC Number': '4.1.1.41', 'SUBSYSTEM': 'Propanoate synthesis', 'Description': '(S)-Methylmalonyl-CoA --&gt; Propanoyl-CoA + CO2', 'Recon2 id': 'MMCDm', 'Recon2 formula': 'h[m] + mmcoa_S[m] -&gt; co2[m] + ppcoa[m]', 'KEGG id': 'R00923', 'GENE_ASSOCIATION': 'ENSG00000093144', 'GENE_LIST': 'ECHDC1', 'HGNC': '21489', 'Gene name': 'ethylmalonyl-CoA decarboxylase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.2579', 'Directionality rationale': 'CO2 production, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.1.1.41', 'kegg.reaction': 'R00923'}, '_gpr': cobra.core.gene.GPR('ENSG00000093144'), 'subsystem': 'Propanoate synthesis', '_genes': {<Gene ENSG00000093144 at 0x176ea1c10>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite mmcoa_S_m at 0x29d0ed090>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite ppcoa_m at 0x29e61e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'RE2649M', 'name': 'R_RE2649M H20 + Propanoyl-CoA  -> Propanoate + CoA + H+', 'notes': {'Model Reaction Number': '164', 'EC Number': '3.1.2.2', 'SUBSYSTEM': 'Propanoate synthesis', 'Description': 'H20 + Propanoyl-CoA  -&gt; Propanoate + CoA + H+', 'Recon2 id': 'RE2649M', 'Recon2 formula': 'h2o[m] + ppcoa[m] -&gt; h[m] + coa[m] + ppa[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000119673', 'GENE_LIST': 'ACOT2', 'HGNC': '18431', 'Gene name': 'acyl-CoA thioesterase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.6365', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '3.1.2.2'}, '_gpr': cobra.core.gene.GPR('ENSG00000119673'), 'subsystem': 'Propanoate synthesis', '_genes': {<Gene ENSG00000119673 at 0x176e92e50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite ppcoa_m at 0x29e61e590>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite ppa_m at 0x29e604450>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'THRD_L', 'name': 'R_THRD_L L-Threonine --> 2-Oxobutanoate + Ammonia', 'notes': {'Model Reaction Number': '165', 'EC Number': '4.3.1.19', 'SUBSYSTEM': 'Threonine degradation', 'Description': 'L-Threonine --&gt; 2-Oxobutanoate + Ammonia', 'Recon2 id': 'THRD_L', 'Recon2 formula': 'thr_L[c] -&gt; nh4[c] + 2obut[c]', 'KEGG id': 'R00996', 'GENE_ASSOCIATION': 'ENSG00000135094 or ENSG00000139410', 'GENE_LIST': 'SDS or SDSL', 'HGNC': '10691 or HGNC:30404', 'Gene name': 'serine dehydratase', 'MitoCarta': 'FALSE', 'MitoCarta score': '3.7304', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 5.7kJ/mol', 'HPA mRNA level in heart tissue': 'Not detected or Low', 'HPA protein level in heart tissue': 'Medium or High'}, '_annotation': {'ec-code': '4.3.1.19', 'kegg.reaction': 'R00996'}, '_gpr': cobra.core.gene.GPR('ENSG00000135094 or ENSG00000139410'), 'subsystem': 'Threonine degradation', '_genes': {<Gene ENSG00000139410 at 0x176ea2290>, <Gene ENSG00000135094 at 0x176ea1f50>}, '_metabolites': {<Metabolite thr_L_c at 0x29d729490>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite 2obut_c at 0x17b23ead0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1155', 'name': 'R_r1155 2-oxobutanoate transport into mitochondrion', 'notes': {'Model Reaction Number': '166', 'EC Number': 'N/A', 'SUBSYSTEM': 'Threonine degradation', 'Description': '2-oxobutanoate transport into mitochondrion', 'Recon2 id': 'r1155', 'Recon2 formula': 'h[c] + 2obut[c] -&gt; h[m] + 2obut[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Directionality rationale': 'Modelling purposes'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Threonine degradation', '_genes': set(), '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.18, <Metabolite 2obut_c at 0x17b23ead0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.18, <Metabolite 2obut_m at 0x291930390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1154', 'name': 'R_r1154 2-Oxobutanoate + CoA + NAD+ --> Propanoyl-CoA + NADH + CO2', 'notes': {'Model Reaction Number': '167', 'EC Number': '1.2.4.4 + 2.3.1.168 + 1.8.1.4', 'SUBSYSTEM': 'Threonine and Methionine degradation', 'Description': '2-Oxobutanoate + CoA + NAD+ --&gt; Propanoyl-CoA + NADH + CO2', 'Recon2 id': 'r1154', 'Recon2 formula': 'nad[m] + 2obut[m] + coa[m] --&gt; nadh[m] + co2[m] + ppcoa[m]', 'KEGG id': 'R10996, R10997, R10997, R07618', 'GENE_ASSOCIATION': 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'GENE_LIST': 'BCKDHA and BCKDHB and DBT and DLD', 'HGNC': '986 and HGNC:987 and HGNC:2698 and HGNC:2898', 'Gene name': 'branched-chain -ketoacid dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '34.1724, 32.7672, 39.2448, 31.2307', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Medium and Medium and Low and High', 'HPA protein level in heart tissue': 'High and Low and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.4', '2.3.1.168', '1.8.1.4'], 'kegg.reaction': ['R10996', 'R10997', 'R07618']}, '_gpr': cobra.core.gene.GPR('ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140'), 'subsystem': 'Threonine and Methionine degradation', '_genes': {<Gene ENSG00000083123 at 0x176ecee90>, <Gene ENSG00000137992 at 0x176ecf1d0>, <Gene ENSG00000248098 at 0x176ecf4d0>, <Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite 2obut_m at 0x291930390>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite ppcoa_m at 0x29e61e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2HBO', 'name': 'R_2HBO 2-Hydroxybutanoic acid + NAD+ --> 2-Oxobutanoate + NADH + H+', 'notes': {'Model Reaction Number': '168', 'EC Number': '1.1.1.27', 'SUBSYSTEM': 'Threonine and Methionine degradation', 'Description': '2-Hydroxybutanoic acid + NAD+ --&gt; 2-Oxobutanoate + NADH + H+', 'Recon2 id': '2HBO', 'Recon2 formula': 'nad[c] + 2hb[c] --&gt; h[c] + nadh[c] + 2obut[c]', 'KEGG id': 'R01000', 'GENE_ASSOCIATION': 'ENSG00000111716', 'GENE_LIST': 'LDHB', 'HGNC': '6541', 'Gene name': 'L-lactate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.2343', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.27', 'kegg.reaction': 'R01000'}, '_gpr': cobra.core.gene.GPR('ENSG00000111716'), 'subsystem': 'Threonine and Methionine degradation', '_genes': {<Gene ENSG00000111716 at 0x28abc9990>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite 2hb_c at 0x2a2585d10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite 2obut_c at 0x17b23ead0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'METAT', 'name': 'R_METAT ATP + L-Methionine + H2O --> Orthophosphate + Diphosphate + S-Adenosyl-L-methionine', 'notes': {'Model Reaction Number': '169', 'EC Number': '2.5.1.6', 'SUBSYSTEM': 'Methionine degradation', 'Description': 'ATP + L-Methionine + H2O --&gt; Orthophosphate + Diphosphate + S-Adenosyl-L-methionine', 'Recon2 id': 'METAT', 'Recon2 formula': 'h2o[c] + atp[c] + met_L[c] -&gt; pi[c] + amet[c] + ppi[c]', 'KEGG id': 'R00177', 'GENE_ASSOCIATION': 'ENSG00000168906', 'GENE_LIST': 'MAT2A', 'HGNC': '6904', 'Gene name': 'methionine adenosyltransferase II, alpha', 'MitoCarta': 'FALSE', 'MitoCarta score': '-8.2811', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.5.1.6', 'kegg.reaction': 'R00177'}, '_gpr': cobra.core.gene.GPR('ENSG00000168906'), 'subsystem': 'Methionine degradation', '_genes': {<Gene ENSG00000168906 at 0x176ea2590>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite met_L_c at 0x29d011150>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite amet_c at 0x2974f9190>: 1.0, <Metabolite ppi_c at 0x29e6fdf10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'METAT2_MitoCore', 'name': 'R_METAT2_MitoCore S-Adenosyl-L-methionine + demethylated methyl donor --> S-Adenosyl-L-homocysteine + methyl donor', 'notes': {'Model Reaction Number': '170', 'EC Number': '2.1.1.-', 'SUBSYSTEM': 'Methionine degradation', 'Description': 'S-Adenosyl-L-methionine + demethylated methyl donor --&gt; S-Adenosyl-L-homocysteine + methyl donor', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'N/A', 'GENE_LIST': 'N/A', 'Gene name': 'Multiple methyltransferases', 'Directionality rationale': 'KEGG, HumanCyc'}, '_annotation': {'ec-code': '2.1.1.-'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Methionine degradation', '_genes': set(), '_metabolites': {<Metabolite amet_c at 0x2974f9190>: -1.0, <Metabolite ahcys_c at 0x2ae078a90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AHC', 'name': 'R_AHC S-Adenosyl-L-homocysteine + H2O --> Adenosine + L-Homocysteine', 'notes': {'Model Reaction Number': '171', 'EC Number': '3.3.1.1', 'SUBSYSTEM': 'Methionine degradation', 'Description': 'S-Adenosyl-L-homocysteine + H2O --&gt; Adenosine + L-Homocysteine', 'Recon2 id': 'AHC', 'Recon2 formula': 'h2o[c] + ahcys[c] --&gt; adn[c] + hcys_L[c]', 'KEGG id': 'R00192', 'GENE_ASSOCIATION': 'ENSG00000101444 or ENSG00000168710 or ENSG00000158467', 'GENE_LIST': 'AHCY or AHCYL1 or AHCYL2', 'HGNC': '343 or HGNC:344 or HGNC:22204', 'Gene name': 'adenosylhomocysteinase', 'MitoCarta': 'FALSE, TRUE, FALSE', 'MitoCarta score': '2.8985, 7.642, -4.7411', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 5.7kJ/mol', 'HPA mRNA level in heart tissue': 'Medium or Medium or Low', 'HPA protein level in heart tissue': 'Low or Not detected or Not detected'}, '_annotation': {'ec-code': '3.3.1.1', 'kegg.reaction': 'R00192'}, '_gpr': cobra.core.gene.GPR('ENSG00000101444 or ENSG00000168710 or ENSG00000158467'), 'subsystem': 'Methionine degradation', '_genes': {<Gene ENSG00000158467 at 0x176ea2e90>, <Gene ENSG00000101444 at 0x176ea2b90>, <Gene ENSG00000168710 at 0x176ea25d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite ahcys_c at 0x2ae078a90>: -1.0, <Metabolite adn_c at 0x17b4c0590>: 1.0, <Metabolite hcys_L_c at 0x29c7453d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ADNK1', 'name': 'R_ADNK1 ATP + Adenosine --> ADP + AMP', 'notes': {'Model Reaction Number': '172', 'EC Number': '2.7.1.20', 'SUBSYSTEM': 'Methionine degradation', 'Description': 'ATP + Adenosine --&gt; ADP + AMP', 'Recon2 id': 'ADNK1', 'Recon2 formula': 'atp[c] + adn[c] -&gt; h[c] + adp[c] + amp[c]', 'KEGG id': 'R00185', 'GENE_ASSOCIATION': 'ENSG00000156110', 'GENE_LIST': 'ADK', 'HGNC': '257', 'Gene name': 'adenosine kinase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-7.1255', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.7.1.20', 'kegg.reaction': 'R00185'}, '_gpr': cobra.core.gene.GPR('ENSG00000156110'), 'subsystem': 'Methionine degradation', '_genes': {<Gene ENSG00000156110 at 0x176ea3150>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite adn_c at 0x17b4c0590>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYSTS', 'name': 'R_CYSTS L-Serine + L-Homocysteine --> L-Cystathionine + H2O', 'notes': {'Model Reaction Number': '173', 'EC Number': '4.2.1.22', 'SUBSYSTEM': 'Methionine and Serine degradation', 'Description': 'L-Serine + L-Homocysteine --&gt; L-Cystathionine + H2O', 'Recon2 id': 'CYSTS', 'Recon2 formula': 'hcys_L[c] + ser_L[c] -&gt; h2o[c] + cyst_L[c]', 'KEGG id': 'R01290', 'GENE_ASSOCIATION': 'ENSG00000160200', 'GENE_LIST': 'CBS', 'HGNC': '1550', 'Gene name': 'cystathionine-beta-synthase', 'MitoCarta': 'FALSE', 'MitoCarta score': '3.7481', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.2.1.22', 'kegg.reaction': 'R01290'}, '_gpr': cobra.core.gene.GPR('ENSG00000160200'), 'subsystem': 'Methionine and Serine degradation', '_genes': {<Gene ENSG00000160200 at 0x176ea3410>}, '_metabolites': {<Metabolite hcys_L_c at 0x29c7453d0>: -1.0, <Metabolite ser_L_c at 0x29d206f50>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite cyst_L_c at 0x133bd0b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYSTGL', 'name': 'R_CYSTGL L-Cystathionine + H2O --> L-Cysteine + Ammonia + 2-Oxobutanoate', 'notes': {'Model Reaction Number': '174', 'EC Number': '4.4.1.1', 'SUBSYSTEM': 'Methionine and Serine degradation', 'Description': 'L-Cystathionine + H2O --&gt; L-Cysteine + Ammonia + 2-Oxobutanoate', 'Recon2 id': 'CYSTGL', 'Recon2 formula': 'h2o[c] + cyst_L[c] -&gt; nh4[c] + 2obut[c] + cys_L[c]', 'KEGG id': 'R01001', 'GENE_ASSOCIATION': 'ENSG00000116761', 'GENE_LIST': 'CTH', 'HGNC': '2501', 'Gene name': 'cystathionine gamma-lyase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.3441', 'Directionality rationale': 'HumanCyc, NH4 production, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.4.1.1', 'kegg.reaction': 'R01001'}, '_gpr': cobra.core.gene.GPR('ENSG00000116761'), 'subsystem': 'Methionine and Serine degradation', '_genes': {<Gene ENSG00000116761 at 0x176ea3750>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite cyst_L_c at 0x133bd0b90>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite 2obut_c at 0x17b23ead0>: 1.0, <Metabolite cys_L_c at 0x29f32a9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYSO', 'name': 'R_CYSO L-Cysteine + Oxygen --> 3-Sulfino-L-alanine', 'notes': {'Model Reaction Number': '175', 'EC Number': '1.13.11.20', 'SUBSYSTEM': 'Cysteine degradation I', 'Description': 'L-Cysteine + Oxygen --&gt; 3-Sulfino-L-alanine', 'Recon2 id': 'CYSO', 'Recon2 formula': 'o2[c] + cys_L[c] -&gt; 2.0 h[c] + 3sala[c]', 'KEGG id': 'R00893', 'GENE_ASSOCIATION': 'ENSG00000129596', 'GENE_LIST': 'CDO1', 'HGNC': '1795', 'Gene name': 'cysteine dioxygenase type 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.1848', 'Directionality rationale': 'HumanCyc, Oxygen consumption, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.13.11.20', 'kegg.reaction': 'R00893'}, '_gpr': cobra.core.gene.GPR('ENSG00000129596'), 'subsystem': 'Cysteine degradation I', '_genes': {<Gene ENSG00000129596 at 0x176ea3a50>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite cys_L_c at 0x29f32a9d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite 3sala_c at 0x2ae05e6d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '3SALATAi', 'name': 'R_3SALATAi 3-Sulfino-L-alanine + 2-Oxoglutarate --> 3-Sulfinylpyruvate + L-Glutamate', 'notes': {'Model Reaction Number': '176', 'EC Number': '2.6.1.1', 'SUBSYSTEM': 'Cysteine degradation I', 'Description': '3-Sulfino-L-alanine + 2-Oxoglutarate --&gt; 3-Sulfinylpyruvate + L-Glutamate', 'Recon2 id': '3SALATAi', 'Recon2 formula': 'h[c] + akg[c] + 3sala[c] -&gt; glu_L[c] + 3snpyr[c]', 'KEGG id': 'R02619', 'GENE_ASSOCIATION': 'ENSG00000120053', 'GENE_LIST': 'GOT1', 'HGNC': '4432', 'Gene name': 'glutamic-oxaloacetic transaminase 1, soluble', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.9098', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '2.6.1.1', 'kegg.reaction': 'R02619'}, '_gpr': cobra.core.gene.GPR('ENSG00000120053'), 'subsystem': 'Cysteine degradation I', '_genes': {<Gene ENSG00000120053 at 0x176eb6c10>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite 3sala_c at 0x2ae05e6d0>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite 3snpyr_c at 0x17927a650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': '3SPYRSP', 'name': 'R_3SPYRSP 3-Sulfinylpyruvate + H2O --> Sulfite + Pyruvate', 'notes': {'Model Reaction Number': '177', 'EC Number': 'Non-Enzymatic', 'SUBSYSTEM': 'Cysteine degradation I', 'Description': '3-Sulfinylpyruvate + H2O --&gt; Sulfite + Pyruvate', 'Recon2 id': '3SPYRSP', 'Recon2 formula': 'h2o[c] + 3snpyr[c] -&gt; h[c] + pyr[c] + so3[c]', 'KEGG id': 'R04861', 'GENE_ASSOCIATION': 'Non-Enzymatic', 'GENE_LIST': 'Non-Enzymatic', 'Gene name': 'Non-Enzymatic'}, '_annotation': {'kegg.reaction': 'R04861'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Cysteine degradation I', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite 3snpyr_c at 0x17927a650>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite so3_c at 0x297508cd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYSTA', 'name': 'R_CYSTA L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate', 'notes': {'Model Reaction Number': '178', 'EC Number': '2.6.1.1', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'L-Cysteine + 2-Oxoglutarate --&gt; Mercaptopyruvate + Glutamate', 'Recon2 id': 'CYSTA', 'Recon2 formula': 'akg[c] + cys_L[c] --&gt; glu_L[c] + mercppyr[c]', 'KEGG id': 'R00895', 'GENE_ASSOCIATION': 'ENSG00000120053', 'GENE_LIST': 'GOT1', 'HGNC': '4432', 'Gene name': 'glutamic-oxaloacetic transaminase 1, soluble', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.9098', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '2.6.1.1', 'kegg.reaction': 'R00895'}, '_gpr': cobra.core.gene.GPR('ENSG00000120053'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000120053 at 0x176eb6c10>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite cys_L_c at 0x29f32a9d0>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite mercppyr_c at 0x2ae48d0d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYSTAm', 'name': 'R_CYSTAm L-Cysteine + 2-Oxoglutarate --> Mercaptopyruvate + Glutamate', 'notes': {'Model Reaction Number': '179', 'EC Number': '2.6.1.1', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'L-Cysteine + 2-Oxoglutarate --&gt; Mercaptopyruvate + Glutamate', 'Recon2 id': 'CYSTAm', 'Recon2 formula': 'akg[m] + cys_L[m] --&gt; glu_L[m] + mercppyr[m]', 'KEGG id': 'R00895', 'GENE_ASSOCIATION': 'ENSG00000125166', 'GENE_LIST': 'GOT2', 'HGNC': '4433', 'Gene name': 'glutamic-oxaloacetic transaminase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '8.8086', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.6.1.1', 'kegg.reaction': 'R00895'}, '_gpr': cobra.core.gene.GPR('ENSG00000125166'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000125166 at 0x176eb7c50>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite cys_L_m at 0x13356df50>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite mercppyr_m at 0x29d010f90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MCPST', 'name': 'R_MCPST Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate', 'notes': {'Model Reaction Number': '180', 'EC Number': '2.8.1.2', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'Hydrogen cyanide + Mercaptopyruvate --&gt; Thiocyanate + Pyruvate', 'Recon2 id': 'MCPST', 'Recon2 formula': 'cyan[c] + mercppyr[c] -&gt; h[c] + pyr[c] + tcynt[c]', 'KEGG id': 'R03106', 'GENE_ASSOCIATION': 'ENSG00000128309', 'GENE_LIST': 'MPST', 'HGNC': '7223', 'Gene name': 'mercaptopyruvate sulfurtransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.759', 'Directionality rationale': 'HumanCyc, Delta G 20.5kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.8.1.2', 'kegg.reaction': 'R03106'}, '_gpr': cobra.core.gene.GPR('ENSG00000128309'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000128309 at 0x179d664d0>}, '_metabolites': {<Metabolite cyan_c at 0x29f362dd0>: -1.0, <Metabolite mercppyr_c at 0x2ae48d0d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite tcynt_c at 0x29d77d2d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MCPSTm_MitoCore', 'name': 'R_MCPSTm_MitoCore Hydrogen cyanide + Mercaptopyruvate --> Thiocyanate + Pyruvate', 'notes': {'Model Reaction Number': '181', 'EC Number': '2.8.1.2', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'Hydrogen cyanide + Mercaptopyruvate --&gt; Thiocyanate + Pyruvate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R03106', 'GENE_ASSOCIATION': 'ENSG00000128309', 'GENE_LIST': 'MPST', 'HGNC': '7223', 'Gene name': 'mercaptopyruvate sulfurtransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.759', 'Directionality rationale': 'HumanCyc, Delta G 20.5kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.8.1.2', 'kegg.reaction': 'R03106'}, '_gpr': cobra.core.gene.GPR('ENSG00000128309'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000128309 at 0x179d664d0>}, '_metabolites': {<Metabolite cyan_m at 0x17b4fae10>: -1.0, <Metabolite mercppyr_m at 0x29d010f90>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite tcynt_m at 0x29d77a9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0595m_MitoCore', 'name': 'R_r0595m_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate', 'notes': {'Model Reaction Number': '182', 'EC Number': '2.8.1.2', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'Mercaptopyruvate + Sulfite --&gt; Thiosulfate + Pyruvate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R03105', 'GENE_ASSOCIATION': 'ENSG00000128309', 'GENE_LIST': 'MPST', 'HGNC': '7223', 'Gene name': 'mercaptopyruvate sulfurtransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.759', 'Directionality rationale': 'Delta G 10.6kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.8.1.2', 'kegg.reaction': 'R03105'}, '_gpr': cobra.core.gene.GPR('ENSG00000128309'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000128309 at 0x179d664d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite so3_m at 0x29d7ada50>: -1.0, <Metabolite mercppyr_m at 0x29d010f90>: -1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite tsul_m at 0x29d719b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0595B_MitoCore', 'name': 'R_r0595B_MitoCore Mercaptopyruvate + Sulfite --> Thiosulfate + Pyruvate', 'notes': {'Model Reaction Number': '183', 'EC Number': '2.8.1.2', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': 'Mercaptopyruvate + Sulfite --&gt; Thiosulfate + Pyruvate', 'Recon2 id': 'r0595', 'Recon2 formula': 'h[c] + so3[c] + mercppyr[c] -&gt; pyr[c] + HC01501[c]', 'KEGG id': 'R03105', 'GENE_ASSOCIATION': 'ENSG00000128309', 'GENE_LIST': 'MPST', 'HGNC': '7223', 'Gene name': 'mercaptopyruvate sulfurtransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.759', 'Directionality rationale': 'Delta G 10.6kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.8.1.2', 'kegg.reaction': 'R03105'}, '_gpr': cobra.core.gene.GPR('ENSG00000128309'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000128309 at 0x179d664d0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite so3_c at 0x297508cd0>: -1.0, <Metabolite mercppyr_c at 0x2ae48d0d0>: -1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite tsul_c at 0x29d746e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MCLOR', 'name': 'R_MCLOR 3-Mercaptolactate + NAD+ --> Mercaptopyruvate + NADH + H+', 'notes': {'Model Reaction Number': '184', 'EC Number': '1.1.1.27', 'SUBSYSTEM': 'Cysteine degradation II', 'Description': '3-Mercaptolactate + NAD+ --&gt; Mercaptopyruvate + NADH + H+', 'Recon2 id': 'MCLOR', 'Recon2 formula': 'nad[c] + mercplac[c] --&gt; h[c] + nadh[c] + mercppyr[c]', 'KEGG id': 'R03104', 'GENE_ASSOCIATION': 'ENSG00000111716', 'GENE_LIST': 'LDHB', 'HGNC': '6541', 'Gene name': 'L-lactate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.2343', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.1.1.27', 'kegg.reaction': 'R03104'}, '_gpr': cobra.core.gene.GPR('ENSG00000111716'), 'subsystem': 'Cysteine degradation II', '_genes': {<Gene ENSG00000111716 at 0x28abc9990>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite mercplac_c at 0x176225110>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite mercppyr_c at 0x2ae48d0d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0193', 'name': 'R_r0193 L-Cysteine + H2O --> Hydrogen sulfide + Pyruvate + Ammonia', 'notes': {'Model Reaction Number': '185', 'EC Number': '4.4.1.1', 'SUBSYSTEM': 'Cysteine degradation III', 'Description': 'L-Cysteine + H2O --&gt; Hydrogen sulfide + Pyruvate + Ammonia', 'Recon2 id': 'r0193', 'Recon2 formula': 'h2o[c] + cys_L[c] --&gt; nh4[c] + h[c] + pyr[c] + HC00250[c]', 'KEGG id': 'R00782', 'GENE_ASSOCIATION': 'ENSG00000116761', 'GENE_LIST': 'CTH', 'HGNC': '2501', 'Gene name': 'cystathionine gamma-lyase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.3441', 'Directionality rationale': 'NH4 production, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.4.1.1', 'kegg.reaction': 'R00782'}, '_gpr': cobra.core.gene.GPR('ENSG00000116761'), 'subsystem': 'Cysteine degradation III', '_genes': {<Gene ENSG00000116761 at 0x176ea3750>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite cys_L_c at 0x29f32a9d0>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0, <Metabolite HC00250_c at 0x29c72efd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TRPO2', 'name': 'R_TRPO2 L-Tryptophan + Oxygen --> L-Formylkynurenine', 'notes': {'Model Reaction Number': '186', 'EC Number': '1.13.11.11 or 1.13.11.52', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': 'L-Tryptophan + Oxygen --&gt; L-Formylkynurenine', 'Recon2 id': 'TRPO2', 'Recon2 formula': 'o2[c] + trp_L[c] -&gt; Lfmkynr[c]', 'KEGG id': 'R00678', 'GENE_ASSOCIATION': 'ENSG00000151790 or ENSG00000131203', 'GENE_LIST': 'TDO2 or IDO1', 'HGNC': '11708 or HGNC:6059', 'Gene name': 'tryptophan 2,3-dioxygenase', 'MitoCarta': 'FALSE, FALSE', 'MitoCarta score': '-8.0704, -8.0704', 'Directionality rationale': 'KEGG, HumanCyc , Delta G 92.1kJ/mol, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Not detected or Not detected', 'HPA protein level in heart tissue': 'Not detected or Not detected'}, '_annotation': {'ec-code': ['1.13.11.11', '1.13.11.52'], 'kegg.reaction': 'R00678'}, '_gpr': cobra.core.gene.GPR('ENSG00000151790 or ENSG00000131203'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000151790 at 0x28b1cf590>, <Gene ENSG00000131203 at 0x29d4bbbd0>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite trp_L_c at 0x29d7a42d0>: -1.0, <Metabolite Lfmkynr_c at 0x29d0cdc10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FKYNH', 'name': 'R_FKYNH L-Formylkynurenine + H2O --> Formate + L-Kynurenine', 'notes': {'Model Reaction Number': '187', 'EC Number': '3.5.1.9', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': 'L-Formylkynurenine + H2O --&gt; Formate + L-Kynurenine', 'Recon2 id': 'FKYNH', 'Recon2 formula': 'h2o[c] + Lfmkynr[c] -&gt; h[c] + for[c] + Lkynr[c]', 'KEGG id': 'R01959', 'GENE_ASSOCIATION': 'ENSG00000183077', 'GENE_LIST': 'AFMID', 'HGNC': '20910', 'Gene name': 'Kynurenine formamidase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.3441', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '3.5.1.9', 'kegg.reaction': 'R01959'}, '_gpr': cobra.core.gene.GPR('ENSG00000183077'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000183077 at 0x176ed05d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite Lfmkynr_c at 0x29d0cdc10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite for_c at 0x13352d2d0>: 1.0, <Metabolite Lkynr_c at 0x29d0cc8d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'KYN3OX', 'name': 'R_KYN3OX L-Kynurenine + Oxygen + NADPH + H+ --> 3-Hydroxy-L-kynurenine + NADP+ + H2O', 'notes': {'Model Reaction Number': '188', 'EC Number': '1.14.13.9', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': 'L-Kynurenine + Oxygen + NADPH + H+ --&gt; 3-Hydroxy-L-kynurenine + NADP+ + H2O', 'Recon2 id': 'KYN3OX', 'Recon2 formula': 'o2[c] + h[c] + nadph[c] + Lkynr[c] -&gt; h2o[c] + nadp[c] + hLkynr[c]', 'KEGG id': 'R01960', 'GENE_ASSOCIATION': 'ENSG00000117009', 'GENE_LIST': 'KMO', 'HGNC': '6381', 'Gene name': 'Kynurenine 3-monooxygenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.6003', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 95.8kJ/mol, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.14.13.9', 'kegg.reaction': 'R01960'}, '_gpr': cobra.core.gene.GPR('ENSG00000117009'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000117009 at 0x176ed08d0>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite Lkynr_c at 0x29d0cc8d0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite hLkynr_c at 0x29c7d4310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HKYNH', 'name': 'R_HKYNH 3-Hydroxy-L-kynurenine + H2O --> 3-Hydroxyanthranilate + L-Alanine', 'notes': {'Model Reaction Number': '189', 'EC Number': '3.7.1.3', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '3-Hydroxy-L-kynurenine + H2O --&gt; 3-Hydroxyanthranilate + L-Alanine', 'Recon2 id': 'HKYNH', 'Recon2 formula': 'h2o[c] + hLkynr[c] -&gt; 3hanthrn[c] + ala_L[c]', 'KEGG id': 'R02668', 'GENE_ASSOCIATION': 'ENSG00000115919', 'GENE_LIST': 'KYNU', 'HGNC': '6469', 'Gene name': 'kynureninase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-6.2297', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.7.1.3', 'kegg.reaction': 'R02668'}, '_gpr': cobra.core.gene.GPR('ENSG00000115919'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000115919 at 0x2a2fef890>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite hLkynr_c at 0x29c7d4310>: -1.0, <Metabolite 3hanthrn_c at 0x2ae3eea10>: 1.0, <Metabolite ala_L_c at 0x17aef1250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': '3HAO', 'name': 'R_3HAO 3-Hydroxyanthranilate + Oxygen --> 2-Amino-3-carboxymuconate semialdehyde', 'notes': {'Model Reaction Number': '190', 'EC Number': '1.13.11.6', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '3-Hydroxyanthranilate + Oxygen --&gt; 2-Amino-3-carboxymuconate semialdehyde', 'Recon2 id': '3HAO', 'Recon2 formula': 'o2[c] + 3hanthrn[c] -&gt; h[c] + cmusa[c]', 'KEGG id': 'R02665', 'GENE_ASSOCIATION': 'ENSG00000162882', 'GENE_LIST': 'HAAO', 'HGNC': '4796', 'Gene name': '3-hydroxyanthranilate 3,4-dioxygenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.376', 'Directionality rationale': 'Delta G 72kJ/mol, Oxygen consumption, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.13.11.6', 'kegg.reaction': 'R02665'}, '_gpr': cobra.core.gene.GPR('ENSG00000162882'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000162882 at 0x176ed0e50>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite 3hanthrn_c at 0x2ae3eea10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite cmusa_c at 0x1692b2810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCLAD', 'name': 'R_PCLAD 2-Amino-3-carboxymuconate semialdehyde --> 2-Aminomuconate semialdehyde + CO2', 'notes': {'Model Reaction Number': '191', 'EC Number': '4.1.1.45', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '2-Amino-3-carboxymuconate semialdehyde --&gt; 2-Aminomuconate semialdehyde + CO2', 'Recon2 id': 'PCLAD', 'Recon2 formula': 'h[c] + cmusa[c] -&gt; co2[c] + am6sa[c]', 'KEGG id': 'R04323', 'GENE_ASSOCIATION': 'ENSG00000153086', 'GENE_LIST': 'ACMSD', 'HGNC': '19288', 'Gene name': 'aminocarboxymuconate semialdehyde decarboxylase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'Directionality rationale': 'KEGG, HumanCyc, CO2 production', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.1.1.45', 'kegg.reaction': 'R04323'}, '_gpr': cobra.core.gene.GPR('ENSG00000153086'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000153086 at 0x176ed1150>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite cmusa_c at 0x1692b2810>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite am6sa_c at 0x2a2b03c50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0645', 'name': 'R_r0645 2-Aminomuconate semialdehyde + NAD+ + H2O --> 2-Aminomuconate + H+ + NADH', 'notes': {'Model Reaction Number': '192', 'EC Number': '1.2.1.32', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '2-Aminomuconate semialdehyde + NAD+ + H2O --&gt; 2-Aminomuconate + H+ + NADH', 'Recon2 id': 'r0645', 'Recon2 formula': 'h2o[c] + nad[c] + am6sa[c] --&gt; 2.0 h[c] + nadh[c] + amuco[c]', 'KEGG id': 'R03889', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 11kJ/mol'}, '_annotation': {'ec-code': '1.2.1.32', 'kegg.reaction': 'R03889'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Tryptophan Metabolism', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite am6sa_c at 0x2a2b03c50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite amuco_c at 0x178c0b650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AMCOXO', 'name': 'R_AMCOXO 2-Aminomuconate + NADPH + H+ + H2O --> 2-Oxoadipate + Ammonia + NADP+', 'notes': {'Model Reaction Number': '193', 'EC Number': '1.5.1.-', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '2-Aminomuconate + NADPH + H+ + H2O --&gt; 2-Oxoadipate + Ammonia + NADP+', 'Recon2 id': 'AMCOXO', 'Recon2 formula': 'h2o[c] + h[c] + nadph[c] + amuco[c] -&gt; nh4[c] + nadp[c] + 2oxoadp[c]', 'KEGG id': 'R01938', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'HumanCyc, Delta G 17.8kJ/mol, KEGG'}, '_annotation': {'ec-code': '1.5.1.-', 'kegg.reaction': 'R01938'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Tryptophan Metabolism', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite amuco_c at 0x178c0b650>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite 2oxoadp_c at 0x28b16c550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AMCOXO2_MitoCore', 'name': 'R_AMCOXO2_MitoCore 2-Aminomuconate + NADH + H+ + H2O --> 2-Oxoadipate + NH3 + NAD+', 'notes': {'Model Reaction Number': '194', 'EC Number': '1.5.1.-', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': '2-Aminomuconate + NADH + H+ + H2O --&gt; 2-Oxoadipate + NH3 + NAD+', 'Recon2 id': 'n/a', 'Recon2 formula': 'n/a', 'KEGG id': 'R01937', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 17.8kJ/mol'}, '_annotation': {'ec-code': '1.5.1.-', 'kegg.reaction': 'R01937'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Tryptophan Metabolism', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadh_c at 0x29d2efa10>: -1.0, <Metabolite amuco_c at 0x178c0b650>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite nad_c at 0x29d264bd0>: 1.0, <Metabolite 2oxoadp_c at 0x28b16c550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2OXOADPTmB_MitoCore', 'name': 'R_2OXOADPTmB_MitoCore oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)', 'notes': {'Model Reaction Number': '195', 'EC Number': 'n/a', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': 'oxodicarboxylate carrier (ODC) - oxoadipate (in) for oxoglutarate (out)', 'Recon2 id': '2OXOADPTm', 'Recon2 formula': 'akg[m] + 2oxoadp[c] --&gt; akg[c] + 2oxoadp[m]', 'KEGG id': 'n/a', 'GENE_ASSOCIATION': 'ENSG00000183032', 'GENE_LIST': 'SLC25A21', 'HGNC': '14411', 'Gene name': '2-oxoadipate shuttle (cytosol/mitochondria)', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183032'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000183032 at 0x176ed1450>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite 2oxoadp_c at 0x28b16c550>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite akg_c at 0x16ac21410>: 1.0, <Metabolite 2oxoadp_m at 0x28b554890>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2OXOADPTmC_MitoCore', 'name': 'R_2OXOADPTmC_MitoCore oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)', 'notes': {'Model Reaction Number': '196', 'EC Number': 'n/a', 'SUBSYSTEM': 'Tryptophan Metabolism', 'Description': 'oxodicarboxylate carrier (ODC) reverse - oxoadipate (out) for oxoglutarate (in)', 'Recon2 id': '2OXOADPTm', 'Recon2 formula': 'akg[m] + 2oxoadp[c] --&gt; akg[c] + 2oxoadp[m]', 'KEGG id': 'n/a', 'GENE_ASSOCIATION': 'ENSG00000183032', 'GENE_LIST': 'SLC25A21', 'HGNC': '14411', 'Gene name': '2-oxoadipate shuttle (cytosol/mitochondria)', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183032'), 'subsystem': 'Tryptophan Metabolism', '_genes': {<Gene ENSG00000183032 at 0x176ed1450>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite 2oxoadp_m at 0x28b554890>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite akg_m at 0x2919313d0>: 1.0, <Metabolite 2oxoadp_c at 0x28b16c550>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2OXOADOXm', 'name': 'R_2OXOADOXm 2-Oxoadipate + CoA + NAD+ --> Glutaryl-CoA + CO2 + NADH + H+', 'notes': {'Model Reaction Number': '197', 'EC Number': '1.2.4.2 and 2.3.1.61 and 1.8.1.4', 'SUBSYSTEM': 'Tryptophan / Lysine Metabolism', 'Description': '2-Oxoadipate + CoA + NAD+ --&gt; Glutaryl-CoA + CO2 + NADH + H+', 'Recon2 id': '2OXOADOXm', 'Recon2 formula': 'nad[m] + 2oxoadp[m] + coa[m] -&gt; nadh[m] + co2[m] + glutcoa[m]', 'KEGG id': 'R01933', 'GENE_ASSOCIATION': 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'GENE_LIST': 'OGDH and DLST and DLD', 'HGNC': '8124 and HGNC:2911 and HGNC:2898', 'Gene name': '2-oxoadipate dehydrogenase complex', 'MitoCarta': 'TRUE, TRUE, TRUE', 'MitoCarta score': '40.5677, 37.6758, 31.2307', 'Directionality rationale': 'KEGG, HumanCyc, CO2 production', 'HPA mRNA level in heart tissue': 'High and High and High', 'HPA protein level in heart tissue': 'High and Medium and High'}, '_annotation': {'ec-code': ['1.2.4.2', '2.3.1.61', '1.8.1.4'], 'kegg.reaction': 'R01933'}, '_gpr': cobra.core.gene.GPR('ENSG00000105953 and ENSG00000119689 and ENSG00000091140'), 'subsystem': 'Tryptophan / Lysine Metabolism', '_genes': {<Gene ENSG00000105953 at 0x28ab18cd0>, <Gene ENSG00000091140 at 0x28abca110>, <Gene ENSG00000119689 at 0x28ab18fd0>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite 2oxoadp_m at 0x28b554890>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite glutcoa_m at 0x29c7378d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0541', 'name': 'R_r0541 Glutaryl-CoA + Q --> QH2 + Crotonoyl-CoA + CO2', 'notes': {'Model Reaction Number': '198', 'EC Number': '1.3.8.6', 'SUBSYSTEM': 'Tryptophan / Lysine Metabolism', 'Description': 'Glutaryl-CoA + Q --&gt; QH2 + Crotonoyl-CoA + CO2', 'Recon2 id': 'r0541', 'Recon2 formula': 'h[m] + glutcoa[m] + q10[m] --&gt; co2[m] + b2coa[m] + q10h2[m]', 'KEGG id': 'R02487', 'GENE_ASSOCIATION': 'ENSG00000105607', 'GENE_LIST': 'GCDH', 'HGNC': '4189', 'Gene name': 'Glutaryl-CoA dehydrogenase, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.6769', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.3.8.6', 'kegg.reaction': 'R02487'}, '_gpr': cobra.core.gene.GPR('ENSG00000105607'), 'subsystem': 'Tryptophan / Lysine Metabolism', '_genes': {<Gene ENSG00000105607 at 0x16da24b50>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite glutcoa_m at 0x29c7378d0>: -1.0, <Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite b2coa_m at 0x2976c9ed0>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SACCD3m', 'name': 'R_SACCD3m L-Lysine + 2-Oxoglutarate + NADPH + H+ --> N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O', 'notes': {'Model Reaction Number': '199', 'EC Number': '1.5.1.8', 'SUBSYSTEM': 'Lysine degradation (saccharopine pathway)', 'Description': 'L-Lysine + 2-Oxoglutarate + NADPH + H+ --&gt; N6-(L-1,3-Dicarboxypropyl)-L-lysine + NADP+ + H2O', 'Recon2 id': 'SACCD3m', 'Recon2 formula': 'h[m] + nadph[m] + akg[m] + lys_L[m] -&gt; h2o[m] + nadp[m] + saccrp_L[m]', 'KEGG id': 'R00716', 'GENE_ASSOCIATION': 'ENSG00000008311', 'GENE_LIST': 'AASS', 'HGNC': '17366', 'Gene name': 'aminoadipate-semialdehyde synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.1893', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.8', 'kegg.reaction': 'R00716'}, '_gpr': cobra.core.gene.GPR('ENSG00000008311'), 'subsystem': 'Lysine degradation (saccharopine pathway)', '_genes': {<Gene ENSG00000008311 at 0x169def5d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite lys_L_m at 0x29d021590>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite saccrp_L_m at 0x29d205710>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0525', 'name': 'R_r0525 N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O-->L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+', 'notes': {'Model Reaction Number': '200', 'EC Number': '1.5.1.9', 'SUBSYSTEM': 'Lysine degradation (saccharopine pathway)', 'Description': 'N6-(L-1,3-Dicarboxypropyl)-L-lysine+NAD++H2O--&gt;L-Glutamate+L-2-Aminoadipate6-semialdehyde+NADH+H+', 'Recon2 id': 'r0525', 'Recon2 formula': 'h2o[m] + nad[m] + saccrp_L[m] --&gt; h[m] + nadh[m] + glu_L[m] + L2aadp6sa[m]', 'KEGG id': 'R02313', 'GENE_ASSOCIATION': 'ENSG00000008311', 'GENE_LIST': 'AASS', 'HGNC': '17366', 'Gene name': 'aminoadipate-semialdehyde synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.1893', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.9', 'kegg.reaction': 'R02313'}, '_gpr': cobra.core.gene.GPR('ENSG00000008311'), 'subsystem': 'Lysine degradation (saccharopine pathway)', '_genes': {<Gene ENSG00000008311 at 0x169def5d0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite saccrp_L_m at 0x29d205710>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite L2aadp6sa_m at 0x29d0ab110>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AASAD3m', 'name': 'R_AASAD3m L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+', 'notes': {'Model Reaction Number': '201', 'EC Number': '1.2.1.31', 'SUBSYSTEM': 'Lysine degradation (saccharopine pathway)', 'Description': 'L-2-Aminoadipate6-semialdehyde+NAD++H2O--&gt;L-2-Aminoadipate+NADH+H+', 'Recon2 id': 'AASAD3m', 'Recon2 formula': 'h2o[m] + nad[m] + L2aadp6sa[m] -&gt; 2.0 h[m] + nadh[m] + L2aadp[m]', 'KEGG id': 'R03102', 'GENE_ASSOCIATION': 'ENSG00000164904', 'GENE_LIST': 'ALDH7A1', 'HGNC': '877', 'Gene name': 'aldehyde dehydrogenase 7 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.8432', 'Directionality rationale': 'Delta G 9.8kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.2.1.31', 'kegg.reaction': 'R03102'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904'), 'subsystem': 'Lysine degradation (saccharopine pathway)', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite L2aadp6sa_m at 0x29d0ab110>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite L2aadp_m at 0x29d0ab850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'R03103_MitoCore', 'name': 'R_R03103_MitoCore L-2-Aminoadipate6-semialdehyde+NADP++H2O-->L-2-Aminoadipate+NADPH+H+', 'notes': {'Model Reaction Number': '202', 'EC Number': '1.2.1.31', 'SUBSYSTEM': 'Lysine degradation (saccharopine pathway)', 'Description': 'L-2-Aminoadipate6-semialdehyde+NADP++H2O--&gt;L-2-Aminoadipate+NADPH+H+', 'Recon2 id': 'r0594', 'Recon2 formula': 'h2o[c] + nad[c] + L2aadp6sa[c] -&gt; 2.0 h[c] + nadh[c] + L2aadp[c]', 'KEGG id': 'R03103', 'GENE_ASSOCIATION': 'ENSG00000164904', 'GENE_LIST': 'ALDH7A1', 'HGNC': '877', 'Gene name': 'aldehyde dehydrogenase 7 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.8432', 'Directionality rationale': 'Delta G 9.8kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.2.1.31', 'kegg.reaction': 'R03103'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904'), 'subsystem': 'Lysine degradation (saccharopine pathway)', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite L2aadp6sa_m at 0x29d0ab110>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite L2aadp_m at 0x29d0ab850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0450', 'name': 'R_r0450 L-2-Aminoadipate+2-Oxoglutarate-->2-Oxoadipate+L-Glutamate', 'notes': {'Model Reaction Number': '203', 'EC Number': '2.6.1.39', 'SUBSYSTEM': 'Lysine degradation (saccharopine pathway)', 'Description': 'L-2-Aminoadipate+2-Oxoglutarate--&gt;2-Oxoadipate+L-Glutamate', 'Recon2 id': 'r0450', 'Recon2 formula': 'L2aadp[m] + akg[m] --&gt; 2oxoadp[m] + glu_L[m]', 'KEGG id': 'R01939', 'GENE_ASSOCIATION': 'ENSG00000109576', 'GENE_LIST': 'AADAT', 'HGNC': '17929', 'Gene name': 'aminoadipate aminotransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.7676', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.6.1.39', 'kegg.reaction': 'R01939'}, '_gpr': cobra.core.gene.GPR('ENSG00000109576'), 'subsystem': 'Lysine degradation (saccharopine pathway)', '_genes': {<Gene ENSG00000109576 at 0x176ed2010>}, '_metabolites': {<Metabolite L2aadp_m at 0x29d0ab850>: -1.0, <Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite 2oxoadp_m at 0x28b554890>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LYSOXc_MitoCore', 'name': 'R_LYSOXc_MitoCore L-Lysine + Oxygen + H2O --> 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide', 'notes': {'Model Reaction Number': '204', 'EC Number': '1.4.3.14', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'L-Lysine + Oxygen + H2O --&gt; 6-Amino-2-oxohexanoate + Ammonia + Hydrogen peroxide', 'Recon2 id': 'LYSOXp', 'Recon2 formula': 'h2o[x] + o2[x] + lys_L[x] -&gt; h2o2[x] + nh4[x] + 6a2ohxnt[x]', 'KEGG id': 'R00447', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'Oxygen consumption, Delta G 23kJ/mol, KEGG, HumanCyc'}, '_annotation': {'ec-code': '1.4.3.14', 'kegg.reaction': 'R00447'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite lys_L_c at 0x29d022890>: -1.0, <Metabolite h2o2_c at 0x29c784c10>: 1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite 6a2ohxnt_c at 0x2ae48e510>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPD2CSPc_MitoCore', 'name': 'R_PPD2CSPc_MitoCore 6-Amino-2-oxohexanoate --> delta1-Piperideine-2-carboxylate + H2O', 'notes': {'Model Reaction Number': '205', 'EC Number': 'Non-Enzymatic', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': '6-Amino-2-oxohexanoate --&gt; delta1-Piperideine-2-carboxylate + H2O', 'Recon2 id': 'PPD2CSPp', 'Recon2 formula': '6a2ohxnt[x] -&gt; h2o[x] + 1pipdn2c[x]', 'KEGG id': 'R04175', 'GENE_ASSOCIATION': 'Non-Enzymatic', 'GENE_LIST': 'Non-Enzymatic', 'Gene name': 'Non-Enzymatic'}, '_annotation': {'kegg.reaction': 'R04175'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': set(), '_metabolites': {<Metabolite 6a2ohxnt_c at 0x2ae48e510>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite 1pipdn2c_c at 0x17b75ec10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': '1PPDCRc_MitoCore', 'name': 'R_1PPDCRc_MitoCore delta1-Piperideine-2-carboxylate + NADH + H+ --> L-Pipecolate + NAD+', 'notes': {'Model Reaction Number': '206', 'EC Number': '1.5.1.1 or 1.5.1.21', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'delta1-Piperideine-2-carboxylate + NADH + H+ --&gt; L-Pipecolate + NAD+', 'Recon2 id': '1PPDCRp', 'Recon2 formula': 'h[x] + 1pipdn2c[x] + nadh[x] -&gt; Lpipecol[x] + nad[x]', 'KEGG id': 'R02201', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'HumanCyc, KEGG, Modelling purposes'}, '_annotation': {'ec-code': ['1.5.1.1', '1.5.1.21'], 'kegg.reaction': 'R02201'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': set(), '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite 1pipdn2c_c at 0x17b75ec10>: -1.0, <Metabolite nadh_c at 0x29d2efa10>: -1.0, <Metabolite Lpipecol_c at 0x2a0555210>: 1.0, <Metabolite nad_c at 0x29d264bd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '1PPDCRc_NADPH_MitoCore', 'name': 'R_1PPDCRc_NADPH_MitoCore delta1-Piperideine-2-carboxylate + NADPH + H+ --> L-Pipecolate + NADP+', 'notes': {'Model Reaction Number': '207', 'EC Number': '1.5.1.1 or 1.5.1.21', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'delta1-Piperideine-2-carboxylate + NADPH + H+ --&gt; L-Pipecolate + NADP+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R02203', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'HumanCyc, KEGG, Modelling purposes'}, '_annotation': {'ec-code': ['1.5.1.1', '1.5.1.21'], 'kegg.reaction': 'R02203'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': set(), '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite 1pipdn2c_c at 0x17b75ec10>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite Lpipecol_c at 0x2a0555210>: 1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LPCOXc_MitoCore', 'name': 'R_LPCOXc_MitoCore L-Pipecolate + Oxygen --> (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide', 'notes': {'Model Reaction Number': '208', 'EC Number': '1.5.3.7', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'L-Pipecolate + Oxygen --&gt; (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + Hydrogen peroxide', 'Recon2 id': 'LPCOXp', 'Recon2 formula': 'o2[x] + Lpipecol[x] -&gt; h2o2[x] + h[x] + thp2c[x]', 'KEGG id': 'R02204', 'GENE_ASSOCIATION': 'ENSG00000179761', 'GENE_LIST': 'PIPOX', 'HGNC': '17804', 'Gene name': 'pipecolic acid oxidase', 'MitoCarta': 'FALSE', 'MitoCarta score': '1.4709', 'Directionality rationale': 'Oxygen consumption, Delta G 32.7kJ/mol, HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.3.7', 'kegg.reaction': 'R02204'}, '_gpr': cobra.core.gene.GPR('ENSG00000179761'), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': {<Gene ENSG00000179761 at 0x176ed2310>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite Lpipecol_c at 0x2a0555210>: -1.0, <Metabolite h2o2_c at 0x29c784c10>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite thp2c_c at 0x29d730890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'RE1254C', 'name': 'R_RE1254C (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --> L-2-Aminoadipate 6-semialdehyde', 'notes': {'Model Reaction Number': '209', 'EC Number': '1.4.1.18', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': '(S)-2,3,4,5-Tetrahydropyridine-2-carboxylate + H2O --&gt; L-2-Aminoadipate 6-semialdehyde', 'Recon2 id': 'RE1254C', 'Recon2 formula': 'h2o[c] + h[c] + thp2c[c] --&gt; L2aadp6sa[c]', 'KEGG id': 'R02317', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {'ec-code': '1.4.1.18', 'kegg.reaction': 'R02317'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite thp2c_c at 0x29d730890>: -1.0, <Metabolite L2aadp6sa_c at 0x29d0aadd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0594', 'name': 'R_r0594 L-2-Aminoadipate6-semialdehyde+NAD++H2O-->L-2-Aminoadipate+NADH+H+', 'notes': {'Model Reaction Number': '210', 'EC Number': '1.2.1.31', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'L-2-Aminoadipate6-semialdehyde+NAD++H2O--&gt;L-2-Aminoadipate+NADH+H+', 'Recon2 id': 'r0594', 'Recon2 formula': 'h2o[c] + nad[c] + L2aadp6sa[c] -&gt; 2.0 h[c] + nadh[c] + L2aadp[c]', 'KEGG id': 'R03102', 'GENE_ASSOCIATION': 'ENSG00000164904', 'GENE_LIST': 'ALDH7A1', 'HGNC': '877', 'Gene name': 'aldehyde dehydrogenase 7 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.8432', 'Directionality rationale': 'Delta G 9.8kJ/mol, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.2.1.31', 'kegg.reaction': 'R03102'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904'), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite L2aadp6sa_c at 0x29d0aadd0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite L2aadp_c at 0x29c760310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2AMADPTmB_MitoCore', 'name': 'R_2AMADPTmB_MitoCore Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)', 'notes': {'Model Reaction Number': '211', 'EC Number': 'N/A', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'Oxodicarboxylate carrier (ODC) - L-aminoadipate (in) for oxoglutarate (out)', 'Recon2 id': '2AMADPTm', 'Recon2 formula': 'L2aadp[c] + akg[m] -&gt; L2aadp[m] + akg[c]', 'KEGG id': 'n/a', 'GENE_ASSOCIATION': 'ENSG00000183032', 'GENE_LIST': 'SLC25A21', 'HGNC': '14411', 'Gene name': '2-oxoadipate shuttle (cytosol/mitochondria)', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183032'), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': {<Gene ENSG00000183032 at 0x176ed1450>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite L2aadp_c at 0x29c760310>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite akg_c at 0x16ac21410>: 1.0, <Metabolite L2aadp_m at 0x29d0ab850>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '2AMADPTmC_MitoCore', 'name': 'R_2AMADPTmC_MitoCore Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)', 'notes': {'Model Reaction Number': '212', 'EC Number': 'N/A', 'SUBSYSTEM': 'Lysine degradation (L-pipecolate pathway)', 'Description': 'Oxodicarboxylate carrier (ODC) reverse - L-aminoadipate (out) for oxoglutarate (in)', 'Recon2 id': '2AMADPTm', 'Recon2 formula': 'L2aadp[c] + akg[m] -&gt; L2aadp[m] + akg[c]', 'KEGG id': 'n/a', 'GENE_ASSOCIATION': 'ENSG00000183032', 'GENE_LIST': 'SLC25A21', 'HGNC': '14411', 'Gene name': '2-oxoadipate shuttle (cytosol/mitochondria)', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183032'), 'subsystem': 'Lysine degradation (L-pipecolate pathway)', '_genes': {<Gene ENSG00000183032 at 0x176ed1450>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite L2aadp_m at 0x29d0ab850>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.18, <Metabolite akg_m at 0x2919313d0>: 1.0, <Metabolite L2aadp_c at 0x29c760310>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.18}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PROD2mB_MitoCore', 'name': 'R_PROD2mB_MitoCore Proline + Q --> (S)-1-Pyrroline-5-carboxylate + QH2', 'notes': {'Model Reaction Number': '213', 'EC Number': '1.5.5.2', 'SUBSYSTEM': 'Proline degradation', 'Description': 'Proline + Q --&gt; (S)-1-Pyrroline-5-carboxylate + QH2', 'Recon2 id': 'PROD2m', 'Recon2 formula': 'fad[m] + pro_L[m] -&gt; h[m] + fadh2[m] + 1pyr5c[m]', 'KEGG id': 'R01253', 'GENE_ASSOCIATION': 'ENSG00000100033', 'GENE_LIST': 'PRODH', 'HGNC': '9453', 'Gene name': 'Proline dehydrogenase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '24.4195', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.5.2', 'kegg.reaction': 'R01253'}, '_gpr': cobra.core.gene.GPR('ENSG00000100033'), 'subsystem': 'Proline degradation', '_genes': {<Gene ENSG00000100033 at 0x176ed2610>}, '_metabolites': {<Metabolite q10_m at 0x29d292950>: -1.0, <Metabolite pro_L_m at 0x1696f8b10>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite q10h2_m at 0x29d2b0790>: 1.0, <Metabolite 1pyr5c_m at 0x179d85610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'G5SADrm', 'name': 'R_G5SADrm L-Glutamate 5-semialdehyde --> (S)-1-Pyrroline-5-carboxylate + H2O', 'notes': {'Model Reaction Number': '214', 'EC Number': 'Non-Enzymatic', 'SUBSYSTEM': 'Proline degradation/synthesis', 'Description': 'L-Glutamate 5-semialdehyde --&gt; (S)-1-Pyrroline-5-carboxylate + H2O', 'Recon2 id': 'G5SADrm', 'Recon2 formula': 'glu5sa[m] --&gt; h2o[m] + h[m] + 1pyr5c[m]', 'KEGG id': 'R03314', 'GENE_ASSOCIATION': 'Non-Enzymatic', 'GENE_LIST': 'Non-Enzymatic', 'Gene name': 'Non-Enzymatic'}, '_annotation': {'kegg.reaction': 'R03314'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Proline degradation/synthesis', '_genes': set(), '_metabolites': {<Metabolite glu5sa_m at 0x29c7fcd90>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite 1pyr5c_m at 0x179d85610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0074', 'name': 'R_r0074 L-Glutamate 5-semialdehyde + NAD+ + H2O --> L-Glutamate + NADH + H+', 'notes': {'Model Reaction Number': '215', 'EC Number': '1.2.1.88', 'SUBSYSTEM': 'Proline / ornithine degradation', 'Description': 'L-Glutamate 5-semialdehyde + NAD+ + H2O --&gt; L-Glutamate + NADH + H+', 'Recon2 id': 'r0074', 'Recon2 formula': 'h2o[m] + nad[m] + glu5sa[m] --&gt; 2.0 h[m] + nadh[m] + glu_L[m]', 'KEGG id': 'R00245', 'GENE_ASSOCIATION': 'ENSG00000159423', 'GENE_LIST': 'ALDH4A1', 'HGNC': '406', 'Gene name': 'aldehyde dehydrogenase 4 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '25.5079', 'Directionality rationale': 'Brenda, Literature, HumanCyc, Delta G 9.8kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.2.1.88', 'kegg.reaction': 'R00245'}, '_gpr': cobra.core.gene.GPR('ENSG00000159423'), 'subsystem': 'Proline / ornithine degradation', '_genes': {<Gene ENSG00000159423 at 0x176ed2910>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite glu5sa_m at 0x29c7fcd90>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLU5Km', 'name': 'R_GLU5Km ATP + L-Glutamate --> ADP + L-Glutamyl 5-phosphate', 'notes': {'Model Reaction Number': '216', 'EC Number': '2.7.2.11', 'SUBSYSTEM': 'Proline synthesis', 'Description': 'ATP + L-Glutamate --&gt; ADP + L-Glutamyl 5-phosphate', 'Recon2 id': 'GLU5Km', 'Recon2 formula': 'atp[m] + glu_L[m] -&gt; adp[m] + glu5p[m]', 'KEGG id': 'R00239', 'GENE_ASSOCIATION': 'ENSG00000059573', 'GENE_LIST': 'ALDH18A1', 'HGNC': '9722', 'Gene name': 'aldehyde dehydrogenase 18 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.6874', 'Directionality rationale': 'ATP consumption, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.7.2.11', 'kegg.reaction': 'R00239'}, '_gpr': cobra.core.gene.GPR('ENSG00000059573'), 'subsystem': 'Proline synthesis', '_genes': {<Gene ENSG00000059573 at 0x17aa14910>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite glu5p_m at 0x29c7fdb90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'G5SDym', 'name': 'R_G5SDym L-Glutamyl 5-phosphate + NADPH + H+ --> L-Glutamate 5-semialdehyde + Orthophosphate + NADP+', 'notes': {'Model Reaction Number': '217', 'EC Number': '1.2.1.41', 'SUBSYSTEM': 'Proline synthesis', 'Description': 'L-Glutamyl 5-phosphate + NADPH + H+ --&gt; L-Glutamate 5-semialdehyde + Orthophosphate + NADP+', 'Recon2 id': 'G5SDym', 'Recon2 formula': 'h[m] + nadph[m] + glu5p[m] -&gt; nadp[m] + pi[m] + glu5sa[m]', 'KEGG id': 'R03313', 'GENE_ASSOCIATION': 'ENSG00000059573', 'GENE_LIST': 'ALDH18A1', 'HGNC': '9722', 'Gene name': 'aldehyde dehydrogenase 18 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.6874', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.2.1.41', 'kegg.reaction': 'R03313'}, '_gpr': cobra.core.gene.GPR('ENSG00000059573'), 'subsystem': 'Proline synthesis', '_genes': {<Gene ENSG00000059573 at 0x17aa14910>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite glu5p_m at 0x29c7fdb90>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite glu5sa_m at 0x29c7fcd90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'P5CRm', 'name': 'R_P5CRm (S)-1-Pyrroline-5-carboxylate + NADPH + H+ --> L-Proline + NADP+', 'notes': {'Model Reaction Number': '218', 'EC Number': '1.5.1.2', 'SUBSYSTEM': 'Proline synthesis', 'Description': '(S)-1-Pyrroline-5-carboxylate + NADPH + H+ --&gt; L-Proline + NADP+', 'Recon2 id': 'P5CRm', 'Recon2 formula': '2.0 h[m] + nadph[m] + 1pyr5c[m] -&gt; nadp[m] + pro_L[m]', 'KEGG id': 'R01251', 'GENE_ASSOCIATION': 'ENSG00000183010', 'GENE_LIST': 'PYCR1', 'HGNC': '9721', 'Gene name': 'pyrroline-5-carboxylate reductase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '2.4163', 'Directionality rationale': 'Brenda, Literature, HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.2', 'kegg.reaction': 'R01251'}, '_gpr': cobra.core.gene.GPR('ENSG00000183010'), 'subsystem': 'Proline synthesis', '_genes': {<Gene ENSG00000183010 at 0x17b4e4890>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -2.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite 1pyr5c_m at 0x179d85610>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite pro_L_m at 0x1696f8b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'P5CRxm', 'name': 'R_P5CRxm (S)-1-Pyrroline-5-carboxylate + NADH + H+ --> L-Proline + NAD+', 'notes': {'Model Reaction Number': '219', 'EC Number': '1.5.1.2', 'SUBSYSTEM': 'Proline synthesis', 'Description': '(S)-1-Pyrroline-5-carboxylate + NADH + H+ --&gt; L-Proline + NAD+', 'Recon2 id': 'P5CRxm', 'Recon2 formula': '2.0 h[m] + nadh[m] + 1pyr5c[m] -&gt; nad[m] + pro_L[m]', 'KEGG id': 'R01248', 'GENE_ASSOCIATION': 'ENSG00000183010', 'GENE_LIST': 'PYCR1', 'HGNC': '9721', 'Gene name': 'pyrroline-5-carboxylate reductase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '2.4163', 'Directionality rationale': 'Brenda, Literature, HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.2', 'kegg.reaction': 'R01248'}, '_gpr': cobra.core.gene.GPR('ENSG00000183010'), 'subsystem': 'Proline synthesis', '_genes': {<Gene ENSG00000183010 at 0x17b4e4890>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -2.0, <Metabolite nadh_m at 0x29d2af310>: -1.0, <Metabolite 1pyr5c_m at 0x179d85610>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: 1.0, <Metabolite pro_L_m at 0x1696f8b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ORNTArm', 'name': 'R_ORNTArm L-Ornithine + 2-Oxoglutarate --> L-Glutamate 5-semialdehyde + L-Glutamate', 'notes': {'Model Reaction Number': '220', 'EC Number': '2.6.1.13', 'SUBSYSTEM': 'Ornithine degradation', 'Description': 'L-Ornithine + 2-Oxoglutarate --&gt; L-Glutamate 5-semialdehyde + L-Glutamate', 'Recon2 id': 'ORNTArm', 'Recon2 formula': 'akg[m] + orn[m] --&gt; glu_L[m] + glu5sa[m]', 'KEGG id': 'R00667', 'GENE_ASSOCIATION': 'ENSG00000065154', 'GENE_LIST': 'OAT', 'HGNC': '8091', 'Gene name': 'ornithine aminotransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.9076', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.6.1.13', 'kegg.reaction': 'R00667'}, '_gpr': cobra.core.gene.GPR('ENSG00000065154'), 'subsystem': 'Ornithine degradation', '_genes': {<Gene ENSG00000065154 at 0x176ed3110>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite orn_m at 0x29d211b90>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite glu5sa_m at 0x29c7fcd90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ORNDC', 'name': 'R_ORNDC L-Ornithine --> Putrescine + CO2', 'notes': {'Model Reaction Number': '221', 'EC Number': '4.1.1.17', 'SUBSYSTEM': 'Ornithine degradation (cytosolic)', 'Description': 'L-Ornithine --&gt; Putrescine + CO2', 'Recon2 id': 'ORNDC', 'Recon2 formula': 'h[c] + orn[c] -&gt; co2[c] + ptrc[c]', 'KEGG id': 'R00670', 'GENE_ASSOCIATION': 'ENSG00000115758', 'GENE_LIST': 'ODC1', 'HGNC': '8109', 'Gene name': 'ornithine decarboxylase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '1.3904', 'Directionality rationale': 'HumanCyc, KEGG, CO2 production', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.1.1.17', 'kegg.reaction': 'R00670'}, '_gpr': cobra.core.gene.GPR('ENSG00000115758'), 'subsystem': 'Ornithine degradation (cytosolic)', '_genes': {<Gene ENSG00000115758 at 0x176ed3450>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite orn_c at 0x29e2d2250>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ptrc_c at 0x29d0b0950>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PTRCOX1', 'name': 'R_PTRCOX1 Putrescine + Oxygen + H2O --> 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide', 'notes': {'Model Reaction Number': '222', 'EC Number': '1.4.3.22', 'SUBSYSTEM': 'Ornithine degradation (cytosolic)', 'Description': 'Putrescine + Oxygen + H2O --&gt; 4-Aminobutyraldehyde + Ammonia + Hydrogen peroxide', 'Recon2 id': 'PTRCOX1', 'Recon2 formula': 'h2o[c] + o2[c] + ptrc[c] -&gt; h2o2[c] + nh4[c] + 4abutn[c]', 'KEGG id': 'R01151', 'GENE_ASSOCIATION': 'ENSG00000002726', 'GENE_LIST': 'AOC1', 'HGNC': '80', 'Gene name': 'amine oxidase, copper containing 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.2802', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 24kJ/mol, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.4.3.22', 'kegg.reaction': 'R01151'}, '_gpr': cobra.core.gene.GPR('ENSG00000002726'), 'subsystem': 'Ornithine degradation (cytosolic)', '_genes': {<Gene ENSG00000002726 at 0x176ed3750>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite ptrc_c at 0x29d0b0950>: -1.0, <Metabolite h2o2_c at 0x29c784c10>: 1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite 4abutn_c at 0x2ae3ee810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0464c_MitoCore', 'name': 'R_r0464c_MitoCore 4-Aminobutyraldehyde + NADP+ + H2O --> 4-Aminobutanoate + NADPH + H+', 'notes': {'Model Reaction Number': '223', 'EC Number': '1.2.1.3', 'SUBSYSTEM': 'Ornithine degradation (cytosolic)', 'Description': '4-Aminobutyraldehyde + NADP+ + H2O --&gt; 4-Aminobutanoate + NADPH + H+', 'Recon2 id': 'r0464', 'Recon2 formula': 'h2o[m] + nadp[m] + 4abutn[m] -&gt; 2.0 h[m] + nadph[m] + 4abut[m]', 'KEGG id': 'R01986', 'GENE_ASSOCIATION': 'ENSG00000164904 or ENSG00000143149', 'GENE_LIST': 'ALDH7A1 or ALDH9A1', 'HGNC': '877 or HGNC:412', 'Gene name': 'aldehyde dehydrogenase family', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '17.6418, 10.116, 13.8432, 23.4278', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 9.8kJ/mol', 'HPA mRNA level in heart tissue': 'Medium or Medium', 'HPA protein level in heart tissue': 'Low or Low'}, '_annotation': {'ec-code': '1.2.1.3', 'kegg.reaction': 'R01986'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904 or ENSG00000143149'), 'subsystem': 'Ornithine degradation (cytosolic)', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>, <Gene ENSG00000143149 at 0x176ed3a50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite 4abutn_c at 0x2ae3ee810>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite 4abut_c at 0x176226650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ABUTD', 'name': 'R_ABUTD 4-Aminobutyraldehyde + NAD+ + H2O --> 4-Aminobutanoate + NADH + H+', 'notes': {'Model Reaction Number': '224', 'EC Number': '1.2.1.3', 'SUBSYSTEM': 'Ornithine degradation (cytosolic)', 'Description': '4-Aminobutyraldehyde + NAD+ + H2O --&gt; 4-Aminobutanoate + NADH + H+', 'Recon2 id': 'ABUTD', 'Recon2 formula': 'h2o[c] + nad[c] + 4abutn[c] -&gt; 2.0 h[c] + nadh[c] + 4abut[c]', 'KEGG id': 'R02549', 'GENE_ASSOCIATION': 'ENSG00000164904 or ENSG00000143149', 'GENE_LIST': 'ALDH7A1 or ALDH9A1', 'HGNC': '877 or HGNC:412', 'Gene name': 'aldehyde dehydrogenase family', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '17.6418, 10.116, 13.8432, 23.4278', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 9.8kJ/mol', 'HPA mRNA level in heart tissue': 'Medium or Medium', 'HPA protein level in heart tissue': 'Low or Low'}, '_annotation': {'ec-code': '1.2.1.3', 'kegg.reaction': 'R02549'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904 or ENSG00000143149'), 'subsystem': 'Ornithine degradation (cytosolic)', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>, <Gene ENSG00000143149 at 0x176ed3a50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite 4abutn_c at 0x2ae3ee810>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite 4abut_c at 0x176226650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGDCm', 'name': 'R_ARGDCm L-Arginine --> Agmatine + CO2', 'notes': {'Model Reaction Number': '225', 'EC Number': '4.1.1.19', 'SUBSYSTEM': 'Arginine degradation', 'Description': 'L-Arginine --&gt; Agmatine + CO2', 'Recon2 id': 'ARGDCm', 'Recon2 formula': 'h[m] + arg_L[m] -&gt; co2[m] + agm[m]', 'KEGG id': 'R00566', 'GENE_ASSOCIATION': 'ENSG00000142920', 'GENE_LIST': 'AZIN2', 'HGNC': '29957', 'Gene name': 'antizyme inhibitor 2', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.0247', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.1.1.19', 'kegg.reaction': 'R00566'}, '_gpr': cobra.core.gene.GPR('ENSG00000142920'), 'subsystem': 'Arginine degradation', '_genes': {<Gene ENSG00000142920 at 0x176ed3a90>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite arg_L_m at 0x2ae4a81d0>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite agm_m at 0x29dfb1350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AGMTm', 'name': 'R_AGMTm Agmatine + H2O --> Putrescine + Urea', 'notes': {'Model Reaction Number': '226', 'EC Number': '3.5.3.11', 'SUBSYSTEM': 'Arginine degradation', 'Description': 'Agmatine + H2O --&gt; Putrescine + Urea', 'Recon2 id': 'AGMTm', 'Recon2 formula': 'h2o[m] + agm[m] -&gt; ptrc[m] + urea[m]', 'KEGG id': 'R01157', 'GENE_ASSOCIATION': 'ENSG00000116771', 'GENE_LIST': 'AGMAT', 'HGNC': '18407', 'Gene name': 'agmatine ureohydrolase (agmatinase)', 'MitoCarta': 'TRUE', 'MitoCarta score': '4.986', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '3.5.3.11', 'kegg.reaction': 'R01157'}, '_gpr': cobra.core.gene.GPR('ENSG00000116771'), 'subsystem': 'Arginine degradation', '_genes': {<Gene ENSG00000116771 at 0x176ed4d50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite agm_m at 0x29dfb1350>: -1.0, <Metabolite ptrc_m at 0x166240a50>: 1.0, <Metabolite urea_m at 0x29d7c2990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PTRCAT1m_MitoCore', 'name': 'R_PTRCAT1m_MitoCore Acetyl-CoA + Putrescine --> CoA + N-Acetylputrescine', 'notes': {'Model Reaction Number': '227', 'EC Number': '2.3.1.57', 'SUBSYSTEM': 'Arginine degradation', 'Description': 'Acetyl-CoA + Putrescine --&gt; CoA + N-Acetylputrescine', 'Recon2 id': 'PTRCAT1', 'Recon2 formula': 'accoa[c] + ptrc[c] -&gt; h[c] + coa[c] + aprut[c]', 'KEGG id': 'R01154', 'GENE_ASSOCIATION': 'ENSG00000141504', 'GENE_LIST': 'SAT2', 'HGNC': '23160', 'Gene name': 'spermidine/spermine N1-acetyltransferase family member 2', 'MitoCarta': 'FALSE', 'MitoCarta score': '-8.2811', 'Directionality rationale': 'KEGG, HumanCyc, Brenda, Delta G 5.3kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.3.1.57', 'kegg.reaction': 'R01154'}, '_gpr': cobra.core.gene.GPR('ENSG00000141504'), 'subsystem': 'Arginine degradation', '_genes': {<Gene ENSG00000141504 at 0x17aedc950>}, '_metabolites': {<Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite ptrc_m at 0x166240a50>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite aprut_m at 0x2a2989890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'APRTO2m_MitoCore', 'name': 'R_APRTO2m_MitoCore N-Acetylputrescine + H2O + Oxygen --> N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide', 'notes': {'Model Reaction Number': '228', 'EC Number': '1.4.3.4', 'SUBSYSTEM': 'Arginine degradation', 'Description': 'N-Acetylputrescine + H2O + Oxygen --&gt; N4-Acetylaminobutanal + Ammonia + Hydrogen peroxide', 'Recon2 id': 'APRTO2', 'Recon2 formula': 'h2o[c] + o2[c] + aprut[c] -&gt; h2o2[c] + nh4[c] + n4abutn[c]', 'KEGG id': 'R04025', 'GENE_ASSOCIATION': 'ENSG00000069535', 'GENE_LIST': 'MAOB', 'HGNC': '6834', 'Gene name': 'monoamine oxidase B', 'MitoCarta': 'TRUE', 'MitoCarta score': '4.5081', 'Directionality rationale': 'Oxygen consumption, Ammonia production, KEGG, HumanCyc, Delta G 24kJ/mol', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.4.3.4', 'kegg.reaction': 'R04025'}, '_gpr': cobra.core.gene.GPR('ENSG00000069535'), 'subsystem': 'Arginine degradation', '_genes': {<Gene ENSG00000069535 at 0x29d0b3510>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite o2_m at 0x17b2554d0>: -1.0, <Metabolite aprut_m at 0x2a2989890>: -1.0, <Metabolite h2o2_m at 0x29c72f810>: 1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite n4abutn_m at 0x29d2e3190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NABTNOm', 'name': 'R_NABTNOm N4-Acetylaminobutanal + NAD+ + H2O --> 4-Acetamidobutanoate + NADH + H+', 'notes': {'Model Reaction Number': '229', 'EC Number': '1.2.1.3', 'SUBSYSTEM': 'Arginine degradation', 'Description': 'N4-Acetylaminobutanal + NAD+ + H2O --&gt; 4-Acetamidobutanoate + NADH + H+', 'Recon2 id': 'NABTNOm', 'Recon2 formula': 'h2o[m] + nad[m] + n4abutn[m] -&gt; 2.0 h[m] + nadh[m] + 4aabutn[m]', 'KEGG id': 'R05050', 'GENE_ASSOCIATION': 'ENSG00000137124 or ENSG00000072210 or ENSG00000164904 or ENSG00000143149', 'GENE_LIST': 'ALDH1B1 or ALDH3A2 or ALDH7A1 or ALDH9A1', 'HGNC': '407 or HGNC:403 or HGNC:877 or HGNC:412', 'Gene name': 'aldehyde dehydrogenase family', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '17.6418, 10.116, 13.8432, 23.4278', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 9.8kJ/mol', 'HPA mRNA level in heart tissue': 'Low or Medium or Medium or Medium', 'HPA protein level in heart tissue': 'High or Medium or Low or Low'}, '_annotation': {'ec-code': '1.2.1.3', 'kegg.reaction': 'R05050'}, '_gpr': cobra.core.gene.GPR('ENSG00000137124 or ENSG00000072210 or ENSG00000164904 or ENSG00000143149'), 'subsystem': 'Arginine degradation', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>, <Gene ENSG00000072210 at 0x176ed7c10>, <Gene ENSG00000137124 at 0x176ed4f50>, <Gene ENSG00000143149 at 0x176ed3a50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite n4abutn_m at 0x29d2e3190>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite 4aabutn_m at 0x16d5d06d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '4aabutn_MitoCore', 'name': 'R_4aabutn_MitoCore 4-Acetamidobutanoate + H2O --> 4-Aminobutanoate + Acetate', 'notes': {'Model Reaction Number': '230', 'EC Number': '3.5.1.63', 'SUBSYSTEM': 'Arginine degradation', 'Description': '4-Acetamidobutanoate + H2O --&gt; 4-Aminobutanoate + Acetate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R01987', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'KEGG, HumanCyc'}, '_annotation': {'ec-code': '3.5.1.63', 'kegg.reaction': 'R01987'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Arginine degradation', '_genes': set(), '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite 4aabutn_m at 0x16d5d06d0>: -1.0, <Metabolite 4abut_m at 0x1792bb910>: 1.0, <Metabolite ac_m at 0x29c7a0490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUDC', 'name': 'R_GLUDC L-Glutamate --> 4-Aminobutanoate + CO2', 'notes': {'Model Reaction Number': '231', 'EC Number': '4.1.1.15', 'SUBSYSTEM': 'GABA shunt', 'Description': 'L-Glutamate --&gt; 4-Aminobutanoate + CO2', 'Recon2 id': 'GLUDC', 'Recon2 formula': 'h[c] + glu_L[c] -&gt; co2[c] + 4abut[c]', 'KEGG id': 'R00261', 'GENE_ASSOCIATION': 'ENSG00000136750', 'GENE_LIST': 'GAD2', 'HGNC': '4093', 'Gene name': 'glutamate decarboxylase 2 (pancreatic islets and brain, 65kDa)', 'MitoCarta': 'FALSE', 'MitoCarta score': '-7.9301', 'Directionality rationale': 'CO2 production, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.1.1.15', 'kegg.reaction': 'R00261'}, '_gpr': cobra.core.gene.GPR('ENSG00000136750'), 'subsystem': 'GABA shunt', '_genes': {<Gene ENSG00000136750 at 0x176ed5a50>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite 4abut_c at 0x176226650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '4ABUTtm', 'name': 'R_4ABUTtm 4-aminobutanoate (GABA) mitochondrial transport', 'notes': {'Model Reaction Number': '232', 'EC Number': 'N/A', 'SUBSYSTEM': 'GABA shunt - transport', 'Description': '4-aminobutanoate (GABA) mitochondrial transport', 'Recon2 id': '4ABUTtm', 'Recon2 formula': '4abut[c] --&gt; 4abut[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'GABA shunt - transport', '_genes': set(), '_metabolites': {<Metabolite 4abut_c at 0x176226650>: -1.0, <Metabolite 4abut_m at 0x1792bb910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ABTArm', 'name': 'R_ABTArm 4-Aminobutanoate + 2-Oxoglutarate --> Succinate semialdehyde + L-Glutamate', 'notes': {'Model Reaction Number': '233', 'EC Number': '2.6.1.19', 'SUBSYSTEM': 'GABA shunt', 'Description': '4-Aminobutanoate + 2-Oxoglutarate --&gt; Succinate semialdehyde + L-Glutamate', 'Recon2 id': 'ABTArm', 'Recon2 formula': 'akg[m] + 4abut[m] --&gt; glu_L[m] + sucsal[m]', 'KEGG id': 'R01648', 'GENE_ASSOCIATION': 'ENSG00000183044', 'GENE_LIST': 'ABAT', 'HGNC': '23', 'Gene name': '4-aminobutyrate aminotransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.6859', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.6.1.19', 'kegg.reaction': 'R01648'}, '_gpr': cobra.core.gene.GPR('ENSG00000183044'), 'subsystem': 'GABA shunt', '_genes': {<Gene ENSG00000183044 at 0x176ed5910>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite 4abut_m at 0x1792bb910>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite sucsal_m at 0x29d77c250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0178', 'name': 'R_r0178 Succinate semialdehyde + NAD+ + H2O --> Succinate + NADH + H+', 'notes': {'Model Reaction Number': '234', 'EC Number': '1.2.1.24', 'SUBSYSTEM': 'GABA shunt', 'Description': 'Succinate semialdehyde + NAD+ + H2O --&gt; Succinate + NADH + H+', 'Recon2 id': 'r0178', 'Recon2 formula': 'h2o[m] + nad[m] + sucsal[m] --&gt; 2.0 h[m] + nadh[m] + succ[m]', 'KEGG id': 'R00713', 'GENE_ASSOCIATION': 'ENSG00000112294', 'GENE_LIST': 'ALDH5A1', 'HGNC': '408', 'Gene name': 'aldehyde dehydrogenase 5 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.6859', 'Directionality rationale': 'HumanCyc, KEGG, Brenda, Delta G 9.8kJ/mol', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.2.1.24', 'kegg.reaction': 'R00713'}, '_gpr': cobra.core.gene.GPR('ENSG00000112294'), 'subsystem': 'GABA shunt', '_genes': {<Gene ENSG00000112294 at 0x176ed6290>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite sucsal_m at 0x29d77c250>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUDxm', 'name': 'R_GLUDxm L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+', 'notes': {'Model Reaction Number': '235', 'EC Number': '1.4.1.3', 'SUBSYSTEM': 'Glutamate degradation/synthesis', 'Description': 'L-Glutamate + NAD+ + H2O --&gt; 2-Oxoglutarate + NH3 + NADH + H+', 'Recon2 id': 'GLUDxm', 'Recon2 formula': 'h2o[m] + nad[m] + glu_L[m] --&gt; nh4[m] + h[m] + nadh[m] + akg[m]', 'KEGG id': 'R00243', 'GENE_ASSOCIATION': 'ENSG00000148672 or ENSG00000182890', 'GENE_LIST': 'GLUD1 or GLUD2', 'HGNC': '4335 or HGNC:4336', 'Gene name': 'glutamate dehydrogenase 1 or 2', 'MitoCarta': 'TRUE, FALSE', 'MitoCarta score': '14.9838, 0.4661', 'Directionality rationale': 'NH4 production, Delta G 8.7kJ/mol', 'HPA mRNA level in heart tissue': 'Medium or Medium', 'HPA protein level in heart tissue': 'Medium and Medium'}, '_annotation': {'ec-code': '1.4.1.3', 'kegg.reaction': 'R00243'}, '_gpr': cobra.core.gene.GPR('ENSG00000148672 or ENSG00000182890'), 'subsystem': 'Glutamate degradation/synthesis', '_genes': {<Gene ENSG00000182890 at 0x176ed7fd0>, <Gene ENSG00000148672 at 0x176ed6650>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: -1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite akg_m at 0x2919313d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUDym', 'name': 'R_GLUDym L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+', 'notes': {'Model Reaction Number': '236', 'EC Number': '1.4.1.3', 'SUBSYSTEM': 'Glutamate degradation/synthesis', 'Description': 'L-Glutamate + NADP+ + H2O --&gt; 2-Oxoglutarate + NH3 + NADPH + H+', 'Recon2 id': 'GLUDym', 'Recon2 formula': 'h2o[m] + nadp[m] + glu_L[m] --&gt; nh4[m] + h[m] + nadph[m] + akg[m]', 'KEGG id': 'R00248', 'GENE_ASSOCIATION': 'ENSG00000148672 or ENSG00000182890', 'GENE_LIST': 'GLUD1 or GLUD2', 'HGNC': '4335 or HGNC:4336', 'Gene name': 'glutamate dehydrogenase 1 or 2', 'MitoCarta': 'TRUE, FALSE', 'MitoCarta score': '14.9838, 0.4661', 'Directionality rationale': 'NH4 production, Delta G 8.7kJ/mol', 'HPA mRNA level in heart tissue': 'Medium or Medium', 'HPA protein level in heart tissue': 'Medium and Medium'}, '_annotation': {'ec-code': '1.4.1.3', 'kegg.reaction': 'R00248'}, '_gpr': cobra.core.gene.GPR('ENSG00000148672 or ENSG00000182890'), 'subsystem': 'Glutamate degradation/synthesis', '_genes': {<Gene ENSG00000182890 at 0x176ed7fd0>, <Gene ENSG00000148672 at 0x176ed6650>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: -1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite akg_m at 0x2919313d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUDxi', 'name': 'R_GLUDxi L-Glutamate + NAD+ + H2O --> 2-Oxoglutarate + NH3 + NADH + H+', 'notes': {'Model Reaction Number': '237', 'EC Number': '1.4.1.3', 'SUBSYSTEM': 'Glutamate degradation/synthesis - cytosolic', 'Description': 'L-Glutamate + NAD+ + H2O --&gt; 2-Oxoglutarate + NH3 + NADH + H+', 'Recon2 id': 'GLUDxi', 'Recon2 formula': 'h2o[c] + nad[c] + glu_L[c] -&gt; nh4[c] + h[c] + nadh[c] + akg[c]', 'KEGG id': 'R00243', 'GENE_ASSOCIATION': 'ENSG00000148672', 'GENE_LIST': 'GLUD1', 'HGNC': '4335', 'Gene name': 'glutamate dehydrogenase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.9838', 'Directionality rationale': 'NH4 production, Delta G 8.7kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.4.1.3', 'kegg.reaction': 'R00243'}, '_gpr': cobra.core.gene.GPR('ENSG00000148672'), 'subsystem': 'Glutamate degradation/synthesis - cytosolic', '_genes': {<Gene ENSG00000148672 at 0x176ed6650>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite akg_c at 0x16ac21410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUDy', 'name': 'R_GLUDy L-Glutamate + NADP+ + H2O --> 2-Oxoglutarate + NH3 + NADPH + H+', 'notes': {'Model Reaction Number': '238', 'EC Number': '1.4.1.3', 'SUBSYSTEM': 'Glutamate degradation/synthesis - cytosolic', 'Description': 'L-Glutamate + NADP+ + H2O --&gt; 2-Oxoglutarate + NH3 + NADPH + H+', 'Recon2 id': 'GLUDy', 'Recon2 formula': 'h2o[c] + nadp[c] + glu_L[c] -&gt; nh4[c] + h[c] + nadph[c] + akg[c]', 'KEGG id': 'R00248', 'GENE_ASSOCIATION': 'ENSG00000148672', 'GENE_LIST': 'GLUD1', 'HGNC': '4335', 'Gene name': 'glutamate dehydrogenase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.9838', 'Directionality rationale': 'NH4 production, Delta G 8.7kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.4.1.3', 'kegg.reaction': 'R00248'}, '_gpr': cobra.core.gene.GPR('ENSG00000148672'), 'subsystem': 'Glutamate degradation/synthesis - cytosolic', '_genes': {<Gene ENSG00000148672 at 0x176ed6650>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite akg_c at 0x16ac21410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLNS', 'name': 'R_GLNS ATP + L-Glutamate + NH3 --> ADP + Orthophosphate + L-Glutamine', 'notes': {'Model Reaction Number': '239', 'EC Number': '6.3.1.2', 'SUBSYSTEM': 'Glutamine synthesis', 'Description': 'ATP + L-Glutamate + NH3 --&gt; ADP + Orthophosphate + L-Glutamine', 'Recon2 id': 'GLNS', 'Recon2 formula': 'nh4[c] + atp[c] + glu_L[c] -&gt; h[c] + adp[c] + pi[c] + gln_L[c]', 'KEGG id': 'R00253', 'GENE_ASSOCIATION': 'ENSG00000135821', 'GENE_LIST': 'GLUL', 'HGNC': '4341', 'Gene name': 'Glutamine synthetase', 'MitoCarta': 'FALSE', 'MitoCarta score': '0.5237', 'Directionality rationale': 'ATP consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '6.3.1.2', 'kegg.reaction': 'R00253'}, '_gpr': cobra.core.gene.GPR('ENSG00000135821'), 'subsystem': 'Glutamine synthesis', '_genes': {<Gene ENSG00000135821 at 0x176ed62d0>}, '_metabolites': {<Metabolite nh4_c at 0x1639455d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite gln_L_c at 0x13354b910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUNm', 'name': 'R_GLUNm L-Glutamine + H2O --> L-Glutamate + NH3', 'notes': {'Model Reaction Number': '240', 'EC Number': '3.5.1.2', 'SUBSYSTEM': 'Glutamine degradation', 'Description': 'L-Glutamine + H2O --&gt; L-Glutamate + NH3', 'Recon2 id': 'GLUNm', 'Recon2 formula': 'h2o[m] + gln_L[m] -&gt; nh4[m] + glu_L[m]', 'KEGG id': 'R00256', 'GENE_ASSOCIATION': 'ENSG00000115419 or ENSG00000135423', 'GENE_LIST': 'GLS or GLS2', 'HGNC': '4331 or HGNC:29570', 'Gene name': 'Glutaminase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '20.9853, 13.0829', 'Directionality rationale': 'Literature, Modelling', 'HPA mRNA level in heart tissue': 'Medium or Not detected', 'HPA protein level in heart tissue': 'Medium or Medium'}, '_annotation': {'ec-code': '3.5.1.2', 'kegg.reaction': 'R00256'}, '_gpr': cobra.core.gene.GPR('ENSG00000115419 or ENSG00000135423'), 'subsystem': 'Glutamine degradation', '_genes': {<Gene ENSG00000135423 at 0x176ed7dd0>, <Gene ENSG00000115419 at 0x176ed45d0>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite gln_L_m at 0x133551a10>: -1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUN_MitoCore', 'name': 'R_GLUN_MitoCore L-Glutamine + H2O --> L-Glutamate + NH3', 'notes': {'Model Reaction Number': '241', 'EC Number': '3.5.1.2', 'SUBSYSTEM': 'Glutamine degradation - cytosolic', 'Description': 'L-Glutamine + H2O --&gt; L-Glutamate + NH3', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00256', 'GENE_ASSOCIATION': 'ENSG00000115419 or ENSG00000084774', 'GENE_LIST': 'GLS or CAD', 'HGNC': '4331 or HGNC:1424', 'Gene name': 'Glutaminase or carbamoyl-phosphate synthetase 2', 'MitoCarta': 'TRUE, FALSE', 'MitoCarta score': '20.9853, 1.0616', 'Directionality rationale': 'Literature, Modelling', 'HPA mRNA level in heart tissue': 'Medium or Low', 'HPA protein level in heart tissue': 'Medium or Medium'}, '_annotation': {'ec-code': '3.5.1.2', 'kegg.reaction': 'R00256'}, '_gpr': cobra.core.gene.GPR('ENSG00000115419 or ENSG00000084774'), 'subsystem': 'Glutamine degradation - cytosolic', '_genes': {<Gene ENSG00000084774 at 0x176ed70d0>, <Gene ENSG00000115419 at 0x176ed45d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite gln_L_c at 0x13354b910>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGCD', 'name': 'R_PGCD 3-Phospho-D-glycerate + NAD+ --> 3-Phosphonooxypyruvate + NADH + H+', 'notes': {'Model Reaction Number': '242', 'EC Number': '1.1.1.95', 'SUBSYSTEM': 'Serine and glycine biosynthesis', 'Description': '3-Phospho-D-glycerate + NAD+ --&gt; 3-Phosphonooxypyruvate + NADH + H+', 'Recon2 id': 'PGCD', 'Recon2 formula': 'nad[c] + 3pg[c] -&gt; h[c] + nadh[c] + 3php[c]', 'KEGG id': 'R01513', 'GENE_ASSOCIATION': 'ENSG00000092621', 'GENE_LIST': 'PHGDH', 'HGNC': '8923', 'Gene name': 'phosphoglycerate dehydrogenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.3441', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.1.1.95', 'kegg.reaction': 'R01513'}, '_gpr': cobra.core.gene.GPR('ENSG00000092621'), 'subsystem': 'Serine and glycine biosynthesis', '_genes': {<Gene ENSG00000092621 at 0x176ed6e50>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite 3pg_c at 0x2ae3ee710>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite 3php_c at 0x179bdb250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PSERT', 'name': 'R_PSERT 3-Phosphonooxypyruvate + L-Glutamate --> O-Phospho-L-serine + 2-Oxoglutarate', 'notes': {'Model Reaction Number': '243', 'EC Number': '2.6.1.52', 'SUBSYSTEM': 'Serine and glycine biosynthesis', 'Description': '3-Phosphonooxypyruvate + L-Glutamate --&gt; O-Phospho-L-serine + 2-Oxoglutarate', 'Recon2 id': 'PSERT', 'Recon2 formula': 'glu_L[c] + 3php[c] -&gt; akg[c] + pser_L[c]', 'KEGG id': 'R04173', 'GENE_ASSOCIATION': 'ENSG00000135069', 'GENE_LIST': 'PSAT1', 'HGNC': '19129', 'Gene name': 'phosphoserine aminotransferase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-4.0561', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.6.1.52', 'kegg.reaction': 'R04173'}, '_gpr': cobra.core.gene.GPR('ENSG00000135069'), 'subsystem': 'Serine and glycine biosynthesis', '_genes': {<Gene ENSG00000135069 at 0x176ed69d0>}, '_metabolites': {<Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite 3php_c at 0x179bdb250>: -1.0, <Metabolite akg_c at 0x16ac21410>: 1.0, <Metabolite pser_L_c at 0x17ae80a90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PSP_L', 'name': 'R_PSP_L O-Phospho-L-serine + H2O --> L-Serine + Orthophosphate', 'notes': {'Model Reaction Number': '244', 'EC Number': '3.1.3.3', 'SUBSYSTEM': 'Serine and glycine biosynthesis', 'Description': 'O-Phospho-L-serine + H2O --&gt; L-Serine + Orthophosphate', 'Recon2 id': 'PSP_L', 'Recon2 formula': 'h2o[c] + pser_L[c] -&gt; pi[c] + ser_L[c]', 'KEGG id': 'R00582', 'GENE_ASSOCIATION': 'ENSG00000146733', 'GENE_LIST': 'PSPH', 'HGNC': '9577', 'Gene name': 'phosphoserine phosphatase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.0112', 'Directionality rationale': 'HumanCyc, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.1.3.3', 'kegg.reaction': 'R00582'}, '_gpr': cobra.core.gene.GPR('ENSG00000146733'), 'subsystem': 'Serine and glycine biosynthesis', '_genes': {<Gene ENSG00000146733 at 0x176ed7a90>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite pser_L_c at 0x17ae80a90>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite ser_L_c at 0x29d206f50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GHMT2r', 'name': 'R_GHMT2r Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O', 'notes': {'Model Reaction Number': '245', 'EC Number': '2.1.2.1', 'SUBSYSTEM': 'Serine and glycine biosynthesis', 'Description': 'Tetrahydrofolate + L-Serine --&gt; 5,10-Methylenetetrahydrofolate + Glycine + H2O', 'Recon2 id': 'GHMT2r', 'Recon2 formula': 'thf[c] + ser_L[c] --&gt; h2o[c] + gly[c] + mlthf[c]', 'KEGG id': 'R00945', 'GENE_ASSOCIATION': 'ENSG00000176974', 'GENE_LIST': 'SHMT1', 'HGNC': '10850', 'Gene name': 'serine hydroxymethyltransferase 1 (soluble)', 'MitoCarta': 'TRUE', 'MitoCarta score': '6.2275', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.1.2.1', 'kegg.reaction': 'R00945'}, '_gpr': cobra.core.gene.GPR('ENSG00000176974'), 'subsystem': 'Serine and glycine biosynthesis', '_genes': {<Gene ENSG00000176974 at 0x176ed50d0>}, '_metabolites': {<Metabolite thf_c at 0x29d7f2310>: -1.0, <Metabolite ser_L_c at 0x29d206f50>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite gly_c at 0x29c737c90>: 1.0, <Metabolite mlthf_c at 0x29d0e5110>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FOLR2', 'name': 'R_FOLR2 Folate + NADPH + H+ --> Dihydrofolate + NADP+', 'notes': {'Model Reaction Number': '246', 'EC Number': '1.5.1.3', 'SUBSYSTEM': 'Folate cytosolic', 'Description': 'Folate + NADPH + H+ --&gt; Dihydrofolate + NADP+', 'Recon2 id': 'FOLR2', 'Recon2 formula': 'nadph[c] + fol[c] -&gt; nadp[c] + dhf[c]', 'KEGG id': 'R02236', 'GENE_ASSOCIATION': 'ENSG00000228716', 'GENE_LIST': 'DHFR', 'HGNC': '2861', 'Gene name': 'dihydrofolate reductase', 'MitoCarta': 'FALSE', 'MitoCarta score': '0.8474', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.3', 'kegg.reaction': 'R02236'}, '_gpr': cobra.core.gene.GPR('ENSG00000228716'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000228716 at 0x176ed4250>}, '_metabolites': {<Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite fol_c at 0x29d072290>: -1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite dhf_c at 0x29d0a5190>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'DHFR', 'name': 'R_DHFR Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+', 'notes': {'Model Reaction Number': '247', 'EC Number': '1.5.1.3', 'SUBSYSTEM': 'Folate cytosolic', 'Description': 'Dihydrofolate + NADPH + H+ --&gt; Tetrahydrofolate + NADP+', 'Recon2 id': 'DHFR', 'Recon2 formula': 'h[c] + nadph[c] + dhf[c] --&gt; nadp[c] + thf[c]', 'KEGG id': 'R00939', 'GENE_ASSOCIATION': 'ENSG00000228716', 'GENE_LIST': 'DHFR', 'HGNC': '2861', 'Gene name': 'dihydrofolate reductase', 'MitoCarta': 'FALSE', 'MitoCarta score': '0.8474', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.3', 'kegg.reaction': 'R00939'}, '_gpr': cobra.core.gene.GPR('ENSG00000228716'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000228716 at 0x176ed4250>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite dhf_c at 0x29d0a5190>: -1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite thf_c at 0x29d7f2310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'MTHFD', 'name': 'R_MTHFD 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH', 'notes': {'Model Reaction Number': '248', 'EC Number': '1.5.1.5', 'SUBSYSTEM': 'Folate cytosolic', 'Description': '5,10-Methylenetetrahydrofolate + NADP+ --&gt; 5,10-Methenyltetrahydrofolate + NADPH', 'Recon2 id': 'MTHFD', 'Recon2 formula': 'nadp[c] + mlthf[c] --&gt; nadph[c] + methf[c]', 'KEGG id': 'R01220', 'GENE_ASSOCIATION': 'ENSG00000100714', 'GENE_LIST': 'MTHFD1', 'HGNC': '7432', 'Gene name': 'methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1, methenyltetrahydrofolate cyclohydrolase, formyltetrahydrofolate synthetase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.7609', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.5.1.5', 'kegg.reaction': 'R01220'}, '_gpr': cobra.core.gene.GPR('ENSG00000100714'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000100714 at 0x176ed4350>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite mlthf_c at 0x29d0e5110>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite methf_c at 0x29d0e6c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MTHFC', 'name': 'R_MTHFC 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+', 'notes': {'Model Reaction Number': '249', 'EC Number': '3.5.4.9', 'SUBSYSTEM': 'Folate cytosolic', 'Description': '5,10-Methenyltetrahydrofolate + H2O --&gt; 10-Formyltetrahydrofolate + H+', 'Recon2 id': 'MTHFC', 'Recon2 formula': 'h2o[c] + methf[c] --&gt; 10fthf[c] + h[c]', 'KEGG id': 'R01655', 'GENE_ASSOCIATION': 'ENSG00000100714 or ENSG00000163738', 'GENE_LIST': 'MTHFD1 or MTHFD2L', 'HGNC': '7432 or HGNC:31865', 'Gene name': 'methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1, methenyltetrahydrofolate cyclohydrolase, formyltetrahydrofolate synthetase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '5.7609, 13.7285', 'HPA mRNA level in heart tissue': 'Medium or Low', 'HPA protein level in heart tissue': 'Low or High'}, '_annotation': {'ec-code': '3.5.4.9', 'kegg.reaction': 'R01655'}, '_gpr': cobra.core.gene.GPR('ENSG00000100714 or ENSG00000163738'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000100714 at 0x176ed4350>, <Gene ENSG00000163738 at 0x176ed4b50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite methf_c at 0x29d0e6c90>: -1.0, <Metabolite 10fthf_c at 0x29755b550>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FTCD', 'name': 'R_FTCD 5-Formiminotetrahydrofolate + H+ --> 5,10-Methenyltetrahydrofolate + Ammonia', 'notes': {'Model Reaction Number': '250', 'EC Number': '4.3.1.4', 'SUBSYSTEM': 'Folate cytosolic', 'Description': '5-Formiminotetrahydrofolate + H+ --&gt; 5,10-Methenyltetrahydrofolate + Ammonia', 'Recon2 id': 'FTCD', 'Recon2 formula': '2.0 h[c] + 5forthf[c] -&gt; nh4[c] + methf[c]', 'KEGG id': 'R02302', 'GENE_ASSOCIATION': 'ENSG00000160282', 'GENE_LIST': 'FTCD', 'HGNC': '3974', 'Gene name': 'formimidoyltransferase cyclodeaminase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.3.1.4', 'kegg.reaction': 'R02302'}, '_gpr': cobra.core.gene.GPR('ENSG00000160282'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000160282 at 0x176ea4290>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -2.0, <Metabolite 5forthf_c at 0x1793229d0>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite methf_c at 0x29d0e6c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FTHFL', 'name': 'R_FTHFL Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate', 'notes': {'Model Reaction Number': '251', 'EC Number': '6.3.4.3', 'SUBSYSTEM': 'Folate cytosolic', 'Description': 'Tetrahydrofolate + Formate + ATP --&gt; ADP + Orthophosphate + 10-Formyltetrahydrofolate', 'Recon2 id': 'FTHFL', 'Recon2 formula': 'atp[c] + thf[c] + for[c] -&gt; 10fthf[c] + adp[c] + pi[c]', 'KEGG id': 'R00943', 'GENE_ASSOCIATION': 'ENSG00000100714', 'GENE_LIST': 'MTHFD1', 'HGNC': '7432', 'Gene name': 'methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1, methenyltetrahydrofolate cyclohydrolase, formyltetrahydrofolate synthetase', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.7609', 'Directionality rationale': 'ATP consumption, KEGG, HumanCyc, literature PMID:20645850', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '6.3.4.3', 'kegg.reaction': 'R00943'}, '_gpr': cobra.core.gene.GPR('ENSG00000100714'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000100714 at 0x176ed4350>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite thf_c at 0x29d7f2310>: -1.0, <Metabolite for_c at 0x13352d2d0>: -1.0, <Metabolite 10fthf_c at 0x29755b550>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FTHFDH', 'name': 'R_FTHFDH 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '252', 'EC Number': '1.5.1.6', 'SUBSYSTEM': 'Folate cytosolic', 'Description': '10-Formyltetrahydrofolate + NADP+ + H2O --&gt; Tetrahydrofolate + CO2 + NADPH + H+', 'Recon2 id': 'FTHFDH', 'Recon2 formula': '10fthf[c] + h2o[c] + nadp[c] -&gt; h[c] + nadph[c] + co2[c] + thf[c]', 'KEGG id': 'R00941', 'GENE_ASSOCIATION': 'ENSG00000144908', 'GENE_LIST': 'ALDH1L1', 'HGNC': '3978', 'Gene name': 'aldehyde dehydrogenase 1 family, member L1', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.9576', 'Directionality rationale': 'CO2 production, Delta G 9kJ/mol, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.5.1.6', 'kegg.reaction': 'R00941'}, '_gpr': cobra.core.gene.GPR('ENSG00000144908'), 'subsystem': 'Folate cytosolic', '_genes': {<Gene ENSG00000144908 at 0x176ea45d0>}, '_metabolites': {<Metabolite 10fthf_c at 0x29755b550>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite thf_c at 0x29d7f2310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0060', 'name': 'R_r0060 L-Serine --> Pyruvate + Ammonia', 'notes': {'Model Reaction Number': '253', 'EC Number': '4.3.1.17 or 4.3.1.19', 'SUBSYSTEM': 'Serine degradation', 'Description': 'L-Serine --&gt; Pyruvate + Ammonia', 'Recon2 id': 'r0060', 'Recon2 formula': 'ser_L[c] -&gt; nh4[c] + pyr[c]', 'KEGG id': 'R00220', 'GENE_ASSOCIATION': 'ENSG00000135094 or ENSG00000167720 or ENSG00000139410', 'GENE_LIST': 'SDS or SRR or SDSL', 'HGNC': '10691 or HGNC:14398 or HGNC:30404', 'Gene name': 'Serine dehydratases', 'MitoCarta': 'FALSE, FALSE, TRUE', 'MitoCarta score': '3.7304, 1.1551, 8.1111', 'Directionality rationale': 'HumanCyc, NH4 production, Delta G 7.5kJ/mol', 'HPA mRNA level in heart tissue': 'Not detected or Low or Low', 'HPA protein level in heart tissue': 'Medium or Low or High'}, '_annotation': {'ec-code': ['4.3.1.17', '4.3.1.19'], 'kegg.reaction': 'R00220'}, '_gpr': cobra.core.gene.GPR('ENSG00000135094 or ENSG00000167720 or ENSG00000139410'), 'subsystem': 'Serine degradation', '_genes': {<Gene ENSG00000167720 at 0x176ea4910>, <Gene ENSG00000135094 at 0x176ea1f50>, <Gene ENSG00000139410 at 0x176ea2290>}, '_metabolites': {<Metabolite ser_L_c at 0x29d206f50>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite pyr_c at 0x29d0b3990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GHMT2rm', 'name': 'R_GHMT2rm Tetrahydrofolate + L-Serine --> 5,10-Methylenetetrahydrofolate + Glycine + H2O', 'notes': {'Model Reaction Number': '254', 'EC Number': '2.1.2.1', 'SUBSYSTEM': 'Serine and glycine biosynthesis', 'Description': 'Tetrahydrofolate + L-Serine --&gt; 5,10-Methylenetetrahydrofolate + Glycine + H2O', 'Recon2 id': 'GHMT2rm', 'Recon2 formula': 'thf[m] + ser_L[m] --&gt; h2o[m] + gly[m] + mlthf[m]', 'KEGG id': 'R00945', 'GENE_ASSOCIATION': 'ENSG00000182199', 'GENE_LIST': 'SHMT2', 'HGNC': '10852', 'Gene name': 'serine hydroxymethyltransferase 2 (mitochondrial)', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.6448', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.1.2.1', 'kegg.reaction': 'R00945'}, '_gpr': cobra.core.gene.GPR('ENSG00000182199'), 'subsystem': 'Serine and glycine biosynthesis', '_genes': {<Gene ENSG00000182199 at 0x176ea4c10>}, '_metabolites': {<Metabolite thf_m at 0x29d733810>: -1.0, <Metabolite ser_L_m at 0x29d27c990>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0, <Metabolite gly_m at 0x29c735650>: 1.0, <Metabolite mlthf_m at 0x29d0e4310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GCCam', 'name': 'R_GCCam Glycine + Lipoylprotein --> S-Aminomethyldihydrolipoylprotein + CO2', 'notes': {'Model Reaction Number': '255', 'EC Number': '1.4.4.2', 'SUBSYSTEM': 'Glycine cleavage system (GCS)', 'Description': 'Glycine + Lipoylprotein --&gt; S-Aminomethyldihydrolipoylprotein + CO2', 'Recon2 id': 'GCCam', 'Recon2 formula': 'h[m] + gly[m] + lpro[m] --&gt; co2[m] + alpro[m]', 'KEGG id': 'R03425', 'GENE_ASSOCIATION': 'ENSG00000178445', 'GENE_LIST': 'GLDC', 'HGNC': '4313', 'Gene name': 'glycine dehydrogenase (decarboxylating)', 'MitoCarta': 'TRUE', 'MitoCarta score': '20.0566', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.4.4.2', 'kegg.reaction': 'R03425'}, '_gpr': cobra.core.gene.GPR('ENSG00000178445'), 'subsystem': 'Glycine cleavage system (GCS)', '_genes': {<Gene ENSG00000178445 at 0x176ea4f10>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite gly_m at 0x29c735650>: -1.0, <Metabolite lpro_m at 0x29d023290>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite alpro_m at 0x17ae31b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GCCbim', 'name': 'R_GCCbim S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --> Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia', 'notes': {'Model Reaction Number': '256', 'EC Number': '2.1.2.10', 'SUBSYSTEM': 'Glycine cleavage system (GCS)', 'Description': 'S-Aminomethyldihydrolipoylprotein + Tetrahydrofolate --&gt; Dihydrolipoylprotein + 5,10-Methylenetetrahydrofolate + Ammonia', 'Recon2 id': 'GCCbim', 'Recon2 formula': 'thf[m] + alpro[m] -&gt; nh4[m] + mlthf[m] + dhlpro[m]', 'KEGG id': 'R04125', 'GENE_ASSOCIATION': 'ENSG00000145020', 'GENE_LIST': 'AMT', 'HGNC': '473', 'Gene name': 'aminomethyltransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.9633', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.1.2.10', 'kegg.reaction': 'R04125'}, '_gpr': cobra.core.gene.GPR('ENSG00000145020'), 'subsystem': 'Glycine cleavage system (GCS)', '_genes': {<Gene ENSG00000145020 at 0x176ea5190>}, '_metabolites': {<Metabolite thf_m at 0x29d733810>: -1.0, <Metabolite alpro_m at 0x17ae31b50>: -1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite mlthf_m at 0x29d0e4310>: 1.0, <Metabolite dhlpro_m at 0x29c7112d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GCCcm', 'name': 'R_GCCcm Dihydrolipoylprotein + NAD+ --> Lipoylprotein + NADH + H+', 'notes': {'Model Reaction Number': '257', 'EC Number': '1.8.1.4', 'SUBSYSTEM': 'Glycine cleavage system (GCS)', 'Description': 'Dihydrolipoylprotein + NAD+ --&gt; Lipoylprotein + NADH + H+', 'Recon2 id': 'GCCcm', 'Recon2 formula': 'nad[m] + dhlpro[m] --&gt; h[m] + nadh[m] + lpro[m]', 'KEGG id': 'R03815', 'GENE_ASSOCIATION': 'ENSG00000091140', 'GENE_LIST': 'DLD', 'HGNC': '2898', 'Gene name': 'dihydrolipoamide dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '31.2307', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.8.1.4', 'kegg.reaction': 'R03815'}, '_gpr': cobra.core.gene.GPR('ENSG00000091140'), 'subsystem': 'Glycine cleavage system (GCS)', '_genes': {<Gene ENSG00000091140 at 0x28abca110>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite dhlpro_m at 0x29c7112d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite lpro_m at 0x29d023290>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0514', 'name': 'R_r0514 Folate + NADPH + H+ --> Dihydrofolate + NADP+', 'notes': {'Model Reaction Number': '258', 'EC Number': '1.5.1.3', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': 'Folate + NADPH + H+ --&gt; Dihydrofolate + NADP+', 'Recon2 id': 'r0514', 'Recon2 formula': 'nadph[m] + fol[m] -&gt; nadp[m] + dhf[m]', 'KEGG id': 'R02236', 'GENE_ASSOCIATION': 'ENSG00000178700', 'GENE_LIST': 'DHFRL1', 'HGNC': '27309', 'Gene name': 'dihydrofolate reductase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.9119', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.5.1.3', 'kegg.reaction': 'R02236'}, '_gpr': cobra.core.gene.GPR('ENSG00000178700'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000178700 at 0x176ea5550>}, '_metabolites': {<Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite fol_m at 0x29d007fd0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite dhf_m at 0x29d0a46d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0226', 'name': 'R_r0226 Dihydrofolate + NADPH + H+ --> Tetrahydrofolate + NADP+', 'notes': {'Model Reaction Number': '259', 'EC Number': '1.5.1.3', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': 'Dihydrofolate + NADPH + H+ --&gt; Tetrahydrofolate + NADP+', 'Recon2 id': 'r0226', 'Recon2 formula': 'h[m] + nadph[m] + dhf[m] -&gt; nadp[m] + thf[m]', 'KEGG id': 'R00939', 'GENE_ASSOCIATION': 'ENSG00000178700', 'GENE_LIST': 'DHFRL1', 'HGNC': '27309', 'Gene name': 'dihydrofolate reductase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.9119', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.5.1.3', 'kegg.reaction': 'R00939'}, '_gpr': cobra.core.gene.GPR('ENSG00000178700'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000178700 at 0x176ea5550>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite dhf_m at 0x29d0a46d0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite thf_m at 0x29d733810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MTHFDm', 'name': 'R_MTHFDm 5,10-Methylenetetrahydrofolate + NADP+ --> 5,10-Methenyltetrahydrofolate + NADPH', 'notes': {'Model Reaction Number': '260', 'EC Number': '1.5.1.5', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': '5,10-Methylenetetrahydrofolate + NADP+ --&gt; 5,10-Methenyltetrahydrofolate + NADPH', 'Recon2 id': 'MTHFDm', 'Recon2 formula': 'nadp[m] + mlthf[m] --&gt; nadph[m] + methf[m]', 'KEGG id': 'R01220', 'GENE_ASSOCIATION': 'ENSG00000163738', 'GENE_LIST': 'MTHFD2L', 'HGNC': '31865', 'Gene name': 'methylenetetrahydrofolate dehydrogenase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.7285', 'Directionality rationale': 'Literature', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.5.1.5', 'kegg.reaction': 'R01220'}, '_gpr': cobra.core.gene.GPR('ENSG00000163738'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000163738 at 0x176ed4b50>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite mlthf_m at 0x29d0e4310>: -1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite methf_m at 0x16c8b5410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'MTHFD2m', 'name': 'R_MTHFD2m 5,10-Methylenetetrahydrofolate + NAD+ --> 5,10-Methenyltetrahydrofolate + NADH', 'notes': {'Model Reaction Number': '261', 'EC Number': '1.5.1.15', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': '5,10-Methylenetetrahydrofolate + NAD+ --&gt; 5,10-Methenyltetrahydrofolate + NADH', 'Recon2 id': 'MTHFD2m', 'Recon2 formula': 'nad[m] + mlthf[m] --&gt; nadh[m] + methf[m]', 'KEGG id': 'R01218', 'GENE_ASSOCIATION': 'ENSG00000163738 or ENSG00000065911', 'GENE_LIST': 'MTHFD2L or MTHFD2', 'HGNC': '31865 or HGNC:7434', 'Gene name': 'methylenetetrahydrofolate dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '13.7285, 15.5945', 'Directionality rationale': 'Literature, HumanCyc', 'HPA mRNA level in heart tissue': 'Low, Low', 'HPA protein level in heart tissue': 'High, Not detected'}, '_annotation': {'ec-code': '1.5.1.15', 'kegg.reaction': 'R01218'}, '_gpr': cobra.core.gene.GPR('ENSG00000163738 or ENSG00000065911'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000065911 at 0x176ea5890>, <Gene ENSG00000163738 at 0x176ed4b50>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite mlthf_m at 0x29d0e4310>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite methf_m at 0x16c8b5410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'MTHFCm', 'name': 'R_MTHFCm 5,10-Methenyltetrahydrofolate + H2O --> 10-Formyltetrahydrofolate + H+', 'notes': {'Model Reaction Number': '262', 'EC Number': '3.5.4.9', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': '5,10-Methenyltetrahydrofolate + H2O --&gt; 10-Formyltetrahydrofolate + H+', 'Recon2 id': 'MTHFCm', 'Recon2 formula': 'h2o[m] + methf[m] --&gt; 10fthf[m] + h[m]', 'KEGG id': 'R01655', 'GENE_ASSOCIATION': 'ENSG00000163738 or ENSG00000065911', 'GENE_LIST': 'MTHFD2L or MTHFD2', 'HGNC': '31865 or HGNC:7434', 'Gene name': 'methylenetetrahydrofolate dehydrogenase', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '13.7285, 15.5945', 'HPA mRNA level in heart tissue': 'Low, Low', 'HPA protein level in heart tissue': 'High, Not detected'}, '_annotation': {'ec-code': '3.5.4.9', 'kegg.reaction': 'R01655'}, '_gpr': cobra.core.gene.GPR('ENSG00000163738 or ENSG00000065911'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000065911 at 0x176ea5890>, <Gene ENSG00000163738 at 0x176ed4b50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite methf_m at 0x16c8b5410>: -1.0, <Metabolite 10fthf_m at 0x2a2f062d0>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FTHFLm', 'name': 'R_FTHFLm Tetrahydrofolate + Formate + ATP --> ADP + Orthophosphate + 10-Formyltetrahydrofolate', 'notes': {'Model Reaction Number': '263', 'EC Number': '6.3.4.3', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': 'Tetrahydrofolate + Formate + ATP --&gt; ADP + Orthophosphate + 10-Formyltetrahydrofolate', 'Recon2 id': 'FTHFLm', 'Recon2 formula': 'atp[m] + thf[m] + for[m] -&gt; 10fthf[m] + adp[m] + pi[m]', 'KEGG id': 'R00943', 'GENE_ASSOCIATION': 'ENSG00000120254', 'GENE_LIST': 'MTHFD1L', 'HGNC': '21055', 'Gene name': 'methylenetetrahydrofolate dehydrogenase (NADP+ dependent) 1-like', 'MitoCarta': 'TRUE', 'MitoCarta score': '19.7152', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '6.3.4.3', 'kegg.reaction': 'R00943'}, '_gpr': cobra.core.gene.GPR('ENSG00000120254'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000120254 at 0x176ea5b90>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite thf_m at 0x29d733810>: -1.0, <Metabolite for_m at 0x133547b90>: -1.0, <Metabolite 10fthf_m at 0x2a2f062d0>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FTHFDHm_MitoCore', 'name': 'R_FTHFDHm_MitoCore 10-Formyltetrahydrofolate + NADP+ + H2O --> Tetrahydrofolate + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '264', 'EC Number': '1.5.1.6', 'SUBSYSTEM': 'Folate mitochondrial', 'Description': '10-Formyltetrahydrofolate + NADP+ + H2O --&gt; Tetrahydrofolate + CO2 + NADPH + H+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00941', 'GENE_ASSOCIATION': 'ENSG00000136010', 'GENE_LIST': 'ALDH1L2', 'HGNC': '26777', 'Gene name': 'aldehyde dehydrogenase 1 family, member L2', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.3962', 'Directionality rationale': 'CO2 production, Delta G 9kJ/mol, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.5.1.6', 'kegg.reaction': 'R00941'}, '_gpr': cobra.core.gene.GPR('ENSG00000136010'), 'subsystem': 'Folate mitochondrial', '_genes': {<Gene ENSG00000136010 at 0x176ea5ed0>}, '_metabolites': {<Metabolite 10fthf_m at 0x2a2f062d0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite thf_m at 0x29d733810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYATm', 'name': 'R_GLYATm Acetyl-CoA + Glycine --> CoA + L-2-Amino-3-oxobutanoic acid', 'notes': {'Model Reaction Number': '265', 'EC Number': '2.3.1.29', 'SUBSYSTEM': 'Glycine degradation', 'Description': 'Acetyl-CoA + Glycine --&gt; CoA + L-2-Amino-3-oxobutanoic acid', 'Recon2 id': 'GLYATm', 'Recon2 formula': 'accoa[m] + gly[m] --&gt; coa[m] + 2aobut[m]', 'KEGG id': 'R00371', 'GENE_ASSOCIATION': 'ENSG00000100116', 'GENE_LIST': 'GCAT', 'HGNC': '4188', 'Gene name': 'glycine C-acetyltransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '19.7152', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.29', 'kegg.reaction': 'R00371'}, '_gpr': cobra.core.gene.GPR('ENSG00000100116'), 'subsystem': 'Glycine degradation', '_genes': {<Gene ENSG00000100116 at 0x176ea6210>}, '_metabolites': {<Metabolite accoa_m at 0x164b26550>: -1.0, <Metabolite gly_m at 0x29c735650>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite 2aobut_m at 0x2a29b6bd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AOBUTDsm', 'name': 'R_AOBUTDsm L-2-Amino-3-oxobutanoic acid --> Aminoacetone + CO2', 'notes': {'Model Reaction Number': '266', 'EC Number': 'Non-enzymatic', 'SUBSYSTEM': 'Glycine degradation', 'Description': 'L-2-Amino-3-oxobutanoic acid --&gt; Aminoacetone + CO2', 'Recon2 id': 'AOBUTDsm', 'Recon2 formula': 'h[m] + 2aobut[m] -&gt; co2[m] + aact[m]', 'KEGG id': 'R03758', 'GENE_ASSOCIATION': 'Non-enzymatic', 'GENE_LIST': 'Non-enzymatic', 'Gene name': 'Non-enzymatic', 'Directionality rationale': 'CO2 production, KEGG'}, '_annotation': {'kegg.reaction': 'R03758'}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Glycine degradation', '_genes': set(), '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite 2aobut_m at 0x2a29b6bd0>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite aact_m at 0x289cc6490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AACTOORm_MitoCore', 'name': 'R_AACTOORm_MitoCore Aminoacetone + H2O + Oxygen --> Methylglyoxal + Ammonia + Hydrogen peroxide', 'notes': {'Model Reaction Number': '267', 'EC Number': '1.4.3.4', 'SUBSYSTEM': 'Glycine degradation', 'Description': 'Aminoacetone + H2O + Oxygen --&gt; Methylglyoxal + Ammonia + Hydrogen peroxide', 'Recon2 id': 'AACTOOR', 'Recon2 formula': 'h2o[c] + o2[c] + aact[c] -&gt; h2o2[c] + nh4[c] + mthgxl[c]', 'KEGG id': 'R02529', 'GENE_ASSOCIATION': 'ENSG00000069535', 'GENE_LIST': 'MAOB', 'HGNC': '6834', 'Gene name': 'monoamine oxidase B', 'MitoCarta': 'TRUE', 'MitoCarta score': '4.5081', 'Directionality rationale': 'Oxygen consumption, ammonia production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.4.3.4', 'kegg.reaction': 'R02529'}, '_gpr': cobra.core.gene.GPR('ENSG00000069535'), 'subsystem': 'Glycine degradation', '_genes': {<Gene ENSG00000069535 at 0x29d0b3510>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite o2_m at 0x17b2554d0>: -1.0, <Metabolite aact_m at 0x289cc6490>: -1.0, <Metabolite h2o2_m at 0x29c72f810>: 1.0, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite mthgxl_m at 0x2a2b3d350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LGTHLm_MitoCore', 'name': 'R_LGTHLm_MitoCore Glutathione + Methylglyoxal -->(R)-S-Lactoylglutathione', 'notes': {'Model Reaction Number': '268', 'EC Number': '4.4.1.5', 'SUBSYSTEM': 'Glycine degradation', 'Description': 'Glutathione + Methylglyoxal --&gt;(R)-S-Lactoylglutathione', 'Recon2 id': 'LGTHL', 'Recon2 formula': 'mthgxl[c] + gthrd[c] -&gt; lgt_S[c]', 'KEGG id': 'R02530', 'GENE_ASSOCIATION': 'ENSG00000124767', 'GENE_LIST': 'GLO1', 'HGNC': '4323', 'Gene name': 'glyoxalase I', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.0445', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.4.1.5', 'kegg.reaction': 'R02530'}, '_gpr': cobra.core.gene.GPR('ENSG00000124767'), 'subsystem': 'Glycine degradation', '_genes': {<Gene ENSG00000124767 at 0x176ea6510>}, '_metabolites': {<Metabolite mthgxl_m at 0x2a2b3d350>: -1.0, <Metabolite gthrd_m at 0x29c7e6e10>: -1.0, <Metabolite lgt_S_m at 0x164635e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYOXm', 'name': 'R_GLYOXm (R)-S-Lactoylglutathione + H2O --> Glutathione + (R)-Lactate', 'notes': {'Model Reaction Number': '269', 'EC Number': '3.1.2.6', 'SUBSYSTEM': 'Glycine degradation', 'Description': '(R)-S-Lactoylglutathione + H2O --&gt; Glutathione + (R)-Lactate', 'Recon2 id': 'GLYOXm', 'Recon2 formula': 'h2o[m] + lgt_S[m] -&gt; h[m] + gthrd[m] + lac_D[m]', 'KEGG id': 'R01736', 'GENE_ASSOCIATION': 'ENSG00000063854', 'GENE_LIST': 'HAGH', 'HGNC': '4805', 'Gene name': 'hydroxyacylglutathione hydrolase', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.5442', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 5.8kJ/mol', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '3.1.2.6', 'kegg.reaction': 'R01736'}, '_gpr': cobra.core.gene.GPR('ENSG00000063854'), 'subsystem': 'Glycine degradation', '_genes': {<Gene ENSG00000063854 at 0x176ea6850>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite lgt_S_m at 0x164635e50>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite gthrd_m at 0x29c7e6e10>: 1.0, <Metabolite lac_D_m at 0x29d0a9910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LDH_Dm_MitoCore', 'name': 'R_LDH_Dm_MitoCore (R)-Lactate + NAD+ --> Pyruvate + NADH + H+', 'notes': {'Model Reaction Number': '270', 'EC Number': '1.1.1.28', 'SUBSYSTEM': 'Glycine degradation', 'Description': '(R)-Lactate + NAD+ --&gt; Pyruvate + NADH + H+', 'Recon2 id': 'LDH_D', 'Recon2 formula': 'nad[c] + lac_D[c] --&gt; h[c] + nadh[c] + pyr[c]', 'KEGG id': 'R00704', 'GENE_ASSOCIATION': 'ENSG00000166816', 'GENE_LIST': 'LDHD', 'HGNC': '19708', 'Gene name': 'Probable D-lactate dehydrogenase, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.493', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '1.1.1.28', 'kegg.reaction': 'R00704'}, '_gpr': cobra.core.gene.GPR('ENSG00000166816'), 'subsystem': 'Glycine degradation', '_genes': {<Gene ENSG00000166816 at 0x176ea6ad0>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite lac_D_m at 0x29d0a9910>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CBPSam', 'name': 'R_CBPSam 2 ATP + Ammonia + HCO3 --> 2 ADP + Orthophosphate + Carbamoyl phosphate', 'notes': {'Model Reaction Number': '271', 'EC Number': '6.3.4.16', 'SUBSYSTEM': 'Citrulline biosynthesis', 'Description': '2 ATP + Ammonia + HCO3 --&gt; 2 ADP + Orthophosphate + Carbamoyl phosphate', 'Recon2 id': 'CBPSam', 'Recon2 formula': 'nh4[m] + 2.0 atp[m] + hco3[m] -&gt; 2.0 h[m] + 2.0 adp[m] + pi[m] + cbp[m]', 'KEGG id': 'R00149', 'GENE_ASSOCIATION': 'ENSG00000021826', 'GENE_LIST': 'CPS1', 'HGNC': '2323', 'Gene name': 'carbamoyl-phosphate synthase 1, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.5621', 'Directionality rationale': 'HumanCyc, KEGG, Literature, Delta G 7.4kJ/mol', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '6.3.4.16', 'kegg.reaction': 'R00149'}, '_gpr': cobra.core.gene.GPR('ENSG00000021826'), 'subsystem': 'Citrulline biosynthesis', '_genes': {<Gene ENSG00000021826 at 0x176ea6e90>}, '_metabolites': {<Metabolite nh4_m at 0x164ccd590>: -1.0, <Metabolite atp_m at 0x28b1339d0>: -2.0, <Metabolite hco3_m at 0x29c745e50>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite adp_m at 0x2a2a88e50>: 2.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite cbp_m at 0x297624310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'OCBTm', 'name': 'R_OCBTm Carbamoyl phosphate + L-Ornithine --> Orthophosphate + L-Citrulline', 'notes': {'Model Reaction Number': '272', 'EC Number': '2.1.3.3', 'SUBSYSTEM': 'Citrulline biosynthesis', 'Description': 'Carbamoyl phosphate + L-Ornithine --&gt; Orthophosphate + L-Citrulline', 'Recon2 id': 'OCBTm', 'Recon2 formula': 'orn[m] + cbp[m] -&gt; h[m] + pi[m] + citr_L[m]', 'KEGG id': 'R01398', 'GENE_ASSOCIATION': 'ENSG00000036473', 'GENE_LIST': 'OTC', 'HGNC': '8512', 'Gene name': 'Ornithine carbamoyltransferase, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.1641', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 9kJ/mol', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.1.3.3', 'kegg.reaction': 'R01398'}, '_gpr': cobra.core.gene.GPR('ENSG00000036473'), 'subsystem': 'Citrulline biosynthesis', '_genes': {<Gene ENSG00000036473 at 0x176ea71d0>}, '_metabolites': {<Metabolite orn_m at 0x29d211b90>: -1.0, <Metabolite cbp_m at 0x297624310>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite citr_L_m at 0x2ae025890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NOS1', 'name': 'R_NOS1 L-Arginine + Oxygen + NADPH + H+ --> N-(omega)-Hydroxyarginine + NADP+ + H2O', 'notes': {'Model Reaction Number': '273', 'EC Number': '1.14.13.39', 'SUBSYSTEM': 'NOS Signalling', 'Description': 'L-Arginine + Oxygen + NADPH + H+ --&gt; N-(omega)-Hydroxyarginine + NADP+ + H2O', 'Recon2 id': 'NOS1', 'Recon2 formula': 'o2[c] + h[c] + nadph[c] + arg_L[c] -&gt; h2o[c] + nadp[c] + nwharg[c]', 'KEGG id': 'R00558', 'GENE_ASSOCIATION': 'ENSG00000007171', 'GENE_LIST': 'NOS2', 'HGNC': '7873', 'Gene name': 'nitric oxide synthase 2, inducible', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.9426', 'Directionality rationale': 'Oxygen consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.14.13.39', 'kegg.reaction': 'R00558'}, '_gpr': cobra.core.gene.GPR('ENSG00000007171'), 'subsystem': 'NOS Signalling', '_genes': {<Gene ENSG00000007171 at 0x176ea7510>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite arg_L_c at 0x29f3b3410>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite nwharg_c at 0x29d48ced0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NOS2', 'name': 'R_NOS2 NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --> NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O', 'notes': {'Model Reaction Number': '274', 'EC Number': '1.14.13.39', 'SUBSYSTEM': 'NOS Signalling', 'Description': 'NADPH + 2 N-(omega)-Hydroxyarginine + 2 Oxygen + H+ --&gt; NADP+ + 2 Nitric oxide + 2 L-Citrulline + 2 H2O', 'Recon2 id': 'NOS2', 'Recon2 formula': 'o2[c] + nadph[c] + nwharg[c] -&gt; h2o[c] + h[c] + nadp[c] + citr_L[c] + no[c]', 'KEGG id': 'R00111', 'GENE_ASSOCIATION': 'ENSG00000007171', 'GENE_LIST': 'NOS2', 'HGNC': '7873', 'Gene name': 'nitric oxide synthase 2, inducible', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.9426', 'Directionality rationale': 'Oxygen consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.14.13.39', 'kegg.reaction': 'R00111'}, '_gpr': cobra.core.gene.GPR('ENSG00000007171'), 'subsystem': 'NOS Signalling', '_genes': {<Gene ENSG00000007171 at 0x176ea7510>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite nwharg_c at 0x29d48ced0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite citr_L_c at 0x2a2fecf50>: 1.0, <Metabolite no_c at 0x29d41c910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0129', 'name': 'R_r0129 Glutathione + H2O --> Cys-Gly + L-Glutamate', 'notes': {'Model Reaction Number': '275', 'EC Number': '3.4.19.13', 'SUBSYSTEM': 'Glutathione degradation', 'Description': 'Glutathione + H2O --&gt; Cys-Gly + L-Glutamate', 'Recon2 id': 'r0129', 'Recon2 formula': 'h2o[c] + gthrd[c] --&gt; glu_L[c] + cgly[c]', 'KEGG id': 'R00494', 'GENE_ASSOCIATION': 'ENSG00000167741 or ENSG00000131067', 'GENE_LIST': 'GGT6 or GGT7', 'HGNC': '26891 or HGNC:4259', 'Gene name': 'gamma-glutamyltransferase 6', 'MitoCarta': 'FALSE, FALSE', 'MitoCarta score': '-10.6961, -9.2075', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected or Low', 'HPA protein level in heart tissue': 'Medium or Low'}, '_annotation': {'ec-code': '3.4.19.13', 'kegg.reaction': 'R00494'}, '_gpr': cobra.core.gene.GPR('ENSG00000167741 or ENSG00000131067'), 'subsystem': 'Glutathione degradation', '_genes': {<Gene ENSG00000167741 at 0x176ea7b90>, <Gene ENSG00000131067 at 0x176ea7850>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite gthrd_c at 0x169ded990>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite cgly_c at 0x17b50a690>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AMPTASECG', 'name': 'R_AMPTASECG Cys-Gly + H2O --> L-Cysteine + Glycine', 'notes': {'Model Reaction Number': '276', 'EC Number': '3.4.11.2', 'SUBSYSTEM': 'Glutathione degradation', 'Description': 'Cys-Gly + H2O --&gt; L-Cysteine + Glycine', 'Recon2 id': 'AMPTASECG', 'Recon2 formula': 'h2o[c] + cgly[c] -&gt; gly[c] + cys_L[c]', 'KEGG id': 'R00899', 'GENE_ASSOCIATION': 'ENSG00000166825', 'GENE_LIST': 'ANPEP', 'HGNC': '500', 'Gene name': 'alanyl (membrane) aminopeptidase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.5916', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '3.4.11.2', 'kegg.reaction': 'R00899'}, '_gpr': cobra.core.gene.GPR('ENSG00000166825'), 'subsystem': 'Glutathione degradation', '_genes': {<Gene ENSG00000166825 at 0x176ea7ed0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite cgly_c at 0x17b50a690>: -1.0, <Metabolite gly_c at 0x29c737c90>: 1.0, <Metabolite cys_L_c at 0x29f32a9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLUCYS', 'name': 'R_GLUCYS ATP + L-Glutamate + L-Cysteine --> ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine', 'notes': {'Model Reaction Number': '277', 'EC Number': '6.3.2.2', 'SUBSYSTEM': 'Glutathione synthesis', 'Description': 'ATP + L-Glutamate + L-Cysteine --&gt; ADP + Orthophosphate + gamma-L-Glutamyl-L-cysteine', 'Recon2 id': 'GLUCYS', 'Recon2 formula': 'atp[c] + glu_L[c] + cys_L[c] -&gt; h[c] + adp[c] + pi[c] + glucys[c]', 'KEGG id': 'R00894', 'GENE_ASSOCIATION': 'ENSG00000001084', 'GENE_LIST': 'GCLC', 'HGNC': '4311', 'Gene name': 'glutamate-cysteine ligase, catalytic subunit', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.1848', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 6.4kJ/mol, ATP consumption', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '6.3.2.2', 'kegg.reaction': 'R00894'}, '_gpr': cobra.core.gene.GPR('ENSG00000001084'), 'subsystem': 'Glutathione synthesis', '_genes': {<Gene ENSG00000001084 at 0x176de0250>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite cys_L_c at 0x29f32a9d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite glucys_c at 0x29c7fc390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GTHS', 'name': 'R_GTHS ATP + gamma-L-Glutamyl-L-cysteine + Glycine --> ADP + Orthophosphate + Glutathione', 'notes': {'Model Reaction Number': '278', 'EC Number': '6.3.2.3', 'SUBSYSTEM': 'Glutathione synthesis', 'Description': 'ATP + gamma-L-Glutamyl-L-cysteine + Glycine --&gt; ADP + Orthophosphate + Glutathione', 'Recon2 id': 'GTHS', 'Recon2 formula': 'atp[c] + gly[c] + glucys[c] -&gt; h[c] + adp[c] + pi[c] + gthrd[c]', 'KEGG id': 'R00497', 'GENE_ASSOCIATION': 'ENSG00000100983', 'GENE_LIST': 'GSS', 'HGNC': '4624', 'Gene name': 'Glutathione synthetase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-8.0704', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 6.4kJ/mol, ATP consumption', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '6.3.2.3', 'kegg.reaction': 'R00497'}, '_gpr': cobra.core.gene.GPR('ENSG00000100983'), 'subsystem': 'Glutathione synthesis', '_genes': {<Gene ENSG00000100983 at 0x176de05d0>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite gly_c at 0x29c737c90>: -1.0, <Metabolite glucys_c at 0x29c7fc390>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite gthrd_c at 0x169ded990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0399', 'name': 'R_r0399 Tetrahydrobiopterin + L-Phenylalanine + Oxygen --> Dihydrobiopterin + L-Tyrosine + H2O', 'notes': {'Model Reaction Number': '279', 'EC Number': '1.14.16.1', 'SUBSYSTEM': 'Phenylalanine degradation', 'Description': 'Tetrahydrobiopterin + L-Phenylalanine + Oxygen --&gt; Dihydrobiopterin + L-Tyrosine + H2O', 'Recon2 id': 'r0399', 'Recon2 formula': 'o2[c] + thbpt[c] + phe_L[c] --&gt; h2o[c] + dhbpt[c] + tyr_L[c]', 'KEGG id': 'R01795', 'GENE_ASSOCIATION': 'ENSG00000171759', 'GENE_LIST': 'PAH', 'HGNC': '8582', 'Gene name': 'phenylalanine hydroxylase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.849', 'Directionality rationale': 'Oxygen consumption, KEGG, HumanCyc, Brenda', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.14.16.1', 'kegg.reaction': 'R01795'}, '_gpr': cobra.core.gene.GPR('ENSG00000171759'), 'subsystem': 'Phenylalanine degradation', '_genes': {<Gene ENSG00000171759 at 0x176de0890>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite thbpt_c at 0x29d7f2990>: -1.0, <Metabolite phe_L_c at 0x29e2c27d0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite dhbpt_c at 0x2a2717cd0>: 1.0, <Metabolite tyr_L_c at 0x29d719b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'DHPR', 'name': 'R_DHPR Dihydrobiopterin + NADH + H+ --> Tetrahydrobiopterin + NAD+', 'notes': {'Model Reaction Number': '280', 'EC Number': '1.5.1.34', 'SUBSYSTEM': 'Phenylalanine degradation', 'Description': 'Dihydrobiopterin + NADH + H+ --&gt; Tetrahydrobiopterin + NAD+', 'Recon2 id': 'DHPR', 'Recon2 formula': 'h[c] + nadh[c] + dhbpt[c] -&gt; nad[c] + thbpt[c]', 'KEGG id': 'R01793', 'GENE_ASSOCIATION': 'ENSG00000151552', 'GENE_LIST': 'QDPR', 'HGNC': '9752', 'Gene name': 'quinoid dihydropteridine reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '6.1455', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.5.1.34', 'kegg.reaction': 'R01793'}, '_gpr': cobra.core.gene.GPR('ENSG00000151552'), 'subsystem': 'Phenylalanine degradation', '_genes': {<Gene ENSG00000151552 at 0x176de0bd0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadh_c at 0x29d2efa10>: -1.0, <Metabolite dhbpt_c at 0x2a2717cd0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: 1.0, <Metabolite thbpt_c at 0x29d7f2990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TYRTA', 'name': 'R_TYRTA L-Tyrosine + 2-Oxoglutarate --> 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate', 'notes': {'Model Reaction Number': '281', 'EC Number': '2.6.1.1 or 2.6.1.5', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': 'L-Tyrosine + 2-Oxoglutarate --&gt; 3-(4-Hydroxyphenyl)pyruvate + L-Glutamate', 'Recon2 id': 'TYRTA', 'Recon2 formula': 'akg[c] + tyr_L[c] --&gt; 34hpp[c] + glu_L[c]', 'KEGG id': 'R00734', 'GENE_ASSOCIATION': 'ENSG00000120053', 'GENE_LIST': 'GOT1', 'HGNC': '4432', 'Gene name': 'glutamic-oxaloacetic transaminase 1, soluble', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.9098', 'Directionality rationale': 'HumanCyc', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': ['2.6.1.1', '2.6.1.5'], 'kegg.reaction': 'R00734'}, '_gpr': cobra.core.gene.GPR('ENSG00000120053'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000120053 at 0x176eb6c10>}, '_metabolites': {<Metabolite akg_c at 0x16ac21410>: -1.0, <Metabolite tyr_L_c at 0x29d719b50>: -1.0, <Metabolite 34hpp_c at 0x29f5b0110>: 1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TYRTB_MitoCore', 'name': 'R_TYRTB_MitoCore L-Tyrosine + H2O + Oxygen --> 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide', 'notes': {'Model Reaction Number': '282', 'EC Number': '1.4.3.2', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': 'L-Tyrosine + H2O + Oxygen --&gt; 3-(4-Hydroxyphenyl)pyruvate + Ammonia + Hydrogen peroxide', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00729', 'GENE_ASSOCIATION': 'ENSG00000104951', 'GENE_LIST': 'IL4I1', 'HGNC': '19094', 'Gene name': 'interleukin 4 induced 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-6.2297', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 23kJ/mol', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.4.3.2', 'kegg.reaction': 'R00729'}, '_gpr': cobra.core.gene.GPR('ENSG00000104951'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000104951 at 0x176de0f10>}, '_metabolites': {<Metabolite tyr_L_c at 0x29d719b50>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite h2o2_c at 0x29c784c10>: 1.0, <Metabolite 34hpp_c at 0x29f5b0110>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '34HPPOR', 'name': 'R_34HPPOR 3-(4-Hydroxyphenyl)pyruvate + Oxygen --> Homogentisate + CO2', 'notes': {'Model Reaction Number': '283', 'EC Number': '1.13.11.27', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': '3-(4-Hydroxyphenyl)pyruvate + Oxygen --&gt; Homogentisate + CO2', 'Recon2 id': '34HPPOR', 'Recon2 formula': 'o2[c] + 34hpp[c] -&gt; co2[c] + hgentis[c]', 'KEGG id': 'R02521', 'GENE_ASSOCIATION': 'ENSG00000158104', 'GENE_LIST': 'HPD', 'HGNC': '5147', 'Gene name': '4-hydroxyphenylpyruvate dioxygenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-9.2075', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 109.7kJ/mol, CO2 production, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.13.11.27', 'kegg.reaction': 'R02521'}, '_gpr': cobra.core.gene.GPR('ENSG00000158104'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000158104 at 0x176de1250>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite 34hpp_c at 0x29f5b0110>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite hgentis_c at 0x29c7d6b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HGNTOR', 'name': 'R_HGNTOR Homogentisate + Oxygen --> 4-Maleylacetoacetate', 'notes': {'Model Reaction Number': '284', 'EC Number': '1.13.11.5', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': 'Homogentisate + Oxygen --&gt; 4-Maleylacetoacetate', 'Recon2 id': 'HGNTOR', 'Recon2 formula': 'o2[c] + hgentis[c] -&gt; h[c] + 4mlacac[c]', 'KEGG id': 'R02519', 'GENE_ASSOCIATION': 'ENSG00000113924', 'GENE_LIST': 'HGD', 'HGNC': '4892', 'Gene name': 'homogentisate 1,2-dioxygenase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.1848', 'Directionality rationale': 'HumanCyc, Brenda, KEGG, Delta G 81.9kJ/mol, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.13.11.5', 'kegg.reaction': 'R02519'}, '_gpr': cobra.core.gene.GPR('ENSG00000113924'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000113924 at 0x176de1590>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite hgentis_c at 0x29c7d6b10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite 4mlacac_c at 0x28b14e590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MACACI', 'name': 'R_MACACI 4-Maleylacetoacetate --> 4-Fumarylacetoacetate', 'notes': {'Model Reaction Number': '285', 'EC Number': '5.2.1.2', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': '4-Maleylacetoacetate --&gt; 4-Fumarylacetoacetate', 'Recon2 id': 'MACACI', 'Recon2 formula': '4mlacac[c] -&gt; 4fumacac[c]', 'KEGG id': 'R03181', 'GENE_ASSOCIATION': 'ENSG00000100577', 'GENE_LIST': 'GSTZ1', 'HGNC': '4643', 'Gene name': 'glutathione S-transferase zeta 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '14.1262', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '5.2.1.2', 'kegg.reaction': 'R03181'}, '_gpr': cobra.core.gene.GPR('ENSG00000100577'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000100577 at 0x176de1890>}, '_metabolites': {<Metabolite 4mlacac_c at 0x28b14e590>: -1.0, <Metabolite 4fumacac_c at 0x2a2b98650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FUMAC', 'name': 'R_FUMAC 4-Fumarylacetoacetate + H2O --> Acetoacetate + Fumarate', 'notes': {'Model Reaction Number': '286', 'EC Number': '3.7.1.2', 'SUBSYSTEM': 'Tyrosine degradation', 'Description': '4-Fumarylacetoacetate + H2O --&gt; Acetoacetate + Fumarate', 'Recon2 id': 'FUMAC', 'Recon2 formula': 'h2o[c] + 4fumacac[c] -&gt; h[c] + acac[c] + fum[c]', 'KEGG id': 'R01364', 'GENE_ASSOCIATION': 'ENSG00000103876', 'GENE_LIST': 'FAH', 'HGNC': '3579', 'Gene name': 'fumarylacetoacetate hydrolase (fumarylacetoacetase)', 'MitoCarta': 'FALSE', 'MitoCarta score': '-0.3604', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 12.8kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '3.7.1.2', 'kegg.reaction': 'R01364'}, '_gpr': cobra.core.gene.GPR('ENSG00000103876'), 'subsystem': 'Tyrosine degradation', '_genes': {<Gene ENSG00000103876 at 0x176de1bd0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite 4fumacac_c at 0x2a2b98650>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite acac_c at 0x28b116d10>: 1.0, <Metabolite fum_c at 0x133597890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASNS1', 'name': 'R_ASNS1 ATP + L-Aspartate + L-Glutamine + H2O --> AMP + Diphosphate + L-Asparagine + L-Glutamate', 'notes': {'Model Reaction Number': '287', 'EC Number': '6.3.5.4', 'SUBSYSTEM': 'Asparagine synthesis', 'Description': 'ATP + L-Aspartate + L-Glutamine + H2O --&gt; AMP + Diphosphate + L-Asparagine + L-Glutamate', 'Recon2 id': 'ASNS1', 'Recon2 formula': 'h2o[c] + atp[c] + asp_L[c] + gln_L[c] -&gt; h[c] + glu_L[c] + amp[c] + ppi[c] + asn_L[c]', 'KEGG id': 'R00578', 'GENE_ASSOCIATION': 'ENSG00000070669', 'GENE_LIST': 'ASNS', 'HGNC': '753', 'Gene name': 'asparagine synthetase (glutamine-hydrolyzing)', 'MitoCarta': 'FALSE', 'MitoCarta score': '-6.2297', 'Directionality rationale': 'HumanCyc, ATP consumption, Delta G 8.2kJ/mol', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '6.3.5.4', 'kegg.reaction': 'R00578'}, '_gpr': cobra.core.gene.GPR('ENSG00000070669'), 'subsystem': 'Asparagine synthesis', '_genes': {<Gene ENSG00000070669 at 0x176de1c10>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite gln_L_c at 0x13354b910>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite ppi_c at 0x29e6fdf10>: 1.0, <Metabolite asn_L_c at 0x161ab19d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0127', 'name': 'R_r0127 L-Asparagine + H2O --> L-Aspartate + Ammonia', 'notes': {'Model Reaction Number': '288', 'EC Number': '3.5.1.1', 'SUBSYSTEM': 'Asparagine degradation', 'Description': 'L-Asparagine + H2O --&gt; L-Aspartate + Ammonia', 'Recon2 id': 'r0127', 'Recon2 formula': 'h2o[c] + asn_L[c] -&gt; nh4[c] + asp_L[c]', 'KEGG id': 'R00485', 'GENE_ASSOCIATION': 'ENSG00000166183', 'GENE_LIST': 'ASPG', 'HGNC': '20123', 'Gene name': 'asparaginase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-1.1966', 'Directionality rationale': 'Ammonia production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.5.1.1', 'kegg.reaction': 'R00485'}, '_gpr': cobra.core.gene.GPR('ENSG00000166183'), 'subsystem': 'Asparagine degradation', '_genes': {<Gene ENSG00000166183 at 0x176de2210>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite asn_L_c at 0x161ab19d0>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite asp_L_c at 0x16a60b310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HISD', 'name': 'R_HISD L-Histidine --> Urocanate + Ammonia', 'notes': {'Model Reaction Number': '289', 'EC Number': '4.3.1.3', 'SUBSYSTEM': 'Histidine degradation', 'Description': 'L-Histidine --&gt; Urocanate + Ammonia', 'Recon2 id': 'HISD', 'Recon2 formula': 'his_L[c] -&gt; nh4[c] + urcan[c]', 'KEGG id': 'R01168', 'GENE_ASSOCIATION': 'ENSG00000084110', 'GENE_LIST': 'HAL', 'HGNC': '4806', 'Gene name': 'histidine ammonia-lyase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-8.0704', 'Directionality rationale': 'Ammonia production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.3.1.3', 'kegg.reaction': 'R01168'}, '_gpr': cobra.core.gene.GPR('ENSG00000084110'), 'subsystem': 'Histidine degradation', '_genes': {<Gene ENSG00000084110 at 0x176de24d0>}, '_metabolites': {<Metabolite his_L_c at 0x29c7d6210>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite urcan_c at 0x29d773ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'URCN', 'name': 'R_URCN Urocanate + H2O --> 4-Imidazolone-5-propanoate', 'notes': {'Model Reaction Number': '290', 'EC Number': '4.2.1.49', 'SUBSYSTEM': 'Histidine degradation', 'Description': 'Urocanate + H2O --&gt; 4-Imidazolone-5-propanoate', 'Recon2 id': 'URCN', 'Recon2 formula': 'h2o[c] + urcan[c] -&gt; 4izp[c]', 'KEGG id': 'R02914', 'GENE_ASSOCIATION': 'ENSG00000159650', 'GENE_LIST': 'UROC1', 'HGNC': '26444', 'Gene name': 'urocanate hydratase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-7.3264', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '4.2.1.49', 'kegg.reaction': 'R02914'}, '_gpr': cobra.core.gene.GPR('ENSG00000159650'), 'subsystem': 'Histidine degradation', '_genes': {<Gene ENSG00000159650 at 0x176de2810>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite urcan_c at 0x29d773ad0>: -1.0, <Metabolite 4izp_c at 0x1792e6b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'IZPN', 'name': 'R_IZPN 4-Imidazolone-5-propanoate + H2O --> N-Formimino-L-glutamate', 'notes': {'Model Reaction Number': '291', 'EC Number': '3.5.2.7', 'SUBSYSTEM': 'Histidine degradation', 'Description': '4-Imidazolone-5-propanoate + H2O --&gt; N-Formimino-L-glutamate', 'Recon2 id': 'IZPN', 'Recon2 formula': 'h2o[c] + 4izp[c] -&gt; h[c] + forglu[c]', 'KEGG id': 'R02288', 'GENE_ASSOCIATION': 'ENSG00000139344', 'GENE_LIST': 'AMDHD1', 'HGNC': '28577', 'Gene name': 'amidohydrolase domain containing 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'Directionality rationale': 'Delta G 14.2kJ/mol, KEGG, HumanCyc, Brenda', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '3.5.2.7', 'kegg.reaction': 'R02288'}, '_gpr': cobra.core.gene.GPR('ENSG00000139344'), 'subsystem': 'Histidine degradation', '_genes': {<Gene ENSG00000139344 at 0x176de2b50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite 4izp_c at 0x1792e6b10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite forglu_c at 0x13359f910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GluForTx', 'name': 'R_GluForTx 5-Formiminotetrahydrofolate + L-Glutamate --> Tetrahydrofolate + N-Formimino-L-glutamate', 'notes': {'Model Reaction Number': '292', 'EC Number': '2.1.2.5', 'SUBSYSTEM': 'Histidine degradation', 'Description': '5-Formiminotetrahydrofolate + L-Glutamate --&gt; Tetrahydrofolate + N-Formimino-L-glutamate', 'Recon2 id': 'GluForTx', 'Recon2 formula': 'h[c] + thf[c] + forglu[c] -&gt; glu_L[c] + 5forthf[c]', 'KEGG id': 'R02287', 'GENE_ASSOCIATION': 'ENSG00000160282', 'GENE_LIST': 'FTCD', 'HGNC': '3974', 'Gene name': 'formimidoyltransferase cyclodeaminase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.1.2.5', 'kegg.reaction': 'R02287'}, '_gpr': cobra.core.gene.GPR('ENSG00000160282'), 'subsystem': 'Histidine degradation', '_genes': {<Gene ENSG00000160282 at 0x176ea4290>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite thf_c at 0x29d7f2310>: -1.0, <Metabolite forglu_c at 0x13359f910>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0, <Metabolite 5forthf_c at 0x1793229d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'APAT2rm', 'name': 'R_APAT2rm beta-Alanine + 2-Oxoglutarate --> 3-Oxopropanoate + L-Glutamate', 'notes': {'Model Reaction Number': '293', 'EC Number': '2.6.1.19', 'SUBSYSTEM': 'Beta-alanine degradation', 'Description': 'beta-Alanine + 2-Oxoglutarate --&gt; 3-Oxopropanoate + L-Glutamate', 'Recon2 id': 'APAT2rm', 'Recon2 formula': 'akg[m] + ala_B[m] --&gt; glu_L[m] + msa[m]', 'KEGG id': 'R00908', 'GENE_ASSOCIATION': 'ENSG00000183044', 'GENE_LIST': 'ABAT', 'HGNC': '23', 'Gene name': '4-aminobutyrate aminotransferase', 'MitoCarta': 'TRUE', 'MitoCarta score': '10.6859', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.6.1.19', 'kegg.reaction': 'R00908'}, '_gpr': cobra.core.gene.GPR('ENSG00000183044'), 'subsystem': 'Beta-alanine degradation', '_genes': {<Gene ENSG00000183044 at 0x176ed5910>}, '_metabolites': {<Metabolite akg_m at 0x2919313d0>: -1.0, <Metabolite ala_B_m at 0x178dca3d0>: -1.0, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0, <Metabolite msa_m at 0x29d0eef50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMSAD3m', 'name': 'R_MMSAD3m 3-Oxopropanoate + CoA + NAD+ --> Acetyl-CoA + CO2 + NADH + H+', 'notes': {'Model Reaction Number': '294', 'EC Number': '1.2.1.18', 'SUBSYSTEM': 'Beta-alanine degradation', 'Description': '3-Oxopropanoate + CoA + NAD+ --&gt; Acetyl-CoA + CO2 + NADH + H+', 'Recon2 id': 'MMSAD3m', 'Recon2 formula': 'nad[m] + coa[m] + msa[m] -&gt; nadh[m] + co2[m] + accoa[m]', 'KEGG id': 'R00705', 'GENE_ASSOCIATION': 'ENSG00000119711', 'GENE_LIST': 'ALDH6A1', 'HGNC': '7179', 'Gene name': 'aldehyde dehydrogenase 6 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '23.2735', 'Directionality rationale': 'CO2 production, Delta G 9kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.2.1.18', 'kegg.reaction': 'R00705'}, '_gpr': cobra.core.gene.GPR('ENSG00000119711'), 'subsystem': 'Beta-alanine degradation', '_genes': {<Gene ENSG00000119711 at 0x176ea0d90>}, '_metabolites': {<Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite msa_m at 0x29d0eef50>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MMSAD3m2_MitoCore', 'name': 'R_MMSAD3m2_MitoCore 3-Oxopropanoate + CoA + NADP+ --> Acetyl-CoA + CO2 + NADPH + H+', 'notes': {'Model Reaction Number': '295', 'EC Number': '1.2.1.18', 'SUBSYSTEM': 'Beta-alanine degradation', 'Description': '3-Oxopropanoate + CoA + NADP+ --&gt; Acetyl-CoA + CO2 + NADPH + H+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R00706', 'GENE_ASSOCIATION': 'ENSG00000119711', 'GENE_LIST': 'ALDH6A1', 'HGNC': '7179', 'Gene name': 'aldehyde dehydrogenase 6 family, member A1', 'MitoCarta': 'TRUE', 'MitoCarta score': '23.2735', 'Directionality rationale': 'CO2 production, Delta G 9kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.2.1.18', 'kegg.reaction': 'R00706'}, '_gpr': cobra.core.gene.GPR('ENSG00000119711'), 'subsystem': 'Beta-alanine degradation', '_genes': {<Gene ENSG00000119711 at 0x176ea0d90>}, '_metabolites': {<Metabolite nadp_m at 0x29d4f6490>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite msa_m at 0x29d0eef50>: -1.0, <Metabolite nadph_m at 0x29d475590>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite accoa_m at 0x164b26550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASP1DC', 'name': 'R_ASP1DC L-Aspartate --> beta-Alanine + CO2', 'notes': {'Model Reaction Number': '296', 'EC Number': '4.1.1.11', 'SUBSYSTEM': 'Beta-alanine synthesis', 'Description': 'L-Aspartate --&gt; beta-Alanine + CO2', 'Recon2 id': 'ASP1DC', 'Recon2 formula': 'h[c] + asp_L[c] -&gt; co2[c] + ala_B[c]', 'KEGG id': 'R00489', 'GENE_ASSOCIATION': 'ENSG00000144644', 'GENE_LIST': 'GADL1', 'HGNC': '27949', 'Gene name': 'glutamate decarboxylase-like 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'Directionality rationale': 'Evidence for high expression for this in skeletal muscle but poor for heart', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '4.1.1.11', 'kegg.reaction': 'R00489'}, '_gpr': cobra.core.gene.GPR('ENSG00000144644'), 'subsystem': 'Beta-alanine synthesis', '_genes': {<Gene ENSG00000144644 at 0x176de2e50>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0, <Metabolite ala_B_c at 0x178fb9910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'CKc', 'name': 'R_CKc ATP + Creatine --> ADP + Phosphocreatine', 'notes': {'Model Reaction Number': '297', 'EC Number': '2.7.3.2', 'SUBSYSTEM': 'Creatine ATP buffer', 'Description': 'ATP + Creatine --&gt; ADP + Phosphocreatine', 'Recon2 id': 'CKc', 'Recon2 formula': 'atp[c] + creat[c] --&gt; adp[c] + pcreat[c]', 'KEGG id': 'R01881', 'GENE_ASSOCIATION': 'ENSG00000166165 or ENSG00000104879', 'GENE_LIST': 'CKB or CKM', 'HGNC': '1991 or HGNC:1994', 'Gene name': 'Creatine kinase', 'MitoCarta': 'FALSE, FALSE', 'MitoCarta score': '0.8985, -3.4637', 'HPA mRNA level in heart tissue': 'Medium or High', 'HPA protein level in heart tissue': 'Low or high'}, '_annotation': {'ec-code': '2.7.3.2', 'kegg.reaction': 'R01881'}, '_gpr': cobra.core.gene.GPR('ENSG00000166165 or ENSG00000104879'), 'subsystem': 'Creatine ATP buffer', '_genes': {<Gene ENSG00000166165 at 0x176de3150>, <Gene ENSG00000104879 at 0x176de3450>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite creat_c at 0x1781b9150>: -1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pcreat_c at 0x29e23cb10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CK', 'name': 'R_CK ATP + Creatine --> ADP + Phosphocreatine', 'notes': {'Model Reaction Number': '298', 'EC Number': '2.7.3.2', 'SUBSYSTEM': 'Creatine ATP buffer', 'Description': 'ATP + Creatine --&gt; ADP + Phosphocreatine', 'Recon2 id': 'CK', 'Recon2 formula': 'atp[m] + creat[m] --&gt; adp[m] + pcreat[m]', 'KEGG id': 'R01881', 'GENE_ASSOCIATION': 'ENSG00000131730', 'GENE_LIST': 'CKMT2', 'HGNC': '1996', 'Gene name': 'Creatine kinase S-type, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '18.2617', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.7.3.2', 'kegg.reaction': 'R01881'}, '_gpr': cobra.core.gene.GPR('ENSG00000131730'), 'subsystem': 'Creatine ATP buffer', '_genes': {<Gene ENSG00000131730 at 0x176de36d0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite creat_m at 0x28b172990>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite pcreat_m at 0x29e20c450>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACOAHi', 'name': 'R_ACOAHi Acetyl-CoA + H2O --> CoA + Acetate', 'notes': {'Model Reaction Number': '299', 'EC Number': '3.1.2.1', 'SUBSYSTEM': 'Acetate production', 'Description': 'Acetyl-CoA + H2O --&gt; CoA + Acetate', 'Recon2 id': 'ACOAHi', 'Recon2 formula': 'h2o[c] + accoa[c] -&gt; h[c] + coa[c] + ac[c]', 'KEGG id': 'R00227', 'GENE_ASSOCIATION': 'ENSG00000172497', 'GENE_LIST': 'ACOT12', 'HGNC': '24436', 'Gene name': 'acyl-CoA thioesterase 12', 'MitoCarta': 'FALSE', 'MitoCarta score': '-0.3604', 'Directionality rationale': 'HumanCyc, KEGG, Delta G 7.3kJ/mol', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '3.1.2.1', 'kegg.reaction': 'R00227'}, '_gpr': cobra.core.gene.GPR('ENSG00000172497'), 'subsystem': 'Acetate production', '_genes': {<Gene ENSG00000172497 at 0x176de3790>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite accoa_c at 0x17b2485d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite coa_c at 0x17818b0d0>: 1.0, <Metabolite ac_c at 0x179379450>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALCD2yf', 'name': 'R_ALCD2yf Ethanol + NADP+ --> Acetaldehyde + NADPH + H+', 'notes': {'Model Reaction Number': '300', 'EC Number': '1.1.1.2', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'Ethanol + NADP+ --&gt; Acetaldehyde + NADPH + H+', 'Recon2 id': 'ALCD2yf', 'Recon2 formula': 'nadp[c] + etoh[c] -&gt; h[c] + nadph[c] + acald[c]', 'KEGG id': 'R00746', 'GENE_ASSOCIATION': 'ENSG00000117448', 'GENE_LIST': 'AKR1A1', 'HGNC': '380', 'Gene name': 'aldo-keto reductase family 1, member A1 (aldehyde reductase)', 'MitoCarta': 'FALSE', 'MitoCarta score': '-6.6912', 'Directionality rationale': 'To prevent interconversion of NADPH and NADH', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.1.1.2', 'kegg.reaction': 'R00746'}, '_gpr': cobra.core.gene.GPR('ENSG00000117448'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000117448 at 0x176de3d90>}, '_metabolites': {<Metabolite nadp_c at 0x17b53d250>: -1.0, <Metabolite etoh_c at 0x1337c2e50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadph_c at 0x2ae0fd610>: 1.0, <Metabolite acald_c at 0x133575e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALCD2if', 'name': 'R_ALCD2if Ethanol + NAD+ --> Acetaldehyde + NADH + H+', 'notes': {'Model Reaction Number': '301', 'EC Number': '1.1.1.1', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'Ethanol + NAD+ --&gt; Acetaldehyde + NADH + H+', 'Recon2 id': 'ALCD2if', 'Recon2 formula': 'nad[c] + etoh[c] -&gt; h[c] + nadh[c] + acald[c]', 'KEGG id': 'R00754', 'GENE_ASSOCIATION': 'ENSG00000248144 and ENSG00000197894', 'GENE_LIST': 'ADH1C and ADH5', 'HGNC': '251 and HGNC:253', 'Gene name': 'alcohol dehydrogenase', 'MitoCarta': 'FALSE, FALSE', 'MitoCarta score': '1.294, 0.7016', 'Directionality rationale': 'To prevent interconversion of NADPH and NADH', 'HPA mRNA level in heart tissue': 'Low and Medium', 'HPA protein level in heart tissue': 'Low and Medium'}, '_annotation': {'ec-code': '1.1.1.1', 'kegg.reaction': 'R00754'}, '_gpr': cobra.core.gene.GPR('ENSG00000248144 and ENSG00000197894'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000248144 at 0x176e60110>, <Gene ENSG00000197894 at 0x176e60450>}, '_metabolites': {<Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite etoh_c at 0x1337c2e50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite acald_c at 0x133575e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACALDtm', 'name': 'R_ACALDtm Acetaldehyde diffusion into mitochondrion', 'notes': {'Model Reaction Number': '302', 'EC Number': 'N/A', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'Acetaldehyde diffusion into mitochondrion', 'Recon2 id': 'ACALDtm', 'Recon2 formula': 'acald[m] --&gt; acald[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'N/A', 'GENE_LIST': 'N/A', 'Gene name': 'N/A', 'MitoCarta': 'N/A', 'MitoCarta score': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Alcohol metabolism', '_genes': set(), '_metabolites': {<Metabolite acald_m at 0x29e1b6590>: -1.0, <Metabolite acald_c at 0x133575e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALDD2xm', 'name': 'R_ALDD2xm Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+', 'notes': {'Model Reaction Number': '303', 'EC Number': '1.2.1.3', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'Acetaldehyde + NAD+ + H2O --&gt; Acetate + NADH + H+', 'Recon2 id': 'ALDD2xm', 'Recon2 formula': 'h2o[m] + nad[m] + acald[m] -&gt; 2.0 h[m] + nadh[m] + ac[m]', 'KEGG id': 'R00710', 'GENE_ASSOCIATION': 'ENSG00000137124 or ENSG00000072210 or ENSG00000164904 or ENSG00000143149', 'GENE_LIST': 'ALDH1B1 or ALDH3A2 or ALDH7A1 or ALDH9A1', 'HGNC': '407 or HGNC:403 or HGNC:877 or HGNC:412', 'Gene name': 'aldehyde dehydrogenase family', 'MitoCarta': 'TRUE, TRUE, TRUE, TRUE', 'MitoCarta score': '17.6418, 10.116, 13.8432, 23.4278', 'Directionality rationale': 'Delta G 12kJ/mol, HumanCyc', 'HPA mRNA level in heart tissue': 'Low or Medium or Medium or Medium', 'HPA protein level in heart tissue': 'High or Medium or Low or Low'}, '_annotation': {'ec-code': '1.2.1.3', 'kegg.reaction': 'R00710'}, '_gpr': cobra.core.gene.GPR('ENSG00000137124 or ENSG00000072210 or ENSG00000164904 or ENSG00000143149'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>, <Gene ENSG00000072210 at 0x176ed7c10>, <Gene ENSG00000137124 at 0x176ed4f50>, <Gene ENSG00000143149 at 0x176ed3a50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: -1.0, <Metabolite acald_m at 0x29e1b6590>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite nadh_m at 0x29d2af310>: 1.0, <Metabolite ac_m at 0x29c7a0490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALDD2x', 'name': 'R_ALDD2x Acetaldehyde + NAD+ + H2O --> Acetate + NADH + H+', 'notes': {'Model Reaction Number': '304', 'EC Number': '1.2.1.3', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'Acetaldehyde + NAD+ + H2O --&gt; Acetate + NADH + H+', 'Recon2 id': 'ALDD2x', 'Recon2 formula': 'h2o[c] + nad[c] + acald[c] -&gt; 2.0 h[c] + nadh[c] + ac[c]', 'KEGG id': 'R00710', 'GENE_ASSOCIATION': 'ENSG00000164904 or ENSG00000143149', 'GENE_LIST': 'ALDH7A1 or ALDH9A1', 'HGNC': '877 or HGNC:412', 'Gene name': 'aldehyde dehydrogenase family', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '13.8432, 23.4278', 'Directionality rationale': 'Delta G 12kJ/mol, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium or Medium', 'HPA protein level in heart tissue': 'Low or Low'}, '_annotation': {'ec-code': '1.2.1.3', 'kegg.reaction': 'R00710'}, '_gpr': cobra.core.gene.GPR('ENSG00000164904 or ENSG00000143149'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000164904 at 0x176ed1cd0>, <Gene ENSG00000143149 at 0x176ed3a50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: -1.0, <Metabolite acald_c at 0x133575e90>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0, <Metabolite ac_c at 0x179379450>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACSm', 'name': 'R_ACSm ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA', 'notes': {'Model Reaction Number': '305', 'EC Number': '6.2.1.1', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'ATP + Acetate + CoA --&gt; AMP + Diphosphate + Acetyl-CoA', 'Recon2 id': 'ACSm', 'Recon2 formula': 'atp[m] + coa[m] + ac[m] -&gt; accoa[m] + amp[m] + ppi[m]', 'KEGG id': 'R00235 (R00316 + R00236)', 'GENE_ASSOCIATION': 'ENSG00000154930', 'GENE_LIST': 'ACSS1', 'HGNC': '16091', 'Gene name': 'acyl-CoA synthetase short-chain family member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.7551', 'Directionality rationale': 'HumanCyc, KEGG, ATP consumption', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '6.2.1.1', 'kegg.reaction': ['R00235', 'R00316', 'R00236']}, '_gpr': cobra.core.gene.GPR('ENSG00000154930'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000154930 at 0x176e60790>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite coa_m at 0x178167cd0>: -1.0, <Metabolite ac_m at 0x29c7a0490>: -1.0, <Metabolite accoa_m at 0x164b26550>: 1.0, <Metabolite amp_m at 0x2975598d0>: 1.0, <Metabolite ppi_m at 0x29e6fcb90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACS', 'name': 'R_ACS ATP + Acetate + CoA --> AMP + Diphosphate + Acetyl-CoA', 'notes': {'Model Reaction Number': '306', 'EC Number': '6.2.1.1', 'SUBSYSTEM': 'Alcohol metabolism', 'Description': 'ATP + Acetate + CoA --&gt; AMP + Diphosphate + Acetyl-CoA', 'Recon2 id': 'ACS', 'Recon2 formula': 'atp[c] + coa[c] + ac[c] -&gt; accoa[c] + amp[c] + ppi[c]', 'KEGG id': 'R00235 (R00316 + R00236)', 'GENE_ASSOCIATION': 'ENSG00000131069', 'GENE_LIST': 'ACSS2', 'HGNC': '15814', 'Gene name': 'acyl-CoA synthetase short-chain family member 2', 'MitoCarta': 'FALSE', 'MitoCarta score': '4.4743', 'Directionality rationale': 'HumanCyc, KEGG, ATP consumption', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '6.2.1.1', 'kegg.reaction': ['R00235', 'R00316', 'R00236']}, '_gpr': cobra.core.gene.GPR('ENSG00000131069'), 'subsystem': 'Alcohol metabolism', '_genes': {<Gene ENSG00000131069 at 0x176e60a90>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite coa_c at 0x17818b0d0>: -1.0, <Metabolite ac_c at 0x179379450>: -1.0, <Metabolite accoa_c at 0x17b2485d0>: 1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite ppi_c at 0x29e6fdf10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ADSL1', 'name': 'R_ADSL1 N6-(1,2-Dicarboxyethyl)-AMP --> Fumarate + AMP', 'notes': {'Model Reaction Number': '307', 'EC Number': '4.3.2.2', 'SUBSYSTEM': 'Purine Nucleotide Cycle', 'Description': 'N6-(1,2-Dicarboxyethyl)-AMP --&gt; Fumarate + AMP', 'Recon2 id': 'ADSL1', 'Recon2 formula': 'dcamp[c] -&gt; amp[c] + fum[c]', 'KEGG id': 'R01083', 'GENE_ASSOCIATION': 'ENSG00000239900', 'GENE_LIST': 'ADSL', 'HGNC': '291', 'Gene name': 'adenylosuccinate lyase', 'MitoCarta': 'FALSE', 'MitoCarta score': '4.9634', 'Directionality rationale': 'KEGG, HumanCyc, Delta G 7.3kJ/mol', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.3.2.2', 'kegg.reaction': 'R01083'}, '_gpr': cobra.core.gene.GPR('ENSG00000239900'), 'subsystem': 'Purine Nucleotide Cycle', '_genes': {<Gene ENSG00000239900 at 0x176e60dd0>}, '_metabolites': {<Metabolite dcamp_c at 0x16ca1ab10>: -1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite fum_c at 0x133597890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ADSS', 'name': 'R_ADSS GTP + IMP + L-Aspartate --> GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP', 'notes': {'Model Reaction Number': '308', 'EC Number': '6.3.4.4', 'SUBSYSTEM': 'Purine Nucleotide Cycle', 'Description': 'GTP + IMP + L-Aspartate --&gt; GDP + Orthophosphate + N6-(1,2-Dicarboxyethyl)-AMP', 'Recon2 id': 'ADSS', 'Recon2 formula': 'asp_L[c] + gtp[c] + imp[c] -&gt; 2.0 h[c] + pi[c] + gdp[c] + dcamp[c]', 'KEGG id': 'R01135', 'GENE_ASSOCIATION': 'ENSG00000035687 or ENSG00000185100', 'GENE_LIST': 'ADSS or ADSSL1', 'HGNC': '292 or HGNC:20093', 'Gene name': 'adenylosuccinate synthase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-6.8878, -3.5181', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low or High', 'HPA protein level in heart tissue': 'Not detected or ND'}, '_annotation': {'ec-code': '6.3.4.4', 'kegg.reaction': 'R01135'}, '_gpr': cobra.core.gene.GPR('ENSG00000035687 or ENSG00000185100'), 'subsystem': 'Purine Nucleotide Cycle', '_genes': {<Gene ENSG00000185100 at 0x176e61110>, <Gene ENSG00000035687 at 0x176e61450>}, '_metabolites': {<Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite gtp_c at 0x29c7e6250>: -1.0, <Metabolite imp_c at 0x29c761850>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 2.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite gdp_c at 0x1335da410>: 1.0, <Metabolite dcamp_c at 0x16ca1ab10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AMPD1', 'name': 'R_AMPD1 AMP + H2O --> IMP + Ammonia', 'notes': {'Model Reaction Number': '309', 'EC Number': '3.5.4.6', 'SUBSYSTEM': 'Purine Nucleotide Cycle', 'Description': 'AMP + H2O --&gt; IMP + Ammonia', 'Recon2 id': 'AMPDA', 'Recon2 formula': 'h2o[c] + h[c] + amp[c] -&gt; nh4[c] + imp[c]', 'KEGG id': 'R00181', 'GENE_ASSOCIATION': 'ENSG00000116748', 'GENE_LIST': 'AMPD1', 'HGNC': '468', 'Gene name': 'adenosine monophosphate deaminase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-10.6961', 'Directionality rationale': 'NH4 production, Delta G 8.5kJ/mol, KEGG, HumanCyc, Brenda', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.5.4.6', 'kegg.reaction': 'R00181'}, '_gpr': cobra.core.gene.GPR('ENSG00000116748'), 'subsystem': 'Purine Nucleotide Cycle', '_genes': {<Gene ENSG00000116748 at 0x176e61790>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite amp_c at 0x2ae087b50>: -1.0, <Metabolite nh4_c at 0x1639455d0>: 1.0, <Metabolite imp_c at 0x29c761850>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGN', 'name': 'R_ARGN L-Arginine + H2O --> L-Ornithine + Urea', 'notes': {'Model Reaction Number': '310', 'EC Number': '3.5.3.1', 'SUBSYSTEM': 'Urea Cycle', 'Description': 'L-Arginine + H2O --&gt; L-Ornithine + Urea', 'Recon2 id': 'ARGN', 'Recon2 formula': 'h2o[c] + arg_L[c] -&gt; orn[c] + urea[c]', 'KEGG id': 'R00551', 'GENE_ASSOCIATION': 'ENSG00000118520', 'GENE_LIST': 'ARG1', 'HGNC': '663', 'Gene name': 'arginase 1', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.9098', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '3.5.3.1', 'kegg.reaction': 'R00551'}, '_gpr': cobra.core.gene.GPR('ENSG00000118520'), 'subsystem': 'Urea Cycle', '_genes': {<Gene ENSG00000118520 at 0x176e61ad0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite arg_L_c at 0x29f3b3410>: -1.0, <Metabolite orn_c at 0x29e2d2250>: 1.0, <Metabolite urea_c at 0x29d763f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGSL', 'name': 'R_ARGSL N-(L-Arginino)succinate --> Fumarate + L-Arginine', 'notes': {'Model Reaction Number': '311', 'EC Number': '4.3.2.1', 'SUBSYSTEM': 'Urea Cycle', 'Description': 'N-(L-Arginino)succinate --&gt; Fumarate + L-Arginine', 'Recon2 id': 'ARGSL', 'Recon2 formula': 'argsuc[c] --&gt; fum[c] + arg_L[c]', 'KEGG id': 'R01086', 'GENE_ASSOCIATION': 'ENSG00000126522', 'GENE_LIST': 'ASL', 'HGNC': '746', 'Gene name': 'argininosuccinate lyase', 'MitoCarta': 'FALSE', 'MitoCarta score': '1.1487', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.3.2.1', 'kegg.reaction': 'R01086'}, '_gpr': cobra.core.gene.GPR('ENSG00000126522'), 'subsystem': 'Urea Cycle', '_genes': {<Gene ENSG00000126522 at 0x176e61e10>}, '_metabolites': {<Metabolite argsuc_c at 0x2a05c9090>: -1.0, <Metabolite fum_c at 0x133597890>: 1.0, <Metabolite arg_L_c at 0x29f3b3410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGSS', 'name': 'R_ARGSS ATP + L-Citrulline + L-Aspartate --> AMP + Diphosphate + N-(L-Arginino)succinate', 'notes': {'Model Reaction Number': '312', 'EC Number': '6.3.4.5', 'SUBSYSTEM': 'Urea Cycle', 'Description': 'ATP + L-Citrulline + L-Aspartate --&gt; AMP + Diphosphate + N-(L-Arginino)succinate', 'Recon2 id': 'ARGSS', 'Recon2 formula': 'atp[c] + asp_L[c] + citr_L[c] -&gt; h[c] + amp[c] + ppi[c] + argsuc[c]', 'KEGG id': 'R01954', 'GENE_ASSOCIATION': 'ENSG00000130707', 'GENE_LIST': 'ASS1', 'HGNC': '758', 'Gene name': 'argininosuccinate synthase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-4.3044', 'Directionality rationale': 'ATP production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '6.3.4.5', 'kegg.reaction': 'R01954'}, '_gpr': cobra.core.gene.GPR('ENSG00000130707'), 'subsystem': 'Urea Cycle', '_genes': {<Gene ENSG00000130707 at 0x176e62150>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite citr_L_c at 0x2a2fecf50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite ppi_c at 0x29e6fdf10>: 1.0, <Metabolite argsuc_c at 0x2a05c9090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGNm', 'name': 'R_ARGNm L-Arginine + H2O --> L-Ornithine + Urea', 'notes': {'Model Reaction Number': '313', 'EC Number': '3.5.3.1', 'SUBSYSTEM': 'Urea Cycle', 'Description': 'L-Arginine + H2O --&gt; L-Ornithine + Urea', 'Recon2 id': 'ARGNm', 'Recon2 formula': 'h2o[m] + arg_L[m] -&gt; orn[m] + urea[m]', 'KEGG id': 'R00551', 'GENE_ASSOCIATION': 'ENSG00000081181', 'GENE_LIST': 'ARG2', 'HGNC': '664', 'Gene name': 'arginase 2', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.9705', 'Directionality rationale': 'KEGG, HumanCyc, Brenda', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '3.5.3.1', 'kegg.reaction': 'R00551'}, '_gpr': cobra.core.gene.GPR('ENSG00000081181'), 'subsystem': 'Urea Cycle', '_genes': {<Gene ENSG00000081181 at 0x176e62490>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite arg_L_m at 0x2ae4a81d0>: -1.0, <Metabolite orn_m at 0x29d211b90>: 1.0, <Metabolite urea_m at 0x29d7c2990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ALASm', 'name': 'R_ALASm Succinyl-CoA + Glycine --> 5-Aminolevulinate + CoA + CO2', 'notes': {'Model Reaction Number': '314', 'EC Number': '2.3.1.37', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Succinyl-CoA + Glycine --&gt; 5-Aminolevulinate + CoA + CO2', 'Recon2 id': 'ALASm', 'Recon2 formula': 'h[m] + gly[m] + succoa[m] -&gt; coa[m] + co2[m] + 5aop[m]', 'KEGG id': 'R00830', 'GENE_ASSOCIATION': 'ENSG00000023330', 'GENE_LIST': 'ALAS1', 'HGNC': '396', 'Gene name': '5&apos;-aminolevulinate synthase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.3146', 'Directionality rationale': 'CO2 production, Brenda, KEGG', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.37', 'kegg.reaction': 'R00830'}, '_gpr': cobra.core.gene.GPR('ENSG00000023330'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000023330 at 0x176e627d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite gly_m at 0x29c735650>: -1.0, <Metabolite succoa_m at 0x29d7ddd10>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite co2_m at 0x1781348d0>: 1.0, <Metabolite 5aop_m at 0x29f306750>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': '5AOPtm', 'name': 'R_5AOPtm 5-Aminolevulinate transport out of mitochondrion', 'notes': {'Model Reaction Number': '315', 'EC Number': 'N/A', 'SUBSYSTEM': 'Haem Synthesis', 'Description': '5-Aminolevulinate transport out of mitochondrion', 'Recon2 id': '5AOPtm', 'Recon2 formula': '5aop[c] --&gt; 5aop[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'Modelling - reverse would require proton motive force coupling'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Haem Synthesis', '_genes': set(), '_metabolites': {<Metabolite 5aop_m at 0x29f306750>: -1.0, <Metabolite 5aop_c at 0x2a2fed050>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPBNGS', 'name': 'R_PPBNGS 2 5-Aminolevulinate --> Porphobilinogen + 2 H2O', 'notes': {'Model Reaction Number': '316', 'EC Number': '4.2.1.24', 'SUBSYSTEM': 'Haem Synthesis', 'Description': '2 5-Aminolevulinate --&gt; Porphobilinogen + 2 H2O', 'Recon2 id': 'PPBNGS', 'Recon2 formula': '2.0 5aop[c] -&gt; 2.0 h2o[c] + h[c] + ppbng[c]', 'KEGG id': 'R00036', 'GENE_ASSOCIATION': 'ENSG00000148218', 'GENE_LIST': 'ALAD', 'HGNC': '395', 'Gene name': 'aminolevulinate dehydratase', 'MitoCarta': 'TRUE', 'MitoCarta score': '1.5686', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.2.1.24', 'kegg.reaction': 'R00036'}, '_gpr': cobra.core.gene.GPR('ENSG00000148218'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000148218 at 0x176e62b10>}, '_metabolites': {<Metabolite 5aop_c at 0x2a2fed050>: -2.0, <Metabolite h2o_c at 0x29c7876d0>: 2.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite ppbng_c at 0x29e61dc90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HMBS', 'name': 'R_HMBS 4 Porphobilinogen + H2O --> Hydroxymethylbilane + 4 Ammonia', 'notes': {'Model Reaction Number': '317', 'EC Number': '2.5.1.61', 'SUBSYSTEM': 'Haem Synthesis', 'Description': '4 Porphobilinogen + H2O --&gt; Hydroxymethylbilane + 4 Ammonia', 'Recon2 id': 'HMBS', 'Recon2 formula': 'h2o[c] + 4.0 ppbng[c] -&gt; 4.0 nh4[c] + hmbil[c]', 'KEGG id': 'R00084', 'GENE_ASSOCIATION': 'ENSG00000256269', 'GENE_LIST': 'HMBS', 'HGNC': '4982', 'Gene name': 'hydroxymethylbilane synthase', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.258', 'Directionality rationale': 'Ammonia production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.5.1.61', 'kegg.reaction': 'R00084'}, '_gpr': cobra.core.gene.GPR('ENSG00000256269'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000256269 at 0x176e62e50>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite ppbng_c at 0x29e61dc90>: -4.0, <Metabolite nh4_c at 0x1639455d0>: 4.0, <Metabolite hmbil_c at 0x17af3dd10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'UPP3S', 'name': 'R_UPP3S Hydroxymethylbilane --> Uroporphyrinogen III + H2O', 'notes': {'Model Reaction Number': '318', 'EC Number': '4.2.1.75', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Hydroxymethylbilane --&gt; Uroporphyrinogen III + H2O', 'Recon2 id': 'UPP3S', 'Recon2 formula': 'hmbil[c] -&gt; h2o[c] + uppg3[c]', 'KEGG id': 'R03165', 'GENE_ASSOCIATION': 'ENSG00000188690', 'GENE_LIST': 'UROS', 'HGNC': '12592', 'Gene name': 'uroporphyrinogen III synthase', 'MitoCarta': 'FALSE', 'MitoCarta score': '-5.1848', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '4.2.1.75', 'kegg.reaction': 'R03165'}, '_gpr': cobra.core.gene.GPR('ENSG00000188690'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000188690 at 0x176e63190>}, '_metabolites': {<Metabolite hmbil_c at 0x17af3dd10>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0, <Metabolite uppg3_c at 0x29d773dd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'UPPDC1', 'name': 'R_UPPDC1 Uroporphyrinogen III --> Coproporphyrinogen III + 4 CO2', 'notes': {'Model Reaction Number': '319', 'EC Number': '4.1.1.37', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Uroporphyrinogen III --&gt; Coproporphyrinogen III + 4 CO2', 'Recon2 id': 'UPPDC1', 'Recon2 formula': '4.0 h[c] + uppg3[c] -&gt; 4.0 co2[c] + cpppg3[c]', 'KEGG id': 'R03197', 'GENE_ASSOCIATION': 'ENSG00000126088', 'GENE_LIST': 'UROD', 'HGNC': '12591', 'Gene name': 'uroporphyrinogen decarboxylase', 'MitoCarta': 'FALSE', 'MitoCarta score': '1.2625', 'Directionality rationale': 'CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '4.1.1.37', 'kegg.reaction': 'R03197'}, '_gpr': cobra.core.gene.GPR('ENSG00000126088'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000126088 at 0x176e634d0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -4.0, <Metabolite uppg3_c at 0x29d773dd0>: -1.0, <Metabolite co2_c at 0x17810ba50>: 4.0, <Metabolite cpppg3_c at 0x1781ad290>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CPPPGO', 'name': 'R_CPPPGO Coproporphyrinogen III + Oxygen --> Protoporphyrinogen IX + 2 CO2 + 2 H2O', 'notes': {'Model Reaction Number': '320', 'EC Number': '1.3.3.3', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Coproporphyrinogen III + Oxygen --&gt; Protoporphyrinogen IX + 2 CO2 + 2 H2O', 'Recon2 id': 'CPPPGO', 'Recon2 formula': 'o2[c] + 2.0 h[c] + cpppg3[c] -&gt; 2.0 h2o[c] + 2.0 co2[c] + pppg9[c]', 'KEGG id': 'R03220', 'GENE_ASSOCIATION': 'ENSG00000080819', 'GENE_LIST': 'CPOX', 'HGNC': '2321', 'Gene name': 'coproporphyrinogen oxidase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.3434', 'Directionality rationale': 'Oxygen consumption, CO2 production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.3.3.3', 'kegg.reaction': 'R03220'}, '_gpr': cobra.core.gene.GPR('ENSG00000080819'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000080819 at 0x176e63810>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -2.0, <Metabolite cpppg3_c at 0x1781ad290>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 2.0, <Metabolite co2_c at 0x17810ba50>: 2.0, <Metabolite pppg9_c at 0x29e6fd810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPPGOmB_MitoCore', 'name': 'R_PPPGOmB_MitoCore Protoporphyrinogen IX + 3 Oxygen --> Protoporphyrin + 3 Hydrogen peroxide', 'notes': {'Model Reaction Number': '321', 'EC Number': '1.3.3.4', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Protoporphyrinogen IX + 3 Oxygen --&gt; Protoporphyrin + 3 Hydrogen peroxide', 'Recon2 id': 'PPPGOm', 'Recon2 formula': '3.0 o2[m] + 2.0 pppg9[m] -&gt; 6.0 h2o[m] + 2.0 ppp9[m]', 'KEGG id': 'R03222', 'GENE_ASSOCIATION': 'ENSG00000143224', 'GENE_LIST': 'PPOX', 'HGNC': '9280', 'Gene name': 'protoporphyrinogen oxidase', 'MitoCarta': 'TRUE', 'MitoCarta score': '13.5423', 'Directionality rationale': 'Oxygen consumption, Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.3.3.4', 'kegg.reaction': 'R03222'}, '_gpr': cobra.core.gene.GPR('ENSG00000143224'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000143224 at 0x176e63b50>}, '_metabolites': {<Metabolite o2_m at 0x17b2554d0>: -3.0, <Metabolite pppg9_c at 0x29e6fd810>: -2.0, <Metabolite h2o_m at 0x29c7848d0>: 6.0, <Metabolite ppp9_m at 0x29e6ff4d0>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FCLTm', 'name': 'R_FCLTm Protoporphyrin + Fe2+ --> Heme + 2 H+', 'notes': {'Model Reaction Number': '322', 'EC Number': '4.99.1.1', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Protoporphyrin + Fe2+ --&gt; Heme + 2 H+', 'Recon2 id': 'FCLTm', 'Recon2 formula': 'fe2[m] + ppp9[m] -&gt; 2.0 h[m] + pheme[m]', 'KEGG id': 'R00310', 'GENE_ASSOCIATION': 'ENSG00000066926', 'GENE_LIST': 'FECH', 'HGNC': '3647', 'Gene name': 'ferrochelatase', 'MitoCarta': 'TRUE', 'MitoCarta score': '30.2083', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '4.99.1.1', 'kegg.reaction': 'R00310'}, '_gpr': cobra.core.gene.GPR('ENSG00000066926'), 'subsystem': 'Haem Synthesis', '_genes': {<Gene ENSG00000066926 at 0x176e63e90>}, '_metabolites': {<Metabolite fe2_m at 0x1335dd950>: -1.0, <Metabolite ppp9_m at 0x29e6ff4d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 2.0, <Metabolite pheme_m at 0x29e63ee10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PHEMEtm', 'name': 'R_PHEMEtm Heme transport out of mitochondrion', 'notes': {'Model Reaction Number': '323', 'EC Number': 'N/A', 'SUBSYSTEM': 'Haem Synthesis', 'Description': 'Heme transport out of mitochondrion', 'Recon2 id': 'PHEMEtm', 'Recon2 formula': 'pheme[m] --&gt; pheme[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Haem Synthesis', '_genes': set(), '_metabolites': {<Metabolite pheme_m at 0x29e63ee10>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.64, <Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite pheme_c at 0x29e63ecd0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.64, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HOXG', 'name': 'R_HOXG Heme + 3 NADPH + 3 Oxygen --> Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O', 'notes': {'Model Reaction Number': '324', 'EC Number': '1.14.99.3', 'SUBSYSTEM': 'Haem Degradation', 'Description': 'Heme + 3 NADPH + 3 Oxygen --&gt; Globin + Biliverdin + CO + Fe2+ + 3 NADP + 3 H2O', 'Recon2 id': 'HOXG', 'Recon2 formula': '3.0 o2[c] + 5.0 h[c] + 3.0 nadph[c] + pheme[c] -&gt; 3.0 h2o[c] + 3.0 nadp[c] + biliverd[c] + co[c] + fe2[c]', 'KEGG id': 'R00311', 'GENE_ASSOCIATION': 'ENSG00000100292 or ENSG00000103415', 'GENE_LIST': 'HMOX1 or HMOX2', 'HGNC': '5013 or HGNC:5014', 'Gene name': 'heme oxygenase 1 or 2', 'MitoCarta': 'N/A', 'MitoCarta score': 'N/A', 'Directionality rationale': 'HumanCyc, Oxygen consumption', 'HPA mRNA level in heart tissue': 'Low or Medium', 'HPA protein level in heart tissue': 'Not detected or Low'}, '_annotation': {'ec-code': '1.14.99.3', 'kegg.reaction': 'R00311'}, '_gpr': cobra.core.gene.GPR('ENSG00000100292 or ENSG00000103415'), 'subsystem': 'Haem Degradation', '_genes': {<Gene ENSG00000100292 at 0x176e65690>, <Gene ENSG00000103415 at 0x176e64450>}, '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -3.0, <Metabolite h_c at 0x29c7e4b50>: -5.0, <Metabolite nadph_c at 0x2ae0fd610>: -3.0, <Metabolite pheme_c at 0x29e63ecd0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 3.0, <Metabolite nadp_c at 0x17b53d250>: 3.0, <Metabolite biliverd_c at 0x16d5d2010>: 1.0, <Metabolite co_c at 0x16d65ce90>: 1.0, <Metabolite fe2_c at 0x133815550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'BILIRED', 'name': 'R_BILIRED Biliverdin + NADPH + H+ --> Bilirubin + NADP+', 'notes': {'Model Reaction Number': '325', 'EC Number': '1.3.1.24', 'SUBSYSTEM': 'Haem Degradation', 'Description': 'Biliverdin + NADPH + H+ --&gt; Bilirubin + NADP+', 'Recon2 id': 'BILIRED', 'Recon2 formula': 'h[c] + nadph[c] + biliverd[c] -&gt; nadp[c] + bilirub[c]', 'KEGG id': 'R02393', 'GENE_ASSOCIATION': 'ENSG00000090013', 'GENE_LIST': 'BLVRB', 'HGNC': '1063', 'Gene name': 'biliverdin reductase B', 'MitoCarta': 'FALSE', 'MitoCarta score': '-2.7787', 'Directionality rationale': 'Delta G 11.9kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.3.1.24', 'kegg.reaction': 'R02393'}, '_gpr': cobra.core.gene.GPR('ENSG00000090013'), 'subsystem': 'Haem Degradation', '_genes': {<Gene ENSG00000090013 at 0x176e65290>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite biliverd_c at 0x16d5d2010>: -1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite bilirub_c at 0x16d951e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'BILIRED2_MitoCore', 'name': 'R_BILIRED2_MitoCore Biliverdin + NADH + H+ --> Bilirubin + NAD+', 'notes': {'Model Reaction Number': '326', 'EC Number': '1.3.1.24', 'SUBSYSTEM': 'Haem Degradation', 'Description': 'Biliverdin + NADH + H+ --&gt; Bilirubin + NAD+', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R02391', 'GENE_ASSOCIATION': 'ENSG00000106605', 'GENE_LIST': 'BLVRA', 'HGNC': '1062', 'Gene name': 'biliverdin reductase A', 'MitoCarta': 'FALSE', 'MitoCarta score': '-9.2075', 'Directionality rationale': 'Delta G 11.9kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.3.1.24', 'kegg.reaction': 'R02391'}, '_gpr': cobra.core.gene.GPR('ENSG00000106605'), 'subsystem': 'Haem Degradation', '_genes': {<Gene ENSG00000106605 at 0x176e64810>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadh_c at 0x29d2efa10>: -1.0, <Metabolite biliverd_c at 0x16d5d2010>: -1.0, <Metabolite nad_c at 0x29d264bd0>: 1.0, <Metabolite bilirub_c at 0x16d951e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCHOLPm_hs', 'name': 'R_PCHOLPm_hs Phosphatidylcholine + H2O --> Phosphatidate + Choline', 'notes': {'Model Reaction Number': '327', 'EC Number': '3.1.4.4', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'Phosphatidylcholine + H2O --&gt; Phosphatidate + Choline', 'Recon2 id': 'PCHOLPm_hs', 'Recon2 formula': 'h2o[m] + pchol_hs[m] -&gt; h[m] + chol[m] + pa_hs[m]', 'KEGG id': 'R01310', 'GENE_ASSOCIATION': 'ENSG00000105223', 'GENE_LIST': 'PLD3', 'HGNC': '9067', 'Gene name': 'phospholipase D1, phosphatidylcholine-specific', 'MitoCarta': 'FALSE', 'MitoCarta score': '-3.8147', 'Directionality rationale': 'HumanCyc, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '3.1.4.4', 'kegg.reaction': 'R01310'}, '_gpr': cobra.core.gene.GPR('ENSG00000105223'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000105223 at 0x176e65110>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite pchol_hs_m at 0x29e23c350>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite chol_m at 0x16c52cfd0>: 1.0, <Metabolite pa_hs_m at 0x29d2102d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYK', 'name': 'R_GLYK ATP + Glycerol --> ADP + sn-Glycerol 3-phosphate', 'notes': {'Model Reaction Number': '328', 'EC Number': '2.7.1.30', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'ATP + Glycerol --&gt; ADP + sn-Glycerol 3-phosphate', 'Recon2 id': 'GLYK', 'Recon2 formula': 'atp[c] + glyc[c] -&gt; h[c] + adp[c] + glyc3p[c]', 'KEGG id': 'R00847', 'GENE_ASSOCIATION': 'ENSG00000198814', 'GENE_LIST': 'GK', 'HGNC': '4289', 'Gene name': 'glycerol kinase', 'MitoCarta': 'TRUE', 'MitoCarta score': '18.158', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.7.1.30', 'kegg.reaction': 'R00847'}, '_gpr': cobra.core.gene.GPR('ENSG00000198814'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000198814 at 0x176e67e10>}, '_metabolites': {<Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite glyc_c at 0x29c734b50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite glyc3p_c at 0x29c7f5a10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYC3Ptm', 'name': 'R_GLYC3Ptm Glycerol 3-phosphate transport - diffusion', 'notes': {'Model Reaction Number': '329', 'EC Number': 'N/A', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'Glycerol 3-phosphate transport - diffusion', 'Recon2 id': 'GLYC3Ptm', 'Recon2 formula': 'glyc3p[c] -&gt; glyc3p[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Cardiolipin synthesis', '_genes': set(), '_metabolites': {<Metabolite glyc3p_c at 0x29c7f5a10>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.64, <Metabolite glyc3p_m at 0x29c7bdf90>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.64}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GPAMm_hsB_MitoCore', 'name': 'R_GPAMm_hsB_MitoCore sn-Glycerol 3-phosphate + Acyl-CoA --> 1-Acyl-sn-glycerol 3-phosphate + CoA', 'notes': {'Model Reaction Number': '330', 'EC Number': '2.3.1.15', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'sn-Glycerol 3-phosphate + Acyl-CoA --&gt; 1-Acyl-sn-glycerol 3-phosphate + CoA', 'Recon2 id': 'GPAMm_hs', 'Recon2 formula': 'Rtotalcoa[m] + glyc3p[m] -&gt; coa[m] + alpa_hs[m]', 'KEGG id': 'R00851', 'GENE_ASSOCIATION': 'ENSG00000186281', 'GENE_LIST': 'GPAT2', 'HGNC': '27168', 'Gene name': 'glycerol-3-phosphate acyltransferase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '2.7219', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.3.1.15', 'kegg.reaction': 'R00851'}, '_gpr': cobra.core.gene.GPR('ENSG00000186281'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000186281 at 0x176e65450>}, '_metabolites': {<Metabolite pmtcoa_m at 0x29e61b310>: -1.0, <Metabolite glyc3p_m at 0x29c7bdf90>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite alpa_hs_m at 0x1792d7310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'AGPAT1B_MitoCore', 'name': 'R_AGPAT1B_MitoCore 1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --> Phosphatidate + CoA', 'notes': {'Model Reaction Number': '331', 'EC Number': '2.3.1.51', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': '1-Acyl-sn-glycerol 3-phosphate + Acyl-CoA --&gt; Phosphatidate + CoA', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R02241', 'GENE_ASSOCIATION': 'ENSG00000155189', 'GENE_LIST': 'AGPAT5', 'HGNC': '20886', 'Gene name': '1-acylglycerol-3-phosphate O-acyltransferase 5', 'MitoCarta': 'TRUE', 'MitoCarta score': '5.9986', 'Directionality rationale': 'KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.3.1.51', 'kegg.reaction': 'R02241'}, '_gpr': cobra.core.gene.GPR('ENSG00000155189'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000155189 at 0x176e65190>}, '_metabolites': {<Metabolite pmtcoa_m at 0x29e61b310>: -1.0, <Metabolite alpa_hs_m at 0x1792d7310>: -1.0, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite pa_hs_m at 0x29d2102d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CDSm', 'name': 'R_CDSm CTP + Phosphatidate --> Diphosphate + CDP-diacylglycerol', 'notes': {'Model Reaction Number': '332', 'EC Number': '2.7.7.41', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'CTP + Phosphatidate --&gt; Diphosphate + CDP-diacylglycerol', 'Recon2 id': 'CDSm', 'Recon2 formula': 'h[m] + pa_hs[m] + ctp[m] -&gt; ppi[m] + cdpdag_hs[m]', 'KEGG id': 'R01799', 'GENE_ASSOCIATION': 'ENSG00000144559', 'GENE_LIST': 'TAMM41', 'HGNC': '25187', 'Gene name': 'TAM41, mitochondrial translocator assembly and maintenance protein, homolog (S. cerevisiae)', 'MitoCarta': 'TRUE', 'MitoCarta score': '18.7491', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '2.7.7.41', 'kegg.reaction': 'R01799'}, '_gpr': cobra.core.gene.GPR('ENSG00000144559'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000144559 at 0x176e64c90>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite pa_hs_m at 0x29d2102d0>: -1.0, <Metabolite ctp_m at 0x28b1716d0>: -1.0, <Metabolite ppi_m at 0x29e6fcb90>: 1.0, <Metabolite cdpdag_hs_m at 0x2a0d4cc50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGPPTm', 'name': 'R_PGPPTm CDP-diacylglycerol + sn-Glycerol 3-phosphate --> CMP + Phosphatidylglycerophosphate', 'notes': {'Model Reaction Number': '333', 'EC Number': '2.7.8.5', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'CDP-diacylglycerol + sn-Glycerol 3-phosphate --&gt; CMP + Phosphatidylglycerophosphate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R01801', 'GENE_ASSOCIATION': 'ENSG00000087157', 'GENE_LIST': 'PGS1', 'HGNC': '30029', 'Gene name': 'phosphatidylglycerophosphate synthase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '8.591', 'Directionality rationale': 'CDP-diacylglycerol consumption, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '2.7.8.5', 'kegg.reaction': 'R01801'}, '_gpr': cobra.core.gene.GPR('ENSG00000087157'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000087157 at 0x176e65b10>}, '_metabolites': {<Metabolite cdpdag_hs_m at 0x2a0d4cc50>: -1.0, <Metabolite glyc3p_m at 0x29c7bdf90>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite cmp_m at 0x1780c8e50>: 1.0, <Metabolite pgp_hs_m at 0x29e2f2790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PGPP_hsm_MitoCore', 'name': 'R_PGPP_hsm_MitoCore Phosphatidylglycerophosphate + H2O --> Phosphatidylglycerol + Orthophosphate', 'notes': {'Model Reaction Number': '334', 'EC Number': '3.1.3.27', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'Phosphatidylglycerophosphate + H2O --&gt; Phosphatidylglycerol + Orthophosphate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R02029', 'GENE_ASSOCIATION': 'ENSG00000110536', 'GENE_LIST': 'PTPMT1', 'HGNC': '26965', 'Gene name': 'protein tyrosine phosphatase, mitochondrial 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '7.3937', 'Directionality rationale': 'Phosphate production, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {'ec-code': '3.1.3.27', 'kegg.reaction': 'R02029'}, '_gpr': cobra.core.gene.GPR('ENSG00000110536'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000110536 at 0x176e65d50>}, '_metabolites': {<Metabolite h2o_m at 0x29c7848d0>: -1.0, <Metabolite pgp_hs_m at 0x29e2f2790>: -1.0, <Metabolite pi_m at 0x29e6c7690>: 1.0, <Metabolite pglyc_hs_m at 0x29e218610>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CLS_hsm_MitoCore', 'name': 'R_CLS_hsm_MitoCore Phosphatidylglycerol + CDP-diacylglycerol --> Cardiolipin + CMP', 'notes': {'Model Reaction Number': '335', 'EC Number': '2.7.8.41', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'Phosphatidylglycerol + CDP-diacylglycerol --&gt; Cardiolipin + CMP', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R02030', 'GENE_ASSOCIATION': 'ENSG00000088766', 'GENE_LIST': 'CRLS1', 'HGNC': '16148', 'Gene name': 'cardiolipin synthase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.7412', 'Directionality rationale': 'CDP-diacylglycerol consumption, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '2.7.8.41', 'kegg.reaction': 'R02030'}, '_gpr': cobra.core.gene.GPR('ENSG00000088766'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000088766 at 0x176e67550>}, '_metabolites': {<Metabolite cdpdag_hs_m at 0x2a0d4cc50>: -1.0, <Metabolite pglyc_hs_m at 0x29e218610>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite cmp_m at 0x1780c8e50>: 1.0, <Metabolite clpn_hs_m at 0x16a60aed0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CLPN_MitoCore', 'name': 'R_CLPN_MitoCore 2 Phosphatidylglycerol --> Cardiolipin + Glycerol', 'notes': {'Model Reaction Number': '336', 'EC Number': '2.7.8.-', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': '2 Phosphatidylglycerol --&gt; Cardiolipin + Glycerol', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'R07390', 'GENE_ASSOCIATION': 'ENSG00000088766', 'GENE_LIST': 'CRLS1', 'HGNC': '16148', 'Gene name': 'cardiolipin synthase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.7412', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {'ec-code': '2.7.8.-', 'kegg.reaction': 'R07390'}, '_gpr': cobra.core.gene.GPR('ENSG00000088766'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000088766 at 0x176e67550>}, '_metabolites': {<Metabolite pglyc_hs_m at 0x29e218610>: -2.0, <Metabolite clpn_hs_m at 0x16a60aed0>: 1.0, <Metabolite glyc_m at 0x29c7bd390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYTK1m', 'name': 'R_CYTK1m ATP + CMP --> ADP + CDP', 'notes': {'Model Reaction Number': '337', 'EC Number': '2.7.4.14', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'ATP + CMP --&gt; ADP + CDP', 'Recon2 id': 'CYTK1m', 'Recon2 formula': 'atp[m] + cmp[m] --&gt; adp[m] + cdp[m]', 'KEGG id': 'R00512', 'GENE_ASSOCIATION': 'ENSG00000134326', 'GENE_LIST': 'CMPK2', 'HGNC': '27015', 'Gene name': 'cytidine monophosphate (UMP-CMP) kinase 2, mitochondrial', 'MitoCarta': 'TRUE', 'MitoCarta score': '4.9286', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '2.7.4.14', 'kegg.reaction': 'R00512'}, '_gpr': cobra.core.gene.GPR('ENSG00000134326'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000134326 at 0x176e662d0>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite cmp_m at 0x1780c8e50>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite cdp_m at 0x2a26125d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NDPK3m', 'name': 'R_NDPK3m ATP + CDP --> ADP + CTP', 'notes': {'Model Reaction Number': '338', 'EC Number': '2.7.4.6', 'SUBSYSTEM': 'Cardiolipin synthesis', 'Description': 'ATP + CDP --&gt; ADP + CTP', 'Recon2 id': 'NDPK3m', 'Recon2 formula': 'atp[m] + cdp[m] --&gt; adp[m] + ctp[m]', 'KEGG id': 'R00570', 'GENE_ASSOCIATION': 'ENSG00000103202', 'GENE_LIST': 'NME4', 'HGNC': '7852', 'Gene name': 'NME/NM23 nucleoside diphosphate kinase 4', 'MitoCarta': 'TRUE', 'MitoCarta score': '9.3094', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '2.7.4.6', 'kegg.reaction': 'R00570'}, '_gpr': cobra.core.gene.GPR('ENSG00000103202'), 'subsystem': 'Cardiolipin synthesis', '_genes': {<Gene ENSG00000103202 at 0x28ab02e10>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite cdp_m at 0x2a26125d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite ctp_m at 0x28b1716d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SPODMm', 'name': 'R_SPODMm 2 O2.- + 2 H+ --> H2O2 + Oxygen', 'notes': {'Model Reaction Number': '339', 'EC Number': '1.15.1.1', 'SUBSYSTEM': 'ROS defense', 'Description': '2 O2.- + 2 H+ --&gt; H2O2 + Oxygen', 'Recon2 id': 'SPODMm', 'Recon2 formula': '2.0 h[m] + 2.0 o2s[m] -&gt; o2[m] + h2o2[m]', 'KEGG id': 'R00275', 'GENE_ASSOCIATION': 'ENSG00000142168 or ENSG00000112096', 'GENE_LIST': 'SOD1 or SOD2', 'HGNC': '11179 or HGNC:11180', 'Gene name': 'superoxide dismutase', 'MitoCarta': 'TRUE', 'MitoCarta score': '28.1817', 'Directionality rationale': 'Brenda, HumanCyc, Literature', 'HPA mRNA level in heart tissue': 'High or High', 'HPA protein level in heart tissue': 'Low or Medium'}, '_annotation': {'ec-code': '1.15.1.1', 'kegg.reaction': 'R00275'}, '_gpr': cobra.core.gene.GPR('ENSG00000142168 or ENSG00000112096'), 'subsystem': 'ROS defense', '_genes': {<Gene ENSG00000142168 at 0x176e66fd0>, <Gene ENSG00000112096 at 0x176e675d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -2.0, <Metabolite o2s_m at 0x29d4ba350>: -2.0, <Metabolite o2_m at 0x17b2554d0>: 1.0, <Metabolite h2o2_m at 0x29c72f810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GTHP', 'name': 'R_GTHP H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O', 'notes': {'Model Reaction Number': '340', 'EC Number': '1.11.1.9', 'SUBSYSTEM': 'ROS defense', 'Description': 'H2O2 + 2 Glutathione --&gt; Glutathione disulfide + 2 H2O', 'Recon2 id': 'GTHP', 'Recon2 formula': 'h2o2[c] + 2.0 gthrd[c] -&gt; 2.0 h2o[c] + gthox[c]', 'KEGG id': 'R00274', 'GENE_ASSOCIATION': 'ENSG00000176153', 'GENE_LIST': 'GPX2', 'HGNC': '4554', 'Gene name': 'glutathione peroxidase 2', 'MitoCarta': 'FALSE', 'MitoCarta score': '0.7262', 'Directionality rationale': 'Delta G 74.1kJ/mol, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {'ec-code': '1.11.1.9', 'kegg.reaction': 'R00274'}, '_gpr': cobra.core.gene.GPR('ENSG00000176153'), 'subsystem': 'ROS defense', '_genes': {<Gene ENSG00000176153 at 0x176e66510>}, '_metabolites': {<Metabolite h2o2_c at 0x29c784c10>: -1.0, <Metabolite gthrd_c at 0x169ded990>: -2.0, <Metabolite h2o_c at 0x29c7876d0>: 2.0, <Metabolite gthox_c at 0x17b23d9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GTHPm', 'name': 'R_GTHPm H2O2 + 2 Glutathione --> Glutathione disulfide + 2 H2O', 'notes': {'Model Reaction Number': '341', 'EC Number': '1.11.1.9', 'SUBSYSTEM': 'ROS defense', 'Description': 'H2O2 + 2 Glutathione --&gt; Glutathione disulfide + 2 H2O', 'Recon2 id': 'GTHPm', 'Recon2 formula': 'h2o2[m] + 2.0 gthrd[m] -&gt; 2.0 h2o[m] + gthox[m]', 'KEGG id': 'R00274', 'GENE_ASSOCIATION': 'ENSG00000233276', 'GENE_LIST': 'GPX1', 'HGNC': '4553', 'Gene name': 'glutathione peroxidase 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '11.5396', 'Directionality rationale': 'Delta G 74.1kJ/mol, KEGG', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Low'}, '_annotation': {'ec-code': '1.11.1.9', 'kegg.reaction': 'R00274'}, '_gpr': cobra.core.gene.GPR('ENSG00000233276'), 'subsystem': 'ROS defense', '_genes': {<Gene ENSG00000233276 at 0x176e66bd0>}, '_metabolites': {<Metabolite h2o2_m at 0x29c72f810>: -1.0, <Metabolite gthrd_m at 0x29c7e6e10>: -2.0, <Metabolite h2o_m at 0x29c7848d0>: 2.0, <Metabolite gthox_m at 0x29c7bee10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GTHO', 'name': 'R_GTHO Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+', 'notes': {'Model Reaction Number': '342', 'EC Number': '1.8.1.7', 'SUBSYSTEM': 'ROS defense', 'Description': 'Glutathione disulfide + NADPH + H+ --&gt; 2 Glutathione + NADP+', 'Recon2 id': 'GTHO', 'Recon2 formula': 'h[c] + nadph[c] + gthox[c] -&gt; nadp[c] + 2.0 gthrd[c]', 'KEGG id': 'R00115', 'GENE_ASSOCIATION': 'ENSG00000104687', 'GENE_LIST': 'GSR', 'HGNC': '4623', 'Gene name': 'glutathione reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.3163', 'Directionality rationale': 'Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.8.1.7', 'kegg.reaction': 'R00115'}, '_gpr': cobra.core.gene.GPR('ENSG00000104687'), 'subsystem': 'ROS defense', '_genes': {<Gene ENSG00000104687 at 0x176e663d0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite nadph_c at 0x2ae0fd610>: -1.0, <Metabolite gthox_c at 0x17b23d9d0>: -1.0, <Metabolite nadp_c at 0x17b53d250>: 1.0, <Metabolite gthrd_c at 0x169ded990>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GTHOm', 'name': 'R_GTHOm Glutathione disulfide + NADPH + H+ --> 2 Glutathione + NADP+', 'notes': {'Model Reaction Number': '343', 'EC Number': '1.8.1.7', 'SUBSYSTEM': 'ROS defense', 'Description': 'Glutathione disulfide + NADPH + H+ --&gt; 2 Glutathione + NADP+', 'Recon2 id': 'GTHOm', 'Recon2 formula': 'h[m] + nadph[m] + gthox[m] -&gt; nadp[m] + 2.0 gthrd[m]', 'KEGG id': 'R00115', 'GENE_ASSOCIATION': 'ENSG00000104687', 'GENE_LIST': 'GSR', 'HGNC': '4623', 'Gene name': 'glutathione reductase', 'MitoCarta': 'TRUE', 'MitoCarta score': '15.3163', 'Directionality rationale': 'Brenda, KEGG, HumanCyc', 'HPA mRNA level in heart tissue': 'Medium', 'HPA protein level in heart tissue': 'Not detected'}, '_annotation': {'ec-code': '1.8.1.7', 'kegg.reaction': 'R00115'}, '_gpr': cobra.core.gene.GPR('ENSG00000104687'), 'subsystem': 'ROS defense', '_genes': {<Gene ENSG00000104687 at 0x176e663d0>}, '_metabolites': {<Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite nadph_m at 0x29d475590>: -1.0, <Metabolite gthox_m at 0x29c7bee10>: -1.0, <Metabolite nadp_m at 0x29d4f6490>: 1.0, <Metabolite gthrd_m at 0x29c7e6e10>: 2.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CITtamB', 'name': 'R_CITtamB Citrate Carrier (CIC) - Citrate for Malate', 'notes': {'Model Reaction Number': '344', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Citrate Carrier (CIC) - Citrate for Malate', 'Recon2 id': 'CITtam', 'Recon2 formula': 'cit[c] + mal_L[m] --&gt; cit[m] + mal_L[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000100075', 'GENE_LIST': 'SLC25A1', 'HGNC': '10979', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; citrate transporter), member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000100075'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000100075 at 0x176e668d0>}, '_metabolites': {<Metabolite cit_c at 0x17b24a710>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: -1.0, <Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.0, <Metabolite cit_m at 0x2a2f126d0>: 1.0, <Metabolite mal_L_c at 0x29d020d50>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0913', 'name': 'R_r0913 Citrate Carrier (CIC) - Isocitrate for Malate', 'notes': {'Model Reaction Number': '345', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Citrate Carrier (CIC) - Isocitrate for Malate', 'Recon2 id': 'r0913', 'Recon2 formula': 'icit[m] + mal_L[c] --&gt; icit[c] + mal_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000100075', 'GENE_LIST': 'SLC25A1', 'HGNC': '10979', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; citrate transporter), member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000100075'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000100075 at 0x176e668d0>}, '_metabolites': {<Metabolite icit_m at 0x29c7d1a50>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite icit_c at 0x29c7d0350>: 1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CITtbm', 'name': 'R_CITtbm Citrate Carrier (CIC) - Citrate for PEP', 'notes': {'Model Reaction Number': '346', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Citrate Carrier (CIC) - Citrate for PEP', 'Recon2 id': 'CITtbm', 'Recon2 formula': 'cit[m] + pep[c] --&gt; cit[c] + pep[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000100075', 'GENE_LIST': 'SLC25A1', 'HGNC': '10979', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; citrate transporter), member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000100075'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000100075 at 0x176e668d0>}, '_metabolites': {<Metabolite pep_c at 0x29e21af10>: -1.0, <Metabolite cit_m at 0x2a2f126d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.0, <Metabolite pep_m at 0x29e2bc110>: 1.0, <Metabolite cit_c at 0x17b24a710>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0917', 'name': 'R_r0917 Citrate Carrier (CIC) - Citrate for Isocitrate', 'notes': {'Model Reaction Number': '347', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Citrate Carrier (CIC) - Citrate for Isocitrate', 'Recon2 id': 'r0917', 'Recon2 formula': 'cit[c] + icit[m] --&gt; cit[m] + icit[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000100075', 'GENE_LIST': 'SLC25A1', 'HGNC': '10979', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; citrate transporter), member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000100075'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000100075 at 0x176e668d0>}, '_metabolites': {<Metabolite cit_c at 0x17b24a710>: -1.0, <Metabolite icit_m at 0x29c7d1a50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite cit_m at 0x2a2f126d0>: 1.0, <Metabolite icit_c at 0x29c7d0350>: 1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0917b_MitoCore', 'name': 'R_r0917b_MitoCore Citrate Carrier (CIC) - Citrate for Isocitrate', 'notes': {'Model Reaction Number': '348', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Citrate Carrier (CIC) - Citrate for Isocitrate', 'Recon2 id': 'r0917', 'Recon2 formula': 'cit[c] + icit[m] --&gt; cit[m] + icit[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000100075', 'GENE_LIST': 'SLC25A1', 'HGNC': '10979', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; citrate transporter), member 1', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000100075'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000100075 at 0x176e668d0>}, '_metabolites': {<Metabolite cit_c at 0x17b24a710>: -1.0, <Metabolite icit_m at 0x29c7d1a50>: -1.0, <Metabolite h_m at 0x29c7e4250>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.0, <Metabolite cit_m at 0x2a2f126d0>: 1.0, <Metabolite icit_c at 0x29c7d0350>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'PIt2mB_MitoCore', 'name': 'R_PIt2mB_MitoCore Phosphate carrier (PIC) - Symport phosphate and H+ together', 'notes': {'Model Reaction Number': '349', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Phosphate carrier (PIC) - Symport phosphate and H+ together', 'Recon2 id': 'PIt2m', 'Recon2 formula': 'h[c] + pi[c] -&gt; h[m] + pi[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000075415', 'GENE_LIST': 'SLC25A3', 'HGNC': '10989', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; phosphate carrier), member 3', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.2493', 'HPA mRNA level in heart tissue': 'High', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000075415'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000075415 at 0x176e645d0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.18, <Metabolite pi_c at 0x29e6c47d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.18, <Metabolite pi_m at 0x29e6c7690>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ATPtmB_MitoCore', 'name': 'R_ATPtmB_MitoCore ADP/ATP carrier - antiport ATP for ADP', 'notes': {'Model Reaction Number': '350', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'ADP/ATP carrier - antiport ATP for ADP', 'Recon2 id': 'ATPtm', 'Recon2 formula': 'atp[m] + adp[c] -&gt; atp[c] + adp[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000151729 or ENSG00000005022 or ENSG00000169100', 'GENE_LIST': 'SLC25A4 or SLC25A5 or SLC25A6', 'HGNC': '10990 or HGNC:10991 or HGNC:10992', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; adenine nucleotide translocator), member 4, 5 and 6', 'MitoCarta': 'TRUE, TRUE, TRUE', 'MitoCarta score': '30.384, 22.6723, 3.172', 'HPA mRNA level in heart tissue': 'High or High or High', 'HPA protein level in heart tissue': 'ND or Not detected or Not detected'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000151729 or ENSG00000005022 or ENSG00000169100'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000005022 at 0x176e5c990>, <Gene ENSG00000169100 at 0x176e5c650>, <Gene ENSG00000151729 at 0x176e5c310>}, '_metabolites': {<Metabolite atp_m at 0x28b1339d0>: -1.0, <Metabolite adp_c at 0x17918bb10>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite atp_c at 0x2ae181f10>: 1.0, <Metabolite adp_m at 0x2a2a88e50>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HtmB_MitoCore', 'name': 'R_HtmB_MitoCore UCP uncoupling protein - Proton Cyto to Proton Mito', 'notes': {'Model Reaction Number': '351', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'UCP uncoupling protein - Proton Cyto to Proton Mito', 'Recon2 id': 'Htm', 'Recon2 formula': 'h[c] -&gt; h[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000175567 or ENSG00000175564', 'GENE_LIST': 'UCP2 or UCP3', 'HGNC': '12518 or HGNC:12519', 'Gene name': 'uncoupling protein 2 and 3', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '5.9313, 11.3472', 'Directionality rationale': 'Literature, to prevent proton pumping', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'High'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000175567 or ENSG00000175564'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000175564 at 0x176e5cc90>, <Gene ENSG00000175567 at 0x176e5cfd0>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MALtm', 'name': 'R_MALtm Dicarboxylate carrier (DIC) - antiport malate for phosphate', 'notes': {'Model Reaction Number': '352', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport malate for phosphate', 'Recon2 id': 'MALtm', 'Recon2 formula': 'pi[m] + mal_L[c] --&gt; pi[c] + mal_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite pi_m at 0x29e6c7690>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MALSO3tm', 'name': 'R_MALSO3tm Dicarboxylate carrier (DIC) - antiport malate for sulfite', 'notes': {'Model Reaction Number': '353', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport malate for sulfite', 'Recon2 id': 'MALSO3tm', 'Recon2 formula': 'so3[m] + mal_L[c] --&gt; so3[c] + mal_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite so3_m at 0x29d7ada50>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite so3_c at 0x297508cd0>: 1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MALTSULtm', 'name': 'R_MALTSULtm Dicarboxylate carrier (DIC) - antiport malate for thiosulfate', 'notes': {'Model Reaction Number': '354', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport malate for thiosulfate', 'Recon2 id': 'MALTSULtm', 'Recon2 formula': 'mal_L[c] + tsul[m] --&gt; mal_L[m] + tsul[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite tsul_m at 0x29d719b90>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0, <Metabolite tsul_c at 0x29d746e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'MALSO4tm', 'name': 'R_MALSO4tm Dicarboxylate carrier (DIC) - antiport malate for sulfate', 'notes': {'Model Reaction Number': '355', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport malate for sulfate', 'Recon2 id': 'MALSO4tm', 'Recon2 formula': 'mal_L[c] + so4[m] --&gt; mal_L[m] + so4[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite mal_L_c at 0x29d020d50>: -1.0, <Metabolite so4_m at 0x29d7b5450>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: 1.0, <Metabolite so4_c at 0x29d7acdd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SUCCt2m', 'name': 'R_SUCCt2m Dicarboxylate carrier (DIC) - antiport succinate for phosphate', 'notes': {'Model Reaction Number': '356', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport succinate for phosphate', 'Recon2 id': 'SUCCt2m', 'Recon2 formula': 'pi[m] + succ[c] --&gt; pi[c] + succ[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite pi_m at 0x29e6c7690>: -1.0, <Metabolite succ_c at 0x29d7b6c90>: -1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0830', 'name': 'R_r0830 Dicarboxylate carrier (DIC) - antiport succinate for sulfite', 'notes': {'Model Reaction Number': '357', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport succinate for sulfite', 'Recon2 id': 'r0830', 'Recon2 formula': 'so3[m] + succ[c] --&gt; so3[c] + succ[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite so3_m at 0x29d7ada50>: -1.0, <Metabolite succ_c at 0x29d7b6c90>: -1.0, <Metabolite so3_c at 0x297508cd0>: 1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0830B_MitoCore', 'name': 'R_r0830B_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate', 'notes': {'Model Reaction Number': '358', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport succinate for thiosulfate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite succ_c at 0x29d7b6c90>: -1.0, <Metabolite tsul_m at 0x29d719b90>: -1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0, <Metabolite tsul_c at 0x29d746e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0829', 'name': 'R_r0829 Dicarboxylate carrier (DIC) - antiport succinate for sulfate', 'notes': {'Model Reaction Number': '359', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport succinate for sulfate', 'Recon2 id': 'r0829', 'Recon2 formula': 'so4[m] + succ[c] --&gt; so4[c] + succ[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite so4_m at 0x29d7b5450>: -1.0, <Metabolite succ_c at 0x29d7b6c90>: -1.0, <Metabolite so4_c at 0x29d7acdd0>: 1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SUCCt3m_MitoCore', 'name': 'R_SUCCt3m_MitoCore Dicarboxylate carrier (DIC) - antiport succinate for malate', 'notes': {'Model Reaction Number': '360', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Dicarboxylate carrier (DIC) - antiport succinate for malate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000183048', 'GENE_LIST': 'SLC25A10', 'HGNC': '10980', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; dicarboxylate transporter), member 10', 'MitoCarta': 'TRUE', 'MitoCarta score': '21.9806', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000183048'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000183048 at 0x176e5d250>}, '_metabolites': {<Metabolite succ_c at 0x29d7b6c90>: -1.0, <Metabolite mal_L_m at 0x133bf6b90>: -1.0, <Metabolite succ_m at 0x29d7de4d0>: 1.0, <Metabolite mal_L_c at 0x29d020d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'COAtmB_MitoCore', 'name': 'R_COAtmB_MitoCore CoA carrier (CoAPC) - antiport CoA for AMP', 'notes': {'Model Reaction Number': '361', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'CoA carrier (CoAPC) - antiport CoA for AMP', 'Recon2 id': 'COAtm', 'Recon2 formula': 'coa[c] --&gt; coa[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000181035', 'GENE_LIST': 'SLC25A42', 'HGNC': '28380', 'Gene name': 'solute carrier family 25, member 42', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.1091', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000181035'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000181035 at 0x176e5d610>}, '_metabolites': {<Metabolite coa_c at 0x17818b0d0>: -1.0, <Metabolite amp_m at 0x2975598d0>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.64, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite amp_c at 0x2ae087b50>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.64}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'COAtmC_MitoCore', 'name': 'R_COAtmC_MitoCore CoA carrier (CoAPC) - antiport CoA for ADP', 'notes': {'Model Reaction Number': '362', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'CoA carrier (CoAPC) - antiport CoA for ADP', 'Recon2 id': 'COAtm', 'Recon2 formula': 'coa[c] --&gt; coa[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000181035', 'GENE_LIST': 'SLC25A42', 'HGNC': '28380', 'Gene name': 'solute carrier family 25, member 42', 'MitoCarta': 'TRUE', 'MitoCarta score': '26.1091', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000181035'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000181035 at 0x176e5d610>}, '_metabolites': {<Metabolite coa_c at 0x17818b0d0>: -1.0, <Metabolite adp_m at 0x2a2a88e50>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite coa_m at 0x178167cd0>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'GLUt2mB_MitoCore', 'name': 'R_GLUt2mB_MitoCore Glutamate Carrier (GC2) - Symport L-glutamate with proton together', 'notes': {'Model Reaction Number': '363', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Glutamate Carrier (GC2) - Symport L-glutamate with proton together', 'Recon2 id': 'GLUt2m', 'Recon2 formula': 'h[c] + glu_L[c] --&gt; h[m] + glu_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000182902', 'GENE_LIST': 'SLC25A18', 'HGNC': '10988', 'Gene name': 'solute carrier family 25 (glutamate carrier), member 18', 'MitoCarta': 'TRUE', 'MitoCarta score': '17.5999', 'Directionality rationale': 'Required for modelling purposes', 'HPA mRNA level in heart tissue': 'Not detected', 'HPA protein level in heart tissue': 'ND'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000182902'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000182902 at 0x176e5d950>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.18, <Metabolite glu_L_c at 0x29c7fe410>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.18, <Metabolite glu_L_m at 0x29c7fe9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ORNt4mB_MitoCore', 'name': 'R_ORNt4mB_MitoCore Ornithine carrier (ORC1) - antiport ornithine for citrulline', 'notes': {'Model Reaction Number': '364', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Ornithine carrier (ORC1) - antiport ornithine for citrulline', 'Recon2 id': 'ORNt4m', 'Recon2 formula': 'h[c] + orn[m] + citr_L[c] --&gt; h[m] + orn[c] + citr_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000102743', 'GENE_LIST': 'SLC25A15', 'HGNC': '10985', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; ornithine transporter) member 15', 'MitoCarta': 'TRUE', 'MitoCarta score': '20.4855', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000102743'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000102743 at 0x176e5dc90>}, '_metabolites': {<Metabolite orn_c at 0x29e2d2250>: -1.0, <Metabolite citr_L_m at 0x2ae025890>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite orn_m at 0x29d211b90>: 1.0, <Metabolite citr_L_c at 0x2a2fecf50>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r2398B_MitoCore', 'name': 'R_r2398B_MitoCore Ornithine carrier (ORC1) - antiport lysine for citrulline', 'notes': {'Model Reaction Number': '365', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Ornithine carrier (ORC1) - antiport lysine for citrulline', 'Recon2 id': 'r2398', 'Recon2 formula': 'h[c] + lys_L[m] + citr_L[c] -&gt; h[m] + lys_L[c] + citr_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000102743', 'GENE_LIST': 'SLC25A15', 'HGNC': '10985', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; ornithine transporter) member 15', 'MitoCarta': 'TRUE', 'MitoCarta score': '20.4855', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000102743'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000102743 at 0x176e5dc90>}, '_metabolites': {<Metabolite lys_L_c at 0x29d022890>: -1.0, <Metabolite citr_L_m at 0x2ae025890>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite lys_L_m at 0x29d021590>: 1.0, <Metabolite citr_L_c at 0x2a2fecf50>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r2402B_MitoCore', 'name': 'R_r2402B_MitoCore Ornithine carrier (ORC1) - arginine for citrulline', 'notes': {'Model Reaction Number': '366', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Ornithine carrier (ORC1) - arginine for citrulline', 'Recon2 id': 'r2402', 'Recon2 formula': 'h[c] + arg_L[m] + citr_L[c] -&gt; h[m] + arg_L[c] + citr_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000102743', 'GENE_LIST': 'SLC25A15', 'HGNC': '10985', 'Gene name': 'solute carrier family 25 (mitochondrial carrier; ornithine transporter) member 15', 'MitoCarta': 'TRUE', 'MitoCarta score': '20.4855', 'HPA mRNA level in heart tissue': 'Low', 'HPA protein level in heart tissue': 'Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000102743'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000102743 at 0x176e5dc90>}, '_metabolites': {<Metabolite arg_L_c at 0x29f3b3410>: -1.0, <Metabolite citr_L_m at 0x2ae025890>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite arg_L_m at 0x2ae4a81d0>: 1.0, <Metabolite citr_L_c at 0x2a2fecf50>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LYStmB_MitoCore', 'name': 'R_LYStmB_MitoCore Basic amino acid carrier (hBAC) - lysine', 'notes': {'Model Reaction Number': '367', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Basic amino acid carrier (hBAC) - lysine', 'Recon2 id': 'LYStm', 'Recon2 formula': 'h[m] + lys_L[c] --&gt; h[c] + lys_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000197119', 'GENE_LIST': 'SLC25A29', 'HGNC': '20116', 'Gene name': 'solute carrier family 25 (mitochondrial carnitine/acylcarnitine carrier), member 29', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000197119'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000197119 at 0x176e5dfd0>}, '_metabolites': {<Metabolite lys_L_c at 0x29d022890>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite lys_L_m at 0x29d021590>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ORNt3mB_MitoCore', 'name': 'R_ORNt3mB_MitoCore Basic amino acid carrier (hBAC) - ornithine', 'notes': {'Model Reaction Number': '368', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Basic amino acid carrier (hBAC) - ornithine', 'Recon2 id': 'ORNt3m', 'Recon2 formula': 'h[c] + orn[m] --&gt; h[m] + orn[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000197119', 'GENE_LIST': 'SLC25A29', 'HGNC': '20116', 'Gene name': 'solute carrier family 25 (mitochondrial carnitine/acylcarnitine carrier), member 29', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000197119'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000197119 at 0x176e5dfd0>}, '_metabolites': {<Metabolite orn_m at 0x29d211b90>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite orn_c at 0x29e2d2250>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ARGtmB_MitoCore', 'name': 'R_ARGtmB_MitoCore Basic amino acid carrier (hBAC) - arginine', 'notes': {'Model Reaction Number': '369', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Basic amino acid carrier (hBAC) - arginine', 'Recon2 id': 'ARGtm', 'Recon2 formula': 'h[m] + arg_L[c] --&gt; h[c] + arg_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000197119', 'GENE_LIST': 'SLC25A29', 'HGNC': '20116', 'Gene name': 'solute carrier family 25 (mitochondrial carnitine/acylcarnitine carrier), member 29', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000197119'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000197119 at 0x176e5dfd0>}, '_metabolites': {<Metabolite arg_L_c at 0x29f3b3410>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite arg_L_m at 0x2ae4a81d0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1427', 'name': 'R_r1427 Basic amino acid carrier (hBAC) - L-histidine', 'notes': {'Model Reaction Number': '370', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Basic amino acid carrier (hBAC) - L-histidine', 'Recon2 id': 'r1427', 'Recon2 formula': 'h[c] + his_L[c] -&gt; h[m] + his_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000197119', 'GENE_LIST': 'SLC25A29', 'HGNC': '20116', 'Gene name': 'solute carrier family 25 (mitochondrial carnitine/acylcarnitine carrier), member 29', 'MitoCarta': 'TRUE', 'MitoCarta score': '16.6691'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000197119'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000197119 at 0x176e5dfd0>}, '_metabolites': {<Metabolite his_L_c at 0x29c7d6210>: -1.0, <Metabolite his_L_m at 0x2ae3edd50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PYRt2m', 'name': 'R_PYRt2m Pyruvate Carrier - pyruvate transport via proton symport', 'notes': {'Model Reaction Number': '371', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Pyruvate Carrier - pyruvate transport via proton symport', 'Recon2 id': 'PYRt2m', 'Recon2 formula': 'h[c] + pyr[c] --&gt; h[m] + pyr[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000060762 and ENSG00000143158', 'GENE_LIST': 'MPC1 and MPC2', 'HGNC': '21606 and HGNC:24515', 'Gene name': 'mitochondrial pyruvate carrier 1 and 2', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '17.5999, 37.3563', 'Directionality rationale': 'Assumed concentration gradient', 'HPA mRNA level in heart tissue': 'High and High', 'HPA protein level in heart tissue': 'High and Medium'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000060762 and ENSG00000143158'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000060762 at 0x176e5e310>, <Gene ENSG00000143158 at 0x176e5e650>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.18, <Metabolite pyr_c at 0x29d0b3990>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite pyr_m at 0x29d290b10>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.18}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACACt2mB_MitoCore', 'name': 'R_ACACt2mB_MitoCore Pyruvate Carrier - Acetoacetate transport via proton symport', 'notes': {'Model Reaction Number': '372', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'Pyruvate Carrier - Acetoacetate transport via proton symport', 'Recon2 id': 'ACACt2m', 'Recon2 formula': 'h[c] + acac[c] --&gt; h[m] + acac[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000060762 and ENSG00000143158', 'GENE_LIST': 'MPC1 and MPC2', 'HGNC': '21606 and HGNC:24515', 'Gene name': 'mitochondrial pyruvate carrier 1 and 2'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000060762 and ENSG00000143158'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000060762 at 0x176e5e310>, <Gene ENSG00000143158 at 0x176e5e650>}, '_metabolites': {<Metabolite h_c at 0x29c7e4b50>: -1.0, <Metabolite acac_c at 0x28b116d10>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.18, <Metabolite h_m at 0x29c7e4250>: 1.0, <Metabolite acac_m at 0x2a02a65d0>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.18}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FE2tm', 'name': 'R_FE2tm MitoFerrin carrier', 'notes': {'Model Reaction Number': '373', 'SUBSYSTEM': 'Mitochondrial transporters - characterised', 'Description': 'MitoFerrin carrier', 'Recon2 id': 'FE2tm', 'Recon2 formula': 'h[c] + fe2[c] -&gt; h[m] + fe2[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'ENSG00000147454 or ENSG00000155287', 'GENE_LIST': 'SLC25A37 or SLC25A28', 'HGNC': '29786 HGNC:23472', 'Gene name': 'solute carrier family 25 (mitochondrial iron transporter), member 37 or 28', 'MitoCarta': 'TRUE, TRUE', 'MitoCarta score': '5.9313, 5.9313', 'HPA mRNA level in heart tissue': 'Low, Medium', 'HPA protein level in heart tissue': 'High, N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('ENSG00000147454 or ENSG00000155287'), 'subsystem': 'Mitochondrial transporters - characterised', '_genes': {<Gene ENSG00000147454 at 0x176e5e990>, <Gene ENSG00000155287 at 0x176e5ecd0>}, '_metabolites': {<Metabolite PMF_c at 0x29c7879d0>: -1.64, <Metabolite fe2_c at 0x133815550>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: 1.64, <Metabolite fe2_m at 0x1335dd950>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ASNtm', 'name': 'R_ASNtm Unknown amino acid carrier - L-Asparagine', 'notes': {'Model Reaction Number': '374', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Asparagine', 'Recon2 id': 'ASNtm', 'Recon2 formula': 'asn_L[c] -&gt; asn_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite asn_L_c at 0x161ab19d0>: -1.0, <Metabolite asn_L_m at 0x1789aa950>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1437', 'name': 'R_r1437 Unknown amino acid carrier - L-Cysteine', 'notes': {'Model Reaction Number': '375', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Cysteine', 'Recon2 id': 'r1437', 'Recon2 formula': 'cys_L[c] -&gt; cys_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite cys_L_c at 0x29f32a9d0>: -1.0, <Metabolite cys_L_m at 0x13356df50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLNtm', 'name': 'R_GLNtm Unknown amino acid carrier - L-Glutamine', 'notes': {'Model Reaction Number': '376', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Glutamine', 'Recon2 id': 'GLNtm', 'Recon2 formula': 'gln_L[c] -&gt; gln_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite gln_L_c at 0x13354b910>: -1.0, <Metabolite gln_L_m at 0x133551a10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PROtm', 'name': 'R_PROtm Unknown amino acid carrier - L-Proline', 'notes': {'Model Reaction Number': '377', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Proline', 'Recon2 id': 'PROtm', 'Recon2 formula': 'pro_L[c] -&gt; pro_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite pro_L_c at 0x29d22c4d0>: -1.0, <Metabolite pro_L_m at 0x1696f8b10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1078', 'name': 'R_r1078 Unknown amino acid carrier - L-Tyrosine', 'notes': {'Model Reaction Number': '378', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Tyrosine', 'Recon2 id': 'r1078', 'Recon2 formula': 'tyr_L[c] -&gt; tyr_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite tyr_L_c at 0x29d719b50>: -1.0, <Metabolite tyr_L_m at 0x29d7726d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1436', 'name': 'R_r1436 Unknown amino acid carrier - L-Methionine', 'notes': {'Model Reaction Number': '379', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Methionine', 'Recon2 id': 'r1436', 'Recon2 formula': 'met_L[c] -&gt; met_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite met_L_c at 0x29d011150>: -1.0, <Metabolite met_L_m at 0x29d0e7bd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1455', 'name': 'R_r1455 Unknown amino acid carrier - L-Phenylalanine', 'notes': {'Model Reaction Number': '380', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Phenylalanine', 'Recon2 id': 'r1455', 'Recon2 formula': 'h[c] + phe_L[c] -&gt; h[m] + phe_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite phe_L_c at 0x29e2c27d0>: -1.0, <Metabolite phe_L_m at 0x29e60a810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TRPtm_MitoCore', 'name': 'R_TRPtm_MitoCore Unknown amino acid carrier - L-Tryptophan', 'notes': {'Model Reaction Number': '381', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Tryptophan', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite trp_L_c at 0x29d7a42d0>: -1.0, <Metabolite trp_L_m at 0x29d745a10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYtm', 'name': 'R_GLYtm Unknown amino acid carrier - Glycine', 'notes': {'Model Reaction Number': '382', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - Glycine', 'Recon2 id': 'GLYtm', 'Recon2 formula': 'gly[c] --&gt; gly[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite gly_c at 0x29c737c90>: -1.0, <Metabolite gly_m at 0x29c735650>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ILEt5m', 'name': 'R_ILEt5m Unknown amino acid carrier - L-Isoleucine', 'notes': {'Model Reaction Number': '383', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Isoleucine', 'Recon2 id': 'ILEt5m', 'Recon2 formula': 'ile_L[c] --&gt; ile_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite ile_L_c at 0x29c763810>: -1.0, <Metabolite ile_L_m at 0x2ae04e010>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'LEUt5m', 'name': 'R_LEUt5m Unknown amino acid carrier - L-Leucine', 'notes': {'Model Reaction Number': '384', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Leucine', 'Recon2 id': 'LEUt5m', 'Recon2 formula': 'leu_L[c] --&gt; leu_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite leu_L_c at 0x2a29b5ed0>: -1.0, <Metabolite leu_L_m at 0x29d0ce590>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'VALt5m', 'name': 'R_VALt5m Unknown amino acid carrier - L-Valine', 'notes': {'Model Reaction Number': '385', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Valine', 'Recon2 id': 'VALt5m', 'Recon2 formula': 'val_L[c] --&gt; val_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite val_L_c at 0x29d750090>: -1.0, <Metabolite val_L_m at 0x29d753d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1434', 'name': 'R_r1434 Unknown amino acid carrier - L-Alanine', 'notes': {'Model Reaction Number': '386', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Alanine', 'Recon2 id': 'r1434', 'Recon2 formula': 'ala_L[c] -&gt; ala_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite ala_L_c at 0x17aef1250>: -1.0, <Metabolite ala_L_m at 0x2a2a29310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 6e-05}\n",
      "{'_id': 'r1435', 'name': 'R_r1435 Unknown amino acid carrier - L-Serine', 'notes': {'Model Reaction Number': '387', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Serine', 'Recon2 id': 'r1435', 'Recon2 formula': 'ser_L[c] -&gt; ser_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite ser_L_c at 0x29d206f50>: -1.0, <Metabolite ser_L_m at 0x29d27c990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1440', 'name': 'R_r1440 Unknown amino acid carrier - L-Threonine', 'notes': {'Model Reaction Number': '388', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Unknown amino acid carrier - L-Threonine', 'Recon2 id': 'r1440', 'Recon2 formula': 'thr_L[c] -&gt; thr_L[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite thr_L_c at 0x29d729490>: -1.0, <Metabolite thr_L_m at 0x29d72aad0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'BALAtmr', 'name': 'R_BALAtmr Beta-alanine reversible mitochondrial transport (diffusion)', 'notes': {'Model Reaction Number': '389', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Beta-alanine reversible mitochondrial transport (diffusion)', 'Recon2 id': 'BALAtmr', 'Recon2 formula': 'ala_B[c] --&gt; ala_B[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite ala_B_c at 0x178fb9910>: -1.0, <Metabolite ala_B_m at 0x178dca3d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'UREAtm', 'name': 'R_UREAtm Urea transport', 'notes': {'Model Reaction Number': '390', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Urea transport', 'Recon2 id': 'UREAtm', 'Recon2 formula': 'urea[c] --&gt; urea[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite urea_c at 0x29d763f10>: -1.0, <Metabolite urea_m at 0x29d7c2990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FUMtmB_MitoCore', 'name': 'R_FUMtmB_MitoCore Fumarate transport', 'notes': {'Model Reaction Number': '391', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Fumarate transport', 'Recon2 id': 'FUMtm', 'Recon2 formula': 'pi[m] + fum[c] --&gt; pi[c] + fum[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown', 'Directionality rationale': 'Assumed concentration gradient'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite fum_c at 0x133597890>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -1.64, <Metabolite fum_m at 0x133525390>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 1.64}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'BHBtmB_MitoCore', 'name': 'R_BHBtmB_MitoCore (R)-3-Hydroxybutanoate mitochondrial transport', 'notes': {'Model Reaction Number': '392', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': '(R)-3-Hydroxybutanoate mitochondrial transport', 'Recon2 id': 'BHBtm', 'Recon2 formula': 'h[c] + bhb[c] --&gt; h[m] + bhb[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite bhb_c at 0x16aaaff10>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite bhb_m at 0x178671290>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PPAtmB_MitoCore', 'name': 'R_PPAtmB_MitoCore Propanoate mitochondrial transport', 'notes': {'Model Reaction Number': '393', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Propanoate mitochondrial transport', 'Recon2 id': 'PPAtm', 'Recon2 formula': 'ppa[c] -&gt; ppa[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite ppa_c at 0x29e678890>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite ppa_m at 0x29e604450>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'BUTt2mB_MitoCore', 'name': 'R_BUTt2mB_MitoCore Butyrate mitochondrial transport', 'notes': {'Model Reaction Number': '394', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Butyrate mitochondrial transport', 'Recon2 id': 'BUTt2m', 'Recon2 formula': 'h[c] + but[c] --&gt; h[m] + but[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite but_c at 0x17b24acd0>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite but_m at 0x16b692610>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FORt2mB_MitoCore', 'name': 'R_FORt2mB_MitoCore Formate mitochondrial transport', 'notes': {'Model Reaction Number': '395', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Formate mitochondrial transport', 'Recon2 id': 'FORt2m', 'Recon2 formula': 'for[m] --&gt; for[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite for_m at 0x133547b90>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite for_c at 0x13352d2d0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0962B_MitoCore', 'name': 'R_r0962B_MitoCore Folate mitochondrial transport', 'notes': {'Model Reaction Number': '396', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Folate mitochondrial transport', 'Recon2 id': 'r0962', 'Recon2 formula': 'fol[c] -&gt; fol[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite fol_c at 0x29d072290>: -1.0, <Metabolite PMF_m at 0x29c785ad0>: -0.82, <Metabolite fol_m at 0x29d007fd0>: 1.0, <Metabolite PMF_c at 0x29c7879d0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CHLtmB_MitoCore', 'name': 'R_CHLtmB_MitoCore Choline mitochondrial transport', 'notes': {'Model Reaction Number': '397', 'SUBSYSTEM': 'Mitochondrial transporters - uncharacterised', 'Description': 'Choline mitochondrial transport', 'Recon2 id': 'CHLtm', 'Recon2 formula': 'chol[c] --&gt; chol[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': 'Unknown', 'GENE_LIST': 'Unknown', 'Gene name': 'Unknown'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transporters - uncharacterised', '_genes': set(), '_metabolites': {<Metabolite chol_c at 0x16c8b5010>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite chol_m at 0x16c52cfd0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CO2tm', 'name': 'R_CO2tm CO2 transport (diffusion), mitochondrial', 'notes': {'Model Reaction Number': '398', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'CO2 transport (diffusion), mitochondrial', 'Recon2 id': 'CO2tm', 'Recon2 formula': 'co2[c] --&gt; co2[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite co2_c at 0x17810ba50>: -1.0, <Metabolite co2_m at 0x1781348d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'H2Otm', 'name': 'R_H2Otm H2O transport, mitochondrial', 'notes': {'Model Reaction Number': '399', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'H2O transport, mitochondrial', 'Recon2 id': 'H2Otm', 'Recon2 formula': 'h2o[c] --&gt; h2o[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite h2o_m at 0x29c7848d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'O2tm', 'name': 'R_O2tm O2 transport (diffusion)', 'notes': {'Model Reaction Number': '400', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'O2 transport (diffusion)', 'Recon2 id': 'O2tm', 'Recon2 formula': 'o2[c] --&gt; o2[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite o2_c at 0x29d48db10>: -1.0, <Metabolite o2_m at 0x17b2554d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'GLYCtm', 'name': 'R_GLYCtm glycerol transport', 'notes': {'Model Reaction Number': '401', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'glycerol transport', 'Recon2 id': 'GLYCtm', 'Recon2 formula': 'glyc[c] --&gt; glyc[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite glyc_c at 0x29c734b50>: -1.0, <Metabolite glyc_m at 0x29c7bd390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYANtm', 'name': 'R_CYANtm Cyanide transport via diffusion', 'notes': {'Model Reaction Number': '402', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Cyanide transport via diffusion', 'Recon2 id': 'CYANtm', 'Recon2 formula': 'cyan[c] --&gt; cyan[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite cyan_c at 0x29f362dd0>: -1.0, <Metabolite cyan_m at 0x17b4fae10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TCYNTtmB_MitoCore', 'name': 'R_TCYNTtmB_MitoCore Thiocyanate transport via diffusion (mitochondrial)', 'notes': {'Model Reaction Number': '403', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Thiocyanate transport via diffusion (mitochondrial)', 'Recon2 id': 'TCYNTtm', 'Recon2 formula': 'tcynt[m] -&gt; tcynt[c]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite tcynt_m at 0x29d77a9d0>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite tcynt_c at 0x29d77d2d0>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CREATtmdiffir', 'name': 'R_CREATtmdiffir Creatine transport to/from mitochondria', 'notes': {'Model Reaction Number': '404', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Creatine transport to/from mitochondria', 'Recon2 id': 'CREATtmdiffir', 'Recon2 formula': 'creat[c] -&gt; creat[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite creat_c at 0x1781b9150>: -1.0, <Metabolite creat_m at 0x28b172990>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCREATtmdiffirB_MitoCore', 'name': 'R_PCREATtmdiffirB_MitoCore Phosphocreatine transport to/from mitochondria', 'notes': {'Model Reaction Number': '405', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Phosphocreatine transport to/from mitochondria', 'Recon2 id': 'PCREATtmdiffir', 'Recon2 formula': 'pcreat[m] -&gt; pcreat[c]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite pcreat_m at 0x29e20c450>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite pcreat_c at 0x29e23cb10>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0941', 'name': 'R_r0941 Bicarbonate diffusion', 'notes': {'Model Reaction Number': '406', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Bicarbonate diffusion', 'Recon2 id': 'r0941', 'Recon2 formula': 'hco3[c] --&gt; hco3[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite hco3_c at 0x29c747e50>: -1.0, <Metabolite hco3_m at 0x29c745e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0838B_MitoCore', 'name': 'R_r0838B_MitoCore Ammonia free diffusion', 'notes': {'Model Reaction Number': '407', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Ammonia free diffusion', 'Recon2 id': 'r0838', 'Recon2 formula': 'nh4[c] --&gt; nh4[m]', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite nh4_c at 0x1639455d0>: -1.0, <Metabolite PMF_c at 0x29c7879d0>: -0.82, <Metabolite nh4_m at 0x164ccd590>: 1.0, <Metabolite PMF_m at 0x29c785ad0>: 0.82}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'Biomasst_MitoCore', 'name': 'R_Biomasst_MitoCore Biomass pseudo transport', 'notes': {'Model Reaction Number': '408', 'SUBSYSTEM': 'Mitochondrial transport - diffusion / artificial', 'Description': 'Biomass pseudo transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A', 'KEGG id': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Mitochondrial transport - diffusion / artificial', '_genes': set(), '_metabolites': {<Metabolite biomass_c at 0x1696f8b50>: -1.0, <Metabolite biomass_m at 0x169ca8b90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCFLOPm', 'name': 'R_PCFLOPm phosphatidylcholine flippase', 'notes': {'Model Reaction Number': '409', 'SUBSYSTEM': 'Mitochondrial transport - lipids', 'Description': 'phosphatidylcholine flippase', 'Recon2 id': 'PCFLOPm', 'Recon2 formula': 'h2o[c] + atp[c] + pchol_hs[m] -&gt; h[c] + adp[c] + pi[c] + pchol_hs[c]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': '(ENSG00000124406 or ENSG00000143515 or ENSG00000081923 or ENSG00000104043 or ENSG00000054793 or ENSG00000166377 or ENSG00000206190 or ENSG00000145246 or ENSG00000068650 or ENSG00000058063 or ENSG00000101974) and (ENSG00000112697 or ENSG00000182107)', 'GENE_LIST': '(ATP8A1 or ATP8B1 or ATP8B2 or ATP8B4 or ATP9A or ATP9B or ATP10A or ATP10D or ATP11A or ATP11B or ATP11C) and (TMEM30A or TMEM30B)', '(HGNC': '13531 or HGNC:13534 or HGNC:3706 or HGNC:13536 or HGNC:13540 or HGNC:13541 or HGNC:13542 or HGNC:13549 or HGNC:13552 or HGNC:13553 or HGNC:13554) and (HGNC:16667 or HGNC:27254)', 'MitoCarta': 'FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE', 'MitoCarta score': '-5.2802, -10.6961, -8.2811, -10.6961, -8.2811, -7.3264, -6.608, -6.2522, -8.5225, -8.2811, -10.6961, -8.2811, -8.6672', 'Directionality rationale': 'ATP consumption', 'HPA mRNA level in heart tissue': '(Low or Low or Low or Low or Low or Low or Low or Low or Medium or Low or Low) and (Medium or Low)', 'HPA protein level in heart tissue': '(High or Low or High or Low or Not detected or Medium or Medium or Low or Not detected or Not detected or Medium) and (Medium or Low)'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('(ENSG00000124406 and ENSG00000112697) or (ENSG00000124406 and ENSG00000182107) or (ENSG00000143515 and ENSG00000112697) or (ENSG00000143515 and ENSG00000182107) or (ENSG00000081923 and ENSG00000112697) or (ENSG00000081923 and ENSG00000182107) or (ENSG00000104043 and ENSG00000112697) or (ENSG00000104043 and ENSG00000182107) or (ENSG00000054793 and ENSG00000112697) or (ENSG00000054793 and ENSG00000182107) or (ENSG00000166377 and ENSG00000112697) or (ENSG00000166377 and ENSG00000182107) or (ENSG00000206190 and ENSG00000112697) or (ENSG00000206190 and ENSG00000182107) or (ENSG00000145246 and ENSG00000112697) or (ENSG00000145246 and ENSG00000182107) or (ENSG00000068650 and ENSG00000112697) or (ENSG00000068650 and ENSG00000182107) or (ENSG00000058063 and ENSG00000112697) or (ENSG00000058063 and ENSG00000182107) or (ENSG00000101974 and ENSG00000112697) or (ENSG00000101974 and ENSG00000182107)'), 'subsystem': 'Mitochondrial transport - lipids', '_genes': {<Gene ENSG00000143515 at 0x176e5f010>, <Gene ENSG00000206190 at 0x176de8a10>, <Gene ENSG00000068650 at 0x176de8050>, <Gene ENSG00000101974 at 0x176e5f690>, <Gene ENSG00000081923 at 0x176de9090>, <Gene ENSG00000124406 at 0x176de86d0>, <Gene ENSG00000182107 at 0x176e5fd10>, <Gene ENSG00000112697 at 0x176de9710>, <Gene ENSG00000145246 at 0x176e5f350>, <Gene ENSG00000166377 at 0x176de8d50>, <Gene ENSG00000058063 at 0x176de8390>, <Gene ENSG00000104043 at 0x176de93d0>, <Gene ENSG00000054793 at 0x176e5f9d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite pchol_hs_c at 0x29d213cd0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite pchol_hs_m at 0x29e23c350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PSFLIPm', 'name': 'R_PSFLIPm phosphatidylserine flippase', 'notes': {'Model Reaction Number': '410', 'SUBSYSTEM': 'Mitochondrial transport - lipids', 'Description': 'phosphatidylserine flippase', 'Recon2 id': 'PSFLIPm', 'Recon2 formula': 'h2o[c] + atp[c] + ps_hs[c] -&gt; h[c] + adp[c] + pi[c] + ps_hs[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': '(ENSG00000124406 or ENSG00000143515 or ENSG00000081923 or ENSG00000104043 or ENSG00000054793 or ENSG00000166377 or ENSG00000206190 or ENSG00000145246 or ENSG00000068650 or ENSG00000058063 or ENSG00000101974) and (ENSG00000112697 or ENSG00000182107)', 'GENE_LIST': '(ATP8A1 or ATP8B1 or ATP8B2 or ATP8B4 or ATP9A or ATP9B or ATP10A or ATP10D or ATP11A or ATP11B or ATP11C) and (TMEM30A or TMEM30B)', '(HGNC': '13531 or HGNC:13534 or HGNC:3706 or HGNC:13536 or HGNC:13540 or HGNC:13541 or HGNC:13542 or HGNC:13549 or HGNC:13552 or HGNC:13553 or HGNC:13554) and (HGNC:16667 or HGNC:27254)', 'MitoCarta': 'FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE', 'MitoCarta score': '-5.2802, -10.6961, -8.2811, -10.6961, -8.2811, -7.3264, -6.608, -6.2522, -8.5225, -8.2811, -10.6961, -8.2811, -8.6672', 'Directionality rationale': 'ATP consumption', 'HPA mRNA level in heart tissue': '(Low or Low or Low or Low or Low or Low or Low or Low or Medium or Low or Low) and (Medium or Low)', 'HPA protein level in heart tissue': '(High or Low or High or Low or Not detected or Medium or Medium or Low or Not detected or Not detected or Medium) and (Medium or Low)'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('(ENSG00000124406 and ENSG00000112697) or (ENSG00000124406 and ENSG00000182107) or (ENSG00000143515 and ENSG00000112697) or (ENSG00000143515 and ENSG00000182107) or (ENSG00000081923 and ENSG00000112697) or (ENSG00000081923 and ENSG00000182107) or (ENSG00000104043 and ENSG00000112697) or (ENSG00000104043 and ENSG00000182107) or (ENSG00000054793 and ENSG00000112697) or (ENSG00000054793 and ENSG00000182107) or (ENSG00000166377 and ENSG00000112697) or (ENSG00000166377 and ENSG00000182107) or (ENSG00000206190 and ENSG00000112697) or (ENSG00000206190 and ENSG00000182107) or (ENSG00000145246 and ENSG00000112697) or (ENSG00000145246 and ENSG00000182107) or (ENSG00000068650 and ENSG00000112697) or (ENSG00000068650 and ENSG00000182107) or (ENSG00000058063 and ENSG00000112697) or (ENSG00000058063 and ENSG00000182107) or (ENSG00000101974 and ENSG00000112697) or (ENSG00000101974 and ENSG00000182107)'), 'subsystem': 'Mitochondrial transport - lipids', '_genes': {<Gene ENSG00000143515 at 0x176e5f010>, <Gene ENSG00000206190 at 0x176de8a10>, <Gene ENSG00000068650 at 0x176de8050>, <Gene ENSG00000101974 at 0x176e5f690>, <Gene ENSG00000081923 at 0x176de9090>, <Gene ENSG00000124406 at 0x176de86d0>, <Gene ENSG00000182107 at 0x176e5fd10>, <Gene ENSG00000112697 at 0x176de9710>, <Gene ENSG00000145246 at 0x176e5f350>, <Gene ENSG00000166377 at 0x176de8d50>, <Gene ENSG00000058063 at 0x176de8390>, <Gene ENSG00000104043 at 0x176de93d0>, <Gene ENSG00000054793 at 0x176e5f9d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite ps_hs_c at 0x29d20b390>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite ps_hs_m at 0x29d28e350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PEFLIPm', 'name': 'R_PEFLIPm phosphatidylethanolamine flippase', 'notes': {'Model Reaction Number': '411', 'SUBSYSTEM': 'Mitochondrial transport - lipids', 'Description': 'phosphatidylethanolamine flippase', 'Recon2 id': 'PEFLIPm', 'Recon2 formula': 'h2o[c] + atp[c] + pe_hs[c] -&gt; h[c] + adp[c] + pi[c] + pe_hs[m]', 'KEGG id': 'N/A', 'GENE_ASSOCIATION': '(ENSG00000124406 or ENSG00000143515 or ENSG00000081923 or ENSG00000104043 or ENSG00000054793 or ENSG00000166377 or ENSG00000206190 or ENSG00000145246 or ENSG00000068650 or ENSG00000058063 or ENSG00000101974) and (ENSG00000112697 or ENSG00000182107)', 'GENE_LIST': '(ATP8A1 or ATP8B1 or ATP8B2 or ATP8B4 or ATP9A or ATP9B or ATP10A or ATP10D or ATP11A or ATP11B or ATP11C) and (TMEM30A or TMEM30B)', '(HGNC': '13531 or HGNC:13534 or HGNC:3706 or HGNC:13536 or HGNC:13540 or HGNC:13541 or HGNC:13542 or HGNC:13549 or HGNC:13552 or HGNC:13553 or HGNC:13554) and (HGNC:16667 or HGNC:27254)', 'MitoCarta': 'FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, FALSE, FALSE, FALSE, FALSE, FALSE', 'MitoCarta score': '-5.2802, -10.6961, -8.2811, -10.6961, -8.2811, -7.3264, -6.608, -6.2522, -8.5225, -8.2811, -10.6961, -8.2811, -8.6672', 'Directionality rationale': 'ATP consumption', 'HPA mRNA level in heart tissue': '(Low or Low or Low or Low or Low or Low or Low or Low or Medium or Low or Low) and (Medium or Low)', 'HPA protein level in heart tissue': '(High or Low or High or Low or Not detected or Medium or Medium or Low or Not detected or Not detected or Medium) and (Medium or Low)'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR('(ENSG00000124406 and ENSG00000112697) or (ENSG00000124406 and ENSG00000182107) or (ENSG00000143515 and ENSG00000112697) or (ENSG00000143515 and ENSG00000182107) or (ENSG00000081923 and ENSG00000112697) or (ENSG00000081923 and ENSG00000182107) or (ENSG00000104043 and ENSG00000112697) or (ENSG00000104043 and ENSG00000182107) or (ENSG00000054793 and ENSG00000112697) or (ENSG00000054793 and ENSG00000182107) or (ENSG00000166377 and ENSG00000112697) or (ENSG00000166377 and ENSG00000182107) or (ENSG00000206190 and ENSG00000112697) or (ENSG00000206190 and ENSG00000182107) or (ENSG00000145246 and ENSG00000112697) or (ENSG00000145246 and ENSG00000182107) or (ENSG00000068650 and ENSG00000112697) or (ENSG00000068650 and ENSG00000182107) or (ENSG00000058063 and ENSG00000112697) or (ENSG00000058063 and ENSG00000182107) or (ENSG00000101974 and ENSG00000112697) or (ENSG00000101974 and ENSG00000182107)'), 'subsystem': 'Mitochondrial transport - lipids', '_genes': {<Gene ENSG00000143515 at 0x176e5f010>, <Gene ENSG00000206190 at 0x176de8a10>, <Gene ENSG00000068650 at 0x176de8050>, <Gene ENSG00000101974 at 0x176e5f690>, <Gene ENSG00000081923 at 0x176de9090>, <Gene ENSG00000124406 at 0x176de86d0>, <Gene ENSG00000182107 at 0x176e5fd10>, <Gene ENSG00000112697 at 0x176de9710>, <Gene ENSG00000145246 at 0x176e5f350>, <Gene ENSG00000166377 at 0x176de8d50>, <Gene ENSG00000058063 at 0x176de8390>, <Gene ENSG00000104043 at 0x176de93d0>, <Gene ENSG00000054793 at 0x176e5f9d0>}, '_metabolites': {<Metabolite h2o_c at 0x29c7876d0>: -1.0, <Metabolite atp_c at 0x2ae181f10>: -1.0, <Metabolite pe_hs_c at 0x29e291750>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite adp_c at 0x17918bb10>: 1.0, <Metabolite pi_c at 0x29e6c47d0>: 1.0, <Metabolite pe_hs_m at 0x29e2bf250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'Biomass_MitoCore', 'name': 'R_Biomass_MitoCore Biomass Sink', 'notes': {'Model Reaction Number': '412', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'Biomass Sink', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite biomass_e at 0x1785f9f90>: -1.0, <Metabolite biomass_c at 0x1696f8b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'O2t', 'name': 'R_O2t o2 transport (diffusion)', 'notes': {'Model Reaction Number': '413', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'o2 transport (diffusion)', 'Recon2 id': 'O2t', 'Recon2 formula': 'o2[e] --&gt; o2[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite o2_e at 0x29d4fa190>: -1.0, <Metabolite o2_c at 0x29d48db10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1.188}\n",
      "{'_id': 'CO2t', 'name': 'R_CO2t CO2 transporter via diffusion', 'notes': {'Model Reaction Number': '414', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'CO2 transporter via diffusion', 'Recon2 id': 'CO2t', 'Recon2 formula': 'co2[e] --&gt; co2[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite co2_e at 0x178134490>: -1.0, <Metabolite co2_c at 0x17810ba50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'HCO3t_MitoCore', 'name': 'R_HCO3t_MitoCore Bicarbonate diffusion', 'notes': {'Model Reaction Number': '415', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'Bicarbonate diffusion', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite hco3_e at 0x16ca19bd0>: -1.0, <Metabolite hco3_c at 0x29c747e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.1026}\n",
      "{'_id': 'GLCt1r', 'name': 'R_GLCt1r glucose transport', 'notes': {'Model Reaction Number': '416', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'glucose transport', 'Recon2 id': 'GLCt1r', 'Recon2 formula': 'glc_D[e] --&gt; glc_D[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite glc_D_e at 0x13354e010>: -1.0, <Metabolite glc_D_c at 0x2ae06c410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.054}\n",
      "{'_id': 'HDCAtr', 'name': 'R_HDCAtr fatty acid transport via diffusion', 'notes': {'Model Reaction Number': '417', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'fatty acid transport via diffusion', 'Recon2 id': 'HDCAtr', 'Recon2 formula': 'hdca[e] --&gt; hdca[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite hdca_e at 0x29c7d7f10>: -1.0, <Metabolite hdca_c at 0x164ccde10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.06}\n",
      "{'_id': 'HDCAtm_MitoCore', 'name': 'R_HDCAtm_MitoCore fatty acid accumulation - mitochondria', 'notes': {'Model Reaction Number': '418', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'fatty acid accumulation - mitochondria', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite hdca_e at 0x29c7d7f10>: -1.0, <Metabolite hdca_m at 0x16d63ddd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'L_LACt2r', 'name': 'R_L_LACt2r L-lactate reversible transport via proton symport', 'notes': {'Model Reaction Number': '419', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'L-lactate reversible transport via proton symport', 'Recon2 id': 'L_LACt2r', 'Recon2 formula': 'h[e] + lac_L[e] --&gt; h[c] + lac_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite lac_L_e at 0x29d0a8410>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite lac_L_c at 0x17aeddf10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0345}\n",
      "{'_id': 'BHBt', 'name': 'R_BHBt (R)-3-Hydroxybutanoate transport via H+ symport', 'notes': {'Model Reaction Number': '420', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': '(R)-3-Hydroxybutanoate transport via H+ symport', 'Recon2 id': 'BHBt', 'Recon2 formula': 'h[e] + bhb[e] --&gt; h[c] + bhb[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite bhb_e at 0x17b519e90>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite bhb_c at 0x16aaaff10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00288}\n",
      "{'_id': 'ACACt2', 'name': 'R_ACACt2 acetoacetate transport via proton symport', 'notes': {'Model Reaction Number': '421', 'SUBSYSTEM': 'Boundary conditions - core', 'Description': 'acetoacetate transport via proton symport', 'Recon2 id': 'ACACt2', 'Recon2 formula': 'h[e] + acac[e] --&gt; h[c] + acac[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - core', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite acac_e at 0x28b116f50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite acac_c at 0x28b116d10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00684}\n",
      "{'_id': 'ETOHt', 'name': 'R_ETOHt ethanol reversible transport', 'notes': {'Model Reaction Number': '422', 'SUBSYSTEM': 'Boundary conditions - alternative fuels', 'Description': 'ethanol reversible transport', 'Recon2 id': 'ETOHt', 'Recon2 formula': 'etoh[e] --&gt; etoh[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - alternative fuels', '_genes': set(), '_metabolites': {<Metabolite etoh_e at 0x2a2b9ac10>: -1.0, <Metabolite etoh_c at 0x1337c2e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'BUTt2r', 'name': 'R_BUTt2r Butyrate transport via proton symport, reversible', 'notes': {'Model Reaction Number': '423', 'SUBSYSTEM': 'Boundary conditions - alternative fuels', 'Description': 'Butyrate transport via proton symport, reversible', 'Recon2 id': 'BUTt2r', 'Recon2 formula': 'h[e] + but[e] --&gt; h[c] + but[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - alternative fuels', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite but_e at 0x2a2a64cd0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite but_c at 0x17b24acd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00036}\n",
      "{'_id': 'GLYCt', 'name': 'R_GLYCt glycerol transport via channel', 'notes': {'Model Reaction Number': '424', 'SUBSYSTEM': 'Boundary conditions - alternative fuels', 'Description': 'glycerol transport via channel', 'Recon2 id': 'GLYCt', 'Recon2 formula': 'glyc[c] --&gt; glyc[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - alternative fuels', '_genes': set(), '_metabolites': {<Metabolite glyc_c at 0x29c734b50>: -1.0, <Metabolite glyc_e at 0x29c734090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -0.0006, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0942', 'name': 'R_r0942 Creatine storage', 'notes': {'Model Reaction Number': '425', 'SUBSYSTEM': 'Boundary conditions - alternative fuels', 'Description': 'Creatine storage', 'Recon2 id': 'r0942', 'Recon2 formula': 'creat[c] --&gt; creat[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - alternative fuels', '_genes': set(), '_metabolites': {<Metabolite creat_c at 0x1781b9150>: -1.0, <Metabolite creat_e at 0x17b50a8d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0942b_MitoCore', 'name': 'R_r0942b_MitoCore Phosphocreatine storage', 'notes': {'Model Reaction Number': '426', 'SUBSYSTEM': 'Boundary conditions - alternative fuels', 'Description': 'Phosphocreatine storage', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - alternative fuels', '_genes': set(), '_metabolites': {<Metabolite pcreat_c at 0x29e23cb10>: -1.0, <Metabolite pcreat_e at 0x29e24fb90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'HIStiDF', 'name': 'R_HIStiDF L-histidine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '427', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-histidine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'HIStiDF', 'Recon2 formula': 'his_L[e] -&gt; his_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite his_L_e at 0x29c7d5610>: -1.0, <Metabolite his_L_c at 0x29c7d6210>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0006}\n",
      "{'_id': 'ILEtec', 'name': 'R_ILEtec L-isoleucine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '428', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-isoleucine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'ILEtec', 'Recon2 formula': 'ile_L[e] --&gt; ile_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite ile_L_e at 0x29c762e90>: -1.0, <Metabolite ile_L_c at 0x29c763810>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00024}\n",
      "{'_id': 'LEUtec', 'name': 'R_LEUtec L-leucine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '429', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-leucine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'LEUtec', 'Recon2 formula': 'leu_L[e] --&gt; leu_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite leu_L_e at 0x29d0cee10>: -1.0, <Metabolite leu_L_c at 0x2a29b5ed0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00096}\n",
      "{'_id': 'LYStiDF', 'name': 'R_LYStiDF L-lysine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '430', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-lysine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'LYStiDF', 'Recon2 formula': 'lys_L[e] -&gt; lys_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite lys_L_e at 0x29d021d50>: -1.0, <Metabolite lys_L_c at 0x29d022890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0018}\n",
      "{'_id': 'METtec', 'name': 'R_METtec L-methionine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '431', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-methionine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'METtec', 'Recon2 formula': 'met_L[e] --&gt; met_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite met_L_e at 0x29d0e7a10>: -1.0, <Metabolite met_L_c at 0x29d011150>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0003}\n",
      "{'_id': 'PHEtec', 'name': 'R_PHEtec L-phenylalanine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '432', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-phenylalanine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'PHEtec', 'Recon2 formula': 'phe_L[e] --&gt; phe_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite phe_L_e at 0x29e616790>: -1.0, <Metabolite phe_L_c at 0x29e2c27d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00072}\n",
      "{'_id': 'r2534', 'name': 'R_r2534 L-threonine transport', 'notes': {'Model Reaction Number': '433', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-threonine transport', 'Recon2 id': 'r2534', 'Recon2 formula': 'thr_L[e] --&gt; thr_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite thr_L_e at 0x29d728cd0>: -1.0, <Metabolite thr_L_c at 0x29d729490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00072}\n",
      "{'_id': 'TRPt', 'name': 'R_TRPt L-tryptophan transport', 'notes': {'Model Reaction Number': '434', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-tryptophan transport', 'Recon2 id': 'TRPt', 'Recon2 formula': 'trp_L[e] --&gt; trp_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite trp_L_e at 0x29d7a63d0>: -1.0, <Metabolite trp_L_c at 0x29d7a42d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00048}\n",
      "{'_id': 'VALtec', 'name': 'R_VALtec L-valine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '435', 'SUBSYSTEM': 'Boundary conditions - essential amino acids', 'Description': 'L-valine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'VALtec', 'Recon2 formula': 'val_L[e] --&gt; val_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - essential amino acids', '_genes': set(), '_metabolites': {<Metabolite val_L_e at 0x29d751a90>: -1.0, <Metabolite val_L_c at 0x29d750090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.00066}\n",
      "{'_id': 'ARGtiDF', 'name': 'R_ARGtiDF L-arginine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '436', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-arginine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'ARGtiDF', 'Recon2 formula': 'arg_L[e] -&gt; arg_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite arg_L_e at 0x17b2591d0>: -1.0, <Metabolite arg_L_c at 0x29f3b3410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00042}\n",
      "{'_id': 'ASPte', 'name': 'R_ASPte L-aspartate transport', 'notes': {'Model Reaction Number': '437', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-aspartate transport', 'Recon2 id': 'ASPte', 'Recon2 formula': 'asp_L[c] -&gt; asp_L[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite asp_L_c at 0x16a60b310>: -1.0, <Metabolite asp_L_e at 0x2919310d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -0.00924, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYStec', 'name': 'R_CYStec L-cysteine transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '438', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-cysteine transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'CYStec', 'Recon2 formula': 'cys_L[e] --&gt; cys_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite cys_L_e at 0x133bc3910>: -1.0, <Metabolite cys_L_c at 0x29f32a9d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 6e-05}\n",
      "{'_id': 'GLUt_MitoCore', 'name': 'R_GLUt_MitoCore L-glutamate transport', 'notes': {'Model Reaction Number': '439', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-glutamate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite glu_L_e at 0x29c7ff110>: -1.0, <Metabolite glu_L_c at 0x29c7fe410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r2525', 'name': 'R_r2525 L_glutamine transport', 'notes': {'Model Reaction Number': '440', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L_glutamine transport', 'Recon2 id': 'r2525', 'Recon2 formula': 'gln_L[e] --&gt; gln_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite gln_L_e at 0x2ae0ac890>: -1.0, <Metabolite gln_L_c at 0x13354b910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'GLYt2r', 'name': 'R_GLYt2r glycine reversible transport via proton symport', 'notes': {'Model Reaction Number': '441', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'glycine reversible transport via proton symport', 'Recon2 id': 'GLYt2r', 'Recon2 formula': 'h[e] + gly[e] --&gt; h[c] + gly[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite gly_e at 0x29c736110>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite gly_c at 0x29c737c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00054}\n",
      "{'_id': 'PROt2r', 'name': 'R_PROt2r L-proline reversible transport via proton symport', 'notes': {'Model Reaction Number': '442', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-proline reversible transport via proton symport', 'Recon2 id': 'PROt2r', 'Recon2 formula': 'h[e] + pro_L[e] --&gt; h[c] + pro_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite pro_L_e at 0x29d22d590>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite pro_L_c at 0x29d22c4d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00024}\n",
      "{'_id': 'r2526', 'name': 'R_r2526 L-serine transport', 'notes': {'Model Reaction Number': '443', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-serine transport', 'Recon2 id': 'r2526', 'Recon2 formula': 'ser_L[e] --&gt; ser_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite ser_L_e at 0x29d27d250>: -1.0, <Metabolite ser_L_c at 0x29d206f50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00102}\n",
      "{'_id': 'TYRt', 'name': 'R_TYRt L-tyrosine transport', 'notes': {'Model Reaction Number': '444', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-tyrosine transport', 'Recon2 id': 'TYRt', 'Recon2 formula': 'tyr_L[e] --&gt; tyr_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite tyr_L_e at 0x29d771110>: -1.0, <Metabolite tyr_L_c at 0x29d719b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.00066}\n",
      "{'_id': 'r2532', 'name': 'R_r2532 L-asparagine transport', 'notes': {'Model Reaction Number': '445', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-asparagine transport', 'Recon2 id': 'r2532', 'Recon2 formula': 'asn_L[e] --&gt; asn_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite asn_L_e at 0x1789aa550>: -1.0, <Metabolite asn_L_c at 0x161ab19d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0006}\n",
      "{'_id': 'ALAt2r', 'name': 'R_ALAt2r L-alanine reversible transport via proton symport', 'notes': {'Model Reaction Number': '446', 'SUBSYSTEM': 'Boundary conditions - nonessential amino acids', 'Description': 'L-alanine reversible transport via proton symport', 'Recon2 id': 'ALAt2r', 'Recon2 formula': 'h[e] + ala_L[e] --&gt; h[c] + ala_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary conditions - nonessential amino acids', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite ala_L_e at 0x1795972d0>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite ala_L_c at 0x17aef1250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0006}\n",
      "{'_id': 'FUMt_MitoCore', 'name': 'R_FUMt_MitoCore Fumarate transport', 'notes': {'Model Reaction Number': '447', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Fumarate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite fum_e at 0x133596fd0>: -1.0, <Metabolite fum_c at 0x133597890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'SUMt_MitoCore', 'name': 'R_SUMt_MitoCore Succinate transport', 'notes': {'Model Reaction Number': '448', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Succinate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite succ_e at 0x29d7dc7d0>: -1.0, <Metabolite succ_c at 0x29d7b6c90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0817', 'name': 'R_r0817 Citrulline transport', 'notes': {'Model Reaction Number': '449', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Citrulline transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite citr_L_c at 0x2a2fecf50>: -1.0, <Metabolite citr_L_e at 0x17808fa10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NH4t3r', 'name': 'R_NH4t3r ammonia transport via proton antiport', 'notes': {'Model Reaction Number': '450', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'ammonia transport via proton antiport', 'Recon2 id': 'NH4t3r', 'Recon2 formula': 'nh4[c] + h[e] --&gt; nh4[e] + h[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite nh4_c at 0x1639455d0>: -1.0, <Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite nh4_e at 0x29d497f90>: 1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'ACt2r', 'name': 'R_ACt2r acetate reversible transport via proton symport', 'notes': {'Model Reaction Number': '451', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'acetate reversible transport via proton symport', 'Recon2 id': 'ACt2r', 'Recon2 formula': 'h[e] + ac[e] --&gt; h[c] + ac[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite ac_e at 0x169dee450>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite ac_c at 0x179379450>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -0.0006, '_upper_bound': 0.0006}\n",
      "{'_id': 'PPAt', 'name': 'R_PPAt Propanoate transport, diffusion', 'notes': {'Model Reaction Number': '452', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Propanoate transport, diffusion', 'Recon2 id': 'PPAt', 'Recon2 formula': 'ppa[e] -&gt; ppa[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite ppa_e at 0x29e605590>: -1.0, <Metabolite ppa_c at 0x29e678890>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': '2HBt2', 'name': 'R_2HBt2 2-hydroxybutyrate cotransport with proton', 'notes': {'Model Reaction Number': '453', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': '2-hydroxybutyrate cotransport with proton', 'Recon2 id': '2HBt2', 'Recon2 formula': 'h[e] + 2hb[e] --&gt; h[c] + 2hb[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite 2hb_e at 0x164636a90>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite 2hb_c at 0x2a2585d10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'CHOLtu', 'name': 'R_CHOLtu Choline uniport', 'notes': {'Model Reaction Number': '454', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Choline uniport', 'Recon2 id': 'CHOLtu', 'Recon2 formula': 'chol[e] --&gt; chol[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite chol_e at 0x2974fafd0>: -1.0, <Metabolite chol_c at 0x16c8b5010>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r1088', 'name': 'R_r1088 Citrate transport', 'notes': {'Model Reaction Number': '455', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Citrate transport', 'Recon2 id': 'r1088', 'Recon2 formula': 'h[e] + cit[e] --&gt; h[c] + cit[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite cit_e at 0x17802eb90>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0, <Metabolite cit_c at 0x17b24a710>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'ICITt_MitoCore', 'name': 'R_ICITt_MitoCore Isocitrate transport', 'notes': {'Model Reaction Number': '456', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Isocitrate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite icit_e at 0x29c7d1450>: -1.0, <Metabolite icit_c at 0x29c7d0350>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'UREAt', 'name': 'R_UREAt Urea transport via facilitate diffusion', 'notes': {'Model Reaction Number': '457', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Urea transport via facilitate diffusion', 'Recon2 id': 'UREAt', 'Recon2 formula': 'urea[e] --&gt; urea[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite urea_e at 0x29d761490>: -1.0, <Metabolite urea_c at 0x29d763f10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r1512', 'name': 'R_r1512 Bilirubin transport', 'notes': {'Model Reaction Number': '458', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Bilirubin transport', 'Recon2 id': 'r1512', 'Recon2 formula': 'bilirub[e] -&gt; bilirub[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite bilirub_e at 0x297bc4250>: -1.0, <Metabolite bilirub_c at 0x16d951e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'ARGSUCt_MitoCore', 'name': 'R_ARGSUCt_MitoCore Argininosuccinate transport', 'notes': {'Model Reaction Number': '459', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Argininosuccinate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite argsuc_e at 0x2a298b390>: -1.0, <Metabolite argsuc_c at 0x2a05c9090>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'MAL_Lte', 'name': 'R_MAL_Lte Malate transport', 'notes': {'Model Reaction Number': '460', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Malate transport', 'Recon2 id': 'MAL_Lte', 'Recon2 formula': 'mal_L[e] --&gt; mal_L[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite mal_L_e at 0x29d0200d0>: -1.0, <Metabolite mal_L_c at 0x29d020d50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'OAAt_MitoCore', 'name': 'R_OAAt_MitoCore Oxaloacetate transport', 'notes': {'Model Reaction Number': '461', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': 'Oxaloacetate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite oaa_e at 0x29e285f50>: -1.0, <Metabolite oaa_c at 0x29f50d490>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'AKGt_MitoCore', 'name': 'R_AKGt_MitoCore 2-Oxoglutarate transport', 'notes': {'Model Reaction Number': '462', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': '2-Oxoglutarate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite akg_e at 0x16af677d0>: -1.0, <Metabolite akg_c at 0x16ac21410>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'MERCPLACt_MitoCore', 'name': 'R_MERCPLACt_MitoCore 3-Mercaptolactate transport', 'notes': {'Model Reaction Number': '463', 'SUBSYSTEM': 'Boundary Conditions - Biomarkers', 'Description': '3-Mercaptolactate transport', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Biomarkers', '_genes': set(), '_metabolites': {<Metabolite mercplac_e at 0x29d011f10>: -1.0, <Metabolite mercplac_c at 0x176225110>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'r0899', 'name': 'R_r0899 Beta-alanine', 'notes': {'Model Reaction Number': '464', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Beta-alanine', 'Recon2 id': 'r0899', 'Recon2 formula': 'ala_B[c] --&gt; ala_B[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite ala_B_c at 0x178fb9910>: -1.0, <Metabolite ala_B_e at 0x2a2ef1290>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'FE2t', 'name': 'R_FE2t iron (II) transport', 'notes': {'Model Reaction Number': '465', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'iron (II) transport', 'Recon2 id': 'FE2t', 'Recon2 formula': 'fe2[e] -&gt; fe2[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite fe2_e at 0x133816390>: -1.0, <Metabolite fe2_c at 0x133815550>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'H2Ot', 'name': 'R_H2Ot H2O transport via diffusion', 'notes': {'Model Reaction Number': '466', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'H2O transport via diffusion', 'Recon2 id': 'H2Ot', 'Recon2 formula': 'h2o[e] --&gt; h2o[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite h2o_e at 0x29c7872d0>: -1.0, <Metabolite h2o_c at 0x29c7876d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'Hct_MitoCore', 'name': 'R_Hct_MitoCore Proton escape - cytosolic', 'notes': {'Model Reaction Number': '467', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Proton escape - cytosolic', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite h_c at 0x29c7e4b50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'Hmt_MitoCore', 'name': 'R_Hmt_MitoCore Proton escape - mitochondrial', 'notes': {'Model Reaction Number': '468', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Proton escape - mitochondrial', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite h_e at 0x29c786a50>: -1.0, <Metabolite h_m at 0x29c7e4250>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'SO3t_MitoCore', 'name': 'R_SO3t_MitoCore Sulfite', 'notes': {'Model Reaction Number': '469', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Sulfite', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite so3_e at 0x29d7acb50>: -1.0, <Metabolite so3_c at 0x297508cd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TSULt_MitoCore', 'name': 'R_TSULt_MitoCore Thiosulfate', 'notes': {'Model Reaction Number': '470', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Thiosulfate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite tsul_e at 0x29d746c50>: -1.0, <Metabolite tsul_c at 0x29d746e90>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r0940', 'name': 'R_r0940 hydrogen sulfide', 'notes': {'Model Reaction Number': '471', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'hydrogen sulfide', 'Recon2 id': 'r0940', 'Recon2 formula': 'HC00250[c] --&gt; HC00250[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite HC00250_c at 0x29c72efd0>: -1.0, <Metabolite HC00250_e at 0x29c72e290>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'CYANt', 'name': 'R_CYANt Cyanide transport via diffusion (extracellular to cytosol)', 'notes': {'Model Reaction Number': '472', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Cyanide transport via diffusion (extracellular to cytosol)', 'Recon2 id': 'CYANt', 'Recon2 formula': 'cyan[e] --&gt; cyan[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite cyan_e at 0x17b24aa10>: -1.0, <Metabolite cyan_c at 0x29f362dd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'TCYNTt', 'name': 'R_TCYNTt Thiocyanate', 'notes': {'Model Reaction Number': '473', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Thiocyanate', 'Recon2 id': 'TCYNTt', 'Recon2 formula': 'tcynt[c] -&gt; tcynt[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite tcynt_c at 0x29d77d2d0>: -1.0, <Metabolite tcynt_e at 0x29d77ef50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'r1423', 'name': 'R_r1423 Phosphate facilitated diffusion', 'notes': {'Model Reaction Number': '474', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Phosphate facilitated diffusion', 'Recon2 id': 'r1423', 'Recon2 formula': 'pi[c] -&gt; pi[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite pi_c at 0x29e6c47d0>: -1.0, <Metabolite pi_e at 0x29e6c4fd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'FORt_MitoCore', 'name': 'R_FORt_MitoCore Formate', 'notes': {'Model Reaction Number': '475', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Formate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite for_e at 0x13352c710>: -1.0, <Metabolite for_c at 0x13352d2d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 0.0}\n",
      "{'_id': 'FOLt_MitoCore', 'name': 'R_FOLt_MitoCore Folate', 'notes': {'Model Reaction Number': '476', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'Folate', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite fol_e at 0x29d0067d0>: -1.0, <Metabolite fol_c at 0x29d072290>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NADHt_MitoCore', 'name': 'R_NADHt_MitoCore NADH pool', 'notes': {'Model Reaction Number': '477', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'NADH pool', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite nadh_e at 0x2a0b01bd0>: -1.0, <Metabolite nadh_c at 0x29d2efa10>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'NADt_MitoCore', 'name': 'R_NADt_MitoCore NAD+ pool', 'notes': {'Model Reaction Number': '478', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'NAD+ pool', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite nad_e at 0x29d2e94d0>: -1.0, <Metabolite nad_c at 0x29d264bd0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'NADHtm_MitoCore', 'name': 'R_NADHtm_MitoCore NADH pool mitochondrial', 'notes': {'Model Reaction Number': '479', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'NADH pool mitochondrial', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite nadh_e at 0x2a0b01bd0>: -1.0, <Metabolite nadh_m at 0x29d2af310>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'NADtm_MitoCore', 'name': 'R_NADtm_MitoCore NAD+ pool mitochondrial', 'notes': {'Model Reaction Number': '480', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'NAD+ pool mitochondrial', 'Recon2 id': 'N/A', 'Recon2 formula': 'N/A'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite nad_e at 0x29d2e94d0>: -1.0, <Metabolite nad_m at 0x29c7a06d0>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': 0.0, '_upper_bound': 0.0}\n",
      "{'_id': 'COt', 'name': 'R_COt CO transporter via diffusion', 'notes': {'Model Reaction Number': '481', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'CO transporter via diffusion', 'Recon2 id': 'COt', 'Recon2 formula': 'co[c] --&gt; co[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite co_c at 0x16d65ce90>: -1.0, <Metabolite co_e at 0x29f362e50>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'NOt', 'name': 'R_NOt NO transport (diffusion)', 'notes': {'Model Reaction Number': '482', 'SUBSYSTEM': 'Boundary Conditions - Misc', 'Description': 'NO transport (diffusion)', 'Recon2 id': 'NOt', 'Recon2 formula': 'no[e] --&gt; no[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Misc', '_genes': set(), '_metabolites': {<Metabolite no_e at 0x2a0565550>: -1.0, <Metabolite no_c at 0x29d41c910>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PCHOLHSTDe', 'name': 'R_PCHOLHSTDe Phosphatidylcholine', 'notes': {'Model Reaction Number': '483', 'SUBSYSTEM': 'Boundary Conditions - Lipids', 'Description': 'Phosphatidylcholine', 'Recon2 id': 'PCHOLHSTDe', 'Recon2 formula': 'pchol_hs[c] -&gt; pchol_hs[e]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Lipids', '_genes': set(), '_metabolites': {<Metabolite pchol_hs_c at 0x29d213cd0>: -1.0, <Metabolite pchol_hs_e at 0x17b24d790>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PSt3', 'name': 'R_PSt3 Phosphatidylserine', 'notes': {'Model Reaction Number': '484', 'SUBSYSTEM': 'Boundary Conditions - Lipids', 'Description': 'Phosphatidylserine', 'Recon2 id': 'PSt3', 'Recon2 formula': 'ps_hs[e] --&gt; ps_hs[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Lipids', '_genes': set(), '_metabolites': {<Metabolite ps_hs_e at 0x29d28ddd0>: -1.0, <Metabolite ps_hs_c at 0x29d20b390>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n",
      "{'_id': 'PEt', 'name': 'R_PEt Phosphatidylethanolamine', 'notes': {'Model Reaction Number': '485', 'SUBSYSTEM': 'Boundary Conditions - Lipids', 'Description': 'Phosphatidylethanolamine', 'Recon2 id': 'PEt', 'Recon2 formula': 'pe_hs[e] --&gt; pe_hs[c]'}, '_annotation': {}, '_gpr': cobra.core.gene.GPR(''), 'subsystem': 'Boundary Conditions - Lipids', '_genes': set(), '_metabolites': {<Metabolite pe_hs_e at 0x2975456d0>: -1.0, <Metabolite pe_hs_c at 0x29e291750>: 1.0}, '_model': <Model S1SBMLmodel at 0x16d5c3a90>, '_lower_bound': -1000.0, '_upper_bound': 1000.0}\n"
     ]
    }
   ],
   "source": [
    "#visualization_model_notes('Mitocore_0.xml', 'genes')\n",
    "visualization_model_general('Mitocore_Curated_Emanuel.xml', 'reactions')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a9fb8586",
   "metadata": {},
   "outputs": [],
   "source": [
    "visualization_model_annotations('Mitocore_Curated_Emanuel.xml', 'reactions')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "71f472a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def get_model_ids_for_databases(model, component_type, dict, databases):\n",
    "    \"\"\"\n",
    "    Extract ids from model annotations for various databases across a specified component type.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - dict: dictionary where database ids are located ('annotations', 'notes')\n",
    "    - databases: list of str, e.g., ['kegg', 'bigg', 'metanetx', 'uniprot'].\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': []}\n",
    "\n",
    "    counters= {db:0 for db in databases}\n",
    "\n",
    "    for db in databases: \n",
    "        ids_dict[db] = []\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "\n",
    "        for db in databases:\n",
    "            value = None\n",
    "            annotation = getattr(comp, dict, {})\n",
    "\n",
    "            # Use several naming conventions\n",
    "            for key in [db, \n",
    "                        f\"{db}.reaction\", \n",
    "                        f\"{db}.compound\", \n",
    "                        f\"{db}.metabolite\", \n",
    "                        f\"{db}.chemical\", \n",
    "                        f\"{db}.genes\",\n",
    "                        f\"{db}.gene\",\n",
    "                        f\"{db} id\",\n",
    "                        f\"{db.upper()} ID\",\n",
    "                        f\"{db.lower()} ID\"\n",
    "                        ]:\n",
    "                if key in annotation and annotation[key] not in [None, '', 'N/A']:\n",
    "                    counters[db] += 1\n",
    "                    value = annotation[key]\n",
    "                    break\n",
    "\n",
    "\n",
    "            ids_dict[db].append(value)\n",
    "            #pd.DataFrame(ids_dict)\n",
    "    \n",
    "    return counters\n",
    "\n",
    "def get_model_ids_for_attr(model, component_type, attr_name):\n",
    "    \"\"\"\n",
    "    Extract attribute in the model in the target core-component.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: Core component of GEMs, str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - attr_name: name of the dictionary where target object is located ('notes', '_gpr')\n",
    "\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': [], attr_name:[]}\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "        value = comp.__dict__.get(attr_name, None)\n",
    "        ids_dict[attr_name].append(value)\n",
    "\n",
    "    return pd.DataFrame(ids_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "39b9a0e2",
   "metadata": {},
   "source": [
    "# Metabolites"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "4ae9a212",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'KEGG': 436, 'RECON2': 436, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 1st curation {'kegg': 436, 'bigg': 325, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'kegg': 436, 'bigg': 325, 'metanetx': 436, 'Human1': 421}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'metabolites', 'notes', ['KEGG', 'RECON2', 'metanetx', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'metabolites', '_annotation', ['kegg','bigg', 'metanetx','Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'metabolites', '_annotation', ['kegg','bigg', 'metanetx','Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "92809075",
   "metadata": {},
   "source": [
    "# Reactions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "154cb325",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'KEGG': 328, 'Recon2': 432, 'EC Number': 328, 'Human1': 0}\n",
      "Mitocore 1st curation {'kegg': 324, 'bigg': 0, 'ec-code': 322, 'metanetx': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'kegg': 328, 'bigg': 432, 'ec-code': 323, 'metanetx': 335, 'Human1': 395}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'reactions', 'notes', ['KEGG', 'Recon2','EC Number', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'reactions', '_annotation', ['kegg', 'bigg', 'ec-code', 'metanetx', 'Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'reactions', '_annotation', ['kegg', 'bigg', 'ec-code', 'metanetx', 'Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "70aae5d3",
   "metadata": {},
   "source": [
    "# Genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "8dc8ff94",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mitocore original {'Uniprot': 0, 'kegg': 0, 'Human1': 0}\n",
      "Mitocore 1st curation {'uniprot': 387, 'kegg': 0, 'Human1': 0}\n",
      "Mitocore 2nd curation {'uniprot': 595, 'kegg': 587, 'Human1': 595}\n"
     ]
    }
   ],
   "source": [
    "mitocore0_react= get_model_ids_for_databases(mitocore0, 'genes', 'notes', ['Uniprot', 'kegg', 'Human1'])\n",
    "mitocore1_react= get_model_ids_for_databases(mitocore1, 'genes', '_annotation', ['uniprot','kegg', 'Human1'])\n",
    "mitocore2_react= get_model_ids_for_databases(mitocore2, 'genes', '_annotation', ['uniprot', 'kegg', 'Human1'])\n",
    "\n",
    "\n",
    "print('Mitocore original', mitocore0_react)\n",
    "print('Mitocore 1st curation', mitocore1_react)\n",
    "print('Mitocore 2nd curation', mitocore2_react)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "52fd4bad",
   "metadata": {},
   "source": [
    "# --- ec-codes -------"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "43464c7e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mitocore original: 339 ['2.7.1.1 or 2.7.1.2', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '2.7.2.3', '5.4.2.1', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.49', '3.1.1.31', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.4.1, 2.3.1.12, 1.8.1.4', '2.3.3.1', '4.2.1.3', '1.1.1.41', '1.1.1.42', '1.2.4.2, 1.8.1.4, 2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.37', '1.6.5.3, 1.6.5.8, 1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14, 3.6.3.10, 3.6.3.6', '4.1.1.32', '6.4.1.1', '1.1.1.40', '1.1.1.38', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.3', '1.1.1.40', '2.6.1.2', '2.7.4.6', '4.2.1.2', '2.7.4.3', '1.1.1.42', '4.2.1.3', '2.3.3.8', '2.6.1.1', '1.1.1.37', 'N/A', 'N/A', '2.6.1.1', '1.1.1.8', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.6.1.1', 'N/A', '2.3.1.21', '3.6.1.1', '3.1.2.2', '1.3.8.8, 1.5.5.1', '1.3.1.38', '4.2.1.74, 1.1.1.211, 2.3.1.16', '1.3.8.8, 1.5.5.1', '1.3.1.38', '4.2.1.74, 1.1.1.211, 2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7, 1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.1, 1.5.5.1', '4.2.1.17', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.39', '2.3.1.38', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.2', '1.1.1.30', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.1', '4.2.1.17', '1.1.1.35', '2.3.1.16', '2.6.1.42', '1.2.4.4, 2.3.1.168, 1.8.1.4', '1.3.8.1', '4.2.1.17', '3.2.1.4', '1.1.1.31 or 1.1.1.35', '6.2.1.1', '1.2.1.27', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '3.1.2.2', '4.3.1.19', 'N/A', '1.2.4.4 + 2.3.1.168 + 1.8.1.4', '1.1.1.27', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.20', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', 'Non-Enzymatic', '2.6.1.1', '2.6.1.1', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.4.1.1', '1.13.11.11 or 1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.5.1.-', 'n/a', 'n/a', '1.2.4.2 and 2.3.1.61 and 1.8.1.4', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', 'Non-Enzymatic', '1.5.1.1 or 1.5.1.21', '1.5.1.1 or 1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', 'N/A', 'N/A', '1.5.5.2', 'Non-Enzymatic', '1.2.1.88', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '2.6.1.13', '4.1.1.17', '1.4.3.22', '1.2.1.3', '1.2.1.3', '4.1.1.19', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', 'N/A', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.3', '1.4.1.3', '1.4.1.3', '6.3.1.2', '3.5.1.2', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.3', '1.5.1.5', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17 or 4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.3', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', 'Non-enzymatic', '1.4.3.4', '4.4.1.5', '3.1.2.6', '1.1.1.28', '6.3.4.16', '2.1.3.3', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.2', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1 or 2.6.1.5', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '2.6.1.19', '1.2.1.18', '1.2.1.18', '4.1.1.11', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', 'N/A', '1.2.1.3', '1.2.1.3', '6.2.1.1', '6.2.1.1', '4.3.2.2', '6.3.4.4', '3.5.4.6', '3.5.3.1', '4.3.2.1', '6.3.4.5', '3.5.3.1', '2.3.1.37', 'N/A', '4.2.1.24', '2.5.1.61', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', 'N/A', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', 'N/A', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.8.1.7']\n",
      "mitocore 1 curation: 358 ['2.7.1.1', '2.7.1.2', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '2.7.2.3', '5.4.2.1', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.49', '3.1.1.31', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.4.1', '2.3.1.12', '1.8.1.4', '2.3.3.1', '4.2.1.3', '1.1.1.41', '1.1.1.42', '1.2.4.2', '1.8.1.4', '2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.37', '1.6.5.3', '1.6.5.8', '1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14', '3.6.3.10', '3.6.3.6', '4.1.1.32', '6.4.1.1', '1.1.1.40', '1.1.1.38', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.3', '1.1.1.40', '2.6.1.2', '2.7.4.6', '4.2.1.2', '2.7.4.3', '1.1.1.42', '4.2.1.3', '2.3.3.8', '2.6.1.1', '1.1.1.37', '2.6.1.1', '1.1.1.8', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.6.1.1', '2.3.1.21', '3.6.1.1', '3.1.2.2', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.1', '1.5.5.1', '4.2.1.17', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.39', '2.3.1.38', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.2', '1.1.1.30', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.1', '4.2.1.17', '1.1.1.35', '2.3.1.16', '2.6.1.42', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.3.8.1', '4.2.1.17', '3.2.1.4', '1.1.1.31', '1.1.1.35', '6.2.1.1', '1.2.1.27', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '3.1.2.2', '4.3.1.19', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.1.1.27', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.20', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', '2.6.1.1', '2.6.1.1', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.4.1.1', '1.13.11.11', '1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.5.1.-', '1.2.4.2', '2.3.1.61', '1.8.1.4', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', '1.5.1.1', '1.5.1.21', '1.5.1.1', '1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', '1.5.5.2', '1.2.1.88', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '2.6.1.13', '4.1.1.17', '1.4.3.22', '1.2.1.3', '1.2.1.3', '4.1.1.19', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.3', '1.4.1.3', '1.4.1.3', '6.3.1.2', '3.5.1.2', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.3', '1.5.1.5', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17', '4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.3', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', '1.4.3.4', '4.4.1.5', '3.1.2.6', '1.1.1.28', '6.3.4.16', '2.1.3.3', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.2', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1', '2.6.1.5', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '2.6.1.19', '1.2.1.18', '1.2.1.18', '4.1.1.11', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', '1.2.1.3', '1.2.1.3', '6.2.1.1', '6.2.1.1', '4.3.2.2', '6.3.4.4', '3.5.4.6', '3.5.3.1', '4.3.2.1', '6.3.4.5', '3.5.3.1', '2.3.1.37', '4.2.1.24', '2.5.1.61', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.8.1.7']\n",
      "mitocore curation 2: 582 ['2.7.1.1', '2.7.1.2', '2.7.1.4', '2.7.1.55', '2.7.1.59', '2.7.1.60', '3.1.3.1', '3.1.3.104', '3.1.3.2', '3.1.3.23', '3.1.3.25', '3.1.3.29', '3.1.3.58', '3.1.3.9', '5.3.1.9', '2.7.1.11', '3.1.3.11', '4.1.2.13', '5.3.1.1', '1.2.1.12', '1.2.1.59', '2.7.2.10', '2.7.2.3', '5.4.2.1', '5.4.2.4', '4.2.1.11', '2.7.1.40', '2.7.1.40', '4.1.1.32', '1.1.1.27', '1.1.1.28', '1.1.1.363', '1.1.1.47', '1.1.1.49', '3.1.1.31', '1.1.1.351', '1.1.1.44', '5.3.1.6', '5.1.3.1', '2.2.1.1', '2.2.1.2', '2.2.1.1', '1.2.1.104', '1.2.1.51', '1.2.4.1', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.12', '2.3.1.168', '2.3.1.61', '2.3.3.1', '2.3.3.14', '2.3.3.16', '2.3.3.3', '4.1.3.7', '4.2.1.3', '1.1.1.286', '1.1.1.41', '1.1.1.42', '1.1.1.87', '1.1.1.42', '1.2.1.105', '1.2.1.52', '1.2.4.2', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.3.1.61', '6.2.1.4', '6.2.1.5', '4.2.1.2', '1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82', '1.6.5.3', '1.6.5.8', '1.6.5.9', '1.3.5.1', '1.10.2.2', '1.9.3.1', '3.6.3.14', '7.1.2.2', '4.1.1.32', '4.1.1.49', '6.4.1.1', '1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42', '1.1.1.38', '1.1.1.39', '1.1.1.40', '2.6.1.2', '2.3.3.8', '2.7.4.6', '1.6.1.2', '2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8', '1.1.1.38', '1.1.1.39', '1.1.1.40', '1.1.1.42', '2.6.1.1', '2.6.1.2', '2.6.1.5', '2.7.4.6', '4.2.1.2', '2.7.4.10', '2.7.4.3', '2.7.4.4', '2.7.4.8', '1.1.1.42', '4.2.1.3', '2.3.3.8', '4.1.3.6', '2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57', '1.1.1.27', '1.1.1.299', '1.1.1.37', '1.1.1.82', '2.6.1.1', '2.6.1.19', '2.6.1.42', '2.6.1.5', '2.6.1.57', '1.1.1.8', '1.1.1.94', '1.1.5.3', '1.1.5.3', '6.2.1.3', '2.3.1.21', '3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1', '2.3.1.21', '3.1.3.1', '3.1.3.2', '3.6.1.1', '3.6.2.1', '3.1.2.2', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.8', '1.5.5.1', '1.3.1.38', '4.2.1.74', '1.1.1.211', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.17', '1.1.1.35', '2.3.1.16', '1.3.8.7', '1.5.5.1', '1.3.1.10', '1.3.1.104', '1.3.1.38', '1.3.1.8', '4.2.1.17', '4.2.1.74', '1.1.1.211', '1.1.1.35', '2.3.1.16', '2.3.1.9', '1.3.8.7', '1.5.5.1', '1.3.1.38', '4.2.1.150', '4.2.1.17', '4.2.1.74', '1.1.1.211', '1.1.1.35', '2.3.1.16', '2.3.1.9', '1.3.8.1', '1.5.5.1', '4.2.1.116', '4.2.1.150', '4.2.1.17', '4.2.1.55', '1.1.1.178', '1.1.1.211', '1.1.1.35', '2.3.1.9', '6.4.1.2', '2.3.1.0', '2.3.1.39', '2.3.1.85', '2.3.1.86', '2.3.1.179', '2.3.1.180', '2.3.1.38', '2.3.1.41', '2.3.1.85', '2.3.1.86', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '2.3.1.85', '3.1.2.14', '6.2.1.1', '6.2.1.17', '6.2.1.2', '6.2.1.3', '1.1.1.30', '2.8.3.18', '2.8.3.5', '2.3.3.10', '4.1.3.4', '2.6.1.42', '2.6.1.6', '2.6.1.67', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '1.3.8.4', '6.4.1.4', '4.2.1.18', '2.6.1.42', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '1.3.8.1', '4.2.1.150', '4.2.1.17', '1.1.1.178', '1.1.1.35', '2.3.1.16', '2.3.1.9', '2.6.1.42', '1.2.1.25', '1.2.4.4', '1.8.1.4', '2.3.1.168', '2.7.11.4', '1.3.8.1', '4.2.1.150', '4.2.1.17', '3.1.2.4', '1.1.1.31', '6.2.1.1', '6.2.1.17', '1.2.1.27', '6.4.1.2', '6.4.1.3', '5.1.99.1', '5.4.99.2', '4.1.1.41', '4.1.1.9', '4.1.1.94', '7.2.4.3', '2.3.1.226', '2.3.1.30', '2.3.3.13', '3.1.2.17', '3.1.2.18', '3.1.2.2', '3.1.2.20', '3.1.2.23', '4.3.1.17', '4.3.1.19', '1.2.4.4', '2.3.1.168', '1.8.1.4', '1.1.1.27', '1.1.1.272', '1.1.1.28', '2.5.1.6', '2.1.1.-', '3.3.1.1', '2.7.1.11', '2.7.1.20', '2.7.1.74', '4.2.1.22', '4.4.1.1', '1.13.11.20', '2.6.1.1', '2.6.1.1', '2.6.1.3', '2.6.1.1', '2.6.1.3', '2.8.1.2', '2.8.1.2', '2.8.1.2', '2.8.1.2', '1.1.1.27', '4.1.99.1', '4.4.1.1', '4.4.1.11', '4.4.1.13', '4.4.1.28', '4.4.1.3', '4.4.1.8', '1.13.11.11', '1.13.11.52', '3.5.1.9', '1.14.13.9', '3.7.1.3', '1.13.11.6', '4.1.1.45', '1.2.1.32', '1.5.1.-', '1.2.1.105', '1.2.1.52', '1.2.4.2', '1.8.1.4', '2.3.1.61', '1.3.8.6', '1.5.1.8', '1.5.1.9', '1.2.1.31', '1.2.1.31', '2.6.1.39', '1.4.3.14', '1.4.3.2', '1.5.1.1', '1.5.1.21', '1.5.1.49', '1.5.1.1', '1.5.1.21', '1.5.3.7', '1.4.1.18', '1.2.1.31', '1.5.99.8', '1.2.1.24', '1.2.1.5', '1.2.1.88', '1.5.1.12', '2.7.2.11', '1.2.1.41', '1.5.1.2', '1.5.1.2', '1.5.99.8', '2.6.1.11', '2.6.1.13', '4.1.1.17', '4.1.1.19', '4.1.1.86', '1.4.3.10', '1.4.3.21', '1.4.3.22', '1.4.3.6', '1.2.1.3', '1.2.1.19', '1.2.1.3', '4.1.1.19', '3.5.3.1', '3.5.3.11', '2.3.1.57', '1.4.3.4', '1.2.1.3', '3.5.1.63', '4.1.1.15', '2.6.1.19', '1.2.1.24', '1.4.1.3', '1.4.1.13', '1.4.1.3', '1.4.1.4', '1.4.1.14', '1.4.1.2', '1.4.1.3', '1.4.1.4', '1.4.1.13', '1.4.1.3', '1.4.1.4', '6.3.1.2', '1.4.1.13', '1.4.1.14', '1.4.7.1', '2.4.2.14', '3.5.1.2', '3.5.1.38', '4.3.3.6', '6.3.4.2', '6.3.5.2', '6.3.5.4', '6.3.5.5', '6.3.5.7', '3.5.1.2', '1.1.1.95', '2.6.1.52', '3.1.3.2', '3.1.3.29', '3.1.3.3', '2.1.2.1', '1.5.1.3', '1.5.1.33', '1.5.1.3', '1.5.1.5', '1.5.1.15', '1.5.1.5', '1.5.1.15', '3.5.4.9', '4.3.1.4', '6.3.4.3', '1.5.1.6', '4.3.1.17', '4.3.1.19', '2.1.2.1', '1.4.4.2', '2.1.2.10', '1.8.1.4', '1.5.1.3', '1.5.1.33', '1.5.1.3', '1.5.1.15', '1.5.1.5', '1.5.1.15', '1.5.1.5', '1.5.1.15', '3.5.4.9', '6.3.4.3', '1.5.1.6', '2.3.1.29', '2.3.1.37', '1.1.1.381', '1.4.3.21', '1.4.3.4', '1.4.3.6', '4.4.1.5', '3.1.2.12', '3.1.2.6', '1.1.1.27', '1.1.1.28', '6.3.4.16', '6.3.5.5', '2.1.3.3', '2.1.3.6', '1.14.13.39', '1.14.13.39', '3.4.19.13', '3.4.11.1', '3.4.11.2', '3.4.11.23', '3.4.13.18', '6.3.2.2', '6.3.2.3', '1.14.16.1', '1.5.1.34', '2.6.1.1', '2.6.1.27', '2.6.1.42', '2.6.1.5', '2.6.1.57', '2.6.1.9', '1.4.3.2', '1.13.11.27', '1.13.11.5', '5.2.1.2', '3.7.1.2', '6.3.5.4', '3.5.1.1', '4.3.1.3', '4.2.1.49', '3.5.2.7', '2.1.2.5', '4.3.1.4', '2.6.1.19', '2.6.1.55', '1.2.1.18', '1.2.1.27', '1.2.1.18', '4.1.1.11', '4.1.1.15', '2.7.3.2', '2.7.3.2', '3.1.2.1', '1.1.1.2', '1.1.1.1', '1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88', '1.2.1.19', '1.2.1.24', '1.2.1.3', '1.2.1.31', '1.2.1.36', '1.2.1.48', '1.2.1.5', '1.2.1.69', '1.2.1.88', '6.2.1.1', '6.2.1.17', '6.2.1.1', '6.2.1.17', '4.3.2.2', '6.3.4.4', '3.5.4.17', '3.5.4.6', '3.5.3.1', '3.5.3.7', '4.3.2.1', '6.3.4.5', '3.5.3.1', '3.5.3.7', '2.3.1.37', '4.2.1.24', '2.5.1.61', '4.3.1.8', '4.2.1.75', '4.1.1.37', '1.3.3.3', '1.3.3.4', '4.99.1.1', '1.14.14.18', '1.14.99.3', '1.3.1.24', '1.3.1.24', '3.1.4.4', '2.7.1.30', '2.3.1.15', '2.3.1.51', '2.7.7.41', '2.7.8.5', '3.1.3.27', '2.7.8.41', '2.7.8.-', '2.7.4.14', '2.7.4.25', '2.7.4.6', '1.15.1.1', '1.11.1.9', '1.11.1.9', '1.8.1.7', '1.6.4.2', '1.8.1.7', '1.8.4.2', '2.7.4.6']\n"
     ]
    }
   ],
   "source": [
    "curation_0_ec = []\n",
    "\n",
    "for reaction in mitocore0.reactions:\n",
    "    annotations = getattr(reaction, 'notes', {})\n",
    "    ec_data = annotations.get('EC Number', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "\n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_0_ec.append(ec_code)\n",
    "\n",
    "print('mitocore original:', len(curation_0_ec), curation_0_ec)\n",
    "\n",
    "curation_1_ec = []\n",
    "\n",
    "for reaction in mitocore1.reactions:\n",
    "    annotations = getattr(reaction, '_annotation', {})\n",
    "    ec_data = annotations.get('ec-code', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "\n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_1_ec.append(ec_code)\n",
    "\n",
    "print('mitocore 1 curation:', len(curation_1_ec), curation_1_ec)\n",
    "\n",
    "\n",
    "curation_2_ec=[]\n",
    "\n",
    "for reaction in mitocore2.reactions:\n",
    "    annotations = getattr(reaction, '_annotation', {})\n",
    "    ec_data = annotations.get('ec-code', [])\n",
    "\n",
    "    # Normalize: ensure ec_data is always a list\n",
    "    if isinstance(ec_data, str):\n",
    "        ec_data = [ec_data]\n",
    "        \n",
    "    # iterate over str (ec-code) in a list made of str (ec_data)\n",
    "    for ec_code in ec_data:\n",
    "        if isinstance(ec_code, str):\n",
    "            curation_2_ec.append(ec_code)\n",
    "\n",
    "print('mitocore curation 2:', len(curation_2_ec), curation_2_ec)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "83b70828",
   "metadata": {},
   "source": [
    "# ---- GPR rules ----\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "2e01d1af",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_model_ids_for_attr(model, component_type, attr_name):\n",
    "    \"\"\"\n",
    "    Extract attribute in the model in the target core-component.\n",
    "    \n",
    "    Parameters:\n",
    "    - model: COBRApy model object (mitocore)\n",
    "    - component_type: Core component of GEMs, str, one of 'reactions', 'metabolites', 'genes'\n",
    "    - attr_name: name of the dictionary where target object is located ('notes', '_gpr')\n",
    "\n",
    "    \n",
    "    Returns:\n",
    "    - DataFrame with component ID and extracted database ids.\n",
    "    \"\"\"\n",
    "    # Initialize dictionary\n",
    "    ids_dict = {'model_id': [], attr_name:[]}\n",
    "\n",
    "    # Access model component (e.g., reactions, genes)\n",
    "    components = getattr(model, component_type)\n",
    "\n",
    "    for comp in components:\n",
    "        ids_dict['model_id'].append(comp.id)\n",
    "        value = comp.__dict__.get(attr_name, None)\n",
    "        ids_dict[attr_name].append(value)\n",
    "\n",
    "    return pd.DataFrame(ids_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "b440ba4d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "GPR Mitocore 0:     model_id                                 _gpr\n",
      "150    r0638                      ENSG00000116353\n",
      "151    r0660                      ENSG00000127884\n",
      "152    r0722                      ENSG00000138796\n",
      "153    r0724                      ENSG00000167315\n",
      "154    r1451  ENSG00000117054 and ENSG00000171503\n",
      "GPR Mitocore 1:     model_id                                 _gpr\n",
      "150    r0638                      ENSG00000116353\n",
      "151    r0660                      ENSG00000127884\n",
      "152    r0722                      ENSG00000138796\n",
      "153    r0724                      ENSG00000167315\n",
      "154    r1451  ENSG00000117054 and ENSG00000171503\n",
      "GPR Mitocore 2:     model_id                                               _gpr\n",
      "150    r0638                                    ENSG00000116353\n",
      "151    r0660  ENSG00000127884 or (ENSG00000084754 and ENSG00...\n",
      "152    r0722  ENSG00000138796 or (ENSG00000060971 and ENSG00...\n",
      "153    r0724  ENSG00000167315 or (ENSG00000060971 and ENSG00...\n",
      "154    r1451  (ENSG00000117054 and ENSG00000171503) or ENSG0...\n"
     ]
    }
   ],
   "source": [
    "gpr_mit_0=get_model_ids_for_attr(mitocore0, 'reactions', '_gpr')\n",
    "gpr_mit_1=get_model_ids_for_attr(mitocore1, 'reactions', '_gpr')\n",
    "gpr_mit_2= get_model_ids_for_attr(mitocore2, 'reactions', '_gpr')\n",
    "\n",
    "print('GPR Mitocore 0:', gpr_mit_0[150:155])\n",
    "print('GPR Mitocore 1:', gpr_mit_1[150:155])\n",
    "print('GPR Mitocore 2:', gpr_mit_2[150:155])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4157a6c7",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Split GPR on 'or' \n",
    "def gpr_counte(df, model):\n",
    "\n",
    "    gpr_absolute= []\n",
    "\n",
    "    for index, row in df.iterrows():\n",
    "        gpr = row['_gpr']\n",
    "        elements_in_gpr = str(gpr).split(' or ')\n",
    "\n",
    "        for element in elements_in_gpr:\n",
    "            if element not in [None, 'N/A','']:\n",
    "                gpr_absolute.append(element)\n",
    "    print(gpr_absolute)\n",
    "    return print(f\" {model}, has {len(gpr_absolute)} GPRs\" )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "8ffbded1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000102144', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000067225', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000160211', 'ENSG00000130313', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000163931', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000166411 and ENSG00000101365 and ENSG00000067829', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000163541 and ENSG00000172340', 'ENSG00000136143', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', 'ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295', 'ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581', 'ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and (ENSG00000159199', 'ENSG00000135390', 'ENSG00000154518) and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253', 'ENSG00000100889', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000082212', 'ENSG00000166123', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000065833', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000014641', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000205560', 'ENSG00000180817', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000138777', 'ENSG00000119673', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000122971 and ENSG00000171503', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000075239', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000161267', 'ENSG00000083720', 'ENSG00000134240', 'ENSG00000117305', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000128928', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000111058', 'ENSG00000119711', 'ENSG00000175198 and ENSG00000114054', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'Unknown', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000168906', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000120053', 'Non-Enzymatic', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000111716', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'Unknown', 'Unknown', 'Unknown', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'Unknown', 'Non-Enzymatic', 'Unknown', 'Unknown', 'ENSG00000179761', 'Unknown', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'Non-Enzymatic', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000183010', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000069535', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'Unknown', 'ENSG00000136750', 'Unknown', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000148672', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000228716', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000178445', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000178700', 'ENSG00000163738', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000120254', 'ENSG00000136010', 'ENSG00000100116', 'Non-enzymatic', 'ENSG00000069535', 'ENSG00000124767', 'ENSG00000063854', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000007171', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000166825', 'ENSG00000001084', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000172497', 'ENSG00000117448', 'ENSG00000248144 and ENSG00000197894', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000154930', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000023330', 'Unknown', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'Unknown', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000105223', 'ENSG00000198814', 'ENSG00000186281', 'ENSG00000155189', 'ENSG00000144559', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000103202', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000176153', 'ENSG00000233276', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000147454', 'ENSG00000155287', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', 'Unknown', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)', '(ENSG00000124406', 'ENSG00000143515', 'ENSG00000081923', 'ENSG00000104043', 'ENSG00000054793', 'ENSG00000166377', 'ENSG00000206190', 'ENSG00000145246', 'ENSG00000068650', 'ENSG00000058063', 'ENSG00000101974) and (ENSG00000112697', 'ENSG00000182107)']\n",
      " Mitocore0, has 489 GPRs\n",
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000102144', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000067225', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000160211', 'ENSG00000130313', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000163931', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000166411 and ENSG00000101365 and ENSG00000067829', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000163541 and ENSG00000172340', 'ENSG00000136143', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', 'ENSG00000198727 and ENSG00000179091 and ENSG00000172115 and ENSG00000169021 and ENSG00000156467 and ENSG00000173660 and ENSG00000140740 and ENSG00000010256 and ENSG00000127540 and ENSG00000184076 and ENSG00000011295', 'ENSG00000198804 and ENSG00000198712 and ENSG00000198938 and ENSG00000131143 and ENSG00000131055 and ENSG00000178741 and ENSG00000135940 and ENSG00000111775 and ENSG00000156885 and ENSG00000126267 and ENSG00000160471 and ENSG00000164919 and ENSG00000161281 and ENSG00000112695 and ENSG00000115944 and ENSG00000131174 and ENSG00000127184 and ENSG00000115944 and ENSG00000176340 and ENSG00000187581', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000159199)', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000135390)', '(ENSG00000152234 and ENSG00000110955 and ENSG00000165629 and ENSG00000099624 and ENSG00000124172 and ENSG00000116459 and ENSG00000167863 and ENSG00000169020 and ENSG00000154723 and ENSG00000241468 and ENSG00000167283 and ENSG00000249222 and ENSG00000241837 and ENSG00000198899 and ENSG00000228253 and ENSG00000154518)', 'ENSG00000100889', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000082212', 'ENSG00000166123', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000065833', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000014641', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000205560', 'ENSG00000180817', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000138777', 'ENSG00000119673', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000115361 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000117054 and ENSG00000171503', 'ENSG00000116353', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000122971 and ENSG00000171503', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000075239', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000161267', 'ENSG00000083720', 'ENSG00000134240', 'ENSG00000117305', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000128928', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000167315', 'ENSG00000105552', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000111058', 'ENSG00000119711', 'ENSG00000175198 and ENSG00000114054', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000168906', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000120053', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000128309', 'ENSG00000111716', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'ENSG00000179761', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000183010', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000069535', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000136750', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000148672', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000228716', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000178445', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000178700', 'ENSG00000163738', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000120254', 'ENSG00000136010', 'ENSG00000100116', 'ENSG00000069535', 'ENSG00000124767', 'ENSG00000063854', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000007171', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000166825', 'ENSG00000001084', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000172497', 'ENSG00000117448', 'ENSG00000248144 and ENSG00000197894', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000154930', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000023330', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000105223', 'ENSG00000198814', 'ENSG00000186281', 'ENSG00000155189', 'ENSG00000144559', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000103202', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000176153', 'ENSG00000233276', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000102743', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000197119', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000060762 and ENSG00000143158', 'ENSG00000147454', 'ENSG00000155287', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', '(ENSG00000124406 and ENSG00000112697)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000143515 and ENSG00000112697)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000166377 and ENSG00000112697)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000206190 and ENSG00000112697)', '(ENSG00000206190 and ENSG00000182107)', '(ENSG00000145246 and ENSG00000112697)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)']\n",
      " Mitocore1, has 479 GPRs\n",
      "['ENSG00000156515', 'ENSG00000159399', 'ENSG00000160883', 'ENSG00000106633', 'ENSG00000156510', 'ENSG00000131482', 'ENSG00000105220', 'ENSG00000152556', 'ENSG00000067057', 'ENSG00000141959', 'ENSG00000165140', 'ENSG00000130957', 'ENSG00000149925', 'ENSG00000109107', 'ENSG00000136872', 'ENSG00000111669', 'ENSG00000111640', 'ENSG00000105679', 'ENSG00000102144', 'ENSG00000096006', 'ENSG00000170950', 'ENSG00000171314', 'ENSG00000164708', 'ENSG00000172331', 'ENSG00000226784', 'ENSG00000074800', 'ENSG00000111674', 'ENSG00000108515', 'ENSG00000067225', 'ENSG00000143627', 'ENSG00000067225', 'ENSG00000143627', 'ENSG00000124253', 'ENSG00000111716', 'ENSG00000134333', 'ENSG00000160211', 'ENSG00000049239', 'ENSG00000130313', 'ENSG00000130313', 'ENSG00000049239', 'ENSG00000142657', 'ENSG00000153574', 'ENSG00000235376', 'ENSG00000197713', 'ENSG00000163931', 'ENSG00000007350', 'ENSG00000151005', 'ENSG00000177156', 'ENSG00000163931', 'ENSG00000007350', 'ENSG00000151005', 'ENSG00000131828 and ENSG00000168291 and ENSG00000150768 and ENSG00000091140', 'ENSG00000062485', 'ENSG00000100412', 'ENSG00000067829 and ENSG00000101365 and ENSG00000166411', 'ENSG00000182054', 'ENSG00000105953 and ENSG00000119689 and ENSG00000091140', '(ENSG00000163541 and ENSG00000172340)', '(ENSG00000136143 and ENSG00000163541)', 'ENSG00000136143', '(ENSG00000163541 and ENSG00000172340)', '(ENSG00000136143 and ENSG00000163541)', 'ENSG00000091483', 'ENSG00000146701', 'ENSG00000014641', 'ENSG00000138400', 'ENSG00000115286 and ENSG00000110717 and ENSG00000178127 and ENSG00000213619 and ENSG00000158864 and ENSG00000167792 and ENSG00000023228 and ENSG00000198888 and ENSG00000198763 and ENSG00000198840 and ENSG00000198886 and ENSG00000212907 and ENSG00000198786 and ENSG00000198695 and ENSG00000145494 and ENSG00000184752 and ENSG00000164258 and ENSG00000139180 and ENSG00000004779 and ENSG00000131495 and ENSG00000119013 and ENSG00000128609 and ENSG00000184983 and ENSG00000174886 and ENSG00000147123 and ENSG00000168653 and ENSG00000065518 and ENSG00000186010 and ENSG00000099795 and ENSG00000119421 and ENSG00000147684 and ENSG00000140990 and ENSG00000166136 and ENSG00000151366 and ENSG00000090266 and ENSG00000267855 and ENSG00000170906 and ENSG00000189043 and ENSG00000136521 and ENSG00000183648 and ENSG00000109390 and ENSG00000130414 and ENSG00000185633 and ENSG00000160194 and ENSG00000165264', 'ENSG00000073578 and ENSG00000117118 and ENSG00000143252', '(ENSG00000010256 and ENSG00000011295 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000169021 and ENSG00000172115 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727)', '(ENSG00000010256 and ENSG00000127540 and ENSG00000140740 and ENSG00000156467 and ENSG00000164405 and ENSG00000169021 and ENSG00000173660 and ENSG00000179091 and ENSG00000184076 and ENSG00000198727)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154723 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', '(ENSG00000099624 and ENSG00000110955 and ENSG00000116459 and ENSG00000124172 and ENSG00000135390 and ENSG00000152234 and ENSG00000154518 and ENSG00000154723 and ENSG00000156411 and ENSG00000159199 and ENSG00000165629 and ENSG00000167283 and ENSG00000167863 and ENSG00000169020 and ENSG00000173915 and ENSG00000180389 and ENSG00000198899 and ENSG00000228253 and ENSG00000241468 and ENSG00000241837 and ENSG00000249222)', 'ENSG00000100889', 'ENSG00000124253', 'ENSG00000173599', 'ENSG00000151376', 'ENSG00000013392', 'ENSG00000065833', 'ENSG00000082212', 'ENSG00000151376', 'ENSG00000166123', 'ENSG00000167701', 'ENSG00000131473', 'ENSG00000103202', 'ENSG00000103024', 'ENSG00000143156', 'ENSG00000172113', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000112992', 'ENSG00000004455', 'ENSG00000147853', 'ENSG00000085231', 'ENSG00000106992', 'ENSG00000140057', 'ENSG00000154027', 'ENSG00000155085', 'ENSG00000162433', 'ENSG00000165695', 'ENSG00000273841', 'ENSG00000065833', 'ENSG00000013392', 'ENSG00000167701', 'ENSG00000011052', 'ENSG00000103024', 'ENSG00000172113', 'ENSG00000143156', 'ENSG00000162368', 'ENSG00000106992', 'ENSG00000162433', 'ENSG00000154027', 'ENSG00000140057', 'ENSG00000165695', 'ENSG00000155085', 'ENSG00000103202', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000091483', 'ENSG00000106992', 'ENSG00000004455', 'ENSG00000085231', 'ENSG00000140057', 'ENSG00000147853', 'ENSG00000154027', 'ENSG00000155085', 'ENSG00000162433', 'ENSG00000165695', 'ENSG00000273841', 'ENSG00000138413', 'ENSG00000122729', 'ENSG00000131473', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000014641', 'ENSG00000138400', 'ENSG00000108528', 'ENSG00000115840', 'ENSG00000004864', 'ENSG00000125166', 'ENSG00000167588', 'ENSG00000152642', 'ENSG00000115159', 'ENSG00000151726', 'ENSG00000123983', 'ENSG00000068366', 'ENSG00000103740', 'ENSG00000164398', 'ENSG00000197142', 'ENSG00000130377', 'ENSG00000140284', 'ENSG00000205560', 'ENSG00000110090', 'ENSG00000169169', 'ENSG00000180817', 'ENSG00000107902', 'ENSG00000138777', 'ENSG00000143363', 'ENSG00000178537', 'ENSG00000157184', 'ENSG00000110090', 'ENSG00000169169', 'ENSG00000205560', 'ENSG00000138777', 'ENSG00000107902', 'ENSG00000143363', 'ENSG00000180817', 'ENSG00000119673', 'ENSG00000097021', 'ENSG00000131238', 'ENSG00000136881', 'ENSG00000177465', 'ENSG00000184227', 'ENSG00000205669', 'ENSG00000221988', 'ENSG00000241404', '(ENSG00000115361 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', '(ENSG00000115361 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000087008', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000161533', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000084754 and ENSG00000138029', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000072778', 'ENSG00000115361', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000117054', 'ENSG00000177646', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000117054 and ENSG00000171503)', 'ENSG00000087008', 'ENSG00000161533', 'ENSG00000196177', 'ENSG00000116353', 'ENSG00000127884', '(ENSG00000084754 and ENSG00000113790 and ENSG00000167969 and ENSG00000198721)', 'ENSG00000138796', '(ENSG00000060971 and ENSG00000113790 and ENSG00000133835)', 'ENSG00000167315', '(ENSG00000060971 and ENSG00000167306)', '(ENSG00000122971 and ENSG00000171503)', 'ENSG00000117054', 'ENSG00000122971', 'ENSG00000151498', 'ENSG00000177646', 'ENSG00000196177', 'ENSG00000240303', 'ENSG00000127884', 'ENSG00000138796', '(ENSG00000072506 and ENSG00000084754 and ENSG00000138796)', 'ENSG00000075239', 'ENSG00000120437', 'ENSG00000278540', 'ENSG00000076555', 'ENSG00000169710', 'ENSG00000100294', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000115425', 'ENSG00000116353', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000151093', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000169710', 'ENSG00000152463', 'ENSG00000005187', 'ENSG00000173124', 'ENSG00000066813', 'ENSG00000166743', 'ENSG00000183549', 'ENSG00000183747', 'ENSG00000215009', 'ENSG00000161267', 'ENSG00000164039', 'ENSG00000083720', 'ENSG00000198754', 'ENSG00000134240', 'ENSG00000112972', 'ENSG00000117305', 'ENSG00000146151', 'ENSG00000105552', 'ENSG00000060982', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', '(ENSG00000083123 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000128928', 'ENSG00000196177', 'ENSG00000078070 and ENSG00000131844', 'ENSG00000148090', 'ENSG00000105552', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', '(ENSG00000083123 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000122971', 'ENSG00000196177', 'ENSG00000127884', 'ENSG00000138796', 'ENSG00000072506', 'ENSG00000167315', '(ENSG00000084754 and ENSG00000138029 and ENSG00000167315)', 'ENSG00000105552', 'ENSG00000060982', '(ENSG00000083123 and ENSG00000091140 and ENSG00000137992 and ENSG00000248098)', 'ENSG00000137992', 'ENSG00000122971', 'ENSG00000127884', 'ENSG00000198130', 'ENSG00000106049', 'ENSG00000138796', 'ENSG00000072506', 'ENSG00000111058', 'ENSG00000131069', 'ENSG00000154930', 'ENSG00000119711', 'ENSG00000114054 and ENSG00000175198', 'ENSG00000124370', 'ENSG00000146085', 'ENSG00000093144', 'ENSG00000103150', 'ENSG00000119673', 'ENSG00000135094', 'ENSG00000139410', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000248098 and ENSG00000083123 and ENSG00000137992 and ENSG00000091140', 'ENSG00000111716', 'ENSG00000166796', 'ENSG00000166800', 'ENSG00000171989', 'ENSG00000168906', 'ENSG00000038274', 'ENSG00000151224', 'ENSG00000101444', 'ENSG00000168710', 'ENSG00000158467', 'ENSG00000156110', 'ENSG00000156136', 'ENSG00000160200', 'ENSG00000116761', 'ENSG00000129596', 'ENSG00000116761', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000139133', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000125166', 'ENSG00000120053', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000128309', 'ENSG00000128311', 'ENSG00000111716', 'ENSG00000166796', 'ENSG00000166800', 'ENSG00000171989', 'ENSG00000116761', 'ENSG00000151790', 'ENSG00000131203', 'ENSG00000188676', 'ENSG00000183077', 'ENSG00000117009', 'ENSG00000115919', 'ENSG00000162882', 'ENSG00000153086', 'ENSG00000183032', 'ENSG00000183032', '(ENSG00000091140 and ENSG00000105953 and ENSG00000119689)', '(ENSG00000105953 and ENSG00000119689)', '(ENSG00000119689 and ENSG00000181192)', 'ENSG00000105607', 'ENSG00000008311', 'ENSG00000008311', 'ENSG00000143653', 'ENSG00000164904', 'ENSG00000164904', 'ENSG00000109576', 'ENSG00000179761', 'ENSG00000164904', 'ENSG00000183032', 'ENSG00000183032', 'ENSG00000100033', 'ENSG00000250799', 'ENSG00000159423', 'ENSG00000159423', 'ENSG00000059573', 'ENSG00000059573', 'ENSG00000183010', 'ENSG00000104524', 'ENSG00000143811', 'ENSG00000183010', 'ENSG00000104524', 'ENSG00000143811', 'ENSG00000065154', 'ENSG00000115758', 'ENSG00000002726', 'ENSG00000131471', 'ENSG00000131480', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000072210', 'ENSG00000111275', 'ENSG00000137124', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000072210', 'ENSG00000111275', 'ENSG00000137124', 'ENSG00000142920', 'ENSG00000116771', 'ENSG00000141504', 'ENSG00000130066', 'ENSG00000069535', 'ENSG00000002726', 'ENSG00000131471', 'ENSG00000131480', 'ENSG00000189221', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000111275', 'ENSG00000136750', 'ENSG00000128683', 'ENSG00000183044', 'ENSG00000112294', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000148672', 'ENSG00000182890', 'ENSG00000135821', 'ENSG00000115419', 'ENSG00000135423', 'ENSG00000115419', 'ENSG00000084774', 'ENSG00000092621', 'ENSG00000134882', 'ENSG00000135069', 'ENSG00000146733', 'ENSG00000111237', 'ENSG00000176974', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000100714', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000160282', 'ENSG00000100714', 'ENSG00000144908', 'ENSG00000136010', 'ENSG00000135094', 'ENSG00000167720', 'ENSG00000139410', 'ENSG00000182199', 'ENSG00000176974', 'ENSG00000178445', '(ENSG00000140905 and ENSG00000145020)', 'ENSG00000145020', 'ENSG00000091140', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000178700', 'ENSG00000228716', 'ENSG00000163738', 'ENSG00000100714', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000163738', 'ENSG00000065911', 'ENSG00000100714', 'ENSG00000120254', 'ENSG00000100714', 'ENSG00000136010', 'ENSG00000144908', 'ENSG00000100116', 'ENSG00000069535', 'ENSG00000131471', 'ENSG00000124767', 'ENSG00000167699', 'ENSG00000063854', 'ENSG00000103253', 'ENSG00000166816', 'ENSG00000021826', 'ENSG00000036473', 'ENSG00000007171', 'ENSG00000089250', 'ENSG00000164867', 'ENSG00000007171', 'ENSG00000089250', 'ENSG00000164867', 'ENSG00000167741', 'ENSG00000131067', 'ENSG00000099998', 'ENSG00000100031', 'ENSG00000100121', 'ENSG00000274252', 'ENSG00000166825', 'ENSG00000002549', 'ENSG00000141279', 'ENSG00000001084', '(ENSG00000001084 and ENSG00000023909)', 'ENSG00000100983', 'ENSG00000171759', 'ENSG00000151552', 'ENSG00000120053', 'ENSG00000125166', 'ENSG00000198650', 'ENSG00000104951', 'ENSG00000158104', 'ENSG00000113924', 'ENSG00000100577', 'ENSG00000103876', 'ENSG00000070669', 'ENSG00000166183', 'ENSG00000162174', 'ENSG00000084110', 'ENSG00000159650', 'ENSG00000139344', 'ENSG00000160282', 'ENSG00000183044', 'ENSG00000119711', 'ENSG00000119711', 'ENSG00000144644', 'ENSG00000128683', 'ENSG00000136750', 'ENSG00000166165', 'ENSG00000104879', 'ENSG00000131730', 'ENSG00000223572', 'ENSG00000237289', 'ENSG00000131730', 'ENSG00000104879', 'ENSG00000166165', 'ENSG00000223572', 'ENSG00000237289', 'ENSG00000172497', 'ENSG00000117448', '(ENSG00000197894 and ENSG00000248144)', 'ENSG00000147576', 'ENSG00000172955', 'ENSG00000180011', 'ENSG00000187758', 'ENSG00000196344', 'ENSG00000196616', 'ENSG00000197894', 'ENSG00000198099', 'ENSG00000248144', 'ENSG00000137124', 'ENSG00000072210', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000006534', 'ENSG00000108602', 'ENSG00000132746', 'ENSG00000184254', 'ENSG00000164904', 'ENSG00000143149', 'ENSG00000006534', 'ENSG00000108602', 'ENSG00000132746', 'ENSG00000184254', 'ENSG00000154930', 'ENSG00000111058', 'ENSG00000131069', 'ENSG00000239900', 'ENSG00000100354', 'ENSG00000035687', 'ENSG00000185100', 'ENSG00000116748', 'ENSG00000116337', 'ENSG00000133805', 'ENSG00000118520', 'ENSG00000126522', 'ENSG00000130707', 'ENSG00000081181', 'ENSG00000118520', 'ENSG00000023330', 'ENSG00000158578', 'ENSG00000178773', 'ENSG00000148218', 'ENSG00000256269', 'ENSG00000188690', 'ENSG00000126088', 'ENSG00000080819', 'ENSG00000143224', 'ENSG00000066926', 'ENSG00000076351', 'ENSG00000100292', 'ENSG00000103415', 'ENSG00000090013', 'ENSG00000106605', 'ENSG00000106605', 'ENSG00000090013', 'ENSG00000105223', 'ENSG00000075651', 'ENSG00000129219', 'ENSG00000179598', 'ENSG00000198814', 'ENSG00000175066', 'ENSG00000196475', 'ENSG00000229894', 'ENSG00000186281', 'ENSG00000119927', 'ENSG00000138678', 'ENSG00000158669', 'ENSG00000155189', 'ENSG00000011198', 'ENSG00000026652', 'ENSG00000143797', 'ENSG00000160216', 'ENSG00000169692', 'ENSG00000172197', 'ENSG00000172954', 'ENSG00000204310', 'ENSG00000144559', 'ENSG00000101290', 'ENSG00000163624', 'ENSG00000087157', 'ENSG00000110536', 'ENSG00000088766', 'ENSG00000088766', 'ENSG00000134326', 'ENSG00000162368', 'ENSG00000103202', 'ENSG00000103024', 'ENSG00000143156', 'ENSG00000172113', 'ENSG00000239672', 'ENSG00000243678', 'ENSG00000142168', 'ENSG00000112096', 'ENSG00000109610', 'ENSG00000176153', 'ENSG00000116157', 'ENSG00000117450', 'ENSG00000117592', 'ENSG00000126432', 'ENSG00000164294', 'ENSG00000165672', 'ENSG00000167468', 'ENSG00000167815', 'ENSG00000198704', 'ENSG00000211445', 'ENSG00000224586', 'ENSG00000233276', 'ENSG00000233276', 'ENSG00000116157', 'ENSG00000117450', 'ENSG00000117592', 'ENSG00000126432', 'ENSG00000164294', 'ENSG00000165672', 'ENSG00000167468', 'ENSG00000167815', 'ENSG00000176153', 'ENSG00000198704', 'ENSG00000211445', 'ENSG00000224586', 'ENSG00000104687', 'ENSG00000104687', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000100075', 'ENSG00000075415', 'ENSG00000151729', 'ENSG00000005022', 'ENSG00000169100', 'ENSG00000175567', 'ENSG00000175564', 'ENSG00000102078', 'ENSG00000109424', 'ENSG00000153291', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000183048', 'ENSG00000181035', 'ENSG00000181035', 'ENSG00000182902', 'ENSG00000177542', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', 'ENSG00000102743', 'ENSG00000120329', 'ENSG00000197119', '(ENSG00000060762 and ENSG00000143158)', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', '(ENSG00000060762 and ENSG00000143158)', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000147454', 'ENSG00000155287', 'ENSG00000136010', 'ENSG00000144908', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000125166', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000144659', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000103569', 'ENSG00000183048', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000155380', 'ENSG00000070214', 'ENSG00000112499', 'ENSG00000129353', 'ENSG00000137968', 'ENSG00000143036', 'ENSG00000175003', 'ENSG00000197375', 'ENSG00000204385', 'ENSG00000086159', 'ENSG00000100170', 'ENSG00000103375', 'ENSG00000143595', 'ENSG00000161798', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000167580', 'ENSG00000171885', 'ENSG00000240583', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000130821', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000206190', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000124406', 'ENSG00000206190', '(ENSG00000112697 and ENSG00000124406)', '(ENSG00000124406 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000143515)', '(ENSG00000143515 and ENSG00000182107)', '(ENSG00000081923 and ENSG00000112697)', '(ENSG00000081923 and ENSG00000182107)', '(ENSG00000104043 and ENSG00000112697)', '(ENSG00000104043 and ENSG00000182107)', '(ENSG00000054793 and ENSG00000112697)', '(ENSG00000054793 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000166377)', '(ENSG00000166377 and ENSG00000182107)', '(ENSG00000112697 and ENSG00000206190)', '(ENSG00000182107 and ENSG00000206190)', '(ENSG00000112697 and ENSG00000145246)', '(ENSG00000145246 and ENSG00000182107)', '(ENSG00000068650 and ENSG00000112697)', '(ENSG00000068650 and ENSG00000182107)', '(ENSG00000058063 and ENSG00000112697)', '(ENSG00000058063 and ENSG00000182107)', '(ENSG00000101974 and ENSG00000112697)', '(ENSG00000101974 and ENSG00000182107)', 'ENSG00000124406', 'ENSG00000206190', 'ENSG00000155380', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000130649', 'ENSG00000155380', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000130821', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000125166', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000164363', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000003989', 'ENSG00000092068', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000112394', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', '(ENSG00000092068 and ENSG00000168003)', 'ENSG00000003989', 'ENSG00000103257', 'ENSG00000123643', 'ENSG00000130876', 'ENSG00000139514', 'ENSG00000149150', 'ENSG00000155465', 'ENSG00000165349', 'ENSG00000167703', 'ENSG00000112077', 'ENSG00000100156', 'ENSG00000108932', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000168679', 'ENSG00000170190', 'ENSG00000118596', 'ENSG00000141526', 'ENSG00000155380', 'ENSG00000070214', 'ENSG00000112499', 'ENSG00000129353', 'ENSG00000137968', 'ENSG00000143036', 'ENSG00000175003', 'ENSG00000197375', 'ENSG00000204385', 'ENSG00000103569', 'ENSG00000132874', 'ENSG00000141469', 'ENSG00000018280', 'ENSG00000110911', 'ENSG00000138449', 'ENSG00000086159', 'ENSG00000100170', 'ENSG00000103375', 'ENSG00000143595', 'ENSG00000161798', 'ENSG00000165269', 'ENSG00000165272', 'ENSG00000167580', 'ENSG00000171885', 'ENSG00000240583']\n",
      " Mitocore2, has 1108 GPRs\n"
     ]
    }
   ],
   "source": [
    "gpr_0=gpr_counte(gpr_mit_0, 'Mitocore0')\n",
    "gpr_0=gpr_counte(gpr_mit_1, 'Mitocore1')\n",
    "gpr_0=gpr_counte(gpr_mit_2, 'Mitocore2')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dc7c5c11",
   "metadata": {},
   "outputs": [],
   "source": [
    "mitocore =cobra.io.read_sbml_model('Mitocore_changed_reactions_ec-code.xml')\n",
    "\n",
    "#-------------Genes GPR extraction-----------------\n",
    "gene_list_gpr=[]\n",
    "pattern= r'ENSG\\d{11}'\n",
    "\n",
    "for reaction in mitocore.reactions:\n",
    "    gpr= reaction.gene_reaction_rule.strip()\n",
    "    matches = re.findall(pattern, gpr)\n",
    "    if matches:\n",
    "        gene_list_gpr.extend(matches) # .extend() add the objects as strings and not like a list as .append() does\n",
    "\n",
    "gene_list_gpr = list(gene_list_gpr)\n",
    "print('genes from GPRs:', len(gene_list_gpr), gene_list_gpr)\n",
    "gene_list_gpr = list(set(gene_list_gpr))\n",
    "print('genes from GPRs after removing duplicates:', len(gene_list_gpr), gene_list_gpr)\n",
    "\n",
    "#---------------Genes Mitocore 'genes' compartment--------------\n",
    "gene_list_mito=[]\n",
    "\n",
    "for gene in mitocore.genes:\n",
    "    id= gene.id\n",
    "    gene_list_mito.append(id)\n",
    "\n",
    "gene_list_mito = list(set(gene_list_mito))\n",
    "print('genes in mitocore.genes:',len(gene_list_mito), gene_list_mito )  \n",
    "\n",
    "#---------check how many genes from GPRs are in Genes compartment------------\n",
    "\n",
    "genes_contained=[]\n",
    "genes_not_contained=[]\n",
    "\n",
    "for gene in gene_list_gpr:\n",
    "    if gene in gene_list_mito:\n",
    "        genes_contained.append(gene)\n",
    "    else: \n",
    "        genes_not_contained.append(gene)\n",
    "\n",
    "print(len(genes_not_contained))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "69409060",
   "metadata": {},
   "source": [
    "# GPR comparison Mitocore original, Preliminary curation, Alignment to Human1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6a7de380",
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract GPRs for original // curation1 // final curation into a csv file \n",
    "# for visualization of the code because memote does not show GPRs\n",
    "mitocore0= cobra.io.read_sbml_model('Mitocore_0.xml')\n",
    "mitocore1= cobra.io.read_sbml_model('Mitocore_Curated_Emanuel.xml')\n",
    "mitocore2= cobra.io.read_sbml_model('Mitocore_GPR_Cleaned.xml')\n",
    "\n",
    "gpr_mitocore_0= get_model_ids_for_attr(mitocore0, 'reactions', '_gpr')\n",
    "gpr_mitocore_1= get_model_ids_for_attr(mitocore1, 'reactions', '_gpr')\n",
    "gpr_mitocore_2= get_model_ids_for_attr(mitocore2, 'reactions', '_gpr')\n",
    "\n",
    "\n",
    "# Merge mito0 and mito1 (not possible to merge)\n",
    "merged_01 = pd.merge(\n",
    "    gpr_mitocore_0, \n",
    "    gpr_mitocore_1, \n",
    "    on='model_id', \n",
    "    suffixes=('_original', '_emanuel')\n",
    ")\n",
    "\n",
    "# Merge merged_01 with mito2\n",
    "gpr_mitocore = pd.merge(\n",
    "    merged_01, \n",
    "    gpr_mitocore_2, \n",
    "    on='model_id'\n",
    "    ,suffixes=('_cleaned','' )\n",
    ")\n",
    "\n",
    "gpr_mitocore.to_csv('GPR_comparison.csv', index=False)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
